{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2103e87f",
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "import pandas as pd\n",
    "\n",
    "import sys\n",
    "sys.path.append('../')\n",
    "from translate import translate_baidu_api"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "959f16ba",
   "metadata": {},
   "outputs": [],
   "source": [
    "def print_(texts, msg=''):\n",
    "    print(f\"{msg}相关文献共有{len(texts)}篇\")\n",
    "    print('-'*60)\n",
    "    for text in texts:\n",
    "        print(text)\n",
    "        print(translate_baidu_api(text))\n",
    "        print('-'*60)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c1de0132",
   "metadata": {},
   "source": [
    "# 数据"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "bde74f00",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "相关文献共有40篇\n",
      "stPlus: a reference-based method for the accurate enhancement of spatial transcriptomics\n",
      "stPlus：一种精确增强空间转录组学的参考方法\n",
      "------------------------------------------------------------\n",
      "Statistical approaches for differential expression analysis in metatranscriptomics\n",
      "转录组学差异表达分析的统计方法\n",
      "------------------------------------------------------------\n",
      "A comparison of rule-based and centroid single-sample multiclass predictors for transcriptomic classification\n",
      "转录组分类中基于规则和质心的单样本多类预测因子的比较\n",
      "------------------------------------------------------------\n",
      "Phylotranscriptomic patterns of network stochasticity and pathway dynamics during embryogenesis\n",
      "胚胎发生过程中网络随机性和路径动力学的系统转录组学模式\n",
      "------------------------------------------------------------\n",
      "Differential transcript usage analysis of bulk and single-cell RNA-seq data with DTUrtle\n",
      "用DTUrtle分析批量和单细胞RNA序列数据的差异转录本使用情况\n",
      "------------------------------------------------------------\n",
      "Tfcancer: a manually curated database of transcription factors associated with human cancers\n",
      "Tfcancer：一个与人类癌症相关的转录因子手动管理数据库\n",
      "------------------------------------------------------------\n",
      "utr.annotation: a tool for annotating genomic variants that could influence post-transcriptional regulation\n",
      "utr。注释：用于注释可能影响转录后调节的基因组变体的工具\n",
      "------------------------------------------------------------\n",
      "TPWshiny: an interactive R/Shiny app to explore cell line transcriptional responses to anti-cancer drugs\n",
      "TPWshiny：一款交互式R/Shiny应用程序，用于探索细胞系对抗癌药物的转录反应\n",
      "------------------------------------------------------------\n",
      "TFcancer: a manually curated database of transcription factors associated with human cancer\n",
      "TFcancer：一个与人类癌症相关的转录因子手动管理数据库\n",
      "------------------------------------------------------------\n",
      "Hubness reduction improves clustering and trajectory inference in single-cell transcriptomic data\n",
      "Hubbness约简改进了单细胞转录组数据的聚类和轨迹推断\n",
      "------------------------------------------------------------\n",
      "Weakly supervised learning of RNA modifications from low-resolution epitranscriptome data\n",
      "低分辨率上转录组数据中RNA修饰的弱监督学习\n",
      "------------------------------------------------------------\n",
      "Assessing heterogeneity in spatial data using the HTA index with applications to spatial transcriptomics and imaging\n",
      "使用HTA指数评估空间数据的异质性，并应用于空间转录组学和成像\n",
      "------------------------------------------------------------\n",
      "SPOT: a web-tool enabling swift profiling of transcriptomes\n",
      "SPOT：一个能够快速分析转录组的网络工具\n",
      "------------------------------------------------------------\n",
      "Scalable inference of transcriptional kinetic parameters from MS2 time series data\n",
      "从MS2时间序列数据可伸缩推断转录动力学参数\n",
      "------------------------------------------------------------\n",
      "Predicting mechanism of action of novel compounds using compound structure and transcriptomic signature coembedding\n",
      "利用化合物结构和转录组学特征共嵌入预测新化合物的作用机制\n",
      "------------------------------------------------------------\n",
      "Ryūtō: Improved multi-sample transcript assembly for differential transcript expression analysis and more\n",
      "Ryūtō：改进的多样本转录本组装，用于差异转录本表达分析等\n",
      "------------------------------------------------------------\n",
      "TIMEx: tumor-immune microenvironment deconvolution web-portal for bulk transcriptomics using pan-cancer scRNA-seq signatures\n",
      "TIMEx：肿瘤免疫微环境反褶积门户网站，用于使用泛癌scRNA序列签名的批量转录组学\n",
      "------------------------------------------------------------\n",
      "Erratum to: An information-theoretic approach for measuring the distance of organ tissue samples using their transcriptomic signatures\n",
      "勘误表至：利用转录组特征测量器官组织样本距离的信息论方法\n",
      "------------------------------------------------------------\n",
      "Biphasic regulation of transcriptional surge generated by the gene feedback loop in a two-component system\n",
      "双组分系统中基因反馈环产生的转录激增的双相调节\n",
      "------------------------------------------------------------\n",
      "BloodGen3Module: Blood transcriptional module repertoire analysis and visualization using R\n",
      "BloodGen3模块：使用R\n",
      "------------------------------------------------------------\n",
      "Deep feature extraction of single-cell transcriptomes by generative adversarial network\n",
      "基于生成对抗网络的单细胞转录组深度特征提取\n",
      "------------------------------------------------------------\n",
      "Swan: a library for the analysis and visualization of long-read transcriptomes\n",
      "Swan：一个用于长阅读转录组分析和可视化的库\n",
      "------------------------------------------------------------\n",
      "FuSe: a tool to move RNA-Seq analyses from chromosomal/gene loci to functional grouping of mRNA transcripts\n",
      "FuSe：将RNA-Seq分析从染色体/基因位点转移到mRNA转录本功能分组的工具\n",
      "------------------------------------------------------------\n",
      "A2Sign: Agnostic algorithms for signatures - a universal method for identifying molecular signatures from transcriptomic datasets prior to cell-type deconvolution\n",
      "A2Sign：用于特征的不可知算法-在细胞类型反褶积之前从转录组数据集中识别分子特征的通用方法\n",
      "------------------------------------------------------------\n",
      "Hierarchical Bayesian models of transcriptional and translational regulation processes with delays\n",
      "具有时滞的转录和翻译调控过程的层次贝叶斯模型\n",
      "------------------------------------------------------------\n",
      "MAT2: Manifold alignment of single-cell transcriptomes with cell triplets\n",
      "MAT2：单细胞转录组与细胞三联体的多重比对\n",
      "------------------------------------------------------------\n",
      "Integrarative transcription start site identification with iTiSS\n",
      "整合转录起始位点的ITIS鉴定\n",
      "------------------------------------------------------------\n",
      "MetaTX: deciphering the distribution of mRNA-related features in the presence of isoform ambiguity, with applications in epitranscriptome analysis\n",
      "MetaTX：在存在异构体歧义的情况下解读mRNA相关特征的分布，并应用于外转录组分析\n",
      "------------------------------------------------------------\n",
      "DoubleRecViz: a web-based tool for visualizing transcript-gene-species tree reconciliation\n",
      "DoubleRecViz：一种基于web的可视化转录基因物种树协调工具\n",
      "------------------------------------------------------------\n",
      "BART3D: Inferring transcriptional regulators associated with differential chromatin interactions from Hi-C data\n",
      "BART3D：从Hi-C数据推断与差异染色质相互作用相关的转录调节因子\n",
      "------------------------------------------------------------\n",
      "IsoTV: processing and visualizing functional features of translated transcript isoforms\n",
      "IsoTV：翻译转录物亚型的加工和可视化功能特征\n",
      "------------------------------------------------------------\n",
      "IntAPT: integrated assembly of phenotype-specific transcripts from multiple RNA-seq profiles\n",
      "IntAPT：来自多个RNA序列图谱的表型特异性转录物的整合组装\n",
      "------------------------------------------------------------\n",
      "Prioritizing transcriptomic and epigenomic experiments using an optimization strategy that leverages imputed data\n",
      "使用利用估算数据的优化策略对转录组学和表观基因组学实验进行优先排序\n",
      "------------------------------------------------------------\n",
      "Characterizing therapeutic signatures of transcription factors in cancer by incorporating profiles in compound treated cells\n",
      "通过在化合物处理的细胞中整合谱来表征癌症中转录因子的治疗特征\n",
      "------------------------------------------------------------\n",
      "Erratum to: Assessing heterogeneity in spatial data using the HTA index with applications to spatial transcriptomics and imaging\n",
      "勘误表至：使用HTA指数评估空间数据的异质性，并应用于空间转录组学和成像\n",
      "------------------------------------------------------------\n",
      "GNET2: an R package for constructing gene regulatory networks from transcriptomic data\n",
      "GNET2：从转录组数据构建基因调控网络的R包\n",
      "------------------------------------------------------------\n",
      "FastBMD: an online tool for rapid benchmark dose-response analysis of transcriptomics data\n",
      "FastBMD：转录组学数据快速基准剂量反应分析的在线工具\n",
      "------------------------------------------------------------\n",
      "Transcriptor: a comprehensive platform for annotation of the enzymatic functions of transcripts\n",
      "转录子：转录子酶功能注释的综合平台\n",
      "------------------------------------------------------------\n",
      "An information-theoretic approach for measuring the distance of organ tissue samples using their transcriptomic signatures\n",
      "利用转录组特征测量器官组织样本距离的信息论方法\n",
      "------------------------------------------------------------\n",
      "SplicingFactory-splicing diversity analysis for transcriptome data\n",
      "转录组数据的SplicingFactory剪接多样性分析\n",
      "------------------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "titles = pd.read_csv('../../data/csv-Bioinforma-set.csv')['Title']\n",
    "# print(f\"2021年Bioinformatics发文数量：{len(titles)}\")\n",
    "print_(transcrip_papers)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "54850f15",
   "metadata": {},
   "outputs": [],
   "source": [
    "all_papers = {\n",
    "    \"database\" : [title for title in titles if re.findall('database*', title)], # 40\n",
    "    \"RNA-seq\" : [title for title in titles if re.findall('RNA-seq*', title)],   # 36\n",
    "    \"transcrip*\" : [title for title in titles if re.findall('transcrip*', title)], # 40\n",
    "    \"solub\" : [title for title in titles if re.findall('solub', title)], # 2\n",
    "    \"stabil\" : [title for title in titles if re.findall('stabil', title)], # 7\n",
    "    \"protein\" : [title for title in titles if re.findall('protein', title)], # 104\n",
    "    \"feature\" : [title for title in titles if re.findall('feature', title)],\n",
    "    \"Structur\" : \n",
    "    \"Secondary Structur\" : \n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "3416613f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "相关文献共有26篇\n",
      "XRRpred: Accurate Predictor of Crystal Structure Quality from Protein Sequence\n",
      "XRRpred：从蛋白质序列准确预测晶体结构质量\n",
      "------------------------------------------------------------\n",
      "Socket2: A Program for Locating, Visualising, and Analysing Coiled-coil Interfaces in Protein Structures\n",
      "Socket2：一个用于定位、可视化和分析蛋白质结构中螺旋线圈界面的程序\n",
      "------------------------------------------------------------\n",
      "tSFM 1.0: tRNA Structure-Function Mapper\n",
      "tSFM 1.0:tRNA结构-功能映射器\n",
      "------------------------------------------------------------\n",
      "Structure-aware protein-protein interaction site prediction using deep graph convolutional network\n",
      "基于深度图卷积网络的结构感知蛋白质相互作用位点预测\n",
      "------------------------------------------------------------\n",
      "Accelerating Big Data Quantitative Structure-Activity Prediction through LASSO-Random Forest Algorithm\n",
      "利用LASSO随机森林算法加速大数据定量结构活动预测\n",
      "------------------------------------------------------------\n",
      "Structural discrimination analysis for constraint selection in protein modeling\n",
      "蛋白质建模中约束选择的结构判别分析\n",
      "------------------------------------------------------------\n",
      "ProFitFun: A Protein Tertiary Structure Fitness Function for Quantifying the Accuracies of Model Structures\n",
      "ProFitFun：用于量化模型结构精确度的蛋白质三级结构适应度函数\n",
      "------------------------------------------------------------\n",
      "SPOT-1D-Single: Improving the Single-Sequence-Based Prediction of Protein Secondary Structure, Backbone Angles, Solvent Accessibility and Half-Sphere Exposures using a Large Training Set and Ensembled Deep Learning\n",
      "SPOT-1D-Single：使用大型训练集和集成式深度学习，改进基于单序列的蛋白质二级结构、主干角度、溶剂可及性和半球暴露预测\n",
      "------------------------------------------------------------\n",
      "A Sequential Niche Multimodal Conformational Sampling Algorithm for Protein Structure Prediction\n",
      "用于蛋白质结构预测的序贯小生境多峰构象抽样算法\n",
      "------------------------------------------------------------\n",
      "2DProts: Database of Family-Wide Protein Secondary Structure Diagrams\n",
      "2DProts：全家族蛋白质二级结构图数据库\n",
      "------------------------------------------------------------\n",
      "Structural classification of proteins based on the computationally efficient recurrence quantification analysis and horizontal visibility graphs\n",
      "基于计算效率递归量化分析和水平可见度图的蛋白质结构分类\n",
      "------------------------------------------------------------\n",
      "Structural feature-driven pattern analysis for multitarget modulator landscapes\n",
      "结构特征驱动的多目标模式分析\n",
      "------------------------------------------------------------\n",
      "ConsAlifold: Considering RNA Structural Alignments Improves Prediction Accuracy of RNA Consensus Secondary Structures\n",
      "取消资格：考虑RNA结构比对可提高RNA一致性二级结构的预测准确性\n",
      "------------------------------------------------------------\n",
      "Comparative Evaluation of Shape Retrieval Methods on Macromolecular Surfaces: An Application of Computer Vision Methods in Structural Bioinformatics\n",
      "高分子表面形状检索方法的比较评价：计算机视觉方法在结构生物信息学中的应用\n",
      "------------------------------------------------------------\n",
      "RCSB Protein Data Bank: Improved Annotation, Search, and Visualization of Membrane Protein Structures Archived in the PDB\n",
      "RCSB蛋白质数据库：改进了PDB中存档的膜蛋白结构的注释、搜索和可视化\n",
      "------------------------------------------------------------\n",
      "muCNV: Genotyping Structural Variants for Population-level Sequencing\n",
      "muCNV：用于群体水平测序的基因型结构变异\n",
      "------------------------------------------------------------\n",
      "Accurate Haplotype-Resolved Assembly Reveals The Origin Of Structural Variants For Human Trios\n",
      "精确的单倍型解析组装揭示了人类三联体结构变异的起源\n",
      "------------------------------------------------------------\n",
      "Large-scale Inference of Population Structure in Presence of Missingness using PCA\n",
      "基于主成分分析的缺失种群结构大规模推断\n",
      "------------------------------------------------------------\n",
      "Structural basis of SARS-CoV-2 spike protein induced by ACE2\n",
      "ACE2诱导SARS-CoV-2棘突蛋白的结构基础\n",
      "------------------------------------------------------------\n",
      "OPUS-X: An Open-Source Toolkit for Protein Torsion Angles, Secondary Structure, Solvent Accessibility, Contact Map Predictions, and 3D Folding\n",
      "OPUS-X：蛋白质扭转角、二级结构、溶剂可及性、接触图预测和3D折叠的开源工具包\n",
      "------------------------------------------------------------\n",
      "Study of Real-Valued Distance Prediction for Protein Structure Prediction with Deep Learning\n",
      "基于深度学习的蛋白质结构预测实值距离预测研究\n",
      "------------------------------------------------------------\n",
      "Protein Contact Map Refinement for Improving Structure Prediction Using Generative Adversarial Networks\n",
      "利用生成性对抗网络改进蛋白质接触图的结构预测\n",
      "------------------------------------------------------------\n",
      "De Novo Genome Assembly Of Solanum Sitiens Reveals Structural Variation Associated With Drought And Salinity Tolerance\n",
      "龙葵的从头基因组组装揭示了与耐旱和耐盐性相关的结构变异\n",
      "------------------------------------------------------------\n",
      "Improved RNA Secondary Structure and Tertiary Base-pairing Prediction Using Evolutionary Profile, Mutational Coupling and Two-dimensional Transfer Learning\n",
      "利用进化模式、突变耦合和二维转移学习改进RNA二级结构和三级碱基配对预测\n",
      "------------------------------------------------------------\n",
      "Active Learning to Classify Macromolecular Structures in situ for Less Supervision in Cryo-Electron Tomography\n",
      "主动学习在原位对大分子结构进行分类，以减少冷冻电子断层扫描中的监督\n",
      "------------------------------------------------------------\n",
      "Structural bioinformatics enhances mechanistic interpretation of genomic variation, demonstrated through the analyses of 935 distinct RAS family mutations\n",
      "通过对935个不同RAS家族突变的分析，结构生物信息学增强了对基因组变异的机械解释\n",
      "------------------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "Structur = [title for title in titles if re.findall('Structur', title)]\n",
    "print_(Structur)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f2bb92bf",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
