{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "7bf7cbce",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import sys\n",
    "import re\n",
    "sys.path.append('../')\n",
    "from translate import translate_baidu_api"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fd0fbbb7",
   "metadata": {},
   "outputs": [],
   "source": [
    "def print_(texts, msg='', trans=False):\n",
    "    print(f\"{msg}相关文献共有{len(texts)}篇\")\n",
    "    for text in texts:\n",
    "        print(text)\n",
    "        if trans:\n",
    "            print(translate_baidu_api(text))\n",
    "        print('-'*60)\n",
    "        \n",
    "def find_papers(kws, visual=False):\n",
    "    filter_titles = [title for title in titles if re.findall(kws, title)]\n",
    "    if visual:\n",
    "        print_(filter_titles)\n",
    "#     return filter_titles, [ for index,title in enumerate(titles) if title ]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "6b595f55",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0       On the complexity of haplotyping a microbial c...\n",
       "1                           CoRC - the COPASI R Connector\n",
       "2       UniBioDicts: Unified access to biological dict...\n",
       "3                    Projected t-SNE for batch correction\n",
       "4                                   The COVID-19 Ontology\n",
       "                              ...                        \n",
       "5394      chimeraviz: a tool for visualizing chimeric RNA\n",
       "5395    SNPhood: investigate, quantify and visualise t...\n",
       "5396    MCAST: scanning for cis-regulatory motif clusters\n",
       "5397    Foldalign 2.5: multithreaded implementation fo...\n",
       "5398    Prediction of the permeability of neutral drug...\n",
       "Name: Title, Length: 5399, dtype: object"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "FILE = '../data/2015-2020_Bioinforma-set.csv'\n",
    "df = pd.read_csv(FILE)\n",
    "titles = df['Title']\n",
    "titles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3c955500",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "a10b8902",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "On the complexity of haplotyping a microbial community\n",
      "CoRC - the COPASI R Connector\n",
      "UniBioDicts: Unified access to biological dictionaries\n",
      "Projected t-SNE for batch correction\n",
      "The COVID-19 Ontology\n",
      "SOS: online probability estimation and generation of T-and B-cell receptors\n",
      "TwisTranscripT: stochastic simulation of the transcription-supercoiling coupling\n",
      "matFR: a MATLAB toolbox for feature ranking\n",
      "Codon optimization: a mathematical programing approach\n",
      "Weighted minimizer sampling improves long read mapping\n",
      "Pooled variable scaling for cluster analysis\n",
      "Feasible-metabolic-pathway-exploration technique using chemical latent space\n",
      "TDAview: an online visualization tool for topological data analysis\n",
      "Collecting and managing taxonomic data with NCBI-taxonomist\n",
      "On Biases Of Attention In Scientific Discovery\n",
      "APOD: accurate sequence-based predictor of disordered flexible linkers\n",
      "Enlighten2: molecular dynamics simulations of protein-ligand systems made accessible\n",
      "GlyGen data model and processing workflow\n",
      "Hopper: a mathematically optimal algorithm for sketching biological data\n",
      "aPCoA: covariate adjusted principal coordinates analysis\n",
      "Geometricus represents protein structures as shape-mers derived from moment invariants\n",
      "Phylogenetic double placement of mixed samples\n",
      "T-Gene: improved target gene prediction\n",
      "Chromatin network markers of leukemia\n",
      "ClonArch: visualizing the spatial clonal architecture of tumors\n",
      "Complete populations of virtual patients for in silico clinical trials\n",
      "PathFinder: Bayesian inference of clone migration histories in cancer\n",
      "SHOGUN: a modular, accurate and scalable framework for microbiome quantification\n",
      "A Bayesian framework for inter-cellular information sharing improves dscRNA-seq quantification\n",
      "Portable nanopore analytics: are we there yet?\n",
      "Supervised learning on phylogenetically distributed data\n",
      "SAINT: self-attention augmented inception-inside-inception network improves protein secondary structure prediction\n",
      "VINYL: Variant prIoritizatioN by survivaL analysis\n",
      "ENANO: Encoder for NANOpore FASTQ files\n",
      "Neoantimon: a multifunctional R package for identification of tumor-specific neoantigens\n",
      "PUMA: PANDA Using MicroRNA Associations\n",
      "Treerecs: an integrated phylogenetic tool, from sequences to reconciliations\n",
      "Improved design and analysis of practical minimizers\n",
      "A phylogenetic C interpreter for TNT\n",
      "Artificial-cell-type aware cell-type classification in CITE-seq\n",
      "DockIT: a tool for interactive molecular docking and molecular complex construction\n",
      "A probabilistic graphical model for system-wide analysis of gene regulatory networks\n",
      "Enhancing statistical power in temporal biomarker discovery through representative shapelet mining\n",
      "LDpred2: better, faster, stronger\n",
      "Network analysis of synonymous codon usage\n",
      "Inferring signaling pathways with probabilistic programming\n",
      "GenMap: ultra-fast computation of genome mappability\n",
      "ProVision: a web-based platform for rapid analysis of proteomics data processed by MaxQuant\n",
      "Differentiating isoform functions with collaborative matrix factorization\n",
      "CiteFuse enables multi-modal analysis of CITE-seq data\n",
      "Numt identification and removal with RtN!\n",
      "Combining phenome-driven drug-target interaction prediction with patients' electronic health records-based clinical corroboration toward drug discovery\n",
      "Priors for genotyping polyploids\n",
      "Molywood: streamlining the design and rendering of molecular movies\n",
      "Struct: an R/bioconductor-based framework for standardised metabolomics data analysis and beyond\n",
      "Advancing PICO element detection in biomedical text via deep neural networks\n",
      "ProCaff: protein-carbohydrate complex binding affinity database\n",
      "FBA reveals guanylate kinase as a potential target for antiviral therapies against SARS-CoV-2\n",
      "SVJedi: genotyping structural variations with long reads\n",
      "EpiGEN: an epistasis simulation pipeline\n",
      "Liftoff: accurate mapping of gene annotations\n",
      "The Glycine Receptor Allosteric Ligands Library (GRALL)\n",
      "Epiviz File Server: Query, transform and interactively explore data from indexed genomic files\n",
      "Combinatorial and statistical prediction of gene expression from haplotype sequence\n",
      "SOMM4mC: a second-order Markov model for DNA N4-methylcytosine site prediction in six species\n",
      "HiC-Hiker: a probabilistic model to determine contig orientation in chromosome-length scaffolds with Hi-C\n",
      "Optimization of co-evolution analysis through phylogenetic profiling reveals pathway-specific signals\n",
      "TreeSAPP: the Tree-based Sensitive and Accurate Phylogenetic Profiler\n",
      "Toward heterogeneous information fusion: bipartite graph convolutional networks for in silico drug repurposing\n",
      "Deshrinking ridge regression for genome-wide association studies\n",
      "Identification of conserved evolutionary trajectories in tumors\n",
      "The locality dilemma of Sankoff-like RNA alignments\n",
      "A parallelized strategy for epistasis analysis based on Empirical Bayesian Elastic Net models\n",
      "PolishEM: image enhancement in FIB-SEM\n",
      "DNA Chisel, a versatile sequence optimizer\n",
      "Programmatic access to bacterial regulatory networks with regutools\n",
      "A multimodal deep learning framework for predicting drug-drug interaction events\n",
      "Deep multiple instance learning classifies subtissue locations in mass spectrometry images from tissue-level annotations\n",
      "Protein-sol pKa: prediction of electrostatic frustration, with application to coronaviruses\n",
      "Joint learning dimension reduction and clustering of single-cell RNA-sequencing data\n",
      "Sequoya: multiobjective multiple sequence alignment in Python\n",
      "Padhoc: a computational pipeline for pathway reconstruction on the fly\n",
      "Benchmark of software tools for prokaryotic chromosomal interaction domain identification\n",
      "Ktrim: an extra-fast and accurate adapter- and quality-trimmer for sequencing data\n",
      "Solubility-Weighted Index: fast and accurate prediction of protein solubility\n",
      "Somatic selection distinguishes oncogenes and tumor suppressor genes\n",
      "Graph neural representational learning of RNA secondary structures for predicting RNA-protein interactions\n",
      "dittoSeq: Universal User-Friendly Single-Cell and Bulk RNA Sequencing Visualization Toolkit\n",
      "Higher-order Markov models for metagenomic sequence classification\n",
      "Predicting target genes of non-coding regulatory variants with IRT\n",
      "Simulation, power evaluation and sample size recommendation for single-cell RNA-seq\n",
      "ipcoal: an interactive Python package for simulating and analyzing genealogies and sequences on a species tree or network\n",
      "IDMIL: an alignment-free Interpretable Deep Multiple Instance Learning (MIL) for predicting disease from whole-metagenomic data\n",
      "A path recorder algorithm for Multiple Longest Common Subsequences (MLCS) problems\n",
      "ganon: precise metagenomics classification against large and up-to-date sets of reference sequences\n",
      "Fast and robust ancestry prediction using principal component analysis\n",
      "CloudASM: an ultra-efficient cloud-based pipeline for mapping allele-specific DNA methylation\n",
      "Predicting Nash equilibria for microbial metabolic interactions\n",
      "An efficient framework to identify key miRNA-mRNA regulatory modules in cancer\n",
      "M2IA: a web server for microbiome and metabolome integrative analysis\n",
      "iBioProVis: interactive visualization and analysis of compound bioactivity space\n",
      "Automated inference of Boolean models from molecular interaction maps using CaSQ\n",
      "Discovering and interpreting transcriptomic drivers of imaging traits using neural networks\n",
      "Inference attacks against differentially private query results from genomic datasets including dependent tuples\n",
      "TinGa: fast and flexible trajectory inference with Growing Neural Gas\n",
      "Tally-2.0: upgraded validator of tandem repeat detection in protein sequences\n",
      "A novel normalization and differential abundance test framework for microbiome data\n",
      "MetaSanity: an integrated microbial genome evaluation and annotation pipeline\n",
      "Feature set optimization in biomarker discovery from genome-scale data\n",
      "UniRule: a unified rule resource for automatic annotation in the UniProt Knowledgebase\n",
      "Heterogeneous graph inference with matrix completion for computational drug repositioning\n",
      "TALC: Transcript-level Aware Long-read Correction\n",
      "Apollo: a sequencing-technology-independent, scalable and accurate assembly polishing algorithm\n",
      "ISMB 2020 proceedings\n",
      "CRISPRL and: Interpretable large-scale inference of DNA repair landscape based on a spectral approach\n",
      "Predicting human microbe-drug associations via graph convolutional network with conditional random field\n",
      "A knowledge-based scoring function to assess quaternary associations of proteins\n",
      "Minimally-overlapping words for sequence similarity search\n",
      "Detecting and correcting misclassified sequences in the large-scale public databases\n",
      "Mutational signature learning with supervised negative binomial non-negative matrix factorization\n",
      "DeepCDA: deep cross-domain compound-protein affinity prediction through LSTM and convolutional neural networks\n",
      "BnpC: Bayesian non-parametric clustering of single-cell mutation profiles\n",
      "Distance indexing and seed clustering in sequence graphs\n",
      "EvoLSTM: context-dependent models of sequence evolution using a sequence-to-sequence LSTM\n",
      "Multiple kernel learning for integrative consensus clustering of omic datasets\n",
      "Evaluating individual genome similarity with a topic model\n",
      "alona: a web server for single-cell RNA-seq analysis\n",
      "Exploring high-dimensional biological data with sparse contrastive principal component analysis\n",
      "grünifai: interactive multiparameter optimization of molecules in a continuous vector space\n",
      "MetaBCC-LR: metagenomics binning by coverage and composition for long reads\n",
      "COMUNET: a tool to explore and visualize intercellular communication\n",
      "Sampling and summarizing transmission trees with multi-strain infections\n",
      "A non-linear regression method for estimation of gene-environment heritability\n",
      "CScape-somatic: distinguishing driver and passenger point mutations in the cancer genome\n",
      "scVAE: variational auto-encoders for single-cell gene expression data\n",
      "Cancer subtype classification and modeling by pathway attention and propagation\n",
      "Identification Of Differentially Expressed Gene Modules In Heterogeneous Diseases\n",
      "MONET: a toolbox integrating top-performing methods for network modularization\n",
      "Estimation of dynamic SNP-heritability with Bayesian Gaussian process models\n",
      "GRaSP: a graph-based residue neighborhood strategy to predict binding sites\n",
      "ThETA: transcriptome-driven efficacy estimates for gene-based TArget discovery\n",
      "Privacy-preserving construction of generalized linear mixed model for biomedical computation\n",
      "Detecting evolutionary patterns of cancers using consensus trees\n",
      "Finding orthologous gene blocks in bacteria: the computational hardness of the problem and novel methods to address it\n",
      "LabxDB: versatile databases for genomic sequencing and lab management\n",
      "Tropical principal component analysis on the space of phylogenetic trees\n",
      "Prioritizing genetic variants in GWAS with lasso using permutation-assisted tuning\n",
      "Position-wise binding preference is important for miRNA target site prediction\n",
      "Dementia key gene identification with multi-layered SNP-gene-disease network\n",
      "Cancer mutational signatures representation by large-scale context embedding\n",
      "Overlap detection on long, error-prone sequencing reads via smooth q-gram\n",
      "FASPR: an open-source tool for fast and accurate protein side-chain packing\n",
      "ntJoin: Fast and lightweight assembly-guided scaffolding using minimizer graphs\n",
      "SCIM: universal single-cell matching with unpaired feature sets\n",
      "FastSK: fast sequence analysis with gapped string kernels\n",
      "Using a GTR+Γ substitution model for dating sequence divergence when stationarity and time-reversibility assumptions are violated\n",
      "Adaptive multiorientation resolution analysis of complex filamentous network images\n",
      "Robust partial reference-free cell composition estimation from tissue expression\n",
      "DeepMAsED: evaluating the quality of metagenomic assemblies\n",
      "Reconstructing ribosomal genes from large scale total RNA meta-transcriptomic data\n",
      "GLASSgo in Galaxy: high-throughput, reproducible and easy-to-integrate prediction of sRNA homologs\n",
      "ProteinFishing: a protein complex generator within the ModelX toolsuite\n",
      "Phigaro: high-throughput prophage sequence annotation\n",
      "OPUS-TASS: a protein backbone torsion angles and secondary structure predictor based on ensemble neural networks\n",
      "AnthOligo: automating the design of oligonucleotides for capture/enrichment technologies\n",
      "SparkINFERNO: a scalable high-throughput pipeline for inferring molecular mechanisms of non-coding genetic variants\n",
      "BiCoN: Network-constrained biclustering of patients and omics data\n",
      "Unsupervised topological alignment for single-cell multi-omics integration\n",
      "Deep-learning method for data association in particle tracking\n",
      "DTF: Deep Tensor Factorization for predicting anticancer drug synergy\n",
      "TransformerCPI: improving compound-protein interaction prediction by sequence-based deep learning with self-attention mechanism and label reversal experiments\n",
      "GM-DockZn: a geometry matching-based docking algorithm for zinc proteins\n",
      "ccNetViz: a WebGL-based JavaScript library for visualization of large networks\n",
      "Blood-based multi-tissue gene expression inference with Bayesian ridge regression\n",
      "RecBic: a fast and accurate algorithm recognizing trend-preserving biclusters\n",
      "Impact of lossy compression of nanopore raw signal data on basecalling and consensus accuracy\n",
      "AlphaFamImpute: high-accuracy imputation in full-sib families from genotype-by-sequencing data\n",
      "DeepNano-blitz: a fast base caller for MinION nanopore sequencers\n",
      "High-throughput and efficient multilocus genome-wide association study on longitudinal outcomes\n",
      "Discovery of multi-operon colinear syntenic blocks in microbial genomes\n",
      "TopicNet: a framework for measuring transcriptional regulatory network change\n",
      "APAlyzer: a bioinformatics package for analysis of alternative polyadenylation isoforms\n",
      "FUpred: detecting protein domains through deep-learning-based contact map prediction\n",
      "Dataset-aware multi-task learning approaches for biomedical named entity recognition\n",
      "HECNet: a hierarchical approach to enzyme function classification using a Siamese Triplet Network\n",
      "Efficient toolkit implementing best practices for principal component analysis of population genetic data\n",
      "Rapid screening and detection of inter-type viral recombinants using Phylo-K-Mers\n",
      "Robust and accurate deconvolution of tumor populations uncovers evolutionary mechanisms of breast cancer metastasis\n",
      "DeepEventMine: end-to-end neural nested event extraction from biomedical texts\n",
      "SSIF: Subsumption-based Sub-term Inference Framework to audit Gene Ontology\n",
      "Scirpy: a Scanpy extension for analyzing single-cell T-cell receptor-sequencing data\n",
      "AOP4EUpest: mapping of pesticides in adverse outcome pathways using a text mining tool\n",
      "Unified methods for feature selection in large-scale genomic studies with censored survival outcomes\n",
      "TE-greedy-nester: structure-based detection of LTR retrotransposons and their nesting\n",
      "Unipept CLI 2.0: adding support for visualizations and functional annotations\n",
      "Joint epitope selection and spacer design for string-of-beads vaccines\n",
      "Coevolution-based prediction of protein-protein interactions in polyketide biosynthetic assembly lines\n",
      "ProkSeq for complete analysis of RNA-Seq data from prokaryotes\n",
      "CLPred: a sequence-based protein crystallization predictor using BLSTM neural network\n",
      "Topological and kernel-based microbial phenotype prediction from MALDI-TOF mass spectra\n",
      "TiFoSi: an efficient tool for mechanobiology simulations of epithelia\n",
      "Exploring chromatin conformation and gene co-expression through graph embedding\n",
      "Parasitologist-level classification of apicomplexan parasites and host cell with deep cycle transfer learning (DCTL)\n",
      "The Epigenetic Pacemaker: modeling epigenetic states under an evolutionary framework\n",
      "A Bayesian linear mixed model for prediction of complex traits\n",
      "iMRM: a platform for simultaneously identifying multiple kinds of RNA modifications\n",
      "Vargas: heuristic-free alignment for assessing linear and graph read aligners\n",
      "Geometric potentials from deep learning improve prediction of CDR H3 loop structures\n",
      "RTExtract: time-series NMR spectra quantification based on 3D surface ridge tracking\n",
      "TranspoScope: interactive visualization of retrotransposon insertions\n",
      "VISAR: an interactive tool for dissecting chemical features learned by deep neural network QSAR models\n",
      "CircMiner: accurate and rapid detection of circular RNA through splice-aware pseudo-alignment scheme\n",
      "ScaffoldGraph: an open-source library for the generation and analysis of molecular scaffold networks and scaffold trees\n",
      "Adversarial deconfounding autoencoder for learning robust gene expression embeddings\n",
      "Ensembling graph attention networks for human microbe-drug association prediction\n",
      "yacrd and fpa: upstream tools for long-read genome assembly\n",
      "Genome-wide association studies of brain imaging data via weighted distance correlation\n",
      "DeepCDR: a hybrid graph convolutional network for predicting cancer drug response\n",
      "MDIPA: a microRNA-drug interaction prediction approach based on non-negative matrix factorization\n",
      "Prediction of protein-binding residues: dichotomy of sequence-based methods developed using structured complexes versus disordered proteins\n",
      "Statistical significance of cluster membership for unsupervised evaluation of cell identities\n",
      "Learning context-aware structural representations to predict antigen and antibody binding interfaces\n",
      "GraphBin: refined binning of metagenomic contigs using assembly graphs\n",
      "Network-principled deep generative models for designing drug combinations as graph sets\n",
      "Improved survival analysis by learning shared genomic information from pan-cancer data\n",
      "PathWalks: identifying pathway communities using a disease-related map of integrated information\n",
      "MinReact: a systematic approach for identifying minimal metabolic networks\n",
      "Multipopulation harmony search algorithm for the detection of high-order SNP interactions\n",
      "HPOLabeler: improving prediction of human protein-phenotype associations by learning to rank\n",
      "MirCure: a tool for quality control, filter and curation of microRNAs of animals and plants\n",
      "immuneSIM: tunable multi-feature simulation of B- and T-cell receptor repertoires for immunoinformatics benchmarking\n",
      "AITL: Adversarial Inductive Transfer Learning with input and output space adaptation for pharmacogenomics\n",
      "Gcluster: a simple-to-use tool for visualizing and comparing genome contexts for numerous genomes\n",
      "Fast and accurate prediction of partial charges using Atom-Path-Descriptor-based machine learning\n",
      "The software for interactive evaluation of mass spectra stability and reproducibility\n",
      "BioModels Parameters: a treasure trove of parameter values from published systems biology models\n",
      "A library of coiled-coil domains: from regular bundles to peculiar twists\n",
      "PEDL: extracting protein-protein associations using deep language models and distant supervision\n",
      "FAME: fast and memory efficient multiple sequences alignment tool through compatible chain of roots\n",
      "DNA4mC-LIP: a linear integration method to identify N4-methylcytosine site in multiple species\n",
      "A reformulation of pLSA for uncertainty estimation and hypothesis testing in bio-imaging\n",
      "Can ODE gene regulatory models neglect time lag or measurement scaling?\n",
      "A general near-exact k-mer counting method with low memory consumption enables de novo assembly of 106× human sequence data in 2.7 hours\n",
      "BioPAX-Parser: parsing and enrichment analysis of BioPAX pathways\n",
      "CoMut: visualizing integrated molecular information with comutation plots\n",
      "RSSALib: a library for stochastic simulation of complex biochemical reactions\n",
      "BIRD: identifying cell doublets via biallelic expression from single cells\n",
      "EnrichrBot: Twitter bot tracking tweets about human genes\n",
      "Redundancy-weighting the PDB for detailed secondary structure prediction using deep-learning models\n",
      "DeepLGP: a novel deep learning method for prioritizing lncRNA target genes\n",
      "Dual-dropout graph convolutional network for predicting synthetic lethality in human cancers\n",
      "ProbeRating: a recommender system to infer binding profiles for nucleic acid-binding proteins\n",
      "Targeted domain assembly for fast functional profiling of metagenomic datasets with S3A\n",
      "RCSB Protein Data Bank 1D Tools and Services\n",
      "bGWAS: an R package to perform Bayesian genome wide association studies\n",
      "New mixture models for decoy-free false discovery rate estimation in mass spectrometry proteomics\n",
      "CAFE 5 models variation in evolutionary rates among gene families\n",
      "TCRBuilder: multi-state T-cell receptor structure prediction\n",
      "CRNT4SBML: a Python package for the detection of bistability in biochemical reaction networks\n",
      "A blind and independent benchmark study for detecting differeally methylated regions in plants\n",
      "SneakySnake: A Fast and Accurate Universal Genome Pre-Alignment Filter for CPUs, GPUs, and FPGAs\n",
      "IDDkin: Network-based influence deep diffusion model for enhancing prediction of kinase inhibitors\n",
      "LuxUS: DNA methylation analysis using generalized linear mixed model with spatial correlation\n",
      "scTPA: a web tool for single-cell transcriptome analysis of pathway activation signatures\n",
      "Terminus enables the discovery of data-driven, robust transcript groups from RNA-seq data\n",
      "BioStructures.jl: read, write and manipulate macromolecular structures in Julia\n",
      "An ultra-sensitive T-cell receptor detection method for TCR-Seq and RNA-Seq data\n",
      "The string decomposition problem and its applications to centromere analysis and assembly\n",
      "QuartataWeb: Integrated Chemical-Protein-Pathway Mapping for Polypharmacology and Chemogenomics\n",
      "Identifiability and experimental design in perturbation studies\n",
      "Batch equalization with a generative adversarial network\n",
      "EWASex: an efficient R-package to predict sex in epigenome-wide association studies\n",
      "LogoJS: a Javascript package for creating sequence logos and embedding them in web applications\n",
      "Automatic identification of relevant genes from low-dimensional embeddings of single-cell RNA-seq data\n",
      "TaxoNN: ensemble of neural networks on stratified microbiome data for disease prediction\n",
      "PRIME: a probabilistic imputation method to reduce dropout effects in single-cell RNA sequencing\n",
      "pepFunk: a tool for peptide-centric functional analysis of metaproteomic human gut microbiome studies\n",
      "Accurate prediction of genome-wide RNA secondary structure profile based on extreme gradient boosting\n",
      "Reanalysis of genome sequences of tomato accessions and its wild relatives: development of Tomato Genomic Variation (TGV) database integrating SNPs and INDELs polymorphisms\n",
      "HLPpred-Fuse: improved and robust prediction of hemolytic peptide and its activity by fusing multiple feature representation\n",
      "HAPPI GWAS: Holistic Analysis with Pre- and Post-Integration GWAS\n",
      "GeoBoost2: a natural languageprocessing pipeline for GenBank metadata enrichment for virus phylogeography\n",
      "qSNE: quadratic rate t-SNE optimizer with automatic parameter tuning for large datasets\n",
      "debCAM: a bioconductor R package for fully unsupervised deconvolution of complex tissues\n",
      "Learning transferable deep convolutional neural networks for the classification of bacterial virulence factors\n",
      "rmRNAseq: differential expression analysis for repeated-measures RNA-seq data\n",
      "Interactive visualization and analysis of morphological skeletons of brain vasculature networks with VessMorphoVis\n",
      "Chemical-protein interaction extraction via Gaussian probability distribution and external biomedical knowledge\n",
      "Metaviral SPAdes: assembly of viruses from metagenomic data\n",
      "Leitmotif: protein motif scanning 2.0\n",
      "Bayesian structural equation modeling in multiple omics data with application to circadian genes\n",
      "PROBselect: accurate prediction of protein-binding residues from proteins sequences via dynamic predictor selection\n",
      "Plant 3D (P3D): a plant phenotyping toolkit for 3D point clouds\n",
      "High-throughput modeling and scoring of TCR-pMHC complexes to predict cross-reactive peptides\n",
      "PASSION: an ensemble neural network approach for identifying the binding sites of RBPs on circRNAs\n",
      "EVICAN-a balanced dataset for algorithm development in cell and nucleus segmentation\n",
      "OCSANA+: optimal control and simulation of signaling networks from network analysis\n",
      "MIXnorm: normalizing RNA-seq data from formalin-fixed paraffin-embedded samples\n",
      "DeepTE: a computational method for de novo classification of transposons with convolutional neural network\n",
      "CPVA: a web-based metabolomic tool for chromatographic peak visualization and annotation\n",
      "Integrating biological knowledge and gene expression data using pathway-guided random forests: a benchmarking study\n",
      "Identification of population-level differentially expressed genes in one-phenotype data\n",
      "RobustClone: a robust PCA method for tumor clone and evolution inference from single-cell sequencing data\n",
      "Copy number evolution with weighted aberrations in cancer\n",
      "LinearPartition: linear-time approximation of RNA folding partition function and base-pairing probabilities\n",
      "Efficient exact inference for dynamical systems with noisy measurements using sequential approximate Bayesian computation\n",
      "PStrain: An Iterative Microbial Strains Profiling Algorithm for Shotgun Metagenomic Sequencing Data\n",
      "Atlas: Automatic modeling of regulation of bacterial gene expression and metabolism using rule-based languages\n",
      "GlycoGlyph: a glycan visualizing, drawing and naming application\n",
      "SCHNEL: scalable clustering of high dimensional single-cell data\n",
      "IDRMutPred: predicting disease-associated germline nonsynonymous single nucleotide variants (nsSNVs) in intrinsically disordered regions\n",
      "Galgo: a bi-objective evolutionary meta-heuristic identifies robust transcriptomic classifiers associated with patient outcome across multiple cancer types\n",
      "Factorized embeddings learns rich and biologically meaningful embedding spaces using factorized tensor decomposition\n",
      "Brewery: deep learning and deeper profiles for the prediction of 1D protein structure annotations\n",
      "Gaussian mixture model-based unsupervised nucleotide modification number detection using nanopore-sequencing readouts\n",
      "HALPER facilitates the identification of regulatory element orthologs across species\n",
      "gplas: a comprehensive tool for plasmid analysis using short-read graphs\n",
      "grabseqs: simple downloading of reads and metadata from multiple next-generation sequencing data repositories\n",
      "MOVICS: an R package for multi-omics integration and visualization in cancer subtyping\n",
      "Potential covalent drugs targeting the main protease of the SARS-CoV-2 coronavirus\n",
      "Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes\n",
      "PhISCS-BnB: a fast branch and bound algorithm for the perfect tumor phylogeny reconstruction problem\n",
      "ADFinder: accurate detection of programmed DNA elimination using NGS high-throughput sequencing data\n",
      "Probabilistic graphlets capture biological function in probabilistic molecular networks\n",
      "Phenotype-specific differences in polygenicity and effect size distribution across functional annotation categories revealed by AI-MiXeR\n",
      "DriverGroup: a novel method for identifying driver gene groups\n",
      "BYASE: a Python library for estimating gene and isoform level allele-specific expression\n",
      "panRGP: a pangenome-based method to predict genomic islands and explore their diversity\n",
      "Allowing mutations in maximal matches boosts genome compression performance\n",
      "DeCoDe: degenerate codon design for complete protein-coding DNA libraries\n",
      "crisscrosslinkeR: identification and visualization of protein-RNA and protein-protein interactions from crosslinking mass spectrometry\n",
      "2019 ISCB Overton Prize: Christophe Dessimoz\n",
      "Exploring generative deep learning for omics data using log-linear models\n",
      "An open-source high-content analysis workflow for CFTR function measurements using the forskolin-induced swelling assay\n",
      "Using AnABlast for intergenic sORF prediction in the Caenorhabditis elegans genome\n",
      "CNV-BAC: Copy number Variation Detection in Bacterial Circular Genome\n",
      "iSOM-GSN: an integrative approach for transforming multi-omic data into gene similarity networks via self-organizing maps\n",
      "Methrix: an R/bioconductor package for systematic aggregation and analysis of bisulfite sequencing data\n",
      "κ-helix and the helical lock and key model: a pivotal way of looking at polyproline II\n",
      "m7GHub: deciphering the location, regulation and pathogenesis of internal mRNA N7-methylguanosine (m7G) sites in human\n",
      "A multitask multiple kernel learning formulation for discriminating early- and late-stage cancers\n",
      "DeepNOG: Fast and accurate protein orthologous group assignment\n",
      "Network-based characterization of disease-disease relationships in terms of drugs and therapeutic targets\n",
      "MIAMI--a tool for non-targeted detection of metabolic flux changes for mode of action identification\n",
      "forgeNet: a graph deep neural network model using tree-based ensemble classifiers for feature graph construction\n",
      "Data-driven noise modeling of digital DNA melting analysis enables prediction of sequence discriminating power\n",
      "HaDeX: an R package and web-server for analysis of data from hydrogen-deuterium exchange mass spectrometry experiments\n",
      "Estimating chain length for time delays in dynamical systems using profile likelihood\n",
      "How to get your goat: automated identification of species from MALDI-ToF spectra\n",
      "Issues Of Z-factor and an approach to avoid them for quality control in high-throughput screening studies\n",
      "GLIDE: combining local methods and diffusion state embeddings to predict missing interactions in biological networks\n",
      "The effect of kinship in re-identification attacks against genomic data sharing beacons\n",
      "RAINFOREST: a random forest approach to predict treatment benefit in data from (failed) clinical drug trials\n",
      "Rapid epistatic mixed-model association studies by controlling multiple polygenic effects\n",
      "Improving circRNA-disease association prediction by sequence and ontology representations with convolutional and recurrent neural networks\n",
      "TissUUmaps: interactive visualization of large-scale spatial gene expression and tissue morphology data\n",
      "An in silico model of LINE-1-mediated neoplastic evolution\n",
      "Spathial: an R package for the evolutionary analysis of biological data\n",
      "iPromoter-BnCNN: a novel branched CNN-based predictor for identifying and classifying sigma promoters\n",
      "Improved Protein Model Quality Assessment By Integrating Sequential And Pairwise Features Using Deep Learning\n",
      "Feature selection and classification of noisy proteomics mass spectrometry data based on one-bit perturbed compressed sensing\n",
      "DeepSELEX: inferring DNA-binding preferences from HT-SELEX data using multi-class CNNs\n",
      "SphereCon-a method for precise estimation of residue relative solvent accessible area from limited structural information\n",
      "PPTPP: a novel therapeutic peptide prediction method using physicochemical property encoding and adaptive feature representation learning\n",
      "scDoc: correcting drop-out events in single-cell RNA-seq data\n",
      "ShallowHRD: detection of homologous recombination deficiency from shallow whole genome sequencing\n",
      "Converting networks to predictive logic models from perturbation signalling data with CellNOpt\n",
      "Palantir: a springboard for the analysis of secondary metabolite gene clusters in large-scale genome mining projects\n",
      "Identifying diagnosis-specific genotype-phenotype associations via joint multitask sparse canonical correlation analysis and classification\n",
      "Efficient weighted univariate clustering maps outstanding dysregulated genomic zones in human cancers\n",
      "DeMaSk: a deep mutational scanning substitution matrix and its use for variant impact prediction\n",
      "Basic polar and hydrophobic properties are the main characteristics that affect the binding of transcription factors to methylation sites\n",
      "projectR: an R/Bioconductor package for transfer learning via PCA, NMF, correlation and clustering\n",
      "A neuro-evolution approach to infer a Boolean network from time-series gene expressions\n",
      "Inferring cellular heterogeneity of associations from single cell genomics\n",
      "Fully interpretable deep learning model of transcriptional control\n",
      "New computational model for miRNA-mediated repression reveals novel regulatory roles of miRNA bindings inside the coding region\n",
      "rScudo: an R package for classification of molecular profiles using rank-based signatures\n",
      "corto: a lightweight R package for gene network inference and master regulator analysis\n",
      "Detecting Gene Ontology misannotations using taxon-specific rate ratio comparisons\n",
      "Stereo3D: using stereo images to enrich 3D visualization\n",
      "Benchmarking gene ontology function predictions using negative annotations\n",
      "Prediction of cancer driver genes through network-based moment propagation of mutation scores\n",
      "Annotation of tandem mass spectrometry data using stochastic neural networks in shotgun proteomics\n",
      "Connecting mathematical models to genomes: joint estimation of model parameters and genome-wide marker effects on these parameters\n",
      "FastMulRFS: fast and accurate species tree estimation under generic gene duplication and loss models\n",
      "Integrating multi-OMICS data through sparse canonical correlation analysis for the prediction of complex traits: a comparison study\n",
      "Helixer: Cross-species gene annotation of large eukaryotic genomes using deep learning\n",
      "Graph convolutional networks for epigenetic state prediction using both sequence and 3D genome data\n",
      "Screening novel drug candidates for Alzheimer's disease by an integrated network and transcriptome analysis\n",
      "COMER2: GPU-accelerated sensitive and specific homology searches\n",
      "iATC-FRAKEL: a simple multi-label web server for recognizing anatomical therapeutic chemical classes of drugs with their fingerprints only\n",
      "BeadNet: deep learning-based bead detection and counting in low-resolution microscopy images\n",
      "Serpentine: a flexible 2D binning method for differential Hi-C analysis\n",
      "SCLpred-EMS: subcellular localization prediction of endomembrane system and secretory pathway proteins by Deep N-to-1 Convolutional Neural Networks\n",
      "Matrix (factorization) reloaded: flexible methods for imputing genetic interactions with cross-species and side information\n",
      "OpenBioLink: a benchmarking framework for large-scale biomedical link prediction\n",
      "DNA Features Viewer: a sequence annotation formatting and plotting library for Python\n",
      "Interpretable factor models of single-cell RNA-seq via variational autoencoders\n",
      "EasyVS: a user-friendly web-based tool for molecule library selection and structure-based virtual screening\n",
      "HotSpot3D web server: an integrated resource for mutation analysis in protein 3D structures\n",
      "ASPRAlign: a tool for the alignment of RNA secondary structures with arbitrary pseudoknots\n",
      "Style transfer with variational autoencoders is a promising approach to RNA-Seq data harmonization and analysis\n",
      "TandemTools: mapping long reads and assessing/improving assembly quality in extra-long tandem repeats\n",
      "ClonoMatch: a tool for identifying homologous immunoglobulin and T cell receptor sequences in large databases\n",
      "MobiDB-lite 3.0: fast consensus annotation of intrinsic disorder flavours in proteins\n",
      "Resolving single-cell heterogeneity from hundreds of thousands of cells through sequential hybrid clustering and NMF\n",
      "Identifying tumor clones in sparse single-cell mutation data\n",
      "IgGeneUsage: differential gene usage in immune repertoires\n",
      "AciDB 1.0: a database of acidophilic organisms, their genomic information and associated metadata\n",
      "Reference bias in the Illumina Isaac aligner\n",
      "MAGGIE: leveraging genetic variation to identify DNA sequence motifs mediating transcription factor binding and function\n",
      "A boosted unbiased molecular dynamics method for predicting ligands binding mechanisms: probing the binding pathway of dasatinib to Src-kinase\n",
      "ConfID: an analytical method for conformational characterization of small molecules using molecular dynamics trajectories\n",
      "PLIDflow: an open-source workflow for the online analysis of protein-ligand docking using galaxy\n",
      "Predicting mechanism of action of cellular perturbations with pathway activity signatures\n",
      "Seqminer2: an efficient tool to query and retrieve genotypes for statistical genetics analyses from biobank scale sequence dataset\n",
      "MethPanel: a parallel pipeline and interactive analysis tool for multiplex bisulphite PCR sequencing to assess DNA methylation biomarker panels for disease detection\n",
      "Boosting the accuracy of protein secondary structure prediction through nearest neighbor search and method hybridization\n",
      "SugarPy facilitates the universal, discovery-driven analysis of intact glycopeptides\n",
      "GPress: a framework for querying general feature format (GFF) files and expression files in a compressed form\n",
      "PETAL: a python tool for deep analysis of biological pathways\n",
      "Cytomapper: an R/bioconductor package for visualisation of highly multiplexed imaging data\n",
      "CPS analysis: self-contained validation of biomedical data clustering\n",
      "Integration of gene interaction information into a reweighted Lasso-Cox model for accurate survival prediction\n",
      "GDASC: a GPU parallel-based web server for detecting hidden batch factors\n",
      "powmic: an R package for power assessment in microbiome case-control studies\n",
      "primirTSS: an R package for identifying cell-specific microRNA transcription start sites\n",
      "PROSS 2: a new server for the design of stable and highly expressed protein variants\n",
      "A Siamese neural network model for the prioritization of metabolic disorders by integrating real and simulated data\n",
      "Very Fast Tree: speeding up the estimation of phylogenies for large alignments through parallelization and vectorization strategies\n",
      "MutCombinator: identification of mutated peptides allowing combinatorial mutations using nucleotide-based graph search\n",
      "mirPLS: a partial linear structure identifier method for cancer subtyping using microRNAs\n",
      "MODifieR: an Ensemble R Package for Inference of Disease Modules from Transcriptomics Networks\n",
      "PICKLE 3.0: Enriching the human Meta-database with the mouse protein interactome extended via mouse-human orthology\n",
      "NIHBA: a network interdiction approach for metabolic engineering design\n",
      "Bio2Rxn: sequence-based enzymatic reaction predictions by a consensus strategy\n",
      "dv-trio: a family-based variant calling pipeline using DeepVariant\n",
      "RainbowSTORM: an open-source ImageJ plug-in for spectroscopic single-molecule localization microscopy (sSMLM) data analysis and image reconstruction\n",
      "REINDEER: efficient indexing of k-mer presence and abundance in sequencing datasets\n",
      "aCLImatise: Automated generation of tool definitions for bioinformatics workflows\n",
      "Modeling circRNA expression pattern with integrated sequence and epigenetic features demonstrates the potential involvement of H3K79me2 in circRNA expression\n",
      "Capybara: equivalence ClAss enumeration of coPhylogenY event-BAsed ReconciliAtions\n",
      "MaveQuest: a web resource for planning experimental tests of human variant effects\n",
      "genozip: a fast and efficient compression tool for VCF files\n",
      "Webina: an open-source library and web app that runs AutoDock Vina entirely in the web browser\n",
      "A graph regularized generalized matrix factorization model for predicting links in biomedical bipartite networks\n",
      "Inference of population admixture network from local gene genealogies: a coalescent-based maximum likelihood approach\n",
      "LeafCutterMD: an algorithm for outlier splicing detection in rare diseases\n",
      "proABC-2: PRediction of AntiBody contacts v2 and its application to information-driven docking\n",
      "Coronavirus3D: 3D structural visualization of COVID-19 genomic divergence\n",
      "MultiPaths: a python framework for analyzing multi-layer biological networks using diffusion algorithms\n",
      "PAFway: pairwise associations between functional annotations in biological networks and pathways\n",
      "MorphOT: Transport-based interpolation between EM maps with UCSF ChimeraX\n",
      "tugHall: a simulator of cancer-cell evolution based on the hallmarks of cancer and tumor-related genes\n",
      "Information theoretic generalized Robinson-Foulds metrics for comparing phylogenetic trees\n",
      "Single Cell Systems Analysis: Decision Geometry In Outliers\n",
      "Finding the direct optimal RNA barrier energy and improving pathways with an arbitrary energy model\n",
      "IGD: high-performance search for large-scale genomic interval datasets\n",
      "QDeep: distance-based protein model quality estimation by residue-level ensemble error classifications using stacked deep residual neural networks\n",
      "scHLAcount: allele-specific HLA expression from single-cell gene expression data\n",
      "ECCB2020: the 19th European Conference on Computational Biology\n",
      "Mainzelliste SecureEpiLinker (MainSEL): Privacy-Preserving Record Linkage using Secure Multi-Party Computation\n",
      "annonex2embl: automatic preparation of annotated DNA sequences for bulk submissions to ENA\n",
      "DeepAntigen: a novel method for neoantigen prioritization via 3D genome and deep sparse learning\n",
      "COVID-19 Docking Server: a meta server for docking small molecules, peptides and antibodies against potential targets of COVID-19\n",
      "CorGAT: a tool for the functional annotation of SARS-CoV-2 genomes\n",
      "Automatic classification of single-molecule force spectroscopy traces from heterogeneous samples\n",
      "nanoTRON: a Picasso module for MLP-based classification of super-resolution data\n",
      "2DImpute: imputation in single-cell RNA-seq data from correlations in two dimensions\n",
      "CRiSP: accurate structure prediction of disulfide-rich peptides with cystine-specific sequence alignment and machine learning\n",
      "IBDkin: fast estimation of kinship coefficients from identity by descent segments\n",
      "COVID-2019-associated overexpressed Prevotella proteins mediated host-pathogen interactions and their role in coronavirus outbreak\n",
      "ScaleQC: a scalable lossy to lossless solution for NGS data compression\n",
      "Metasubtract: an R-package to analytically produce leave-one-out meta-analysis GWAS summary statistics\n",
      "PRANC: ML species tree estimation from the ranked gene trees under coalescence\n",
      "ESTIpop: a computational tool to simulate and estimate parameters for continuous-time Markov branching processes\n",
      "FastTargetPred: a program enabling the fast prediction of putative protein targets for input chemical databases\n",
      "Identification of Sub-Golgi protein localization by use of deep representation learning features\n",
      "IsoMaTrix: a framework to visualize the isoclines of matrix games and quantify uncertainty in structured populations\n",
      "GREMA: modelling of emulated gene regulatory networks with confidence levels based on evolutionary intelligence to cope with the underdetermined problem\n",
      "svMIL: predicting the pathogenic effect of TAD boundary-disrupting somatic structural variants through multiple instance learning\n",
      "Accurate calling of homeoallelic genotypes of iSelect markers using inbred structured populations\n",
      "cd2sbgnml: bidirectional conversion between CellDesigner and SBGN formats\n",
      "Bonnie Berger named ISCB 2019 ISCB Accomplishments by a Senior Scientist Award recipient\n",
      "LBERT: Lexically-aware Transformers based Bidirectional Encoder Representation model for learning Universal Bio-Entity Relations\n",
      "Revealing new therapeutic opportunities through drug target prediction: a class imbalance-tolerant machine learning approach\n",
      "iBioProVis: interactive visualization and analysis of compound bioactivity space\n",
      "Erratum: LFQC: A lossless compression algorithm for FASTQ files\n",
      "RNAIndel: discovering somatic coding indels from tumor RNA-Seq data\n",
      "Boosting the extraction of elementary flux modes in genome-scale metabolic networks using the linear programming approach\n",
      "amplimap: a versatile tool to process and analyze targeted NGS data\n",
      "Gene regulation inference from single-cell RNA-seq data with linear differential equations and velocity inference\n",
      "2019 Outstanding Contributions to ISCB Awarded to Barb Bryant\n",
      "2019 ISCB Innovator Award Recognizes William Stafford Noble\n",
      "Discovery of disease- and drug-specific pathways through community structures of a literature network\n",
      "A Blind and Independent Benchmark Study for Detecting Differentially Methylated Regions in Plants\n",
      "Corrigendum to: HALPER facilitates the identification of regulatory element orthologs across species\n",
      "Comment on 'Hayai-Annotation plants: an ultra-fast and comprehensive functional gene annotation system in plants'. The importance of taking the GO graph structure into account\n",
      "Fast and accurate correction of optical mapping data via spaced seeds\n",
      "Phage protein receptors have multiple interaction partners and high expressions\n",
      "6mA-Finder: a novel online tool for predicting DNA N6-methyladenine sites in genomes\n",
      "OrgDyn: feature- and model-based characterization of spatial and temporal organoid dynamics\n",
      "NERDD: a web portal providing access to in silico tools for drug discovery\n",
      "scTSSR: gene expression recovery for single-cell RNA sequencing using two-side sparse self-representation\n",
      "SPsimSeq: semi-parametric simulation of bulk and single-cell RNA-sequencing data\n",
      "About the need to make computational models of biological macromolecules available and discoverable\n",
      "ShinySOM: graphical SOM-based analysis of single-cell cytometry data\n",
      "Occupancy spectrum distribution: application for coalescence simulation with generic mergers\n",
      "pyBedGraph: a python package for fast operations on 1D genomic signal tracks\n",
      "Diversity of rhodopsins in cultivated bacteria of the family Geodermatophilaceae associated with non-aquatic environments\n",
      "An examination of citation-based impact of the computational biology conferences\n",
      "Coolpup.py: versatile pile-up analysis of Hi-C data\n",
      "BioSeqZip: a collapser of NGS redundant reads for the optimization of sequence analysis\n",
      "AptCompare: optimized de novo motif discovery of RNA aptamers via HTS-SELEX\n",
      "DRIMC: an improved drug repositioning approach using Bayesian inductive matrix completion\n",
      "Bivartect: accurate and memory-saving breakpoint detection by direct read comparison\n",
      "The Feature-Viewer: a visualization tool for positional annotations on a sequence\n",
      "Consensus features nested cross-validation\n",
      "CoMeBack: DNA methylation array data analysis for co-methylated regions\n",
      "GSOAP: a tool for visualization of gene set over-representation analysis\n",
      "PARC: ultrafast and accurate clustering of phenotypic data of millions of single cells\n",
      "CytoSeg 2.0: automated extraction of actin filaments\n",
      "Transcompp: understanding phenotypic plasticity by estimating Markov transition rates for cell state transitions\n",
      "Lineage EM algorithm for inferring latent states from cellular lineage trees\n",
      "Transfer index, NetUniFrac and some useful shortest path-based distances for community analysis in sequence similarity networks\n",
      "Visualization of circular RNAs and their internal splicing events from transcriptomic data\n",
      "Corrigendum to: Single-sample landscape entropy reveals the imminent phase transition during disease progression\n",
      "The Protein Imager: a full-featured online molecular viewer interface with server-side HQ-rendering capabilities\n",
      "Nearest-neighbor Projected-Distance Regression (NPDR) for detecting network interactions with adjustments for multiple tests and confounding\n",
      "PISA-SPARKY: an interactive SPARKY plugin to analyze oriented solid-state NMR spectra of helical membrane proteins\n",
      "PamgeneAnalyzeR: open and reproducible pipeline for kinase profiling\n",
      "contamDE-lm: linear model-based differential gene expression analysis using next-generation RNA-seq data from contaminated tumor samples\n",
      "Lep-Anchor: automated construction of linkage map anchored haploid genomes\n",
      "iq: an R package to estimate relative protein abundances from ion quantification in DIA-MS-based proteomics\n",
      "Accurate prediction of protein beta-aggregation with generalized statistical potentials\n",
      "SANPolyA: a deep learning method for identifying Poly(A) signals\n",
      "ipyrad: Interactive assembly and analysis of RADseq datasets\n",
      "Fast zero-inflated negative binomial mixed modeling approach for analyzing longitudinal metagenomics data\n",
      "MatchMixeR: a cross-platform normalization method for gene expression data integration\n",
      "PgRC: pseudogenome-based read compressor\n",
      "qgg: an R package for large-scale quantitative genetic analyses\n",
      "VEF: a variant filtering tool based on ensemble methods\n",
      "CGLFold: a contact-assisted de novo protein structure prediction using global exploration and loop perturbation sampling algorithm\n",
      "Complexity measures of the mature miRNA for improving pre-miRNAs prediction\n",
      "MPF-BML: a standalone GUI-based package for maximum entropy model inference\n",
      "HaploTypo: a variant-calling pipeline for phased genomes\n",
      "MLDSP-GUI: an alignment-free standalone tool with an interactive graphical user interface for DNA sequence comparison and analysis\n",
      "Lean and deep models for more accurate filtering of SNP and INDEL variant calls\n",
      "pyDockEneRes: per-residue decomposition of protein-protein docking energy\n",
      "Toward perfect reads: self-correction of short reads via mapping on de Bruijn graphs\n",
      "LIMES: a tool for comparing species partition\n",
      "DNA Readout Viewer (DRV): visualization of specificity determining patterns of protein-binding DNA segments\n",
      "Dynamic Meta-Storms enables comprehensive taxonomic and phylogenetic comparison of shotgun metagenomes at the species level\n",
      "Phylonium: fast estimation of evolutionary distances from large samples of similar genomes\n",
      "Point-group symmetry detection in three-dimensional charge density of biomolecules\n",
      "CATHER: a novel threading algorithm with predicted contacts\n",
      "Predicting synthetic lethal interactions using heterogeneous data sources\n",
      "Struo: a pipeline for building custom databases for common metagenome profilers\n",
      "APTANI2: update of aptamer selection through sequence-structure analysis\n",
      "SimRVSequences: an R package to simulate genetic sequence data for pedigrees\n",
      "NeoFuse: predicting fusion neoantigens from RNA sequencing data\n",
      "Fuzzy modeling and global optimization to predict novel therapeutic targets in cancer cells\n",
      "Protein docking model evaluation by 3D deep convolutional neural networks\n",
      "Improved linking of motifs to their TFs using domain information\n",
      "KofamKOALA: KEGG Ortholog assignment based on profile HMM and adaptive score threshold\n",
      "BRM: a statistical method for QTL mapping based on bulked segregant analysis by deep sequencing\n",
      "NCutYX: a package for clustering analysis of multilayer omics data\n",
      "DiscoRhythm: an easy-to-use web application and R package for discovering rhythmicity\n",
      "Differential privacy under dependent tuples-the case of genomic privacy\n",
      "SigHotSpotter: scRNA-seq-based computational tool to control cell subpopulation phenotypes for cellular rejuvenation strategies\n",
      "WPMIAS: Whole-degradome-based Plant MicroRNA-Target Interaction Analysis Server\n",
      "Model-based clustering of multi-tissue gene expression data\n",
      "PRODIGY: personalized prioritization of driver genes\n",
      "Jasmine: a Java pipeline for isomiR characterization in miRNA-Seq Data\n",
      "ImSpectR - R package to quantify immune repertoire diversity in spectratype and repertoire sequencing data\n",
      "PlanExp: intuitive integration of complex RNA-seq datasets with planarian omics resources\n",
      "SINC: a scale-invariant deep-neural-network classifier for bulk and single-cell RNA-seq data\n",
      "SMARTS: the social media-based addiction recovery and intervention targeting server\n",
      "DPre: computational identification of differentiation bias and genes underlying cell type conversions\n",
      "SpliceLauncher: a tool for detection, annotation and relative quantification of alternative junctions from RNAseq data\n",
      "An improved encoding of genetic variation in a Burrows-Wheeler transform\n",
      "Exploiting transfer learning for the reconstruction of the human gene regulatory network\n",
      "SubMito-XGBoost: predicting protein submitochondrial localization by fusing multiple feature information and eXtreme gradient boosting\n",
      "SOLart: a structure-based method to predict protein solubility and aggregation\n",
      "Deep-learning approach to identifying cancer subtypes using high-dimensional genomic data\n",
      "FullMeSH: improving large-scale MeSH indexing with full text\n",
      "Eagle: multi-locus association mapping on a genome-wide scale made routine\n",
      "MetumpX-a metabolomics support package for untargeted mass spectrometry\n",
      "Protein-protein interaction site prediction through combining local and global features with deep neural networks\n",
      "NMF-RI: blind spectral unmixing of highly mixed multispectral flow and image cytometry data\n",
      "WASPS: web-assisted symbolic plasmid synteny server\n",
      "Fast and flexible coarse-grained prediction of protein folding routes using ensemble modeling and evolutionary sequence variation\n",
      "Proteome-level assessment of origin, prevalence and function of leucine-aspartic acid (LD) motifs\n",
      "Deep-learning with synthetic data enables automated picking of cryo-EM particle images of biological macromolecules\n",
      "bayNorm: Bayesian gene expression recovery, imputation and normalization for single-cell RNA-sequencing data\n",
      "simurg: simulate bacterial pangenomes in R\n",
      "ChIPseqSpikeInFree: a ChIP-seq normalization approach to reveal global changes in histone modifications without spike-in\n",
      "EvolClust: automated inference of evolutionary conserved gene clusters in eukaryotes\n",
      "Pavian: interactive analysis of metagenomics data for microbiome studies and pathogen identification\n",
      "XMAn v2-a database of Homo sapiens mutated peptides\n",
      "DeepMSPeptide: peptide detectability prediction using deep learning\n",
      "Multifaceted protein-protein interaction prediction based on Siamese residual RCNN\n",
      "Statistical compression of protein sequences and inference of marginal probability landscapes over competing alignments using finite state models and Dirichlet priors\n",
      "MOLI: multi-omics late integration with deep neural networks for drug response prediction\n",
      "PRISM: methylation pattern-based, reference-free inference of subclonal makeup\n",
      "hicGAN infers super resolution Hi-C data with generative adversarial networks\n",
      "Inheritance and variability of kinetic gene expression parameters in microbial cells: modeling and inference from lineage tree data\n",
      "A divide-and-conquer method for scalable phylogenetic network inference from multilocus data\n",
      "Inferring signalling dynamics by integrating interventional with observational data\n",
      "FunDMDeep-m6A: identification and prioritization of functional differential m6A methylation genes\n",
      "Alignment-free filtering for cfNA fusion fragments\n",
      "TideHunter: efficient and sensitive tandem repeat detection from noisy long-reads using seed-and-chain\n",
      "Adversarial domain adaptation for cross data source macromolecule in situ structural classification in cellular electron cryo-tomograms\n",
      "Large-scale inference of competing endogenous RNA networks with sparse partial correlation\n",
      "Estimating the predictability of cancer evolution\n",
      "Representation transfer for differentially private drug sensitivity prediction\n",
      "Drug repositioning based on bounded nuclear norm regularization\n",
      "Selfish: discovery of differential chromatin interactions via a self-similarity measure\n",
      "Fully-sensitive seed finding in sequence graphs using a hybrid index\n",
      "Building large updatable colored de Bruijn graphs via merging\n",
      "cloudSPAdes: assembly of synthetic long reads using de Bruijn graphs\n",
      "ADAPTIVE: leArning DAta-dePendenT, concIse molecular VEctors for fast, accurate metabolite identification from tandem mass spectra\n",
      "CroP-Coordinated Panel visualization for biological networks analysis\n",
      "Applying citizen science to gene, drug and disease relationship extraction from biomedical abstracts\n",
      "VisFeature: a stand-alone program for visualizing and analyzing statistical features of biological sequences\n",
      "Tissue-specific deconvolution of immune cell composition by integrating bulk and single-cell transcriptomes\n",
      "Analysis of several key factors influencing deep learning-based inter-residue contact prediction\n",
      "A Gaussian process-based definition reveals new and bona fide genetic interactions compared to a multiplicative model in the Gram-negative Escherichia coli\n",
      "COMBAT-TB-NeoDB: fostering tuberculosis research through integrative analysis using graph database technologies\n",
      "Accurate loop calling for 3D genomic data with cLoops\n",
      "scds: computational annotation of doublets in single-cell RNA sequencing data\n",
      "MEXCOwalk: mutual exclusion and coverage based random walk to identify cancer modules\n",
      "Machine learning empowers phosphoproteome prediction in cancers\n",
      "ROBOKOP: an abstraction layer and user interface for knowledge graphs to support question answering\n",
      "U-PASS: unified power analysis and forensics for qualitative traits in genetic association studies\n",
      "Performance of gene expression analyses using de novo assembled transcripts in polyploid species\n",
      "G3viz: an R package to interactively visualize genetic mutation data using a lollipop-diagram\n",
      "Integrating ab initio and template-based algorithms for protein-protein complex structure prediction\n",
      "RICOPILI: Rapid Imputation for COnsortias PIpeLIne\n",
      "Tersect: a set theoretical utility for exploring sequence variant data\n",
      "scHinter: imputing dropout events for single-cell RNA-seq data with limited sample size\n",
      "KRAB ZNF explorer-the online tool for the exploration of the transcriptomic profiles of KRAB-ZNF factors in The Cancer Genome Atlas\n",
      "gwasrapidd: an R package to query, download and wrangle GWAS catalog data\n",
      "BLANT-fast graphlet sampling tool\n",
      "OncodriveCLUSTL: a sequence-based clustering method to identify cancer drivers\n",
      "Tailor-made multiple sequence alignments using the PRALINE 2 alignment toolkit\n",
      "Gaussian mixture copulas for high-dimensional clustering and dependency-based subtyping\n",
      "P-value evaluation, variability index and biomarker categorization for adaptively weighted Fisher's meta-analysis method in omics applications\n",
      "CoBAMP: a Python framework for metabolic pathway analysis in constraint-based models\n",
      "ACTINN: automated identification of cell types in single cell RNA sequencing\n",
      "pourRNA-a time- and memory-efficient approach for the guided exploration of RNA energy landscapes\n",
      "DISPOT: a simple knowledge-based protein domain interaction statistical potential\n",
      "The nPYc-Toolbox, a Python module for the pre-processing, quality-control and analysis of metabolic profiling datasets\n",
      "Genetic association testing using the GENESIS R/Bioconductor package\n",
      "How sequence alignment scores correspond to probability models\n",
      "Identifying emerging phenomenon in long temporal phenotyping experiments\n",
      "MM-6mAPred: identifying DNA N6-methyladenine sites based on Markov model\n",
      "HDX-Viewer: interactive 3D visualization of hydrogen-deuterium exchange data\n",
      "VeriNA3d: an R package for nucleic acids data mining\n",
      "Artificial intelligence-based multi-objective optimization protocol for protein structure refinement\n",
      "Inference of differential gene regulatory networks based on gene expression and genetic perturbation data\n",
      "JUCHMME: a Java Utility for Class Hidden Markov Models and Extensions for biological sequence analysis\n",
      "Isoform function prediction based on bi-random walks on a heterogeneous network\n",
      "HUNER: improving biomedical NER with pretraining\n",
      "PhenoScanner V2: an expanded tool for searching human genotype-phenotype associations\n",
      "DEUS: an R package for accurate small RNA profiling based on differential expression of unique sequences\n",
      "KinomeX: a web application for predicting kinome-wide polypharmacology effect of small molecules\n",
      "DeepMito: accurate prediction of protein sub-mitochondrial localization using convolutional neural networks\n",
      "Testing clonal relatedness of two tumors from the same patient based on their mutational profiles: update of the Clonality R package\n",
      "An empirical Bayesian ranking method, with applications to high throughput biology\n",
      "Mycorrhiza: genotype assignment using phylogenetic networks\n",
      "Cloud bursting galaxy: federated identity and access management\n",
      "RAISS: robust and accurate imputation from summary statistics\n",
      "capC-MAP: software for analysis of Capture-C data\n",
      "P3DOCK: a protein-RNA docking webserver based on template-based and template-free docking\n",
      "NX4: a web-based visualization of large multiple sequence alignments\n",
      "AutoDock CrankPep: combining folding and docking to predict protein-peptide complexes\n",
      "RNA-align: quick and accurate alignment of RNA 3D structures based on size-independent TM-scoreRNA\n",
      "eFORGE v2.0: updated analysis of cell type-specific signal in epigenomic data\n",
      "VCF/Plotein: visualization and prioritization of genomic variants from human exome sequencing projects\n",
      "MTTFsite: cross-cell type TF binding site prediction by using multi-task learning\n",
      "relax: the analysis of biomolecular kinetics and thermodynamics using NMR relaxation dispersion data\n",
      "Augmented Interval List: a novel data structure for efficient genomic interval search\n",
      "Path2Surv: Pathway/gene set-based survival analysis using multiple kernel learning\n",
      "PRODIGY-crystal: a web-tool for classification of biological interfaces in protein complexes\n",
      "CNet: a multi-omics approach to detecting clinically associated, combinatory genomic signatures\n",
      "Seq2Feature: a comprehensive web-based feature extraction tool\n",
      "C-InterSecture-a computational tool for interspecies comparison of genome architecture\n",
      "SPar-K: a method to partition NGS signal data\n",
      "Corrigendum to: A new cis-acting regulatory element driving gene expression in the zebrafish pineal gland\n",
      "Cram-JS: reference-based decompression in node and the browser\n",
      "Iterative feature representations improve N4-methylcytosine site prediction\n",
      "ntEdit: scalable genome sequence polishing\n",
      "Prediction of survival risks with adjusted gene expression through risk-gene networks\n",
      "multiPhATE: bioinformatics pipeline for functional annotation of phage isolates\n",
      "CaverDock: a molecular docking-based tool to analyse ligand transport through protein tunnels and channels\n",
      "Efficient multivariate analysis algorithms for longitudinal genome-wide association studies\n",
      "Graphical data mining of cancer mechanisms with SEMA\n",
      "A decision-theoretic approach to the evaluation of machine learning algorithms in computational drug discovery\n",
      "Identification of differentially expressed gene sets using the Generalized Berk-Jones statistic\n",
      "RETRACTED: LFQC: a lossless compression algorithm for FASTQ files\n",
      "LAmbDA: label ambiguous domain adaptation dataset integration reduces batch effects and improves subtype detection\n",
      "Continuous-state HMMs for modeling time-series single-cell RNA-Seq data\n",
      "icHET: interactive visualization of cytoplasmic heteroplasmy\n",
      "scMatch: a single-cell gene expression profile annotation tool using reference datasets\n",
      "DeepSignal: detecting DNA methylation state from Nanopore sequencing reads using deep-learning\n",
      "PEPred-Suite: improved and robust prediction of therapeutic peptides using adaptive feature representation learning\n",
      "Spliceogen: an integrative, scalable tool for the discovery of splice-altering variants\n",
      "PyPore: a python toolbox for nanopore sequencing data handling\n",
      "Biomedical image augmentation using Augmentor\n",
      "SolidBin: improving metagenome binning with semi-supervised normalized cut\n",
      "Accounting for ambiguity in ancestral sequence reconstruction\n",
      "TORMES: an automated pipeline for whole bacterial genome analysis\n",
      "scRNAss: a single-cell RNA-seq assembler via imputing dropouts and combing junctions\n",
      "IntronDB: a database for eukaryotic intron features\n",
      "Network inference performance complexity: a consequence of topological, experimental and algorithmic determinants\n",
      "A Bayesian model integration for mutation calling through data partitioning\n",
      "epic2 efficiently finds diffuse domains in ChIP-seq data\n",
      "rMTA: robust metabolic transformation analysis\n",
      "Graph analysis of fragmented long-read bacterial genome assemblies\n",
      "Neural networks with circular filters enable data efficient inference of sequence motifs\n",
      "parMATT: parallel multiple alignment of protein 3D-structures with translations and twists for distributed-memory systems\n",
      "SummaryAUC: a tool for evaluating the performance of polygenic risk prediction models in validation datasets with only summary level statistics\n",
      "Dissecting differential signals in high-throughput data from complex tissues\n",
      "BrAPI-an application programming interface for plant breeding applications\n",
      "Enhanced Waddington landscape model with cell-cell communication can explain molecular mechanisms of self-organization\n",
      "ORE identifies extreme expression effects enriched for rare variants\n",
      "FLAS: fast and high-throughput algorithm for PacBio long-read self-correction\n",
      "ScanNeo: identifying indel-derived neoantigens using RNA-Seq data\n",
      "A Bayesian model for single cell transcript expression analysis on MERFISH data\n",
      "Adjutant: an R-based tool to support topic discovery for systematic and literature reviews\n",
      "FoldX 5.0: working with RNA, small molecules and a new graphical interface\n",
      "ECOGEMS: efficient compression and retrieve of SNP data of 2058 rice accessions with integer sparse matrices\n",
      "phylostratr: a framework for phylostratigraphy\n",
      "PleioNet: a web-based visualization tool for exploring pleiotropy across complex traits\n",
      "Multiresolution correction of GC bias and application to identification of copy number alterations\n",
      "SCL: a lattice-based approach to infer 3D chromosome structures from single-cell Hi-C data\n",
      "pldist: ecological dissimilarities for paired and longitudinal microbiome association analysis\n",
      "DEFOR: depth- and frequency-based somatic copy number alteration detector\n",
      "A new computational drug repurposing method using established disease-drug pair knowledge\n",
      "eBDIMS server: protein transition pathways with ensemble analysis in 2D-motion spaces\n",
      "MetaMarker: a pipeline for de novo discovery of novel metagenomic biomarkers\n",
      "AutoDock Bias: improving binding mode prediction and virtual screening using known protein-ligand interactions\n",
      "Enhancing the prediction of disease-gene associations with multimodal deep learning\n",
      "deTS: tissue-specific enrichment analysis to decode tissue specificity\n",
      "Surrogate minimal depth as an importance measure for variables in random forests\n",
      "FORESEE: a tool for the systematic comparison of translational drug response modeling pipelines\n",
      "SW-Tandem: a highly efficient tool for large-scale peptide identification with parallel spectrum dot product on Sunway TaihuLight\n",
      "DIFFpop: a stochastic computational approach to simulate differentiation hierarchies with single cell barcoding\n",
      "CausalTAB: the PSI-MITAB 2.8 updated format for signalling data representation and dissemination\n",
      "Toward perfect reads: self-correction of short reads via mapping on de Bruijn graphs\n",
      "SUBSTRA: Supervised Bayesian Patient Stratification\n",
      "Dhaka: Variational Autoencoder for Unmasking Tumor Heterogeneity from Single Cell Genomic Data\n",
      "3DBIONOTES v3.0: crossing molecular and structural biology data with genomic variations\n",
      "Temporal network alignment via GoT-WAVE\n",
      "Integrating hypertension phenotype and genotype with hybrid non-negative matrix factorization\n",
      "Finding de novo methylated DNA motifs\n",
      "KOnezumi: a web application for automating gene disruption strategies to generate knockout mice\n",
      "Miso: an R package for multiple isotope labeling assisted metabolomics data analysis\n",
      "Incorporating alignment uncertainty into Felsenstein's phylogenetic bootstrap to improve its reliability\n",
      "How well do RNA-Seq differential gene expression tools perform in a complex eukaryote? A case study in Arabidopsis thaliana\n",
      "PMC text mining subset in BioC: about three million full-text articles and growing\n",
      "ebGSEA: an improved Gene Set Enrichment Analysis method for Epigenome-Wide-Association Studies\n",
      "PROSSTT: probabilistic simulation of single-cell RNA-seq data for complex differentiation processes\n",
      "Batch-normalization of cerebellar and medulloblastoma gene expression datasets utilizing empirically defined negative control genes\n",
      "Top-Down Garbage Collector: a tool for selecting high-quality top-down proteomics mass spectra\n",
      "A unified approach for sparse dynamical system inference from temporal measurements\n",
      "Calculation of accurate interatomic contact surface areas for the quantitative analysis of non-bonded molecular interactions\n",
      "Aligning optical maps to de Bruijn graphs\n",
      "GRIMM: GRaph IMputation and matching for HLA genotypes\n",
      "DASHR 2.0: integrated database of human small non-coding RNA genes and mature products\n",
      "SRG extractor: a skinny reference genome approach for reduced-representation sequencing\n",
      "InterMineR: an R package for InterMine databases\n",
      "CMEP: a database for circulating microRNA expression profiling\n",
      "AllerCatPro-prediction of protein allergenicity potential from the protein sequence\n",
      "Improved indel detection in DNA and RNA via realignment with ABRA2\n",
      "Sstack: an R package for stacking with applications to scenarios involving sequential addition of samples and features\n",
      "Characterization and identification of long non-coding RNAs based on feature relationship\n",
      "KORP: knowledge-based 6D potential for fast protein and loop modeling\n",
      "Identifying residues that determine palmitoylation using association rule mining\n",
      "COmplexome Profiling ALignment (COPAL) reveals remodeling of mitochondrial protein complexes in Barth syndrome\n",
      "Crisflash: open-source software to generate CRISPR guide RNAs against genomes annotated with individual variation\n",
      "MULTiPly: a novel multi-layer predictor for discovering general and specific types of promoters\n",
      "The DNA walk and its demonstration of deterministic chaos-relevance to genomic alterations in lung cancer\n",
      "CRISPR-DT: designing gRNAs for the CRISPR-Cpf1 system with improved target efficiency and specificity\n",
      "Simulation of heterogeneous tumour genomes with HeteroGenesis and in silico whole exome sequencing\n",
      "DeepCoil-a fast and accurate prediction of coiled-coil domains in protein sequences\n",
      "BreakID: genomics breakpoints identification to detect gene fusion events using discordant pairs and split reads\n",
      "AnnoFly: annotating Drosophila embryonic images based on an attention-enhanced RNN model\n",
      "ACE: absolute copy number estimation from low-coverage whole-genome sequencing data\n",
      "Improved mutant function prediction via PACT: Protein Analysis and Classifier Toolkit\n",
      "CSHAP: efficient haplotype frequency estimation based on sparse representation\n",
      "PconsC4: fast, accurate and hassle-free contact predictions\n",
      "Smooth orientation-dependent scoring function for coarse-grained protein quality assessment\n",
      "Very low-depth whole-genome sequencing in complex trait association studies\n",
      "Exploring drivers of gene expression in the Cancer Genome Atlas\n",
      "lordFAST: sensitive and Fast Alignment Search Tool for LOng noisy Read sequencing Data\n",
      "NeoDTI: neural integration of neighbor information from a heterogeneous network for discovering new drug-target interactions\n",
      "Multiomics modeling of the immunome, transcriptome, microbiome, proteome and metabolome adaptations during human pregnancy\n",
      "pyTFA and matTFA: a Python package and a Matlab toolbox for Thermodynamics-based Flux Analysis\n",
      "GrandPrix: scaling up the Bayesian GPLVM for single-cell data\n",
      "ModL: exploring and restoring regularity when testing for positive selection\n",
      "DeepIsoFun: a deep domain adaptation approach to predict isoform functions\n",
      "Evaluation of methods for generative modeling of cell and nuclear shape\n",
      "MapOptics: a light-weight, cross-platform visualization tool for optical mapping alignment\n",
      "Deciphering the complexity of human non-coding promoter-proximal transcriptome\n",
      "SPRING: a next-generation compressor for FASTQ data\n",
      "BaiHui: cross-species brain-specific network built with hundreds of hand-curated datasets\n",
      "New models of atherosclerosis and multi-drug therapeutic interventions\n",
      "Global Biobank Engine: enabling genotype-phenotype browsing for biobank summary statistics\n",
      "ARGDIT: a validation and integration toolkit for Antimicrobial Resistance Gene Databases\n",
      "ConDo: protein domain boundary prediction using coevolutionary information\n",
      "Computational functional genomics-based reduction of disease-related gene sets to their key components\n",
      "GUV-AP: multifunctional FIJI-based tool for quantitative image analysis of Giant Unilamellar Vesicles\n",
      "A plugin for the Ensembl Variant Effect Predictor that uses MaxEntScan to predict variant spliceogenicity\n",
      "ProteomicsBrowser: MS/proteomics data visualization and investigation\n",
      "Application of machine learning techniques to tuberculosis drug resistance analysis\n",
      "Directional allelic imbalance profiling and visualization from multi-sample data with RECUR\n",
      "BulkVis: a graphical viewer for Oxford nanopore bulk FAST5 files\n",
      "DiPhiSeq: robust comparison of expression levels on RNA-Seq data with large sample sizes\n",
      "Genetic Neural Networks: an artificial neural network architecture for capturing gene expression relationships\n",
      "pyNVR: investigating factors affecting feature selection from scRNA-seq data for lineage reconstruction\n",
      "PAVOOC: designing CRISPR sgRNAs using 3D protein structures and functional domain annotations\n",
      "Pair Matcher (PaM): fast model-based optimization of treatment/case-control matches\n",
      "Metrics for regulated biochemical pathway systems\n",
      "Analysis of single amino acid variations in singlet hot spots of protein-protein interfaces\n",
      "Quantitative trait loci identification for brain endophenotypes via new additive model with random networks\n",
      "CisPi: a transcriptomic score for disclosing cis-acting disease-associated lincRNAs\n",
      "DeepDTA: deep drug-target binding affinity prediction\n",
      "Bayesian inference on stochastic gene transcription from flow cytometry data\n",
      "An accurate and rapid continuous wavelet dynamic time warping algorithm for end-to-end mapping in ultra-long nanopore sequencing\n",
      "iTOP: inferring the topology of omics data\n",
      "PAIPline: pathogen identification in metagenomic and clinical next generation sequencing samples\n",
      "Ontology-based validation and identification of regulatory phenotypes\n",
      "Conditional generative adversarial network for gene expression inference\n",
      "Recognition of CRISPR/Cas9 off-target sites through ensemble learning of uneven mismatch distributions\n",
      "Discovering epistatic feature interactions from neural network models of regulatory DNA sequences\n",
      "Understanding the evolution of functional redundancy in metabolic networks\n",
      "A Bayesian model for single cell transcript expression analysis on MERFISH data\n",
      "TissueEnrich: Tissue-specific gene enrichment analysis\n",
      "PopLDdecay: a fast and effective tool for linkage disequilibrium decay analysis based on variant call format files\n",
      "Expanded functionality, increased accuracy, and enhanced speed in the de novo genotyping-by-sequencing pipeline GBS-SNP-CROP\n",
      "Colocalization analyses of genomic elements: approaches, recommendations and challenges\n",
      "Multimodal network diffusion predicts future disease-gene-chemical associations\n",
      "Towards region-specific propagation of protein functions\n",
      "PrecursorFinder: a customized biosynthetic precursor explorer\n",
      "TIVAN: tissue-specific cis-eQTL single nucleotide variant annotation and prediction\n",
      "A new method bridging graph theory and residue co-evolutionary networks for specificity determinant positions detection\n",
      "cath-resolve-hits: a new tool that resolves domain matches suspiciously quickly\n",
      "Multi-insight visualization of multi-omics data via ensemble dimension reduction and tensor factorization\n",
      "Presenting and sharing clinical data using the eTRIKS Standards Master Tree for tranSMART\n",
      "PRESM: personalized reference editor for somatic mutation discovery in cancer genomics\n",
      "Learning and Imputation for Mass-spec Bias Reduction (LIMBR)\n",
      "Gimpute: an efficient genetic data imputation pipeline\n",
      "RNApuzzler: efficient outerplanar drawing of RNA-secondary structures\n",
      "TCellXTalk facilitates the detection of co-modified peptides for the study of protein post-translational modification cross-talk in T cells\n",
      "GenomeFlow: a comprehensive graphical tool for modeling and analyzing 3D genome structure\n",
      "bcGST-an interactive bias-correction method to identify over-represented gene-sets in boutique arrays\n",
      "emeraLD: rapid linkage disequilibrium estimation with massive datasets\n",
      "KeggExp: a web server for visual integration of KEGG pathways and expression profile data\n",
      "SequencEnG: an interactive knowledge base of sequencing techniques\n",
      "VarSelLCM: an R/C++ package for variable selection in model-based clustering of mixed-data with missing values\n",
      "Inference and visualization of DNA damage patterns using a grade of membership model\n",
      "Co-registration and analysis of multiple imaging mass spectrometry datasets targeting different analytes\n",
      "DeepGSR: an optimized deep-learning structure for the recognition of genomic signals and regions\n",
      "Differential methylation values in differential methylation analysis\n",
      "Precision Lasso: accounting for correlations and linear dependencies in high-dimensional genomic data\n",
      "Understanding and predicting disease relationships through similarity fusion\n",
      "Methods for automatic reference trees and multilevel phylogenetic placement\n",
      "Robust prediction of clinical outcomes using cytometry data\n",
      "Drug Gene Budger (DGB): an application for ranking drugs to modulate a specific gene based on transcriptomic signatures\n",
      "Improving the prediction of protein-nucleic acids binding residues via multiple sequence profiles and the consensus of complementary methods\n",
      "Improving protein function prediction using protein sequence and GO-term similarities\n",
      "PlayMolecule BindScope: large scale CNN-based virtual screening on the web\n",
      "RepeatCraft: a meta-pipeline for repetitive element de-fragmentation and annotation\n",
      "Large-scale comparative analysis of microbial pan-genomes using PanOCT\n",
      "Haplotype matching in large cohorts using the Li and Stephens model\n",
      "Dot2dot: accurate whole-genome tandem repeats discovery\n",
      "A cluster robustness score for identifying cell subpopulations in single cell gene expression datasets from heterogeneous tissues and tumors\n",
      "MOLGENIS research: advanced bioinformatics data software for non-bioinformaticians\n",
      "Characterizing protein-DNA binding event subtypes in ChIP-exo data\n",
      "VARAN-GIE: curation of genomic interval sets\n",
      "Increased sensitivity with automated validation of XL-MS cleavable peptide crosslinks\n",
      "PathwayConnector: finding complementary pathways to enhance functional analysis\n",
      "On entropy and information in gene interaction networks\n",
      "GlycanAnalyzer: software for automated interpretation of N-glycan profiles after exoglycosidase digestions\n",
      "Container-based bioinformatics with Pachyderm\n",
      "Genetic Simulation Resources and the GSR Certification Program\n",
      "MicroPheno: predicting environments and host phenotypes from 16S rRNA gene sequencing using a k-mer based representation of shallow sub-samples\n",
      "On the problem of confounders in modeling gene expression\n",
      "PredMP: a web server for de novo prediction and visualization of membrane proteins\n",
      "Automated workflow composition in mass spectrometry-based proteomics\n",
      "Quantifying heterogeneity of expression data based on principal components\n",
      "Design tools for MPRA experiments\n",
      "DoRes within CellMissy: dose-response analysis on cell migration and related data\n",
      "WebMetabase: cleavage sites analysis tool for natural and unnatural substrates from diverse data source\n",
      "HOGMMNC: a higher order graph matching with multiple network constraints model for gene-drug regulatory modules identification\n",
      "Quantifying the notions of canalizing and master genes in a gene regulatory network-a Boolean network modeling perspective\n",
      "gMCS: fast computation of genetic minimal cut sets in large networks\n",
      "BinDash, software for fast genome distance estimation on a typical personal laptop\n",
      "Heritability estimation and differential analysis of count data with generalized linear mixed models in genomic sequencing studies\n",
      "Automatic recognition of ligands in electron density by machine learning\n",
      "GECO: gene expression correlation analysis after genetic algorithm-driven deconvolution\n",
      "AuTom-dualx: a toolkit for fully automatic fiducial marker-based alignment of dual-axis tilt series with simultaneous reconstruction\n",
      "Metagenomic binning through low-density hashing\n",
      "Accurate multiple alignment of distantly related genome sequences using filtered spaced word matches as anchor points\n",
      "Integrative DNA copy number detection and genotyping from sequencing and array-based platforms\n",
      "PingPongPro: a tool for the detection of piRNA-mediated transposon-silencing in small RNA-Seq data\n",
      "Computational analysis of kinase inhibitor selectivity using structural knowledge\n",
      "SATurn: a modular bioinformatics framework for the design of robust maintainable web-based and standalone applications\n",
      "Unraveling the role of low-frequency mutated genes in breast cancer\n",
      "A scalable estimator of SNP heritability for biobank-scale data\n",
      "The Kappa platform for rule-based modeling\n",
      "Unsupervised embedding of single-cell Hi-C data\n",
      "Inference of species phylogenies from bi-allelic markers using pseudo-likelihood\n",
      "Onto2Vec: joint vector-based representation of biological entities and their ontology-based annotations\n",
      "NeuroMorphoVis: a collaborative framework for analysis and visualization of neuronal morphology skeletons reconstructed from microscopy stacks\n",
      "Modeling polypharmacy side effects with graph convolutional networks\n",
      "A graph-based approach to diploid genome assembly\n",
      "mGPfusion: predicting protein stability changes with Gaussian process kernel learning and data fusion\n",
      "Predicting CTCF-mediated chromatin loops using CTCF-MP\n",
      "Bayesian parameter estimation for biochemical reaction networks using region-based adaptive parallel tempering\n",
      "Classifying tumors by supervised network propagation\n",
      "Convolutional neural networks for classification of alignments of non-coding RNA sequences\n",
      "Viral quasispecies reconstruction via tensor factorization with successive read removal\n",
      "Association mapping in biomedical time series via statistically significant shapelet mining\n",
      "Versatile genome assembly evaluation with QUAST-LG\n",
      "A spectral clustering-based method for identifying clones from high-throughput B cell repertoire sequencing data\n",
      "Bayesian networks for mass spectrometric metabolite identification via molecular fingerprints\n",
      "A unifying framework for joint trait analysis under a non-infinitesimal model\n",
      "A novel methodology on distributed representations of proteins using their interacting ligands\n",
      "AmpUMI: design and analysis of unique molecular identifiers for deep amplicon sequencing\n",
      "Robust clustering of noisy high-dimensional gene expression data for patients subtyping\n",
      "13Check_RNA: a tool to evaluate 13C chemical shift assignments of RNA\n",
      "SpliceRover: interpretable convolutional neural networks for improved splice site prediction\n",
      "htsget: a protocol for securely streaming genomic data\n",
      "Predicting clone genotypes from tumor bulk sequencing of multiple samples\n",
      "Using tree-based methods for detection of gene-gene interactions in the presence of a polygenic signal: simulation study with application to educational attainment in the Generation Scotland Cohort Study\n",
      "HUME: large-scale detection of causal genetic factors of adverse drug reactions\n",
      "Metaxa2 Database Builder: enabling taxonomic identification from metagenomic or metabarcoding data using any genetic marker\n",
      "TiSAn: estimating tissue-specific effects of coding and non-coding variants\n",
      "Identifying differentially methylated sites in samples with varying tumor purity\n",
      "Improved genomic island predictions with IslandPath-DIMOB\n",
      "MARDy: Mycology Antifungal Resistance Database\n",
      "Modelling BioNano optical data and simulation study of genome map assembly\n",
      "AA9int: SNP interaction pattern search using non-hierarchical additive model set\n",
      "Integrating proteomic or transcriptomic data into metabolic models using linear bound flux balance analysis\n",
      "ImmunomeBrowser: a tool to aggregate and visualize complex and heterogeneous epitopes in reference proteins\n",
      "Lost in translation: bioinformatic analysis of variations affecting the translation initiation codon in the human genome\n",
      "2018 ISCB Overton Prize awarded to Cole Trapnell\n",
      "Transfer learning for biomedical named entity recognition with neural networks\n",
      "Predicting Cancer Drug Response using a Recommender System\n",
      "Message from the ISCB: 2018 Outstanding Contributions to ISCB Award: Russ Altman\n",
      "Accelerating flux balance calculations in genome-scale metabolic models by localizing the application of loopless constraints\n",
      "Comparing complex variants in family trios\n",
      "Motif scraper: a cross-platform, open-source tool for identifying degenerate nucleotide motif matches in FASTA files\n",
      "Pathway-structured predictive modeling for multi-level drug response in multiple myeloma\n",
      "LncADeep: an ab initio lncRNA identification and functional annotation tool based on deep learning\n",
      "GateFinder: projection-based gating strategy optimization for flow and mass cytometry\n",
      "The size matters? A computational tool to design bivalent ligands\n",
      "Nextstrain: real-time tracking of pathogen evolution\n",
      "YAMDA: thousandfold speedup of EM-based motif discovery using deep learning libraries and GPU\n",
      "Palimpsest: an R package for studying mutational and structural variant signatures along clonal evolution in cancer\n",
      "Development and evaluation of a deep learning model for protein-ligand binding affinity prediction\n",
      "Minimap2: pairwise alignment for nucleotide sequences\n",
      "ProteomeVis: a web app for exploration of protein properties from structure to sequence evolution across organisms' proteomes\n",
      "VarExp: estimating variance explained by genome-wide GxE summary statistics\n",
      "Sorting cancer karyotypes using double-cut-and-joins, duplications and deletions\n",
      "Self-consistency test reveals systematic bias in programs for prediction change of stability upon mutation\n",
      "AlphaMate: a program for optimizing selection, maintenance of diversity and mate allocation in breeding programs\n",
      "lncFunTK: a toolkit for functional annotation of long noncoding RNAs\n",
      "ImagePy: an open-source, Python-based and platform-independent software package for bioimage analysis\n",
      "Efficient flexible backbone protein-protein docking for challenging targets\n",
      "GenomicScores: seamless access to genomewide position-specific scores from R and Bioconductor\n",
      "A powerful approach reveals numerous expression quantitative trait haplotypes in multiple tissues\n",
      "BNPMDA: Bipartite Network Projection for MiRNA-Disease Association prediction\n",
      "RANGER-DTL 2.0: rigorous reconstruction of gene-family evolution by duplication, transfer and loss\n",
      "PathwaySplice: an R package for unbiased pathway analysis of alternative splicing in RNA-Seq data\n",
      "CTDquerier: a bioconductor R package for Comparative Toxicogenomics DatabaseTM data extraction, visualization and enrichment of environmental and toxicological studies\n",
      "MetaBridge: enabling network-based integrative analysis via direct protein interactors of metabolites\n",
      "iRO-3wPseKNC: identify DNA replication origins by three-window-based PseKNC\n",
      "Efficient population-scale variant analysis and prioritization with VAPr\n",
      "pysster: classification of biological sequences by learning sequence and structure motifs with convolutional neural networks\n",
      "Computation and application of tissue-specific gene set weights\n",
      "PhyloProfile: dynamic visualization and exploration of multi-layered phylogenetic profiles\n",
      "A comprehensive web tool for toehold switch design\n",
      "3DClusterViSu: 3D clustering analysis of super-resolution microscopy data by 3D Voronoi tessellations\n",
      "Reactome Pengine: a web-logic API to the Homo sapiens reactome\n",
      "Efficient simulation of intrinsic, extrinsic and external noise in biochemical systems\n",
      "FaStore: a space-saving solution for raw sequencing data\n",
      "Identification and function of FAR protein family genes from a transcriptome analysis of Aphelenchoides besseyi\n",
      "Building online genomics applications using BioPyramid\n",
      "LSMM: a statistical approach to integrating functional annotations with genome-wide association studies\n",
      "ONETOOL for the analysis of family-based big data\n",
      "POPdemog: visualizing population demographic history from simulation scripts\n",
      "Generalizing biomedical relation classification with neural adversarial domain adaptation\n",
      "Functional annotation of genomic variants in studies of late-onset Alzheimer's disease\n",
      "Learned protein embeddings for machine learning\n",
      "SCOTCH: subtype A coreceptor tropism classification in HIV-1\n",
      "SCRAM: a pipeline for fast index-free small RNA read alignment and visualization\n",
      "NMRNet: a deep learning approach to automated peak picking of protein NMR spectra\n",
      "ISMB/ECCB 2017 proceedings\n",
      "SMMB: a stochastic Markov blanket framework strategy for epistasis detection in GWAS\n",
      "Nimbus: a design-driven analyses suite for amplicon-based NGS data\n",
      "esATAC: an easy-to-use systematic pipeline for ATAC-seq data analysis\n",
      "PWMScan: a fast tool for scanning entire genomes with a position-specific weight matrix\n",
      "CarbonylDB: a curated data-resource of protein carbonylation sites\n",
      "anexVis: visual analytics framework for analysis of RNA expression\n",
      "Updating the 97% identity threshold for 16S ribosomal RNA OTUs\n",
      "MUGAN: Multi-GPU accelerated AmpliconNoise server for rapid microbial diversity assessment\n",
      "Combining co-evolution and secondary structure prediction to improve fragment library generation\n",
      "SecretSanta: flexible pipelines for functional secretome prediction\n",
      "LS-align: an atom-level, flexible ligand structural alignment algorithm for high-throughput virtual screening\n",
      "SEED 2: a user-friendly platform for amplicon high-throughput sequencing data analyses\n",
      "RaMWAS: fast methylome-wide association study pipeline for enrichment platforms\n",
      "Enhancing protein fold determination by exploring the complementary information of chemical cross-linking and coevolutionary signals\n",
      "Hierarchical analysis of RNA-seq reads improves the accuracy of allele-specific expression\n",
      "CrossPlan: systematic planning of genetic crosses to validate mathematical models\n",
      "Spectral clustering based on learning similarity matrix\n",
      "O-GlcNAcPRED-II: an integrated classification algorithm for identifying O-GlcNAcylation sites based on fuzzy undersampling and a K-means PCA oversampling technique\n",
      "FWER and FDR control when testing multiple mediators\n",
      "Principal metabolic flux mode analysis\n",
      "GCPred: a web tool for guanylyl cyclase functional centre prediction from amino acid sequence\n",
      "GEMMER: GEnome-wide tool for Multi-scale Modeling data Extraction and Representation for Saccharomyces cerevisiae\n",
      "A low-complexity add-on score for protein remote homology search with COMER\n",
      "JABAWS 2.2 distributed web services for Bioinformatics: protein disorder, conservation and RNA secondary structure\n",
      "DIBS: a repository of disordered binding sites mediating interactions with ordered proteins\n",
      "TSAPA: identification of tissue-specific alternative polyadenylation sites in plants\n",
      "Computational drug repositioning using low-rank matrix approximation and randomized algorithms\n",
      "ST Spot Detector: a web-based application for automatic spot and tissue detection for spatial Transcriptomics image datasets\n",
      "APAtrap: identification and quantification of alternative polyadenylation sites from RNA-seq data\n",
      "PhenoRank: reducing study bias in gene prioritization through simulation\n",
      "GTC: how to maintain huge genotype collections in a compressed form\n",
      "VIPER: a web application for rapid expert review of variant calls\n",
      "Splice Expression Variation Analysis (SEVA) for inter-tumor heterogeneity of gene isoform usage in cancer\n",
      "QuantumClone: clonal assessment of functional mutations in cancer based on a genotype-aware method for clonal reconstruction\n",
      "A rapid epistatic mixed-model association analysis by linear retransformations of genomic estimated values\n",
      "PyChimera: use UCSF Chimera modules in any Python 2.7 project\n",
      "A benchmark study of scoring methods for non-coding mutations\n",
      "LeNup: learning nucleosome positioning from DNA sequences with improved convolutional neural networks\n",
      "SimulaTE: simulating complex landscapes of transposable elements of populations\n",
      "A network approach to exploring the functional basis of gene-gene epistatic interactions in disease susceptibility\n",
      "Toppar: an interactive browser for viewing association study results\n",
      "Tumor purity quantification by clonal DNA methylation signatures\n",
      "CGManalyzer: an R package for analyzing continuous glucose monitoring studies\n",
      "Protein classification using modified n-grams and skip-grams\n",
      "QUENTIN: reconstruction of disease transmissions from viral quasispecies genomic data\n",
      "Aligning dynamic networks with DynaWAVE\n",
      "ChopStitch: exon annotation and splice graph construction using transcriptome assembly and whole genome sequencing data\n",
      "SV2: accurate structural variation genotyping and de novo mutation detection from whole genomes\n",
      "MIIC online: a web server to reconstruct causal or non-causal networks from non-perturbative data\n",
      "DyNetViewer: a Cytoscape app for dynamic network construction, analysis and visualization\n",
      "Towards self-learning based hypotheses generation in biomedical text domain\n",
      "mTM-align: an algorithm for fast and accurate multiple protein structure alignment\n",
      "Computational identification of binding energy hot spots in protein-RNA complexes using an ensemble approach\n",
      "Genome U-Plot: a whole genome visualization\n",
      "CRNET: an efficient sampling approach to infer functional regulatory networks by integrating large-scale ChIP-seq and time-course RNA-seq data\n",
      "PRAPI: post-transcriptional regulation analysis pipeline for Iso-Seq\n",
      "TCGA-assembler 2: software pipeline for retrieval and processing of TCGA/CPTAC data\n",
      "Delta: a new web-based 3D genome visualization and analysis platform\n",
      "Analyzing large scale genomic data on the cloud with Sparkhit\n",
      "fMLC: fast multi-level clustering and visualization of large molecular datasets\n",
      "CscoreTool: fast Hi-C compartment analysis at high resolution\n",
      "DaMiRseq-an R/Bioconductor package for data mining of RNA-Seq data: normalization, feature selection and classification\n",
      "Gene Graphics: a genomic neighborhood data visualization web application\n",
      "ASAR: visual analysis of metagenomes in R\n",
      "Identifying functionally informative evolutionary sequence profiles\n",
      "GenSSI 2.0: multi-experiment structural identifiability analysis of SBML models\n",
      "MOSAIC: a chemical-genetic interaction data repository and web resource for exploring chemical modes of action\n",
      "Power Analysis for Genetic Association Test (PAGEANT) provides insights to challenges for rare variant association studies\n",
      "Identifying simultaneous rearrangements in cancer genomes\n",
      "PlanNET: homology-based predicted interactome for multiple planarian transcriptomes\n",
      "bioalcidae, samjs and vcffilterjs: object-oriented formatters and filters for bioinformatics files\n",
      "Cpf1-Database: web-based genome-wide guide RNA library design for gene knockout screens using CRISPR-Cpf1\n",
      "CytoCtrlAnalyser: a Cytoscape app for biomolecular network controllability analysis\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The OncoPPi Portal: an integrative resource to explore and prioritize protein-protein interactions for cancer target discovery\n",
      "ASElux: an ultra-fast and accurate allelic reads counter\n",
      "Meta-analytic principal component analysis in integrative omics application\n",
      "Robust inference of population structure from next-generation sequencing data with systematic differences in sequencing\n",
      "Deep learning for tumor classification in imaging mass spectrometry\n",
      "AIRVF: a filtering toolbox for precise variant calling in Ion Torrent sequencing\n",
      "MOAT: efficient detection of highly mutated regions with the Mutations Overburdening Annotations Tool\n",
      "NanoStringNormCNV: pre-processing of NanoString CNV data\n",
      "CommWalker: correctly evaluating modules in molecular networks in light of annotation bias\n",
      "ViraPipe: scalable parallel pipeline for viral metagenome analysis from next generation sequencing reads\n",
      "Orchid: a novel management, annotation and machine learning framework for analyzing cancer mutations\n",
      "pedigreejs: a web-based graphical pedigree editor\n",
      "BEAM web server: a tool for structural RNA motif discovery\n",
      "GRASS: semi-automated NMR-based structure elucidation of saccharides\n",
      "Quantifying biological samples using Linear Poisson Independent Component Analysis for MALDI-ToF mass spectra\n",
      "MutaNET: a tool for automated analysis of genomic mutations in gene regulatory networks\n",
      "pqsfinder: an exhaustive and imperfection-tolerant search tool for potential quadruplex-forming sequences in R\n",
      "Group spike-and-slab lasso generalized linear models for disease prediction and associated genes detection by incorporating pathway information\n",
      "CLIFinder: identification of LINE-1 chimeric transcripts in RNA-seq data\n",
      "A fast fiducial marker tracking model for fully automatic alignment in electron tomography\n",
      "Bayesian data integration for quantifying the contribution of diverse measurements to parameter estimates\n",
      "PROSPERous: high-throughput prediction of substrate cleavage sites for 90 proteases with improved accuracy\n",
      "Cancer progression models and fitness landscapes: a many-to-many relationship\n",
      "Identification of copy number variations and translocations in cancer cells from Hi-C data\n",
      "An omnibus test for differential distribution analysis of microbiome sequencing data\n",
      "MATAM: reconstruction of phylogenetic marker genes from short sequencing reads in metagenomes\n",
      "GLASS: assisted and standardized assessment of gene variations from Sanger sequence trace data\n",
      "ComplexViewer: visualization of curated macromolecular complexes\n",
      "nandb-number and brightness in R with a novel automatic detrending algorithm\n",
      "MethRaFo: MeDIP-seq methylation estimate using a Random Forest Regressor\n",
      "pLoc-mAnimal: predict subcellular localization of animal proteins with both single and multiple sites\n",
      "Forecasting residue-residue contact prediction accuracy\n",
      "Mechanisms to protect the privacy of families when using the transmission disequilibrium test in genome-wide association studies\n",
      "Analysis and prediction of protein folding energy changes upon mutation by element specific persistent homology\n",
      "Motif independent identification of potential RNA G-quadruplexes by G4RNA screener\n",
      "An accurate algorithm for the detection of DNA fragments from dilution pool sequencing experiments\n",
      "SPRINT: an SNP-free toolkit for identifying RNA editing sites\n",
      "The value of prior knowledge in machine learning of complex network systems\n",
      "Prediction and modeling of pre-analytical sampling errors as a strategy to improve plasma NMR metabolomics data\n",
      "MusiteDeep: a deep-learning framework for general and kinase-specific phosphorylation site prediction\n",
      "Gating mass cytometry data by deep learning\n",
      "proFIA: a data preprocessing workflow for flow injection analysis coupled to high-resolution mass spectrometry\n",
      "DIMM-SC: a Dirichlet mixture model for clustering droplet-based single cell transcriptomic data\n",
      "RealityConvert: a tool for preparing 3D models of biochemical structures for augmented and virtual reality\n",
      "Cancer Hallmarks Analytics Tool (CHAT): a text mining approach to organize and evaluate scientific literature on cancer\n",
      "DeepLoc: prediction of protein subcellular localization using deep learning\n",
      "Structure-based prediction of protein- peptide binding regions using Random Forest\n",
      "Detecting presence of mutational signatures in cancer with confidence\n",
      "Phandango: an interactive viewer for bacterial population genomics\n",
      "diffloop: a computational framework for identifying and analyzing differential DNA loops from sequencing data\n",
      "CCmiR: a computational approach for competitive and cooperative microRNA binding prediction\n",
      "TeachEnG: a Teaching Engine for Genomics\n",
      "TSIS: an R package to infer alternative splicing isoform switches for time-series data\n",
      "Comparison of methods for phylogenetic B-cell lineage inference using time-resolved antibody repertoire simulations (AbSim)\n",
      "A comprehensive assessment of long intrinsic protein disorder from the DisProt database\n",
      "SiNoPsis: Single Nucleotide Polymorphisms selection and promoter profiling\n",
      "Temporal probabilistic modeling of bacterial compositions derived from 16S rRNA sequencing\n",
      "SMLocalizer, a GPU accelerated ImageJ plugin for single molecule localization microscopy\n",
      "JEPEGMIX2: improved gene-level joint analysis of eQTLs in cosmopolitan cohorts\n",
      "LightDock: a new multi-scale approach to protein-protein docking\n",
      "FATHMM-XF: accurate prediction of pathogenic point mutations via extended features\n",
      "An automated tool for obtaining QSAR-ready series of compounds using semantic web technologies\n",
      "SINCERITIES: inferring gene regulatory networks from time-stamped single cell transcriptional expression profiles\n",
      "Snaptron: querying splicing patterns across tens of thousands of RNA-seq samples\n",
      "Morphologically constrained and data informed cell segmentation of budding yeast\n",
      "Kfits: a software framework for fitting and cleaning outliers in kinetic measurements\n",
      "A utility maximizing and privacy preserving approach for protecting kinship in genomic databases\n",
      "Prediction of delayed retention of antibodies in hydrophobic interaction chromatography from sequence using machine learning\n",
      "MIRMMR: binary classification of microsatellite instability using methylation and mutations\n",
      "FIRE: functional inference of genetic variants that regulate gene expression\n",
      "Cloud-based interactive analytics for terabytes of genomic variants data\n",
      "LLR: a latent low-rank approach to colocalizing genetic risk variants in multiple GWAS\n",
      "ChAMP: updated methylation analysis pipeline for Illumina BeadChips\n",
      "CESAR 2.0 substantially improves speed and accuracy of comparative gene annotation\n",
      "fastNGSadmix: admixture proportions and principal component analysis of a single NGS sample\n",
      "biospear: an R package for biomarker selection in penalized Cox regression\n",
      "Kollector: transcript-informed, targeted de novo assembly of gene loci\n",
      "Tumor phylogeny inference using tree-constrained importance sampling\n",
      "Integrative deep models for alternative splicing\n",
      "DeepBound: accurate identification of transcript boundaries via deep convolutional neural fields\n",
      "Improved data-driven likelihood factorizations for transcript abundance estimation\n",
      "Exploiting sequence-based features for predicting enhancer-promoter interactions\n",
      "Direct AUC optimization of regulatory motifs\n",
      "Predicting multicellular function through multi-layer tissue networks\n",
      "Alignment of dynamic networks\n",
      "Denoising genome-wide histone ChIP-seq with convolutional neural networks\n",
      "Molecular signatures that can be transferred across different omics platforms\n",
      "BIOSSES: a semantic sentence similarity estimation system for the biomedical domain\n",
      "Deep learning-based subdivision approach for large scale macromolecules structure recovery from electron cryo tomograms\n",
      "Applying meta-analysis to genotype-tissue expression data from multiple tissues to identify eQTLs and increase the number of eGenes\n",
      "SpaceScanner: COPASI wrapper for automated management of global stochastic optimization experiments\n",
      "GPU-powered model analysis with PySB/cupSODA\n",
      "The KSEA App: a web-based tool for kinase activity inference from quantitative phosphoproteomics\n",
      "Partitioned learning of deep Boltzmann machines for SNP data\n",
      "Matrix completion with side information and its applications in predicting the antigenicity of influenza viruses\n",
      "omiXcore: a web server for prediction of protein interactions with large RNA\n",
      "PhylOligo: a package to identify contaminant or untargeted organism sequences in genome assemblies\n",
      "SANDPUMA: ensemble predictions of nonribosomal peptide chemistry reveal biosynthetic diversity across Actinobacteria\n",
      "Message from the ISCB: 2017 ISCB Overton Prize Awarded to Christoph Bock\n",
      "CloudNeo: a cloud pipeline for identifying patient-specific tumor neoantigens\n",
      "Sparse redundancy analysis of high-dimensional genetic and genomic data\n",
      "Comparison of pre-processing methods for Infinium HumanMethylation450 BeadChip array\n",
      "DEIsoM: a hierarchical Bayesian model for identifying differentially expressed isoforms using biological replicates\n",
      "Integrative analysis of multiple genomic variables using a hierarchical Bayesian model\n",
      "ISAMBARD: an open-source computational environment for biomolecular analysis, modelling and design\n",
      "Protein-Sol: a web tool for predicting protein solubility from sequence\n",
      "Moran's I quantifies spatio-temporal pattern formation in neural imaging data\n",
      "karyoploteR: an R/Bioconductor package to plot customizable genomes displaying arbitrary data\n",
      "HEROD: a human ethnic and regional specific omics database\n",
      "CWDPRNP: a tool for cervid prion sequence analysis in program R\n",
      "A fast and exhaustive method for heterogeneity and epistasis analysis based on multi-objective optimization\n",
      "Flexbar 3.0 - SIMD and multicore parallelization\n",
      "ASAP: a web-based platform for the analysis and interactive visualization of single-cell RNA-seq data\n",
      "Molecular heterogeneity at the network level: high-dimensional testing, clustering and a TCGA case study\n",
      "PhyD3: a phylogenetic tree viewer with extended phyloXML support for functional genomics data visualization\n",
      "Single-cell regulome data analysis by SCRAT\n",
      "OMTools: a software package for visualizing and processing optical mapping data\n",
      "MAGenTA: a Galaxy implemented tool for complete Tn-Seq analysis and data visualization\n",
      "BioMake: a GNU make-compatible utility for declarative workflow management\n",
      "McPAS-TCR: a manually curated catalogue of pathology-associated T cell receptor sequences\n",
      "Reliability of algorithmic somatic copy number alteration detection from targeted capture data\n",
      "EBT: a statistic test identifying moderate size of significant features with balanced power and precision for genome-wide rate comparisons\n",
      "modlAMP: Python for antimicrobial peptides\n",
      "Accounting for tumor purity improves cancer subtype classification from DNA methylation data\n",
      "KMC 3: counting and manipulating k-mer statistics\n",
      "Edlib: a C/C ++ library for fast, exact sequence alignment using edit distance\n",
      "PhenoCurve: capturing dynamic phenotype-environment relationships using phenomics data\n",
      "Sphinx: merging knowledge-based and ab initio approaches to improve protein loop prediction\n",
      "veqtl-mapper: variance association mapping for molecular phenotypes\n",
      "A method for learning a sparse classifier in the presence of missing data for high-dimensional biological datasets\n",
      "EigenTHREADER: analogous protein fold recognition by efficient contact map threading\n",
      "A Bayesian group sparse multi-task regression model for imaging genetics\n",
      "Domain prediction with probabilistic directional context\n",
      "miRCat2: accurate prediction of plant and animal microRNAs from next-generation sequencing datasets\n",
      "Incorporating prior information into differential network analysis using non-paranormal graphical models\n",
      "emMAW: computing minimal absent words in external memory\n",
      "mzML2ISA & nmrML2ISA: generating enriched ISA-Tab metadata files from metabolomics XML data\n",
      "PyGOLD: a python based API for docking based virtual screening workflow generation\n",
      "ST Pipeline: an automated pipeline for spatial mapping of unique transcripts\n",
      "Machine learning in computational biology to accelerate high-throughput protein expression\n",
      "BioContainers: an open-source and community-driven framework for software standardization\n",
      "SieveSifter: a web-based tool for visualizing the sieve analyses of HIV-1 vaccine efficacy trials\n",
      "Kart: a divide-and-conquer algorithm for NGS read alignment\n",
      "EUPAN enables pan-genome studies of a large number of eukaryotic genomes\n",
      "Spresso: an ultrafast compound pre-screening method based on compound decomposition\n",
      "DLTree: efficient and accurate phylogeny reconstruction using the dynamical language method\n",
      "rMSI: an R package for MS imaging data handling and visualization\n",
      "Constructing linkage maps in the genomics era with MapDisto 2.0\n",
      "GenomeScope: fast reference-free genome profiling from short reads\n",
      "Cpipe: a comprehensive computational platform for sequence and structure-based analyses of Cysteine residues\n",
      "Enhanced methods to detect haplotypic effects on gene expression\n",
      "Estimating error models for whole genome sequencing using mixtures of Dirichlet-multinomial distributions\n",
      "A Zoom-Focus algorithm (ZFA) to locate the optimal testing region for rare variant association tests\n",
      "PathwayMapper: a collaborative visual web editor for cancer pathways and genomic data\n",
      "Meta-analytic framework for liquid association\n",
      "Discriminating the reaction types of plant type III polyketide synthases\n",
      "DNA Compass: a secure, client-side site for navigating personal genetic information\n",
      "QRank: a novel quantile regression tool for eQTL discovery\n",
      "Systematic inference of functional phosphorylation events in yeast metabolism\n",
      "ctsGE-clustering subgroups of expression data\n",
      "Glimma: interactive graphics for gene expression analysis\n",
      "Integrative analysis identifies co-dependent gene expression regulation of BRG1 and CHD7 at distal regulatory sites in embryonic stem cells\n",
      "PMAnalyzer: a new web interface for bacterial growth curve analysis\n",
      "AptRank: an adaptive PageRank model for protein function prediction on   bi-relational graphs\n",
      "LASER server: ancestry tracing with genotypes or sequence reads\n",
      "Statistical database analysis of the role of loop dynamics for protein-protein complex formation and allostery\n",
      "Succinct colored de Bruijn graphs\n",
      "MBV: a method to solve sample mislabeling and detect technical bias in large combined genotype and sequencing assay datasets\n",
      "Kollector: transcript-informed, targeted de novo assembly of gene loci\n",
      "Comprehensive evaluation of de novo transcriptome assembly programs and their effects on differential gene expression analysis\n",
      "SChloro: directing Viridiplantae proteins to six chloroplastic sub-compartments\n",
      "simPATHy: a new method for simulating data from perturbed biological PATHways\n",
      "Sequence-structure relations of biopolymers\n",
      "Pro54DB: a database for experimentally verified sigma-54 promoters\n",
      "DEF: an automated dead-end filling approach based on quasi-endosymbiosis\n",
      "RelExplain-integrating data and networks to explain biological processes\n",
      "MotifNet: a web-server for network motif analysis\n",
      "PALADIN: protein alignment for functional profiling whole metagenome shotgun data\n",
      "Accurate prediction of human essential genes using only nucleotide composition and association information\n",
      "Supporting precision medicine by data mining across multi-disciplines: an integrative approach for generating comprehensive linkages between single nucleotide variants (SNVs) and drug-binding sites\n",
      "Efficient computation of transfer free energies of amino acids in beta-barrel membrane proteins\n",
      "LIQUID: an-open source software for identifying lipids in LC-MS/MS-based lipidomics data\n",
      "Riborex: fast and flexible identification of differential translation from Ribo-seq data\n",
      "HIPred: an integrative approach to predicting haploinsufficient genes\n",
      "Introducing COCOS: codon consequence scanner for annotating reading frame changes induced by stop-lost and frame shift variants\n",
      "Historian: accurate reconstruction of ancestral sequences and evolutionary rates\n",
      "VCF-kit: assorted utilities for the variant call format\n",
      "Better diagnostic signatures from RNAseq data through use of auxiliary co-data\n",
      "Resolution and reconciliation of non-binary gene trees with transfers, duplications and losses\n",
      "TRFBA: an algorithm to integrate genome-scale metabolic and transcriptional regulatory networks with incorporation of expression data\n",
      "A novel non-parametric method for uncertainty evaluation of correlation-based molecular signatures: its application on PAM50 algorithm\n",
      "ESA-UbiSite: accurate prediction of human ubiquitination sites by identifying a set of effective negatives\n",
      "SPOT-ligand 2: improving structure-based virtual screening by binding-homology search on an expanded structural template library\n",
      "SNP interaction pattern identifier (SIPI): an intensive search for SNP-SNP interaction patterns\n",
      "Using genotype array data to compare multi- and single-sample variant calls and improve variant call sets from deep coverage whole-genome sequencing data\n",
      "QAcon: single model quality assessment using protein structural and contact information with machine learning techniques\n",
      "ACTG: novel peptide mapping onto gene models\n",
      "Falco: a quick and flexible single-cell RNA-seq processing framework on the cloud\n",
      "Computational prediction of species-specific malonylation sites via enhanced characteristic strategy\n",
      "Improving protein disorder prediction by deep bidirectional long short-term memory recurrent neural networks\n",
      "The PPI3D web server for searching, analyzing and modeling protein-protein interactions in the context of 3D structures\n",
      "SeqLib: a C ++ API for rapid BAM manipulation, sequence alignment and sequence assembly\n",
      "Analyzing user-generated online content for drug discovery: development and use of MedCrawler\n",
      "RADER: a RApid DEcoy Retriever to facilitate decoy based assessment of virtual screening\n",
      "T-BAS: Tree-Based Alignment Selector toolkit for phylogenetic-based placement, alignment downloads and metadata visualization: an example with the Pezizomycotina tree of life\n",
      "Xenolog classification\n",
      "SPADEVizR: an R package for visualization, analysis and integration of SPADE results\n",
      "GENIUS: web server to predict local gene networks and key genes for biological functions\n",
      "BoostGAPFILL: improving the fidelity of metabolic network reconstructions through integrated constraint and pattern-based methods\n",
      "LRFragLib: an effective algorithm to identify fragments for de novo protein structure prediction\n",
      "Cancer driver gene discovery through an integrative genomics approach in a non-parametric Bayesian framework\n",
      "RedNemo: topology-based PPI network reconstruction via repeated diffusion with neighborhood modifications\n",
      "Transfer learning across ontologies for phenome-genome association prediction\n",
      "HiLive: real-time mapping of illumina reads while sequencing\n",
      "Biospark: scalable analysis of large numerical datasets from biological simulations and experiments using Hadoop and Spark\n",
      "cisASE: a likelihood-based method for detecting putative cis-regulated allele-specific expression in RNA sequencing data\n",
      "CRISPR-DO for genome-wide CRISPR design and optimization\n",
      "MIMEAnTo: profiling functional RNA in mutational interference mapping experiments\n",
      "New quality measure for SNP array based CNV detection\n",
      "A subpopulation model to analyze heterogeneous cell differentiation dynamics\n",
      "samExploreR: exploring reproducibility and robustness of RNA-seq results based on SAM files\n",
      "Heat*seq: an interactive web tool for high-throughput sequencing experiment comparison with public data\n",
      "Icarus: visualizer for de novo assembly evaluation\n",
      "The SMAL web server: global multiple network alignment from pairwise alignments\n",
      "ConsPred: a rule-based (re-)annotation framework for prokaryotic genomes\n",
      "w4CSeq: software and web application to analyze 4C-seq data\n",
      "Cyclo-lib: a database of computational molecular dynamics simulations of cyclodextrins\n",
      "Site of metabolism prediction for oxidation reactions mediated by oxidoreductases based on chemical bond\n",
      "An accessibility-incorporated method for accurate prediction of RNA-RNA interactions from sequence data\n",
      "TwoPaCo: an efficient algorithm to build the compacted de Bruijn graph from many complete genomes\n",
      "PSSV: a novel pattern-based probabilistic approach for somatic structural variation identification\n",
      "Tissue-specific pathway association analysis using genome-wide association study summaries\n",
      "RiboDiff: detecting changes of mRNA translation efficiency from ribosome footprints\n",
      "Verdant: automated annotation, alignment and phylogenetic analysis of whole chloroplast genomes\n",
      "Reference point insensitive molecular data analysis\n",
      "DM-BLD: differential methylation detection using a hierarchical Bayesian model exploiting local dependency\n",
      "Assessment of cancer and virus antigens for cross-reactivity in human tissues\n",
      "Nanocall: an open source basecaller for Oxford Nanopore sequencing data\n",
      "BeReTa: a systematic method for identifying target transcriptional regulators to enhance microbial production of chemicals\n",
      "ReliableGenome: annotation of genomic regions with high/low variant calling concordance\n",
      "PHYLOViZ 2.0: providing scalable data integration and visualization for multiple phylogenetic inference methods\n",
      "LncVar: a database of genetic variation associated with long non-coding genes\n",
      "OntoBrowser: a collaborative tool for curation of ontologies by subject matter experts\n",
      "Multivariate two-part statistics for analysis of correlated mass spectrometry data from multiple biological specimens\n",
      "HAPRAP: a haplotype-based iterative method for statistical fine mapping using GWAS summary statistics\n",
      "Bayesian parameter estimation for the Wnt pathway: an infinite mixture models approach\n",
      "Logical model specification aided by model-checking techniques: application to the mammalian cell cycle regulation\n",
      "iReMet-flux: constraint-based approach for integrating relative metabolite levels into a stoichiometric metabolic models\n",
      "A weighted exact test for mutually exclusive mutations in cancer\n",
      "Large-scale inference of conjunctive Bayesian networks\n",
      "Patterns of amino acid conservation in human and animal immunodeficiency viruses\n",
      "ModuleAlign: module-based global alignment of protein-protein interaction networks\n",
      "Gene-set association tests for next-generation sequencing data\n",
      "Optimization of signal-to-noise ratio for efficient microarray probe design\n",
      "LuxGLM: a probabilistic covariate model for quantification of DNA methylation modifications with complex experimental designs\n",
      "The Network Library: a framework to rapidly integrate network biology resources\n",
      "Mutual enrichment in aggregated ranked lists with applications to gene expression regulation\n",
      "Simultaneous discovery of cancer subtypes and subtype features by molecular data integration\n",
      "FATSLiM: a fast and robust software to analyze MD simulations of membranes\n",
      "Virtual exploration of early stage atherosclerosis\n",
      "Integrating genomic information with protein sequence and 3D atomic level structure at the RCSB protein data bank\n",
      "SDEAP: a splice graph based differential transcript expression analysis tool for population data\n",
      "ChemTreeMap: an interactive map of biochemical similarity in molecular datasets\n",
      "LCA*: an entropy-based measure for taxonomic assignment within assembled metagenomes\n",
      "NVT: a fast and simple tool for the assessment of RNA-seq normalization strategies\n",
      "WebSTAR3D: a web server for RNA 3D structural alignment\n",
      "genipe: an automated genome-wide imputation pipeline with automatic reporting and statistical tools\n",
      "Predicting G protein-coupled receptor downstream signaling by tissue expression\n",
      "NCMine: Core-peripheral based functional module detection using near-clique mining\n",
      "Excess positional mutual information predicts both local and allosteric mutations affecting beta lactamase drug resistance\n",
      "TopPIC: a software tool for top-down mass spectrometry-based proteoform identification and characterization\n",
      "Online interactive analysis of protein structure ensembles with Bio3D-web\n",
      "VisualGraphX: interactive graph visualization within Galaxy\n",
      "MEMHDX: an interactive tool to expedite the statistical validation and visualization of large HDX-MS datasets\n",
      "sBWT: memory efficient implementation of the hardware-acceleration-friendly Schindler transform for the fast biological sequence mapping\n",
      "Estimating and testing high-dimensional mediation effects in epigenetic studies\n",
      "CellProfiler Analyst: interactive data exploration, analysis and classification of large biological image sets\n",
      "iPTM-mLys: identifying multiple lysine PTM sites and their different types\n",
      "RTCR: a pipeline for complete and accurate recovery of T cell repertoires from high throughput sequencing data\n",
      "AgIn: measuring the landscape of CpG methylation of individual repetitive elements\n",
      "Pseudotime estimation: deconfounding single cell time series\n",
      "MultiQC: summarize analysis results for multiple tools and samples in a single report\n",
      "Analysis of aggregated cell-cell statistical distances within pathways unveils therapeutic-resistance mechanisms in circulating tumor cells\n",
      "What time is it? Deep learning approaches for circadian rhythms\n",
      "BioASF: a framework for automatically generating executable pathway models specified in BioPAX\n",
      "Classifying and segmenting microscopy images with deep multiple instance learning\n",
      "Ensemble-based evaluation for protein structure models\n",
      "Reconstructing the temporal progression of HIV-1 immune response pathways\n",
      "Prediction of ribosome footprint profile shapes from transcript sequences\n",
      "Analysis of differential splicing suggests different modes of short-term splicing regulation\n",
      "ISMB 2016 Proceedings\n",
      "GSA-Lightning: ultra-fast permutation-based gene set analysis\n",
      "Rapid and powerful detection of subtle allelic imbalance from exome sequencing data with hapLOHseq\n",
      "Zerone: a ChIP-seq discretizer for multiple replicates with built-in quality control\n",
      "GHap: an R package for genome-wide haplotyping\n",
      "KCMBT: a k-mer Counter based on Multiple Burst Trees\n",
      "A knowledge-based approach for predicting gene-disease associations\n",
      "pong: fast analysis and visualization of latent clusters in population genetic data\n",
      "Predicting regulatory variants with composite statistic\n",
      "MIA: non-targeted mass isotopolome analysis\n",
      "SETH detects and normalizes genetic variants in text\n",
      "RANKS: a flexible tool for node label ranking and classification in biological networks\n",
      "CART-a chemical annotation retrieval toolkit\n",
      "Differential rhythmicity: detecting altered rhythmicity in biological data\n",
      "seqlm: an MDL based method for identifying differentially methylated regions in high density methylation array data\n",
      "GSP: a web-based platform for designing genome-specific primers in polyploids\n",
      "Haplotype synthesis analysis reveals functional variants underlying known genome-wide associated susceptibility loci\n",
      "ecceTERA: comprehensive gene tree-species tree reconciliation using parsimony\n",
      "Computational clustering for viral reference proteomes\n",
      "Detecting critical state before phase transition of complex biological systems by hidden Markov model\n",
      "repgenHMM: a dynamic programming tool to infer the rules of immune receptor generation from sequence data\n",
      "PinSnps: structural and functional analysis of SNPs in the context of protein interaction networks\n",
      "A web application for sample size and power calculation in case-control microbiome studies\n",
      "ORFanFinder: automated identification of taxonomically restricted orphan genes\n",
      "EPS: an empirical Bayes approach to integrating pleiotropy and tissue-specific information for prioritizing risk genes\n",
      "RIPPER: a framework for MS1 only metabolomics and proteomics label-free relative quantification\n",
      "2dSpAn: semiautomated 2-d segmentation, classification and analysis of hippocampal dendritic spine plasticity\n",
      "RCP: a novel probe design bias correction method for Illumina Methylation BeadChip\n",
      "Bioinformatics programs are 31-fold over-represented among the highest impact scientific papers of the past two decades\n",
      "Combenefit: an interactive platform for the analysis and visualization of drug combinations\n",
      "MEANS: python package for Moment Expansion Approximation, iNference and Simulation\n",
      "densityCut: an efficient and versatile topological approach for automatic clustering of biological data\n",
      "ASAFE: ancestry-specific allele frequency estimation\n",
      "rnaQUAST: a quality assessment tool for de novo transcriptome assemblies\n",
      "DisGeNET-RDF: harnessing the innovative power of the Semantic Web to explore the genetic basis of diseases\n",
      "Bayesian nonparametrics in protein remote homology search\n",
      "OREMPRO web server: orientation and assessment of atomistic and coarse-grained structures of membrane proteins\n",
      "gkmSVM: an R package for gapped-kmer SVM\n",
      "TreeQTL: hierarchical error control for eQTL findings\n",
      "Tools4miRs - one place to gather all the tools for miRNA analysis\n",
      "MOCAT2: a metagenomic assembly, annotation and profiling framework\n",
      "ProtPOS: a python package for the prediction of protein preferred orientation on a surface\n",
      "Dr.seq: a quality control and analysis pipeline for droplet sequencing\n",
      "GenomeRunner web server: regulatory similarity and differences define the functional impact of SNP sets\n",
      "Quantitative phosphoproteomics-based molecular network description for high-resolution kinase-substrate interactome analysis\n",
      "CellMiner Companion: an interactive web application to explore CellMiner NCI-60 data\n",
      "EBglmnet: a comprehensive R package for sparse generalized linear regression models\n",
      "FRODOCK 2.0: fast protein-protein docking server\n",
      "An algorithm for designing minimal microbial communities with desired metabolic capacities\n",
      "Selecting a classification function for class prediction with gene expression data\n",
      "SplAdder: identification, quantification and testing of alternative splicing events from RNA-Seq data\n",
      "An efficient gene-gene interaction test for genome-wide association studies in trio families\n",
      "Predicting physiologically relevant SH3 domain mediated protein-protein interactions in yeast\n",
      "Model-based media selection to minimize the cost of metabolic cooperation in microbial ecosystems\n",
      "Sprites: detection of deletions from sequencing data by re-aligning split reads\n",
      "ASDB: a resource for probing protein functions with small molecules\n",
      "Galaxy Portal: interacting with the galaxy platform through mobile devices\n",
      "BioCircos.js: an interactive Circos JavaScript library for biological data visualization on web applications\n",
      "FILTUS: a desktop GUI for fast and efficient detection of disease-causing variants, including a novel autozygosity detector\n",
      "FamAgg: an R package to evaluate familial aggregation of traits in large pedigrees\n",
      "MS-REDUCE: an ultrafast technique for reduction of big mass spectrometry data for high-throughput processing\n",
      "The TraDIS toolkit: sequencing and analysis for dense transposon mutant libraries\n",
      "Sampletrees and Rsampletrees: sampling gene genealogies conditional on SNP genotype data\n",
      "Piecewise parameter estimation for stochastic models in COPASI\n",
      "lnCaNet: pan-cancer co-expression network for human lncRNA and cancer genes\n",
      "Multi-omics enrichment analysis using the GeneTrail2 web service\n",
      "SIRAH tools: mapping, backmapping and visualization of coarse-grained models\n",
      "Integration of string and de Bruijn graphs for genome assembly\n",
      "MTG2: an efficient algorithm for multivariate linear mixed model analysis based on genomic information\n",
      "Improving microRNA target prediction by modeling with unambiguously identified microRNA-target pairs from CLIP-ligation studies\n",
      "PINCAGE: probabilistic integration of cancer genomics data for perturbed gene identification and sample classification\n",
      "Bayesian variable selection for binary outcomes in high-dimensional genomic studies using non-local priors\n",
      "Prediction of missing sequences and branch lengths in phylogenomic data\n",
      "Path2PPI: an R package to predict protein-protein interaction networks for a set of proteins\n",
      "Robust quantitative scratch assay\n",
      "Authors' response to 'Comment on: ERGC: An efficient Referential Genome Compression Algorithm'\n",
      "DeNovo: virus-host sequence-based protein-protein interaction prediction\n",
      "Computational identification of piRNA targets on mouse mRNAs\n",
      "CG2AA: backmapping protein coarse-grained structures\n",
      "Adaptive gene- and pathway-trait association testing with GWAS summary statistics\n",
      "FuMa: reporting overlap in RNA-seq detected fusion genes\n",
      "CASSIS and SMIPS: promoter-based prediction of secondary metabolite gene clusters in eukaryotic genomes\n",
      "Joint detection of copy number variations in parent-offspring trios\n",
      "Maligner: a fast ordered restriction map aligner\n",
      "Representation and inference of cellular architecture for metabolic reconstruction and modeling\n",
      "Comment on: 'ERGC: an efficient referential genome compression algorithm'\n",
      "Admix'em: a flexible framework for forward-time simulations of hybrid populations with selection and mate choice\n",
      "SVD-phy: improved prediction of protein functional associations through singular value decomposition of phylogenetic profiles\n",
      "RNA-Enrich: a cut-off free functional enrichment testing method for RNA-seq with improved detection power\n",
      "Synchronized navigation and comparative analyses across Ensembl complete bacterial genomes with INSYGHT\n",
      "Genefu: an R/Bioconductor package for computation of gene expression-based signatures in breast cancer\n",
      "ParTIES: a toolbox for Paramecium interspersed DNA elimination studies\n",
      "LocusExplorer: a user-friendly tool for integrated visualization of human genetic association data and biological annotations\n",
      "AccessFold: predicting RNA-RNA interactions with consideration for competing self-structure\n",
      "Prioritizing hypothesis tests for high throughput data\n",
      "JSpeciesWS: a web server for prokaryotic species circumscription based on pairwise genome comparison\n",
      "Global copy number profiling of cancer genomes\n",
      "TCGA2STAT: simple TCGA data access for integrated statistical analysis in R\n",
      "BioPartsBuilder: a synthetic biology tool for combinatorial assembly of biological parts\n",
      "Using de novo protein structure predictions to measure the quality of very large multiple sequence alignments\n",
      "contamDE: differential expression analysis of RNA-seq data for contaminated tumor samples\n",
      "De novo identification of replication-timing domains in the human genome by deep learning\n",
      "A hidden Markov random field-based Bayesian method for the detection of long-range chromosomal interactions in Hi-C data\n",
      "TESS: an R package for efficiently simulating phylogenetic trees and performing Bayesian inference of lineage diversification rates\n",
      "Fit3D: a web application for highly accurate screening of spatial residue patterns in protein structure data\n",
      "CAMUR: Knowledge extraction from RNA-seq cancer data through equivalent classification rules\n",
      "ancGWAS: a post genome-wide association study method for interaction, pathway and ancestry analysis in homogeneous and admixed populations\n",
      "caRpools: an R package for exploratory data analysis and documentation of pooled CRISPR/Cas9 screens\n",
      "Omokage search: shape similarity search service for biomolecular structures in both the PDB and EMDB\n",
      "DARA: a web server for rapid search of structural neighbours using solution small angle X-ray scattering data\n",
      "Pycellerator: an arrow-based reaction-like modelling language for biological simulations\n",
      "GenoWAP: GWAS signal prioritization through integrated analysis of genomic functional annotation\n",
      "iEnhancer-2L: a two-layer predictor for identifying enhancers and their strength by pseudo k-tuple nucleotide composition\n",
      "Computational discovery of specificity-conferring sites in non-ribosomal peptide synthetases\n",
      "BiTrinA--multiscale binarization and trinarization with quality analysis\n",
      "FogLight: an efficient matrix-based approach to construct metabolic pathways by search space reduction\n",
      "The new protein topology graph library web server\n",
      "Rust-Bio: a fast and safe bioinformatics library\n",
      "JEPEGMIX: gene-level joint analysis of functional SNPs in cosmopolitan cohorts\n",
      "regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests\n",
      "ANARCI: antigen receptor numbering and receptor classification\n",
      "XPIWIT--an XML pipeline wrapper for the Insight Toolkit\n",
      "CLUSTERnGO: a user-defined modelling platform for two-stage clustering of time-series data\n",
      "MDD-SOH: exploiting maximal dependence decomposition to identify S-sulfenylation sites with substrate motifs\n",
      "Approximately independent linkage disequilibrium blocks in human populations\n",
      "APTANI: a computational tool to select aptamers through sequence-structure motif analysis of HT-SELEX data\n",
      "NanoOK: multi-reference alignment analysis of nanopore sequencing data, quality and error profiles\n",
      "CLAMMS: a scalable algorithm for calling common and rare copy number variants from exome sequencing data\n",
      "PyPDB: a Python API for the Protein Data Bank\n",
      "Cas-Designer: a web-based tool for choice of CRISPR-Cas9 target sites\n",
      "SeDuS: segmental duplication simulator\n",
      "Group and sparse group partial least square approaches applied in genomics context\n",
      "ResiCon: a method for the identification of dynamic domains, hinges and interfacial regions in proteins\n",
      "BMix: probabilistic modeling of occurring substitutions in PAR-CLIP data\n",
      "Benchmark analysis of algorithms for determining and quantifying full-length mRNA splice forms from RNA-seq data\n",
      "ABIF Manager: simple reading and editing of ABIF formatted files\n",
      "HapCol: accurate and memory-efficient haplotype assembly from long reads\n",
      "GenomeLaser: fast and accurate haplotyping from pedigree genotypes\n",
      "DSviaDRM: an R package for estimating disease similarity via dysfunctional regulation mechanism\n",
      "GMcloser: closing gaps in assemblies accurately with a likelihood-based selection of contig or long-read alignments\n",
      "Improving protein fold recognition with hybrid profiles combining sequence and structure evolution\n",
      "ERC analysis: web-based inference of gene function via evolutionary rate covariation\n",
      "NAM: association studies in multiple populations\n",
      "stringgaussnet: from differentially expressed genes to semantic and Gaussian networks generation\n",
      "MICC: an R package for identifying chromatin interactions from ChIA-PET data\n",
      "BackCLIP: a tool to identify common background presence in PAR-CLIP datasets\n",
      "SNPGenie: estimating evolutionary parameters to detect natural selection using pooled next-generation sequencing data\n",
      "A mutation profile for top-k patient search exploiting Gene-Ontology and orthogonal non-negative matrix factorization\n",
      "Statistical models for RNA-seq data derived from a two-condition 48-replicate experiment\n",
      "RxnSim: a tool to compare biochemical reactions\n",
      "A Bayesian approach for structure learning in oscillating regulatory networks\n",
      "DINGO: differential network analysis in genomics\n",
      "PSIKO2: a fast and versatile tool to infer population stratification on various levels in GWAS\n",
      "al3c: high-performance software for parameter inference using Approximate Bayesian Computation\n",
      "A multivariate Bernoulli model to predict DNaseI hypersensitivity status from haplotype data\n",
      "Integrating full spectrum of sequence features into predicting functional microRNA-mRNA interactions\n",
      "biRte: Bayesian inference of context-specific regulator activities and transcriptional networks\n",
      "Hyperscape: visualization for complex biological networks\n",
      "NCC-AUC: an AUC optimization method to identify multi-biomarker panel for cancer prognosis from genomic and clinical data\n",
      "Rchemcpp: a web service for structural analoging in ChEMBL, Drugbank and the Connectivity Map\n",
      "AmyLoad: website dedicated to amyloidogenic protein fragments\n",
      "A note on the false discovery rate of novel peptides in proteogenomics\n",
      "De novo meta-assembly of ultra-deep sequencing data\n",
      "IgRepertoireConstructor: a novel algorithm for antibody repertoire construction and immunoproteogenomics analysis\n",
      "Using collective expert judgements to evaluate quality measures of mass spectrometry images\n",
      "MeSHLabeler: improving the accuracy of large-scale MeSH indexing by integrating diverse evidence\n",
      "Comparing genomes with rearrangements and segmental duplications\n",
      "MEMCover: integrated analysis of mutual exclusivity and functional network reveals dysregulated pathways across multiple cancer types\n",
      "Cypiripi: exact genotyping of CYP2D6 using high-throughput sequencing data\n",
      "PAGER: constructing PAGs and new PAG-PAG relationships for network biology\n",
      "Genome-wide detection of intervals of genetic heterogeneity associated with complex traits\n",
      "Integrative random forest for gene regulatory network inference\n",
      "Metabolome-scale de novo pathway reconstruction using regioisomer-sensitive graph alignments\n",
      "Finding optimal interaction interface alignments between biological complexes\n",
      "Using kernelized partial canonical correlation analysis to study directly coupled side chains and allostery in small G proteins\n",
      "Large-scale model quality assessment for improving protein tertiary structure prediction\n",
      "ISMB/ECCB 2015\n",
      "EXIMS: an improved data analysis pipeline based on a new peak picking method for EXploring Imaging Mass Spectrometry data\n",
      "Automated profiling of individual cell-cell interactions from high-throughput time-lapse imaging microscopy in nanowell grids (TIMING)\n",
      "BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs\n",
      "The pervasiveness and plasticity of circadian oscillations: the coupled circadian-oscillators framework\n",
      "GeIST: a pipeline for mapping integrated DNA elements\n",
      "Assembling short reads from jumping libraries with large insert sizes\n",
      "RPdb: a database of experimentally verified cellular reprogramming records\n",
      "QVZ: lossy compression of quality values\n",
      "Bayesian mixture analysis for metagenomic community profiling\n",
      "pwOmics: an R package for pathway-based integration of time-series omics data using public database knowledge\n",
      "When less is more: 'slicing' sequencing data improves read decoding accuracy and de novo assembly quality\n",
      "ABC: a tool to identify SNVs causing allele-specific transcription factor binding from ChIP-Seq experiments\n",
      "TRAL: tandem repeat annotation library\n",
      "IMSEQ--a fast and error aware approach to immunogenetic sequence analysis\n",
      "SpeeDB: fast structural protein searches\n",
      "GEO2Enrichr: browser extension and server app to extract gene sets from GEO and analyze them for biological functions\n",
      "Fast parametric time warping of peak lists\n",
      "Interactive analysis of large cancer copy number studies with Copy Number Explorer\n",
      "Using combined evidence from replicates to evaluate ChIP-seq peaks\n",
      "Tax4Fun: predicting functional profiles from metagenomic 16S rRNA data\n",
      "Identification of C2H2-ZF binding preferences from ChIP-seq data using RCADE\n",
      "pez: phylogenetics for the environmental sciences\n",
      "The SwissLipids knowledgebase for lipid biology\n",
      "RVD2: an ultra-sensitive variant detection model for low-depth heterogeneous next-generation sequencing data\n",
      "ESPRESSO: taking into account assessment errors on outcome and exposures in power analysis for association studies\n",
      "MultiMeta: an R package for meta-analyzing multi-phenotype genome-wide association studies\n",
      "Conformational sampling and structure prediction of multiple interacting loops in soluble and β-barrel membrane proteins using multi-loop distance-guided chain-growth Monte Carlo method\n",
      "Cellular phenotype database: a repository for systems microscopy data\n",
      "Overlap and diversity in antimicrobial peptide databases: compiling a non-redundant set of sequences\n",
      "Outlier detection at the transcriptome-proteome interface\n",
      "Assessing allele-specific expression across multiple tissues from RNA-seq read data\n",
      "Identification of a small set of plasma signalling proteins using neural network for prediction of Alzheimer's disease\n",
      "PRROC: computing and visualizing precision-recall and receiver operating characteristic curves in R\n",
      "EW_dmGWAS: edge-weighted dense module search for genome-wide association studies and gene expression profiles\n",
      "Design of shortest double-stranded DNA sequences covering all k-mers with applications to protein-binding microarrays and synthetic enhancers\n",
      "Multiscale reaction-diffusion simulations with Smoldyn\n",
      "Crossfinder-assisted mapping of protein crosslinks formed by site-specifically incorporated crosslinkers\n",
      "Two-group comparisons of zero-inflated intensity values: the choice of test statistic matters\n",
      "WGBSSuite: simulating whole-genome bisulphite sequencing data and benchmarking differential DNA methylation analysis tools\n",
      "Hi-Jack: a novel computational framework for pathway-based inference of host-pathogen interactions\n",
      "Estimating the proportion of true null hypotheses when the statistics are discrete\n",
      "L-GRAAL: Lagrangian graphlet-based network aligner\n",
      "Detection of significant protein coevolution\n",
      "LCR-eXXXplorer: a web platform to search, visualize and share data for low complexity regions in protein sequences\n",
      "Pattern recognition methods to relate time profiles of gene expression with phenotypic data: a comparative study\n",
      "Deep sequencing analysis of viral infection and evolution allows rapid and detailed characterization of viral mutant spectrum\n",
      "Annocript: a flexible pipeline for the annotation of transcriptomes able to identify putative long noncoding RNAs\n",
      "Unified representation of genetic variants\n",
      "Extending P450 site-of-metabolism models with region-resolution data\n",
      "Sambamba: fast processing of NGS alignment formats\n",
      "RAPTR-SV: a hybrid method for the detection of structural variants\n",
      "DOSE: an R/Bioconductor package for disease ontology semantic and enrichment analysis\n",
      "MESMER: minimal ensemble solutions to multiple experimental restraints\n",
      "Detecting differential peaks in ChIP-seq signals with ODIN\n",
      "Large-scale exploration and analysis of drug combinations\n",
      "Whiteboard: a framework for the programmatic visualization of complex biological analyses\n",
      "PANNZER: high-throughput functional annotation of uncharacterized proteins in an error-prone environment\n",
      "Reference-based compression of short-read sequences using path encoding\n",
      "Analysis of nanopore data using hidden Markov models\n",
      "Similarity-based prediction for Anatomical Therapeutic Chemical classification of drugs by integrating multiple data sources\n",
      "SFA-SPA: a suffix array based short peptide assembler for metagenomic data\n",
      "NetExplore: a web server for modeling small network motifs\n",
      "Dizzy-Beats: a Bayesian evidence analysis tool for systems biology\n",
      "Deterministic identification of specific individuals from GWAS results\n",
      "Cooperative development of logical modelling standards and tools with CoLoMoTo\n",
      "protr/ProtrWeb: R package and web server for generating various numerical representation schemes of protein sequences\n",
      "The assembly of miRNA-mRNA-protein regulatory networks using high-throughput expression data\n",
      "Context-specific metabolic network reconstruction of a naphthalene-degrading bacterial community guided by metaproteomic data\n",
      "MethylMix: an R package for identifying DNA methylation-driven genes\n",
      "drexplorer: A tool to explore dose-response relationships and drug-drug interactions\n",
      "QTLMiner: QTL database curation by mining tables in literature\n",
      "NLR-parser: rapid annotation of plant NLR complements\n",
      "Combining tree-based and dynamical systems for the inference of gene regulatory networks\n",
      "DDIG-in: detecting disease-causing genetic variations due to frameshifting indels and nonsense mutations employing sequence and structural properties at nucleotide and protein levels\n",
      "The anisotropic network model web server at 2015 (ANM 2.0)\n",
      "Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models\n",
      "PRSice: Polygenic Risk Score software\n",
      "UProC: tools for ultra-fast protein domain classification\n",
      "RRDistMaps: a UCSF Chimera tool for viewing and comparing protein distance maps\n",
      "PrimerDesign-M: a multiple-alignment based multiple-primer design tool for walking across variable genomes\n",
      "MpTheory Java library: a multi-platform Java library for systems biology based on the Metabolic P theory\n",
      "andi: fast and accurate estimation of evolutionary distances between closely related genomes\n",
      "SMARTS: reconstructing disease response networks from multiple individuals using time series gene expression data\n",
      "Lipid-Pro: a computational lipid identification solution for untargeted lipidomics on data-independent acquisition tandem mass spectrometry platforms\n",
      "A bio-inspired computing model for ovarian carcinoma classification and oncogene detection\n",
      "DISSECT: an assignment-free Bayesian discovery method for species delimitation under the multispecies coalescent\n",
      "Efficient searching and annotation of metabolic networks using chemical similarity\n",
      "Single-cell transcriptional analysis to uncover regulatory circuits driving cell fate decisions in early mouse development\n",
      "Multilevel regularized regression for simultaneous taxa selection and network construction with metagenomic count data\n",
      "Hi-Corrector: a fast, scalable and memory-efficient package for normalizing large-scale Hi-C data\n",
      "MeRP: a high-throughput pipeline for Mendelian randomization analysis\n",
      "What handedness and angles between helices has the studied three-helical protein domain?\n",
      "PyFDAP: automated analysis of fluorescence decay after photoconversion (FDAP) experiments\n",
      "Development of a robust classifier for quality control of reverse-phase protein arrays\n",
      "Computational framework for next-generation sequencing of heterogeneous viral populations using combinatorial pooling\n",
      "Detecting time periods of differential gene expression using Gaussian processes: an application to endothelial cells exposed to radiotherapy dose fraction\n",
      "Semantic Body Browser: graphical exploration of an organism and spatially resolved expression data visualization\n",
      "FlaiMapper: computational annotation of small ncRNA-derived fragments using RNA-seq high-throughput data\n",
      "Introducing dAUTObase: a first step towards the global scale geoepidemiology of autoimmune syndromes and diseases\n",
      "Differential regulation enrichment analysis via the integration of transcriptional regulatory network and gene expression data\n",
      "miRseqViewer: multi-panel visualization of sequence, structure and expression for analysis of microRNA sequencing data\n",
      "Accurate estimation of haplotype frequency from pooled sequencing data and cost-effective identification of rare haplotype carriers by overlapping pool sequencing\n",
      "A stochastic automaton model for simulating kinesin processivity\n",
      "ExomeAI: detection of recurrent allelic imbalance in tumors using whole-exome sequencing data\n",
      "Visual workflows for 13C-metabolic flux analysis\n",
      "FungiFun2: a comprehensive online resource for systematic analysis of gene lists from fungal species\n",
      "DupliPHY-Web: a web server for DupliPHY and DupliPHY-ML\n",
      "KDDN: an open-source Cytoscape app for constructing differential dependency networks with significant rewiring\n",
      "Curation, integration and visualization of bacterial virulence factors in PATRIC\n",
      "HTSeq--a Python framework to work with high-throughput sequencing data\n",
      "EasyStrata: evaluation and visualization of stratified genome-wide association meta-analysis data\n",
      "PINBPA: cytoscape app for network analysis of GWAS data\n",
      "Comprehensive large-scale assessment of intrinsic protein disorder\n",
      "NAIL, a software toolset for inferring, analyzing and visualizing regulatory networks\n",
      "Proteomic analysis and prediction of human phosphorylation sites in subcellular level reveal subcellular specificity\n",
      "Making novel proteins from pseudogenes\n",
      "PseKNC-General: a cross-platform package for generating various modes of pseudo nucleotide compositions\n",
      "EUROCarbDB(CCRC): a EUROCarbDB node for storing glycomics standard data\n",
      "mod_bio: Apache modules for Next-Generation sequencing data\n",
      "Estimation of GFP-tagged RNA numbers from temporal fluorescence intensity data\n",
      "BigDataScript: a scripting language for data pipelines\n",
      "RCSB PDB Mobile: iOS and Android mobile apps to provide data access and visualization to the RCSB Protein Data Bank\n",
      "HiGwas: how to compute longitudinal GWAS data in population designs\n",
      "DeepMSA: constructing deep multiple sequence alignment to improve contact prediction and fold-recognition for distant-homology proteins\n",
      "When a single lineage is not enough: Uncertainty-Aware Tracking for spatio-temporal live-cell image analysis\n",
      "ALTRE: workflow for defining ALTered regulatory elements using chromatin accessibility data\n",
      "Support vector machine model of developmental brain gene expression data for prioritization of Autism risk gene candidates\n",
      "A novel copy number variants kernel association test with application to autism spectrum disorders studies\n",
      "Designing robust watermark barcodes for multiplex long-read sequencing\n",
      "EpimiRBase: a comprehensive database of microRNA-epilepsy associations\n",
      "Fitchi: haplotype genealogy graphs based on the Fitch algorithm\n",
      "TiMEx: a waiting time model for mutually exclusive cancer alterations\n",
      "MESA: automated assessment of synthetic DNA fragments and simulation of DNA synthesis, storage, sequencing and PCR errors\n",
      "Supervised learning is an accurate method for network-based gene classification\n",
      "The bioinformatics wealth of nations\n",
      "V-SVA: an R Shiny application for detecting and annotating hidden sources of variation in single-cell RNA-seq data\n",
      "scRMD: imputation for single cell RNA-seq data via robust matrix decomposition\n",
      "Breeze: an integrated quality control and data analysis application for high-throughput drug screening\n",
      "SynergyFinder: a web application for analyzing drug combination dose-response matrix data\n",
      "PSORTm: a bacterial and archaeal protein subcellular localization prediction tool for metagenomics data\n",
      "rMSIproc: an R package for mass spectrometry imaging data processing\n",
      "StackCPPred: a stacking and pairwise energy content-based prediction of cell-penetrating peptides and their uptake efficiency\n",
      "Ensemble learning for classifying single-cell data and projection across reference atlases\n",
      "Rhapsody: predicting the pathogenicity of human missense variants\n",
      "Composite Kernel Association Test (CKAT) for SNP-set joint assessment of genotype and genotype-by-treatment interaction in Pharmacogenetics studies\n",
      "Proline: an efficient and user-friendly software suite for large-scale proteomics\n",
      "DIMERBOW: exploring possible GPCR dimer interfaces\n",
      "Nubeam-dedup: a fast and RAM-efficient tool to de-duplicate sequencing reads without mapping\n",
      "Protein-ligand binding residue prediction enhancement through hybrid deep heterogeneous learning of sequence and structure data\n",
      "CMF-Impute: an accurate imputation tool for single-cell RNA-seq data\n",
      "A unified ILP framework for core ancestral genome reconstruction problems\n",
      "Methplotlib: analysis of modified nucleotides from nanopore sequencing\n",
      "BionoiNet: ligand-binding site classification with off-the-shelf deep neural network\n",
      "Bayesian inference using qualitative observations of underlying continuous variables\n",
      "InterLig: improved ligand-based virtual screening using topologically independent structural alignments\n",
      "FluPhenotype-a one-stop platform for early warnings of the influenza A virus\n",
      "atomium-a Python structure parser\n",
      "DeepHIT: a deep learning framework for prediction of hERG-induced cardiotoxicity\n",
      "Genesis and Gappa: processing, analyzing and visualizing phylogenetic (placement) data\n",
      "LAMPA, LArge Multidomain Protein Annotator, and its application to RNA virus polyproteins\n",
      "GEMtractor: extracting views into genome-scale metabolic models\n",
      "decorate: differential epigenetic correlation test\n",
      "Tempel: time-series mutation prediction of influenza A viruses via attention-based recurrent neural networks\n",
      "Heterogeneous networks integration for disease-gene prioritization with node kernels\n",
      "iPAC: a genome-guided assembler of isoforms via phasing and combing paths\n",
      "Identifying and removing haplotypic duplication in primary genome assemblies\n",
      "incaRNAfbinv 2.0: a webserver and software with motif control for fragment-based design of RNAs\n",
      "Generalizable sgRNA design for improved CRISPR/Cas9 editing efficiency\n",
      "Differential network analysis of multiple human tissue interactomes highlights tissue-selective processes and genetic disorder genes\n",
      "Simulating trees with millions of species\n",
      "Curation and annotation of planarian gene expression patterns with segmented reference morphologies\n",
      "LigRMSD: a web server for automatic structure matching and RMSD calculations among identical and similar compounds in protein-ligand docking\n",
      "Interaction screening by Kendall's partial correlation for ultrahigh-dimensional data with survival trait\n",
      "Defining data-driven primary transcript annotations with primaryTranscriptAnnotation in R\n",
      "DeepSimulator1.5: a more powerful, quicker and lighter simulator for Nanopore sequencing\n",
      "cd2sbgnml: bidirectional conversion between CellDesigner and SBGN formats\n",
      "PaSiT: a novel approach based on short-oligonucleotide frequencies for efficient bacterial identification and typing\n",
      "Differential Expression Gene Explorer (DrEdGE): a tool for generating interactive online visualizations of gene expression datasets\n",
      "Application of topic models to a compendium of ChIP-Seq datasets uncovers recurrent transcriptional regulatory modules\n",
      "Bringing data from curated pathway resources to Cytoscape with OmniPath\n",
      "Bioinformatics pipeline using JUDI: Just Do It!\n",
      "ShinyGO: a graphical gene-set enrichment tool for animals and plants\n",
      "A deep learning architecture for metabolic pathway prediction\n",
      "Causal network perturbations for instance-specific analysis of single cell and disease samples\n",
      "AQUA-DUCT 1.0: structural and functional analysis of macromolecules from an intramolecular voids perspective\n",
      "PDEparams: parameter fitting toolbox for partial differential equations in python\n",
      "Using deep learning to associate human genes with age-related diseases\n",
      "dSreg: a Bayesian model to integrate changes in splicing and RNA-binding protein activity\n",
      "FFLtool: a web server for transcription factor and miRNA feed forward loop analysis in human\n",
      "Semantic persistence of ambiguous biomedical names in the citation network\n",
      "psSubpathway: a software package for flexible identification of phenotype-specific subpathways in cancer progression\n",
      "ImPLoc: a multi-instance deep learning model for the prediction of protein subcellular localization based on immunohistochemistry images\n",
      "RocaSec: a standalone GUI-based package for robust co-evolutionary analysis of proteins\n",
      "schex avoids overplotting for large single-cell RNA-sequencing datasets\n",
      "The use of missing values in proteomic data-independent acquisition mass spectrometry to enable disease activity discrimination\n",
      "Designing a general method for predicting the regulatory relationships between long noncoding RNAs and protein-coding genes based on multi-omics characteristics\n",
      "PyIOmica: longitudinal omics analysis and trend identification\n",
      "Organoid model of mammographic density displays a higher frequency of aberrant colony formations with radiation exposure\n",
      "SCGid: a consensus approach to contig filtering and genome prediction from single-cell sequencing libraries of uncultured eukaryotes\n",
      "CoMM-S2: a collaborative mixed model using summary statistics in transcriptome-wide association studies\n",
      "All-FIT: allele-frequency-based imputation of tumor purity from high-depth sequencing data\n",
      "Easy-HLA: a validated web application suite to reveal the full details of HLA typing\n",
      "HypercubeME: two hundred million combinatorially complete datasets from a single experiment\n",
      "Link-HD: a versatile framework to explore and integrate heterogeneous microbial communities\n",
      "SBML2Modelica: integrating biochemical models within open-standard simulation ecosystems\n",
      "Cross-lingual semantic annotation of biomedical literature: experiments in Spanish and English\n",
      "RaNA-Seq: Interactive RNA-Seq analysis from FASTQ files to functional analysis\n",
      "GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database\n",
      "ProAffiMuSeq: sequence-based method to predict the binding free energy change of protein-protein complexes upon mutation using functional classification\n",
      "Diversity of rhodopsins in cultivated bacteria of the family Geodermatophilaceae associated with non-aquatic environments\n",
      "Estimation of Eosinophil Cells in Cord Blood with References Based on Blood in Adults via Bayesian Measurement Error Modeling\n",
      "EpiDISH web server: Epigenetic Dissection of Intra-Sample-Heterogeneity with online GUI\n",
      "DrawGlycan-SNFG & gpAnnotate: Rendering glycans and annotating glycopeptide mass spectra\n",
      "BAGSE: a Bayesian hierarchical model approach for gene set enrichment analysis\n",
      "DiNGO: standalone application for Gene Ontology and Human Phenotype Ontology term enrichment analysis\n",
      "HiCBricks: building blocks for efficient handling of large Hi-C datasets\n",
      "The evolution of contact prediction: evidence that contact selection in statistical contact prediction is changing\n",
      "Taxonomic weighting improves the accuracy of a gap-filling algorithm for metabolic models\n",
      "OLOGRAM: Determining significance of total overlap length between genomic regions sets\n",
      "Enzyme annotation in UniProtKB using Rhea\n",
      "Kalign 3: multiple sequence alignment of large data sets\n",
      "Bayesian GWAS with Structured and Non-Local Priors\n",
      "MIAmS: microsatellite instability detection on NGS amplicons data\n",
      "gep2pep: a Bioconductor package for the creation and analysis of pathway-based expression profiles\n",
      "3D flow field estimation and assessment for live cell fluorescence microscopy\n",
      "Inferring subgroup-specific driver genes from heterogeneous cancer samples via subspace learning with subgroup indication\n",
      "µ -ANT: semantic microaggregation-based anonymization tool\n",
      "Computationally predicted gene regulatory networks in molluscan biomineralization identify extracellular matrix production and ion transportation pathways\n",
      "dbInDel: a database of enhancer-associated insertion and deletion variants by analysis of H3K27ac ChIP-Seq\n",
      "Cancer classification of single-cell gene expression data by neural network\n",
      "Expectation pooling: an effective and interpretable pooling method for predicting DNA-protein binding\n",
      "Single-sample landscape entropy reveals the imminent phase transition during disease progression\n",
      "ErrorTracer: an algorithm for identifying the origins of inconsistencies in genome-scale metabolic models\n",
      "Identifying enhancer-promoter interactions with neural network based on pre-trained DNA vectors and attention mechanism\n",
      "EvoEF2: accurate and fast energy function for computational protein design\n",
      "Meltos: multi-sample tumor phylogeny reconstruction for structural variants\n",
      "SCSsim: an integrated tool for simulating single-cell genome sequencing data\n",
      "Evaluation of bottom-up and top-down mass spectrum identifications with different customized protein sequences databases\n",
      "DeepCleave: a deep learning predictor for caspase and matrix metalloprotease substrates and cleavage sites\n",
      "EpiGraph: an open-source platform to quantify epithelial organization\n",
      "Subtype-specific transcriptional regulators in breast tumors subjected to genetic and epigenetic alterations\n",
      "ShaKer: RNA SHAPE prediction using graph kernel\n",
      "Efficient haplotype matching between a query and a panel for genealogical search\n",
      "Collaborative intra-tumor heterogeneity detection\n",
      "A joint method for marker-free alignment of tilt series in electron tomography\n",
      "TreeMerge: a new method for improving the scalability of species tree estimation methods\n",
      "Locality-sensitive hashing for the edit distance\n",
      "NPS: scoring and evaluating the statistical significance of peptidic natural product-spectrum matches\n",
      "Predicting drug-induced transcriptome responses of a wide range of human cell lines by a novel tensor-train decomposition algorithm\n",
      "Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology\n",
      "Summarizing the solution space in tumor phylogeny inference by multiple consensus trees\n",
      "Integrating regulatory DNA sequence and gene expression to predict genome-wide chromatin accessibility across cellular contexts\n",
      "DeepLigand: accurate prediction of MHC class I ligands using peptide embedding\n",
      "Minnow: a principled framework for rapid simulation of dscRNA-seq data at the read level\n",
      "Controlling large Boolean networks with single-step perturbations\n",
      "Model-based optimization of subgroup weights for survival analysis\n",
      "Modeling clinical and molecular covariates of mutational process activity in cancer\n",
      "Comprehensive evaluation of deep learning architectures for prediction of DNA/RNA sequence binding specificities\n",
      "Accurate and efficient cell lineage tree inference from noisy single cell data: the maximum likelihood perfect phylogeny approach\n",
      "Association rule mining to identify transcription factor interactions in genomic regions\n",
      "Semi-blind sparse affine spectral unmixing of autofluorescence-contaminated micrographs\n",
      "Gene relevance based on multiple evidences in complex networks\n",
      "Accurate estimation of microbial sequence diversity with Distanced\n",
      "Partition: a surjective mapping approach for dimensionality reduction\n",
      "CROSSalive: a web server for predicting the in vivo structure of RNA molecules\n",
      "NACHO: an R package for quality control of NanoString nCounter data\n",
      "miComplete: weighted quality evaluation of assembled microbial genomes\n",
      "Knot_pull-python package for biopolymer smoothing and knot detection\n",
      "Spectrum: fast density-aware spectral clustering for single and multi-omic data\n",
      "CRDS: Consensus Reverse Docking System for target fishing\n",
      "Multi-scale structural analysis of proteins by deep semantic segmentation\n",
      "baerhunter: an R package for the discovery and analysis of expressed non-coding regions in bacterial RNA-seq data\n",
      "AntiHIV-Pred: web-resource for in silico prediction of anti-HIV/AIDS activity\n",
      "EPIP: a novel approach for condition-specific enhancer-promoter interaction prediction\n",
      "Alternating EM algorithm for a bilinear model in isoform quantification from RNA-seq data\n",
      "Using multiple measurements of tissue to estimate subject- and cell-type-specific gene expression\n",
      "Graph convolution for predicting associations between miRNA and drug resistance\n",
      "MsPAC: a tool for haplotype-phased structural variant detection\n",
      "PeNGaRoo, a combined gradient boosting and ensemble learning framework for predicting non-classical secreted proteins\n",
      "Genome-phenome explorer (GePhEx): a tool for the visualization and interpretation of phenotypic relationships supported by genetic evidence\n",
      "Protein Homeostasis Database: protein quality control in E.coli\n",
      "Context awareness and embedding for biomedical event extraction\n",
      "CNVRanger: association analysis of CNVs with gene expression and quantitative phenotypes\n",
      "Nanopype: a modular and scalable nanopore data processing pipeline\n",
      "ECHO: an application for detection and analysis of oscillators identifies metabolic regulation on genome-wide circadian output\n",
      "PyRanges: efficient comparison of genomic intervals in Python\n",
      "VirtualCytometry: a webserver for evaluating immune cell differentiation using single-cell RNA sequencing data\n",
      "ArtiFuse-computational validation of fusion gene detection tools without relying on simulated reads\n",
      "circMeta: a unified computational framework for genomic feature annotation and differential expression analysis of circular RNAs\n",
      "Non-parametric individual treatment effect estimation for survival data with random forests\n",
      "scRNABatchQC: multi-samples quality control for single cell RNA-seq data\n",
      "Discovering protein drug targets using knowledge graph embeddings\n",
      "Bayesian estimation of genetic regulatory effects in high-throughput reporter assays\n",
      "PyMethylProcess-convenient high-throughput preprocessing workflow for DNA methylation data\n",
      "geneCo: a visualized comparative genomic method to analyze multiple genome structures\n",
      "Towards accurate high-throughput ligand affinity prediction by exploiting structural ensembles, docking metrics and ligand similarity\n",
      "amplimap: a versatile tool to process and analyze targeted NGS data\n",
      "Bayesian inference of distributed time delay in transcriptional and translational regulation\n",
      "MetaBMF: a scalable binning algorithm for large-scale reference-free metagenomic studies\n",
      "DR-SIP: protocols for higher order structure modeling with distance restraints- and cyclic symmetry-imposed packing\n",
      "GSMA: an approach to identify robust global and test Gene Signatures using Meta-Analysis\n",
      "QPARSE: searching for long-looped or multimeric G-quadruplexes potentially distinctive and druggable\n",
      "PgpRules: a decision tree based prediction server for P-glycoprotein substrates and inhibitors\n",
      "DeePaC: predicting pathogenic potential of novel DNA with reverse-complement neural networks\n",
      "Cooler: scalable storage for Hi-C data and other genomically labeled arrays\n",
      "CytoGPS: a web-enabled karyotype analysis tool for cytogenetics\n",
      "The NEW ESID online database network\n",
      "Modelling cancer progression using Mutual Hazard Networks\n",
      "Comprehensive study of the exposome and omic data using rexposome Bioconductor Packages\n",
      "MiTPeptideDB: a proteogenomic resource for the discovery of novel peptides\n",
      "MaNGA: a novel multi-niche multi-objective genetic algorithm for QSAR modelling\n",
      "CaPSSA: visual evaluation of cancer biomarker genes for patient stratification and survival analysis using mutation and expression data\n",
      "Deep learning on chaos game representation for proteins\n",
      "DepLogo: visualizing sequence dependencies in R\n",
      "Sequencing data discovery with MetaSeek\n",
      "Compositional data network analysis via lasso penalized D-trace loss\n",
      "Estimation of population genetic parameters using an EM algorithm and sequence data from experimental evolution populations\n",
      "Reversible jump MCMC for multi-model inference in Metabolic Flux Analysis\n",
      "PatientExploreR: an extensible application for dynamic visualization of patient clinical history from electronic health records in the OMOP common data model\n",
      "iScore: a novel graph kernel-based function for scoring protein-protein docking models\n",
      "GMSimpute: a generalized two-step Lasso approach to impute missing values in label-free mass spectrum analysis\n",
      "SIGN: similarity identification in gene expression\n",
      "BAMixChecker: an automated checkup tool for matched sample pairs in NGS cohort\n",
      "iTUPA: an online automated application to perform Topographic-Unit Parsimony Analysis\n",
      "DECENT: differential expression with capture efficiency adjustmeNT for single-cell RNA-seq data\n",
      "VarMap: a web tool for mapping genomic coordinates to protein sequence and structure and retrieving protein structural annotations\n",
      "Graphlet Laplacians for topology-function and topology-disease relationships\n",
      "Protein contact prediction using metagenome sequence data and residual neural networks\n",
      "arcasHLA: high-resolution HLA typing from RNAseq\n",
      "DepthFinder: a tool to determine the optimal read depth for reduced-representation sequencing\n",
      "New algorithms for detecting multi-effect and multi-way epistatic interactions\n",
      "WDSPdb: an updated resource for WD40 proteins\n",
      "Inferring clonal heterogeneity in cancer using SNP arrays and whole genome sequencing\n",
      "Full-length de novo viral quasispecies assembly through variation graph construction\n",
      "ExpansionHunter: a sequence-graph-based tool to analyze variation in short tandem repeat regions\n",
      "snakePipes: facilitating flexible, scalable and integrative epigenomic analysis\n",
      "Joint detection of germline and somatic copy number events in matched tumor-normal sample pairs\n",
      "ACME: pan-specific peptide-MHC class I binding prediction through attention-based deep neural networks\n",
      "Statistical test of structured continuous trees based on discordance matrix\n",
      "SeRenDIP: SEquential REmasteriNg to DerIve profiles for fast and accurate predictions of PPI interface positions\n",
      "simuG: a general-purpose genome simulator\n",
      "ARMBIS: accurate and robust matching of brain image sequences from multiple modal imaging techniques\n",
      "PathFXweb: a web application for identifying drug safety and efficacy phenotypes\n",
      "TPES: tumor purity estimation from SNVs\n",
      "YeastSpotter: accurate and parameter-free web segmentation for microscopy images of yeast cells\n",
      "Hayai-Annotation Plants: an ultra-fast and comprehensive functional gene annotation system in plants\n",
      "Is the Bombali virus pathogenic in humans?\n",
      "Protein-ensemble-RNA docking by efficient consideration of protein flexibility through homology models\n",
      "Variant Score Ranker-a web application for intuitive missense variant prioritization\n",
      "Entrezpy: a Python library to dynamically interact with the NCBI Entrez databases\n",
      "Deep learning of the back-splicing code for circular RNA formation\n",
      "SPLATCHE3: simulation of serial genetic data under spatially explicit evolutionary scenarios including long-distance dispersal\n",
      "MINERVA API and plugins: opening molecular network analysis and visualization to the community\n",
      "mirTime: Identifying Condition-Specific Targets of MicroRNA in Time-series Transcript Data using Gaussian Process Model and Spherical Vector Clustering\n",
      "INDRA-IPM: interactive pathway modeling using natural language with automated assembly\n",
      "sbgntikz-a TikZ library to draw SBGN maps\n",
      "RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference\n",
      "ResPRE: high-accuracy protein contact prediction by coupling precision matrix with deep residual neural networks\n",
      "Predicted dynamical couplings of protein residues characterize catalysis, transport and allostery\n",
      "BHap: a novel approach for bacterial haplotype reconstruction\n",
      "VCPA: genomic variant calling pipeline and data management tool for Alzheimer's Disease Sequencing Project\n",
      "Evolution of biosequence search algorithms: a brief survey\n",
      "Hybrid analysis of gene dynamics predicts context-specific expression and offers regulatory insights\n",
      "Automatic curation of large comparative animal MicroRNA datasets\n",
      "Discovering novel mutation signatures by latent Dirichlet allocation with variational Bayes inference\n",
      "Identifying centromeric satellites with dna-brnn\n",
      "ASICS: an R package for a whole analysis workflow of 1D 1H NMR spectra\n",
      "TeraPCA: a fast and scalable software package to study genetic variation in tera-scale genotypes\n",
      "Figure and caption extraction from biomedical documents\n",
      "Structural variant analysis for linked-read sequencing data with gemtools\n",
      "FusionLearn: a biomarker selection algorithm on cross-platform data\n",
      "Simultaneous clustering of multiview biomedical data using manifold optimization\n",
      "PgpRules: a decision tree based prediction server for P-glycoprotein substrates and inhibitors\n",
      "GenomeWarp: an alignment-based variant coordinate transformation\n",
      "SPRINT-Gly: predicting N- and O-linked glycosylation sites of human and mouse proteins by using sequence and predicted structural properties\n",
      "TISIDB: an integrated repository portal for tumor-immune system interactions\n",
      "ngsLD: evaluating linkage disequilibrium using genotype likelihoods\n",
      "Health assistant: answering your questions anytime from biomedical literature\n",
      "Bigmelon: tools for analysing large DNA methylation datasets\n",
      "ST viewer: a tool for analysis and visualization of spatial transcriptomics datasets\n",
      "Significance tests for analyzing gene expression data with small sample sizes\n",
      "Classical scoring functions for docking are unable to exploit large volumes of structural and interaction data\n",
      "SodaPop: a forward simulation suite for the evolutionary dynamics of asexual populations on protein fitness landscapes\n",
      "Probabilistic count matrix factorization for single cell expression data analysis\n",
      "Drug repositioning through integration of prior knowledge and projections of drugs and diseases\n",
      "Using association signal annotations to boost similarity network fusion\n",
      "Using local alignment to enhance single-cell bisulfite sequencing data efficiency\n",
      "Brain annotation toolbox: exploring the functional and genetic associations of neuroimaging results\n",
      "Response envelope analysis for quantitative evaluation of drug combinations\n",
      "ppsPCP: a plant presence/absence variants scanner and pan-genome construction pipeline\n",
      "Interoperable and scalable data analysis with microservices: applications in metabolomics\n",
      "Integration of methylation QTL and enhancer-target gene maps with schizophrenia GWAS summary results identifies novel genes\n",
      "A Gaussian process model and Bayesian variable selection for mapping function-valued quantitative traits with incomplete phenotypic data\n",
      "PhenoPro: a novel toolkit for assisting in the diagnosis of Mendelian disease\n",
      "aphid: an R package for analysis with profile hidden Markov models\n",
      "NFPws: a web server for delineating broadly neutralizing antibody specificities from serum HIV-1 neutralization data\n",
      "VHost-Classifier: virus-host classification using natural language processing\n",
      "DECO: decompose heterogeneous population cohorts for patient stratification and discovery of sample biomarkers using omic data profiling\n",
      "Destin: toolkit for single-cell analysis of chromatin accessibility\n",
      "SpaRC: scalable sequence clustering using Apache Spark\n",
      "QuantiNemo 2: a Swiss knife to simulate complex demographic and genetic scenarios, forward and backward in time\n",
      "PANDA: A comprehensive and flexible tool for quantitative proteomics data analysis\n",
      "BioMethyl: an R package for biological interpretation of DNA methylation data\n",
      "Anduril 2: upgraded large-scale data integration framework\n",
      "LIONS: analysis suite for detecting and quantifying transposable element initiated transcription from RNA-seq\n",
      "ST-Steiner: a spatio-temporal gene discovery algorithm\n",
      "Finding enzyme cofactors in Protein Data Bank\n",
      "Comparison of six breast cancer classifiers using qPCR\n",
      "rMETL: sensitive mobile element insertion detection with long read realignment\n",
      "mCSEA: detecting subtle differentially methylated regions\n",
      "PairedFB: a full hierarchical Bayesian model for paired RNA-seq data with heterogeneous treatment effects\n",
      "Expanded functionality, increased accuracy, and enhanced speed in the de novo genotyping-by-sequencing pipeline GBS-SNP-CROP\n",
      "Two-Tier Mapper, an unbiased topology-based clustering method for enhanced global gene expression analysis\n",
      "Biological sequence modeling with convolutional kernel networks\n",
      "Classifying tumors by supervised network propagation\n",
      "Multispectral tracing in densely labeled mouse brain with nTracer\n",
      "Predicting the optimal growth temperatures of prokaryotes using only genome derived features\n",
      "Tracing the ancestry of operons in bacteria\n",
      "Varanto: variant enrichment analysis and annotation\n",
      "multiHiCcompare: joint normalization and comparative analysis of complex Hi-C experiments\n",
      "SMARTCyp 3.0: enhanced cytochrome P450 site-of-metabolism prediction server\n",
      "Ordino: a visual cancer analysis tool for ranking and exploring genes, cell lines and tissue samples\n",
      "METACLUSTER-an R package for context-specific expression analysis of metabolic gene clusters\n",
      "Flux tope analysis: studying the coordination of reaction directions in metabolic networks\n",
      "Exosomal lncRNAs and cancer: connecting the missing links\n",
      "Characterizing and ranking computed metabolic engineering strategies\n",
      "SMuRF: portable and accurate ensemble prediction of somatic mutations\n",
      "A simple convolutional neural network for prediction of enhancer-promoter interactions with DNA sequence data\n",
      "Deuteros: software for rapid analysis and visualization of data from differential hydrogen deuterium exchange-mass spectrometry\n",
      "Statistical force-field for structural modeling using chemical cross-linking/mass spectrometry distance constraints\n",
      "i6mA-Pred: identifying DNA N6-methyladenine sites in the rice genome\n",
      "polyDFEv2.0: testing for invariance of the distribution of fitness effects within and across species\n",
      "The COP9 signalosome influences the epigenetic landscape of Arabidopsis thaliana\n",
      "Multiview: a software package for multiview pattern recognition methods\n",
      "poreTally: run and publish de novo nanopore assembler benchmarks\n",
      "PINSPlus: a tool for tumor subtype discovery in integrated genomic data\n",
      "TriPOINT: a software tool to prioritize important genes in pathways and their non-coding regulators\n",
      "Pathogen-Host Analysis Tool (PHAT): an integrative platform to analyze next-generation sequencing data\n",
      "ShinyCNV: a Shiny/R application to view and annotate DNA copy number variations\n",
      "Database of evidence for precision oncology portal\n",
      "pyseer: a comprehensive tool for microbial pangenome-wide association studies\n",
      "A novel measure of non-coding genome conservation identifies genomic regulatory blocks within primates\n",
      "Histones predate the split between bacteria and archaea\n",
      "Protein-protein interaction sites prediction by ensemble random forests with synthetic minority oversampling technique\n",
      "In silico error correction improves cfDNA mutation calling\n",
      "iHam and pyHam: visualizing and processing hierarchical orthologous groups\n",
      "GUESS: projecting machine learning scores to well-calibrated probability estimates for clinical decision-making\n",
      "Genome-wide search of nucleosome patterns using visual analytics\n",
      "BlendMol: advanced macromolecular visualization in Blender\n",
      "PhastWeb: a web interface for evolutionary conservation scoring of multiple sequence alignments using phastCons and phyloP\n",
      "RT States: systematic annotation of the human genome using cell type-specific replication timing programs\n",
      "The Microbiome Modeling Toolbox: from microbial interactions to personalized microbial communities\n",
      "Condition-specific series of metabolic sub-networks and its application for gene set enrichment analysis\n",
      "GRNBoost2 and Arboreto: efficient and scalable inference of gene regulatory networks\n",
      "Protein pocket detection via convex hull surface evolution and associated Reeb graph\n",
      "ECCB 2018: The 17th European Conference on Computational Biology\n",
      "FLYCOP: metabolic modeling-based analysis and engineering microbial communities\n",
      "ECCB 2018 Organization\n",
      "A fast adaptive algorithm for computing whole-genome homology maps\n",
      "Insights on the alteration of functionality of a tyrosine kinase 2 variant: a molecular dynamics study\n",
      "CNEFinder: finding conserved non-coding elements in genomes\n",
      "Hierarchical HotNet: identifying hierarchies of altered subnetworks\n",
      "Learning structural motif representations for efficient protein structure search\n",
      "IRSOM, a reliable identifier of ncRNAs based on supervised self-organizing maps with rejection\n",
      "Liquid-chromatography retention order prediction for metabolite identification\n",
      "Semantic Disease Gene Embeddings (SmuDGE): phenotype-based disease gene prioritization without phenotypes\n",
      "DeepDiff: DEEP-learning for predicting DIFFerential gene expression from histone modifications\n",
      "Comparative Network Reconstruction using mixed integer programming\n",
      "Computational enhancement of single-cell sequences for inferring tumor evolution\n",
      "Prioritising candidate genes causing QTL using hierarchical orthologous groups\n",
      "Identifying antimicrobial peptides using word embedding with deep recurrent neural networks\n",
      "MAGpy: a reproducible pipeline for the downstream analysis of metagenome-assembled genomes (MAGs)\n",
      "Pan-genomic analysis provides novel insights into the association of E.coli with human host and its minimal genome\n",
      "Clusterdv: a simple density-based clustering method that is robust, general and automatic\n",
      "High efficiency referential genome compression algorithm\n",
      "Whisper: read sorting allows robust mapping of DNA sequencing data\n",
      "BacPaCS-Bacterial Pathogenicity Classification via Sparse-SVM\n",
      "Index suffix-prefix overlaps by (w, k)-minimizer to generate long contigs for reads compression\n",
      "FAVITES: simultaneous simulation of transmission networks, phylogenetic trees and sequences\n",
      "SJARACNe: a scalable software tool for gene network reverse engineering from big data\n",
      "Bastion3: a two-layer ensemble predictor of type III secreted effectors\n",
      "TPMCalculator: one-step software to quantify mRNA abundance of genomic features\n",
      "A framework for identifying dysregulated chromatin regulators as master regulators in human cancer\n",
      "Flexible design of multiple metagenomics classification pipelines with UGENE\n",
      "Clustermatch: discovering hidden relations in highly diverse kinds of qualitative and quantitative data without standardization\n",
      "Disease gene identification based on generic and disease-specific genome networks\n",
      "A natural upper bound to the accuracy of predicting protein stability changes upon mutations\n",
      "Thalia: semantic search engine for biomedical abstracts\n",
      "farPPI: a webserver for accurate prediction of protein-ligand binding structures for small-molecule PPI inhibitors by MM/PB(GB)SA methods\n",
      "ABC random forests for Bayesian parameter inference\n",
      "ParGenes: a tool for massively parallel model selection and phylogenetic tree inference on thousands of genes\n",
      "A global transcriptomic pipeline decoding core network of genes involved in stages leading to acquisition of drug-resistance to cisplatin in osteosarcoma cells\n",
      "The EVcouplings Python framework for coevolutionary sequence analysis\n",
      "LION LBD: a literature-based discovery system for cancer biology\n",
      "VIRULIGN: fast codon-correct alignment and annotation of viral genomes\n",
      "CIRCOAST: a statistical hypothesis test for cellular colocalization with network structures\n",
      "SemGen: a tool for semantics-based annotation and composition of biosimulation models\n",
      "RMTL: an R library for multi-task learning\n",
      "A systematic method for surveying data visualizations and a resulting genomic epidemiology visualization typology: GEViT\n",
      "A framework and an algorithm to detect low-abundance DNA by a handy sequencer and a palm-sized computer\n",
      "Powerful and efficient SNP-set association tests across multiple phenotypes using GWAS summary data\n",
      "Parkour LIMS: high-quality sample preparation in next generation sequencing\n",
      "BhGLM: Bayesian hierarchical GLMs and survival models, with applications to genomics and epidemiology\n",
      "In vitro versus in vivo compositional landscapes of histone sequence preferences in eucaryotic genomes\n",
      "Dante: genotyping of known complex and expanded short tandem repeats\n",
      "Filtering and inference for stochastic oscillators with distributed delays\n",
      "Development of a protein-ligand extended connectivity (PLEC) fingerprint and its application for binding affinity predictions\n",
      "Ambiguous splice sites distinguish circRNA and linear splicing in the human genome\n",
      "Mixed graphical models for integrative causal analysis with application to chronic lung disease diagnosis and prognosis\n",
      "Phylogenetic approaches to identifying fragments of the same gene, with application to the wheat genome\n",
      "Glutton: a tool for generating structural ensembles of partly disordered proteins from chemical shifts\n",
      "AGILE: an assembled genome mining pipeline\n",
      "metaGraphite-a new layer of pathway annotation to get metabolite networks\n",
      "RNAdetect: efficient computational detection of novel non-coding RNAs\n",
      "PDV: an integrative proteomics data viewer\n",
      "PhysiBoSS: a multi-scale agent-based modelling framework integrating physical dimension and cell signalling\n",
      "Protease target prediction via matrix factorization\n",
      "miES: predicting the essentiality of miRNAs with machine learning and sequence features\n",
      "checkMyIndex: a web-based R/Shiny interface for choosing compatible sequencing indexes\n",
      "PolyQTL: Bayesian multiple eQTL detection with control for population structure and sample relatedness\n",
      "Phylo-PFP: improved automated protein function prediction using phylogenetic distance of distantly related sequences\n",
      "AncestryView: data-driven visualization of whole-genome local-ancestry\n",
      "admetSAR 2.0: web-service for prediction and optimization of chemical ADMET properties\n",
      "The Distance Precision Matrix: computing networks from non-linear relationships\n",
      "Parallel clustering of single cell transcriptomic data with split-merge sampling on Dirichlet process mixtures\n",
      "ISDB: a database toolkit for storing and analyzing viral integration site data\n",
      "Genome Detective: an automated system for virus identification from high-throughput sequencing data\n",
      "CharGer: clinical Characterization of Germline variants\n",
      "Cell membrane proteins with high N-glycosylation, high expression and multiple interaction partners are preferred by mammalian viruses as receptors\n",
      "PrimedRPA: primer design for recombinase polymerase amplification assays\n",
      "A powerful conditional gene-based association approach implicated functionally important genes for schizophrenia\n",
      "LipidFinder on LIPID MAPS: peak filtering, MS searching and statistical analysis for lipidomics\n",
      "MIPUP: minimum perfect unmixed phylogenies for multi-sampled tumors via branchings and ILP\n",
      "Processing of big heterogeneous genomic datasets for tertiary analysis of Next Generation Sequencing data\n",
      "CABS-flex standalone: a simulation environment for fast modeling of protein flexibility\n",
      "KEDDY: a knowledge-based statistical gene set test method to detect differential functional protein-protein interactions\n",
      "VisioProt-MS: interactive 2D maps from intact protein mass spectrometry\n",
      "Effusion: prediction of protein function from sequence similarity networks\n",
      "copMEM: finding maximal exact matches via sampling both genomes\n",
      "zfRegeneration: a database for gene expression profiling during regeneration\n",
      "4mCPred: machine learning methods for DNA N4-methylcytosine sites prediction\n",
      "Distinguishing prognostic and predictive biomarkers: an information theoretic approach\n",
      "MinIONQC: fast and simple quality control for MinION sequencing data\n",
      "Re-identification of individuals in genomic data-sharing beacons via allele inference\n",
      "SonicParanoid: fast, accurate and easy orthology inference\n",
      "Toward fast and accurate SNP genotyping from whole genome sequencing data for bedside diagnostics\n",
      "Systematic discovery of novel and valuable plant gene modules by large-scale RNA-seq samples\n",
      "Random walk with restart on multiplex and heterogeneous biological networks\n",
      "Dynamic compression schemes for graph coloring\n",
      "ACDtool: a web-server for the generic analysis of large data sets of counts\n",
      "Kinome-wide identification of phosphorylation networks in eukaryotic proteomes\n",
      "Simulating Illumina metagenomic data with InSilicoSeq\n",
      "pLoc_bal-mAnimal: predict subcellular localization of animal proteins by balancing training dataset and PseAAC\n",
      "Comparative assessment of different familial aggregation methods in the context of large and unstructured pedigrees\n",
      "DMCM: a Data-adaptive Mutation Clustering Method to identify cancer-related mutation clusters\n",
      "SPUTNIK: an R package for filtering of spatially related peaks in mass spectrometry imaging data\n",
      "CoMSA: compression of protein multiple sequence alignment files\n",
      "BrainImageR: spatiotemporal gene set analysis referencing the human brain\n",
      "Kmer-db: instant evolutionary distance estimation\n",
      "BrainEXP: a database featuring with spatiotemporal expression variations and co-expression organizations in human brains\n",
      "Differential coexpression in human tissues and the confounding effect of mean expression levels\n",
      "LONGO: an R package for interactive gene length dependent analysis for neuronal identity\n",
      "Accurate prediction of orthologs in the presence of divergence after duplication\n",
      "Haplotype phasing in single-cell DNA-sequencing data\n",
      "SIMPLE: Sparse Interaction Model over Peaks of moLEcules for fast, interpretable metabolite identification from tandem mass spectra\n",
      "MicroPheno: predicting environments and host phenotypes from 16S rRNA gene sequencing using a k-mer based representation of shallow sub-samples\n",
      "An evolutionary model motivated by physicochemical properties of amino acids reveals variation among proteins\n",
      "Driver gene mutations based clustering of tumors: methods and applications\n",
      "Improved pathway reconstruction from RNA interference screens by exploiting off-target effects\n",
      "Asymptotically optimal minimizers schemes\n",
      "AnoniMME: bringing anonymity to the Matchmaker Exchange platform for rare disease gene discovery\n",
      "Optimization and profile calculation of ODE models using second order adjoint sensitivity analysis\n",
      "Scalable preprocessing for sparse scRNA-seq data exploiting prior knowledge\n",
      "Enumerating consistent sub-graphs of directed acyclic graphs: an insight into biomedical ontologies\n",
      "Improving genomics-based predictions for precision medicine through active elicitation of expert knowledge\n",
      "Dissecting newly transcribed and old RNA using GRAND-SLAM\n",
      "A pan-genome-based machine learning approach for predicting antimicrobial resistance activities of the Escherichia coli strains\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Gene prioritization using Bayesian matrix factorization with genomic and phenotypic side information\n",
      "DeepFam: deep learning based alignment-free method for protein family modeling and prediction\n",
      "MARSI: metabolite analogues for rational strain improvement\n",
      "Quokka: a comprehensive tool for rapid and accurate prediction of kinase family-specific phosphorylation sites in the human proteome\n",
      "Tightly integrated genomic and epigenomic data mining using tensor decomposition\n",
      "Reverse-engineering flow-cytometry gating strategies for phenotypic labelling and high-performance cell sorting\n",
      "PASTA for proteins\n",
      "Accurate prediction of protein contact maps by coupling residual two-dimensional bidirectional long short-term memory with convolutional neural networks\n",
      "SONiCS: PCR stutter noise correction in genome-scale microsatellites\n",
      "Grimon: graphical interface to visualize multi-omics networks\n",
      "3DPatch: fast 3D structure visualization with residue conservation\n",
      "Operon-mapper: a web server for precise operon identification in bacterial and archaeal genomes\n",
      "Modelling signalling networks from perturbation data\n",
      "Prioritizing predictive biomarkers for gene essentiality in cancer cells with mRNA expression data and DNA copy number profile\n",
      "The ancestral KH peptide at the root of a domain family with three different folds\n",
      "In silico read normalization using set multi-cover optimization\n",
      "GlycoStore: a database of retention properties for glycan analysis\n",
      "Spatial quantification of the synaptic activity phenotype across large populations of neurons with Markov random fields\n",
      "CONICS integrates scRNA-seq with DNA sequencing to map gene expression to tumor sub-clones\n",
      "Knotty: efficient and accurate prediction of complex RNA pseudoknot structures\n",
      "TEcandidates: prediction of genomic origin of expressed transposable elements using RNA-seq data\n",
      "Message from the ISCB: 2018 ISCB Accomplishments by a Senior Scientist Award\n",
      "PREQUAL: detecting non-homologous characters in sets of unaligned homologous sequences\n",
      "The Rényi divergence enables accurate and precise cluster analysis for localization microscopy\n",
      "2018 ISCB Innovator Award recognizes M. Madan Babu\n",
      "A graph-embedded deep feedforward network for disease outcome classification and feature selection using gene expression data\n",
      "NGL viewer: web-based molecular graphics for large complexes\n",
      "Bio-knowledge-based filters improve residue-residue contact prediction accuracy\n",
      "mEBT: multiple-matching evidence-based translator of murine genomic responses for human immunity studies\n",
      "EBIC: an evolutionary-based parallel biclustering algorithm for pattern discovery\n",
      "bcSeq: an R package for fast sequence mapping in high-throughput shRNA and CRISPR screens\n",
      "Exploring drug space with ChemMaps.com\n",
      "GSCALite: a web server for gene set cancer analysis\n",
      "Snakemake-a scalable bioinformatics workflow engine\n",
      "FlexiDot: highly customizable, ambiguity-aware dotplots for visual sequence analyses\n",
      "Indexed variation graphs for efficient and accurate resistome profiling\n",
      "TimeXNet Web: identifying cellular response networks from diverse omics time-course data\n",
      "Two C++ libraries for counting trees on a phylogenetic terrace\n",
      "CAVER Analyst 2.0: analysis and visualization of channels and tunnels in protein structures and molecular dynamics trajectories\n",
      "Distinguishing prognostic and predictive biomarkers: an information theoretic approach\n",
      "PBRpredict-Suite: a suite of models to predict peptide-recognition domain residues from protein sequence\n",
      "MACARON: a python framework to identify and re-annotate multi-base affected codons in whole genome/exome sequence data\n",
      "NGScloud: RNA-seq analysis of non-model species using cloud computing\n",
      "MICAN-SQ: a sequential protein structure alignment program that is applicable to monomers and all types of oligomers\n",
      "Edge-group sparse PCA for network-guided high dimensional data analysis\n",
      "Applying graph theory to protein structures: an Atlas of coiled coils\n",
      "Knoto-ID: a tool to study the entanglement of open protein chains using the concept of knotoids\n",
      "Improved enzyme annotation with EC-specific cutoffs using DETECT v2\n",
      "Covariate-adjusted heatmaps for visualizing biological data via correlation decomposition\n",
      "Universal evolutionary selection for high dimensional silent patterns of information hidden in the redundancy of viral genetic code\n",
      "hts-nim: scripting high-performance genomic analyses\n",
      "D3NER: biomedical named entity recognition using CRF-biLSTM improved with fine-tuned embeddings of various linguistic information\n",
      "MiRGOFS: a GO-based functional similarity measurement for miRNAs, with applications to the prediction of miRNA subcellular localization and miRNA-disease association\n",
      "MetaboDiff: an R package for differential metabolomic analysis\n",
      "GapRepairer: a server to model a structural gap and validate it using topological analysis\n",
      "Predicting gene structure changes resulting from genetic variants via exon definition features\n",
      "BinQuasi: a peak detection method for ChIP-sequencing data with biological replicates\n",
      "CRISPR/Cas9 cleavage efficiency regression through boosting algorithms and Markov sequence profiling\n",
      "AnnotSV: an integrated tool for structural variations annotation\n",
      "GGRaSP: a R-package for selecting representative genomes using Gaussian mixture models\n",
      "MethylMix 2.0: an R package for identifying DNA methylation genes\n",
      "MP-LAMP: parallel detection of statistically significant multi-loci markers on cloud platforms\n",
      "DeepSimulator: a deep simulator for Nanopore sequencing\n",
      "LipidPedia: a comprehensive lipid knowledgebase\n",
      "VSClust: feature-based variance-sensitive clustering of omics data\n",
      "Pleiotropic mapping and annotation selection in genome-wide association studies with penalized Gaussian mixture models\n",
      "Constructing lightweight and flexible pipelines using Plugin-Based Microbiome Analysis (PluMA)\n",
      "MechRNA: prediction of lncRNA mechanisms from RNA-RNA and RNA-protein interactions\n",
      "A reference peptide database for proteome quantification based on experimental mass spectrum response curves\n",
      "Efficient analysis of large-scale genome-wide data with two R packages: bigstatsr and bigsnpr\n",
      "GENEASE: real time bioinformatics tool for multi-omics and disease ontology exploration, analysis and visualization\n",
      "Leveraging known genomic variants to improve detection of variants, especially close-by Indels\n",
      "A comprehensive evaluation of alignment software for reduced representation bisulfite sequencing data\n",
      "EWAS: epigenome-wide association study software 2.0\n",
      "QuimP: analyzing transmembrane signalling in highly deformable cells\n",
      "accuMUlate: a mutation caller designed for mutation accumulation experiments\n",
      "DeepSol: a deep learning framework for sequence-based protein solubility prediction\n",
      "NanoPack: visualizing and processing long-read sequencing data\n",
      "A new method for the high-precision assessment of tumor changes in response to treatment\n",
      "Network-based integration of multi-omics data for prioritizing cancer genes\n",
      "Bastion6: a bioinformatics approach for accurate prediction of type VI secreted effectors\n",
      "Accurate mapping of tRNA reads\n",
      "REStLESS: automated translation of glycan sequences from residue-based notation to SMILES and atomic coordinates\n",
      "Repeat-aware evaluation of scaffolding tools\n",
      "Exploiting and assessing multi-source data for supervised biomedical named entity recognition\n",
      "ddPCRclust: an R package and Shiny app for automated analysis of multiplexed ddPCR data\n",
      "NGSphy: phylogenomic simulation of next-generation sequencing data\n",
      "Kernelized rank learning for personalized drug recommendation\n",
      "Sensitive and robust assessment of ChIP-seq read distribution using a strand-shift profile\n",
      "iFeature: a Python package and web server for features extraction and selection from protein and peptide sequences\n",
      "AltHapAlignR: improved accuracy of RNA-seq analyses through the use of alternative haplotypes\n",
      "GDCRNATools: an R/Bioconductor package for integrative analysis of lncRNA, miRNA and mRNA data in GDC\n",
      "Prediction of potential disease-associated microRNAs using structural perturbation method\n",
      "Isoform-level gene expression patterns in single-cell RNA-sequencing data\n",
      "SECLAF: a webserver and deep neural network design tool for hierarchical biological sequence classification\n",
      "EPIC-CoGe: managing and analyzing genomic data\n",
      "pBRIT: gene prioritization by correlating functional and phenotypic annotations through integrative data fusion\n",
      "StructureMapper: a high-throughput algorithm for analyzing protein sequence locations in structural data\n",
      "ChemDistiller: an engine for metabolite annotation in mass spectrometry\n",
      "Compression of genomic sequencing reads via hash-based reordering: algorithm and analysis\n",
      "findGSE: estimating genome size variation within human and Arabidopsis using k-mer frequencies\n",
      "Squeakr: an exact and approximate k-mer counting system\n",
      "Selenzyme: enzyme selection tool for pathway design\n",
      "Kpax3: Bayesian bi-clustering of large sequence datasets\n",
      "GCevobase: an evolution-based database for GC content in eukaryotic genomes\n",
      "Integrative pipeline for profiling DNA copy number and inferring tumor phylogeny\n",
      "ARGs-OAP v2.0 with an expanded SARG database and Hidden Markov Models for enhancement characterization and quantification of antibiotic resistance genes in environmental metagenomes\n",
      "pymzML v2.0: introducing a highly compressed and seekable gzip format\n",
      "EPS-LASSO: test for high-dimensional regression under extreme phenotype sampling of continuous traits\n",
      "Accurity: accurate tumor purity and ploidy inference from tumor-normal WGS data by jointly modelling somatic copy number alterations and heterozygous germline single-nucleotide-variants\n",
      "HPViewer: sensitive and specific genotyping of human papillomavirus in metagenomic DNA\n",
      "Feature specific quantile normalization enables cross-platform classification of molecular subtypes using gene expression data\n",
      "ADEPTUS: a discovery tool for disease prediction, enrichment and network analysis based on profiles from many diseases\n",
      "OPAL: prediction of MoRF regions in intrinsically disordered protein sequences\n",
      "Computational deconvolution of transcriptomics data from mixed cell populations\n",
      "Optimal water networks in protein cavities with GAsol and 3D-RISM\n",
      "Informational and linguistic analysis of large genomic sequence collections via efficient Hadoop cluster algorithms\n",
      "pyHVis3D: visualising molecular simulation deduced H-bond networks in 3D: application to T-cell receptor interactions\n",
      "myTAI: evolutionary transcriptomics with R\n",
      "Mustguseal: a server for multiple structure-guided sequence alignment of protein families\n",
      "A benchmark for comparing precision medicine methods in thyroid cancer diagnosis using tissue microarrays\n",
      "PINE-SPARKY.2 for automated NMR-based protein structure research\n",
      "R2DGC: threshold-free peak alignment and identification for 2D gas chromatography-mass spectrometry in R\n",
      "Sequential feature selection and inference using multi-variate random forests\n",
      "DeepSynergy: predicting anti-cancer drug synergy with Deep Learning\n",
      "Outlier detection for improved differential splicing quantification from RNA-Seq experiments with replicates\n",
      "Accurate mapping of tRNA reads\n",
      "PipelineDog: a simple and flexible graphic pipeline construction and maintenance tool\n",
      "Reactome diagram viewer: data structures and strategies to boost performance\n",
      "GDSCTools for mining pharmacogenomic interactions in cancer\n",
      "MOCASSIN-prot: a multi-objective clustering approach for protein similarity networks\n",
      "Phenotypic stability and plasticity in GMP-derived cells as determined by their underlying regulatory network\n",
      "Efficiently counting all orbits of graphlets of any order in a graph using autogenerated equations\n",
      "glactools: a command-line toolset for the management of genotype likelihoods and allele counts\n",
      "SAPP: functional genome annotation and analysis through a semantic framework using FAIR principles\n",
      "Classifying next-generation sequencing data using a zero-inflated Poisson model\n",
      "SATORI: a system for ontology-guided visual exploration of biomedical data repositories\n",
      "Reproducible and flexible simulation experiments with ML-Rules and SESSL\n",
      "CALQ: compression of quality values of aligned sequencing data\n",
      "QVZ: lossy compression of quality values\n",
      "Modeling positional effects of regulatory sequences with spline transformations increases prediction accuracy of deep neural networks\n",
      "Membranome 2.0: database for proteome-wide profiling of bitopic proteins and their dimers\n",
      "PERF: an exhaustive algorithm for ultra-fast and efficient identification of microsatellites from large DNA sequences\n",
      "Identifying structural variants using linked-read sequencing data\n",
      "DFAST: a flexible prokaryotic genome annotation pipeline for faster genome publication\n",
      "AlloSigMA: allosteric signaling and mutation analysis server\n",
      "Estimation of QTL heritability based on pooled sequencing data\n",
      "Visualization portal for genetic variation (VizGVar): a tool for interactive visualization of SNPs and somatic mutations in exons, genes and protein domains\n",
      "Predicting protein-DNA binding free energy change upon missense mutations using modified MM/PBSA approach: SAMPDI webserver\n",
      "ndexr-an R package to interface with the network data exchange\n",
      "Dynamic assessment of microbial ecology (DAME): a web app for interactive analysis and visualization of microbial sequencing data\n",
      "TITINdb-a computational tool to assess titin's role as a disease gene\n",
      "ProtDec-LTR2.0: an improved method for protein remote homology detection by combining pseudo protein and supervised Learning to Rank\n",
      "VecScreen_plus_taxonomy: imposing a tax(onomy) increase on vector contamination screening\n",
      "Constructing prediction models from expression profiles for large scale lncRNA-miRNA interaction profiling\n",
      "Evaluation of tools for long read RNA-seq splice-aware alignment\n",
      "ROSC-Pred: web-service for rodent organ-specific carcinogenicity prediction\n",
      "ARCS: scaffolding genome drafts with linked reads\n",
      "Krait: an ultrafast tool for genome-wide survey of microsatellites and primer design\n",
      "Beyond similarity assessment: selecting the optimal model for sequence alignment via the Factorized Asymptotic Bayesian algorithm\n",
      "MFIB: a repository of protein complexes with mutual folding induced by binding\n",
      "PLACNETw: a web-based tool for plasmid reconstruction from bacterial genomes\n",
      "A novel data structure to support ultra-fast taxonomic classification of metagenomic sequences with k-mer signatures\n",
      "HybridSim-VS: a web server for large-scale ligand-based virtual screening using hybrid similarity recognition techniques\n",
      "CrocoBLAST: Running BLAST efficiently in the age of next-generation sequencing\n",
      "gVolante for standardizing completeness assessment of genome and transcriptome assemblies\n",
      "SPATKIN: a simulator for rule-based modeling of biomolecular site dynamics on surfaces\n",
      "SVmine improves structural variation detection by integrative mining of predictions from multiple algorithms\n",
      "Detect differentially methylated regions using non-homogeneous hidden Markov model for methylation array data\n",
      "Irys Extract\n",
      "VICTOR: genome-based phylogeny and classification of prokaryotic viruses\n",
      "SCooP: an accurate and fast predictor of protein stability curves as a function of temperature\n",
      "QuASAR-MPRA: accurate allele-specific analysis for massively parallel reporter assays\n",
      "A new haplotype block detection method for dense genome sequencing data based on interval graph modeling of clusters of highly correlated SNPs\n",
      "A Bayesian framework for the inference of gene regulatory networks from time and pseudo-time series data\n",
      "ZCCHC17 is a master regulator of synaptic gene expression in Alzheimer's disease\n",
      "Tumor origin detection with tissue-specific miRNA and DNA methylation markers\n",
      "Genome-wide pre-miRNA discovery from few labeled examples\n",
      "graphkernels: R and Python packages for graph comparison\n",
      "PyDREAM: high-dimensional parameter inference for biological models in python\n",
      "Canvas SPW: calling de novo copy number variants in pedigrees\n",
      "SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data\n",
      "RINspector: a Cytoscape app for centrality analyses and DynaMine flexibility prediction\n",
      "StereoGene: rapid estimation of genome-wide correlation of continuous or interval feature data\n",
      "Epigenomic annotation-based interpretation of genomic data: from enrichment analysis to machine learning\n",
      "Applying family analyses to electronic health records to facilitate genetic research\n",
      "PySCeSToolbox: a collection of metabolic pathway analysis tools\n",
      "CSAR: a contig scaffolding tool using algebraic rearrangements\n",
      "A graph regularized non-negative matrix factorization method for identifying microRNA-disease associations\n",
      "CRISPR-RT: a web application for designing CRISPR-C2c2 crRNA with improved target specificity\n",
      "FMtree: a fast locating algorithm of FM-indexes for genomic data\n",
      "Omics AnalySIs System for PRecision Oncology (OASISPRO): a web-based omics analysis tool for clinical phenotype prediction\n",
      "RUCS: rapid identification of PCR primers for unique core sequences\n",
      "SNPDelScore: combining multiple methods to score deleterious effects of noncoding mutations in the human genome\n",
      "GradDock: rapid simulation and tailored ranking functions for peptide-MHC Class I docking\n",
      "CCFold: rapid and accurate prediction of coiled-coil structures and application to modelling intermediate filaments\n",
      "regNet: an R package for network-based propagation of gene expression alterations\n",
      "Genome-scale fluxes predicted under the guidance of enzyme abundance using a novel hyper-cube shrink algorithm\n",
      "Phosphonormalizer: an R package for normalization of MS-based label-free phosphoproteomics\n",
      "NavMol 3.0: enabling the representation of metabolic reactions by blind users\n",
      "Real-time demultiplexing Nanopore barcoded sequencing data with npBarcode\n",
      "Improved prediction of breast cancer outcome by identifying heterogeneous biomarkers\n",
      "MAPseq: highly efficient k-mer search with confidence estimates, for rRNA sequence analysis\n",
      "Metagene projection characterizes GEN2.2 and CAL-1 as relevant human plasmacytoid dendritic cell models\n",
      "PyLasso: a PyMOL plugin to identify lassos\n",
      "Improving protein fold recognition by extracting fold-specific features from predicted residue-residue contacts\n",
      "HaploForge: a comprehensive pedigree drawing and haplotype visualization web application\n",
      "4D nucleome Analysis Toolbox: analysis of Hi-C data with abnormal karyotype and time series capabilities\n",
      "Novel approaches for bioinformatic analysis of salivary RNA sequencing data for development\n",
      "Deconvolution of multiple infections in Plasmodium falciparum from high throughput sequencing data\n",
      "Genome-wide association studies using a penalized moving-window regression\n",
      "Orthologue chemical space and its influence on target prediction\n",
      "Holmes: a graphical tool for development, simulation and analysis of Petri net based models of complex biological systems\n",
      "3DBIONOTES v2.0: a web server for the automatic annotation of macromolecular structures\n",
      "Integrative structural and dynamical biology with PLUMED-ISDB\n",
      "WIsH: who is the host? Predicting prokaryotic hosts from metagenomic phage contigs\n",
      "A multi-scenario genome-wide medical population genetics simulation framework\n",
      "Probability Map Viewer: near real-time probability map generator of serial block electron microscopy collections\n",
      "Spliceman2: a computational web server that predicts defects in pre-mRNA splicing\n",
      "POSSUM: a bioinformatics toolkit for generating numerical sequence feature descriptors based on PSSM profiles\n",
      "When loss-of-function is loss of function: assessing mutational signatures and impact of loss-of-function genetic variants\n",
      "Genomes as documents of evolutionary history: a probabilistic macrosynteny model for the reconstruction of ancestral genomes\n",
      "Discovery and genotyping of novel sequence insertions in many sequenced individuals\n",
      "Estimation of time-varying growth, uptake and excretion rates from dynamic metabolomics data\n",
      "Association testing of bisulfite-sequencing methylation data via a Laplace approximation\n",
      "Large-scale structure prediction by improved contact predictions and model quality assessment\n",
      "Abundance estimation and differential testing on strain level in metagenomics data\n",
      "Chromatin accessibility prediction via convolutional long short-term memory networks with k-mer embedding\n",
      "SnapDock-template-based docking by Geometric Hashing\n",
      "iATC-mISF: a multi-label classifier for predicting the classes of anatomical therapeutic chemicals\n",
      "FALCON: a toolbox for the fast contextualization of logical networks\n",
      "CoreTracker: accurate codon reassignment prediction, applied to mitochondrial genomes\n",
      "GfaPy: a flexible and extensible software library for handling sequence graphs in Python\n",
      "rqt: an R package for gene-level meta-analysis\n",
      "runBNG: a software package for BioNano genomic analysis on the command line\n",
      "Fast bootstrapping-based estimation of confidence intervals of expression levels and differential expression from RNA-Seq data\n",
      "Message from the ISCB: 2017 ISCB Accomplishment by a Senior Scientist Award Given to Pavel Pevzner\n",
      "Glycan Reader is improved to recognize most sugar types and chemical modifications in the Protein Data Bank\n",
      "JDINAC: joint density-based non-parametric differential interaction network analysis and classification using high-dimensional sparse omics data\n",
      "FunGAP: Fungal Genome Annotation Pipeline using evidence-based gene model evaluation\n",
      "PASTASpark: multiple sequence alignment meets Big Data\n",
      "Hierarchical probabilistic models for multiple gene/variant associations based on next-generation sequencing data\n",
      "Structurexplor: a platform for the exploration of structural features of RNA secondary structures\n",
      "DeepSite: protein-binding site predictor using 3D-convolutional neural networks\n",
      "MD-TASK: a software suite for analyzing molecular dynamics trajectories\n",
      "GateKeeper: a new hardware architecture for accelerating pre-alignment in DNA short read mapping\n",
      "Tissue-specific network-based genome wide study of amygdala imaging phenotypes to identify functional interaction modules\n",
      "Computational modeling of immune system of the fish for a more effective vaccination in aquaculture\n",
      "On patterns and re-use in bioinformatics databases\n",
      "The neXtProt peptide uniqueness checker: a tool for the proteomics community\n",
      "HTSvis: a web app for exploratory data analysis and visualization of arrayed high-throughput screens\n",
      "PROXiMATE: a database of mutant protein-protein complex thermodynamics and kinetics\n",
      "rMFilter: acceleration of long read-based structure variation calling by chimeric read filtering\n",
      "An efficient graph kernel method for non-coding RNA functional prediction\n",
      "FlashPCA2: principal component analysis of Biobank-scale genotype datasets\n",
      "Exploring spatially adjacent TFBS-clustered regions with Hi-C data\n",
      "The spike-and-slab lasso Cox model for survival prediction and associated genes detection\n",
      "RIblast: an ultrafast RNA-RNA interaction prediction system based on a seed-and-extension approach\n",
      "GDISC: a web portal for integrative analysis of gene-drug interaction for survival in cancer\n",
      "The null hypothesis of GSEA, and a novel statistical model for competitive gene set analysis\n",
      "MAPPI-DAT: data management and analysis for protein-protein interaction data from the high-throughput MAPPIT cell microarray platform\n",
      "ntCard: a streaming algorithm for cardinality estimation in genomics data\n",
      "VEXOR: an integrative environment for prioritization of functional variants in fine-mapping analysis\n",
      "modSaRa: a computationally efficient R package for CNV identification\n",
      "SigSeeker: a peak-calling ensemble approach for constructing epigenetic signatures\n",
      "Neuro-symbolic representation learning on biological knowledge graphs\n",
      "RNAblueprint: flexible multiple target nucleic acid sequence design\n",
      "HLA class I binding prediction via convolutional neural networks\n",
      "CImbinator: a web-based tool for drug synergy analysis in small- and large-scale datasets\n",
      "Mapping genes for calcium signaling and their associated human genetic disorders\n",
      "SVMQA: support-vector-machine-based protein single-model quality assessment\n",
      "HaploMerger2: rebuilding both haploid sub-assemblies from high-heterozygosity diploid genome assembly\n",
      "Network module-based model in the differential expression analysis for RNA-seq\n",
      "FunChIP: an R/Bioconductor package for functional classification of ChIP-seq shapes\n",
      "Systemic evaluation of cellular reprogramming processes exploiting a novel R-tool: eegc\n",
      "Use of AAScatterPlot tool for monitoring the evolution of the hemagglutinin cleavage site in H9 avian influenza viruses\n",
      "DeepEM3D: approaching human-level performance on 3D anisotropic EM image segmentation\n",
      "SQC: secure quality control for meta-analysis of genome-wide association studies\n",
      "BEESEM: estimation of binding energy models using HT-SELEX data\n",
      "SynergyFinder: a web application for analyzing drug combination dose-response matrix data\n",
      "ViPTree: the viral proteomic tree server\n",
      "PARTIE: a partition engine to separate metagenomic and amplicon projects in the Sequence Read Archive\n",
      "The RNASeq-er API-a gateway to systematically updated analysis of public RNA-seq data\n",
      "PhagesDB: the actinobacteriophage database\n",
      "node.dating: dating ancestors in phylogenetic trees in R\n",
      "HLA-MA: simple yet powerful matching of samples using HLA typing results\n",
      "DeepBlueR: large-scale epigenomic analysis in R\n",
      "Estimating gene regulatory networks with pandaR\n",
      "MotifMap-RNA: a genome-wide map of RBP binding sites\n",
      "rMATS-DVR: rMATS discovery of differential variants in RNA\n",
      "swga: a primer design toolkit for selective whole genome amplification\n",
      "RNAscClust: clustering RNA sequences using structure conservation and graph based motifs\n",
      "Identification of genetic outliers due to sub-structure and cryptic relationships\n",
      "AQUA-DUCT: a ligands tracking tool\n",
      "BiRen: predicting enhancers with a deep-learning-based model using the DNA sequence alone\n",
      "BioJava-ModFinder: identification of protein modifications in 3D structures from the Protein Data Bank\n",
      "RefBool: a reference-based algorithm for discretizing gene expression data\n",
      "Pseudoalignment for metagenomic read assignment\n",
      "BCFtools/csq: haplotype-aware variant consequences\n",
      "SANA: simulated annealing far outperforms many other search algorithms for biological network alignment\n",
      "Benchmarking substrate-based kinase activity inference using phosphoproteomic data\n",
      "An approach to infer putative disease-specific mechanisms using neighboring gene networks\n",
      "IRaPPA: information retrieval based integration of biophysical models for protein assembly selection\n",
      "PhosphoPICK-SNP: quantifying the effect of amino acid variants on protein phosphorylation\n",
      "STELLS2: fast and accurate coalescent-based maximum likelihood inference of species trees from gene tree topologies\n",
      "CPSS 2.0: a computational platform update for the analysis of small RNA sequencing data\n",
      "Phyx: phylogenetic tools for unix\n",
      "An efficient concordant integrative analysis of multiple large-scale two-sample expression data sets\n",
      "SEQualyzer: interactive tool for quality control and exploratory analysis of high-throughput RNA structural profiling data\n",
      "CloudPhylo: a fast and scalable tool for phylogeny reconstruction\n",
      "State aggregation for fast likelihood computations in molecular evolution\n",
      "OMBlast: alignment tool for optical mapping using a seed-and-extend approach\n",
      "ARResT/Interrogate: an interactive immunoprofiler for IG/TR NGS data\n",
      "Comparing co-evolution methods and their application to template-free protein structure prediction\n",
      "annoPeak: a web application to annotate and visualize peaks from ChIP-seq/ChIP-exo-seq\n",
      "pETM: a penalized Exponential Tilt Model for analysis of correlated high-dimensional DNA methylation data\n",
      "PyMod 2.0: improvements in protein sequence-structure analysis and homology modeling within PyMOL\n",
      "Large-scale 16S gene assembly using metagenomics shotgun sequences\n",
      "MetaPlotR: a Perl/R pipeline for plotting metagenes of nucleotide modifications and other transcriptomic sites\n",
      "Hierarchical sets: analyzing pangenome structure through scalable set visualizations\n",
      "ANAQUIN: a software toolkit for the analysis of spike-in controls for next generation sequencing\n",
      "Extracting morphologies from third harmonic generation images of structurally normal human brain tissue\n",
      "Genome-wide analysis of DNA methylation profiles in a senescence-accelerated mouse prone 8 brain using whole-genome bisulfite sequencing\n",
      "Ligandbook: an online repository for small and drug-like molecule force field parameters\n",
      "Biomartr: genomic data retrieval with R\n",
      "miRmine: a database of human miRNA expression profiles\n",
      "Multi-rate Poisson tree processes for single-locus species delimitation under maximum likelihood and Markov chain Monte Carlo\n",
      "TimesVector: a vectorized clustering approach to the analysis of time series transcriptome data from multiple phenotypes\n",
      "Feature selection using a one dimensional naïve Bayes' classifier increases the accuracy of support vector machine classification of CDR3 repertoires\n",
      "KNIME4NGS: a comprehensive toolbox for next generation sequencing analysis\n",
      "RECKONER: read error corrector based on KMC\n",
      "The structural bioinformatics library: modeling in biomolecular science and beyond\n",
      "Context-sensitive network-based disease genetics prediction and its implications in drug discovery\n",
      "Algorithm sensitivity analysis and parameter tuning for tissue image segmentation pipelines\n",
      "An informative approach on differential abundance analysis for time-course metagenomic sequencing data\n",
      "DNABarcodes: an R package for the systematic construction of DNA sample tags\n",
      "Lineage-specific mutational clustering in protein structures predicts evolutionary shifts in function\n",
      "Parameter estimation for dynamical systems with discrete events and logical operations\n",
      "AnglerFish: a webserver for defining the geometry of α-helices in membrane proteins\n",
      "The discordant method: a novel approach for differential correlation\n",
      "dAPE: a web server to detect homorepeats and follow their evolution\n",
      "Inference of cellular level signaling networks using single-cell gene expression data in Caenorhabditis elegans reveals mechanisms of cell fate specification\n",
      "Mutual enrichment in aggregated ranked lists with applications to gene expression regulation\n",
      "bnstruct: an R package for Bayesian Network structure learning in the presence of missing data\n",
      "Improving flux predictions by integrating data from multiple strains\n",
      "CymeR: cytometry analysis using KNIME, docker and R\n",
      "Predict drug permeability to blood-brain-barrier from clinical phenotypes: drug side effects and drug indications\n",
      "Hum-mPLoc 3.0: prediction enhancement of human protein subcellular localization through modeling the hidden correlations of gene ontology and functional domain features\n",
      "Recon2Neo4j: applying graph database technologies for managing comprehensive genome-scale networks\n",
      "A comprehensive benchmark of RNA-RNA interaction prediction tools for all domains of life\n",
      "Big Data Smart Socket (BDSS): a system that abstracts data transfer habits from end users\n",
      "JADOPPT: java based AutoDock preparing and processing tool\n",
      "ImpulseDE: detection of differentially expressed genes in time series data using impulse models\n",
      "ViSiBooL-visualization and simulation of Boolean networks with temporal constraints\n",
      "yMap: an automated method to map yeast variants to protein modifications and functional regions\n",
      "gsrc: an R package for genome structure rearrangement calling\n",
      "Efficient detection of differentially methylated regions using DiMmeR\n",
      "FractBias: a graphical tool for assessing fractionation bias following polyploidy\n",
      "Improved methods for multi-trait fine mapping of pleiotropic risk loci\n",
      "signeR: an empirical Bayesian approach to mutational signature discovery\n",
      "SYNBADm: a tool for optimization-based automated design of synthetic gene circuits\n",
      "Performance of protein-structure predictions with the physics-based UNRES force field in CASP11\n",
      "Unbiased classification of spatial strategies in the Barnes maze\n",
      "ChAsE: chromatin analysis and exploration tool\n",
      "MetaPred2CS: a sequence-based meta-predictor for protein-protein interactions of prokaryotic two-component system proteins\n",
      "AMIGO2, a toolbox for dynamic modeling, optimization and control in systems biology\n",
      "cMapper: gene-centric connectivity mapper for EBI-RDF platform\n",
      "AKT: ancestry and kinship toolkit\n",
      "ImmuneDB: a system for the analysis and exploration of high-throughput adaptive immune receptor sequencing data\n",
      "TRI_tool: a web-tool for prediction of protein-protein interactions in human transcriptional regulation\n",
      "DAPAR & ProStaR: software to perform statistical analyses in quantitative discovery proteomics\n",
      "Gene- and pathway-based association tests for multiple traits with GWAS summary statistics\n",
      "Precrec: fast and accurate precision-recall and ROC curve calculations in R\n",
      "popSTR: population-scale detection of STR variants\n",
      "L1 regularization facilitates detection of cell type-specific parameters in dynamical systems\n",
      "PEPSI-Dock: a detailed data-driven protein-protein interaction potential accelerated by polar Fourier correlation\n",
      "SWORD-a highly efficient protein database search\n",
      "PRED-TMBB2: improved topology prediction and detection of beta-barrel outer membrane proteins\n",
      "XGSA: A statistical method for cross-species gene set analysis\n",
      "Detecting horizontal gene transfer by mapping sequencing reads across species boundaries\n",
      "Pathway-based approach using hierarchical components of collapsed rare variants\n",
      "Characterizing leader sequences of CRISPR loci\n",
      "CoLoRMap: Correcting Long Reads by Mapping short reads\n",
      "Improve homology search sensitivity of PacBio data by correcting frameshifts\n",
      "ECCB 2016: The 15th European Conference on Computational Biology\n",
      "A likelihood ratio-based method to predict exact pedigrees for complex families from next-generation sequencing data\n",
      "Prediction of nucleosome positioning by the incorporation of frequencies and distributions of three different nucleotide segment lengths into a general pseudo k-tuple nucleotide composition\n",
      "Discoal: flexible coalescent simulations with selection\n",
      "STAMS: STRING-assisted module search for genome wide association studies and application to autism\n",
      "Predicting synergistic effects between compounds through their structural similarity and effects on transcriptomes\n",
      "Higher classification sensitivity of short metagenomic reads with CLARK-S\n",
      "Selective mutation accumulation: a computational model of the paternal age effect\n",
      "H-PoP and H-PoPG: heuristic partitioning algorithms for single individual haplotyping of polyploids\n",
      "Multivariate Welch t-test on distances\n",
      "SRinversion: a tool for detecting short inversions by splitting and re-aligning poorly mapped and unmapped sequencing reads\n",
      "shinyGEO: a web-based application for analyzing gene expression omnibus datasets\n",
      "PathScore: a web tool for identifying altered pathways in cancer data\n",
      "Bayesian network feature finder (BANFF): an R package for gene network feature selection\n",
      "A profile-based method for identifying functional divergence of orthologous genes in bacterial genomes\n",
      "FARAO: the flexible all-round annotation organizer\n",
      "PcircRNA_finder: a software for circRNA prediction in plants\n",
      "HiPub: translating PubMed and PMC texts to networks for knowledge discovery\n",
      "An efficient method to estimate the optimum regularization parameter in RLDA\n",
      "Drug drug interaction extraction from biomedical literature using syntax convolutional neural network\n",
      "MSAViewer: interactive JavaScript visualization of multiple sequence alignments\n",
      "SSBD: a database of quantitative data of spatiotemporal dynamics of biological phenomena\n",
      "4DGenome: a comprehensive database of chromatin interactions\n",
      "Evaluating the molecule-based prediction of clinical drug responses in cancer\n",
      "GORpipe: a query tool for working with sequence data based on a Genomic Ordered Relational (GOR) architecture\n",
      "SERAPHIM: studying environmental rasters and phylogenetically informed movements\n",
      "PEP_scaffolder: using (homologous) proteins to scaffold genomes\n",
      "ACE: adaptive cluster expansion for maximum entropy graphical model inference\n",
      "Integrated gene set analysis for microRNA studies\n",
      "STRUM: structure-based prediction of protein stability changes upon single-point mutation\n",
      "NegGOA: negative GO annotations selection using ontology structure\n",
      "Assemblytics: a web analytics tool for the detection of variants from an assembly\n",
      "Genome puzzle master (GPM): an integrated pipeline for building and editing pseudomolecules from fragmented sequences\n",
      "TADtool: visual parameter identification for TAD-calling algorithms\n",
      "@MInter: automated text-mining of microbial interactions\n",
      "RADIS: analysis of RAD-seq data for interspecific phylogeny\n",
      "DeepMeSH: deep semantic representation for improving large-scale MeSH indexing\n",
      "SHARAKU: an algorithm for aligning and clustering read mapping profiles of deep sequencing in non-coding RNA processing\n",
      "RCK: accurate and efficient inference of sequence- and structure-based protein-RNA binding models from RNAcompete data\n",
      "CMsearch: simultaneous exploration of protein sequence space and structure space improves not only protein homology detection but also protein structure prediction\n",
      "Structure-based prediction of transcription factor binding specificity using an integrative energy function\n",
      "Finding correct protein-protein docking models using ProQDock\n",
      "A convex optimization approach for identification of human tissue-specific interactomes\n",
      "RapMap: a rapid, sensitive and accurate tool for mapping RNA-seq reads to transcriptomes\n",
      "DrugE-Rank: improving drug-target interaction prediction of new candidate drugs or targets by ensemble learning to rank\n",
      "deBWT: parallel construction of Burrows-Wheeler Transform for large collection of genomes with de Bruijn-branch encoding\n",
      "A network-driven approach for genome-wide association mapping\n",
      "Using genomic annotations increases statistical power to detect eGenes\n",
      "A cross-species bi-clustering approach to identifying conserved co-regulated genes\n",
      "Influence maximization in time bounded network identifies transcription factors regulating perturbed pathways\n",
      "A time-varying group sparse additive model for genome-wide association studies of dynamic complex traits\n",
      "RareVariantVis: new tool for visualization of causative variants in rare monogenic disorders using whole genome sequencing data\n",
      "The Glycome Analytics Platform: an integrative framework for glycobioinformatics\n",
      "bdvis: visualizing biodiversity data in R\n",
      "TaggerOne: joint named entity recognition and normalization with semi-Markov Models\n",
      "R.JIVE for exploration of multi-source molecular data\n",
      "HDTD: analyzing multi-tissue gene expression data\n",
      "Group-combined P-values with applications to genetic association studies\n",
      "MetaFast: fast reference-free graph-based comparison of shotgun metagenomic data\n",
      "Benchmarking the next generation of homology inference tools\n",
      "Message from the ISCB: 2016 Outstanding Contributions to ISCB Award: Burkhard Rost\n",
      "TarPmiR: a new approach for microRNA target site prediction\n",
      "SoFIA: a data integration framework for annotating high-throughput datasets\n",
      "Calculating and scoring high quality multiple flexible protein structure alignments\n",
      "A two-part mixed-effects model for analyzing longitudinal microbiome compositional data\n",
      "SimLoRD: Simulation of Long Read Data\n",
      "Detection and removal of spatial bias in multiwell assays\n",
      "Cas-Database: web-based genome-wide guide RNA library design for gene knockout screens using CRISPR-Cas9\n",
      "Branch: an interactive, web-based tool for testing hypotheses and developing predictive models\n",
      "AlgoRun: a Docker-based packaging system for platform-agnostic implemented algorithms\n",
      "PON-Sol: prediction of effects of amino acid substitutions on protein solubility\n",
      "MetaKTSP: a meta-analytic top scoring pair method for robust cross-study validation of omics prediction analysis\n",
      "Multilevel biological characterization of exomic variants at the protein level significantly improves the identification of their deleterious effects\n",
      "FRED 2: an immunoinformatics framework for Python\n",
      "Simulated single molecule microscopy with SMeagol\n",
      "BLESS 2: accurate, memory-efficient and fast error correction method\n",
      "Direct calculation of minimal cut sets involving a specific reaction knock-out\n",
      "Reaction Decoder Tool (RDT): extracting features from chemical reactions\n",
      "MOLGENIS/connect: a system for semi-automatic integration of heterogeneous phenotype data with applications in biobanks\n",
      "Assembly scaffolding with PE-contaminated mate-pair libraries\n",
      "OpenSegSPIM: a user-friendly segmentation tool for SPIM data\n",
      "Coala: an R framework for coalescent simulation\n",
      "Assessing statistical significance in multivariable genome wide association analysis\n",
      "Goldilocks: a tool for identifying genomic regions that are 'just right'\n",
      "Drug repositioning based on comprehensive similarity measures and Bi-Random walk algorithm\n",
      "Spectral identification of topological domains\n",
      "CaFE: a tool for binding affinity prediction using end-point free energy methods\n",
      "Sparse group factor analysis for biclustering of multiple data sources\n",
      "Pasha: a versatile R package for piling chromatin HTS data\n",
      "Beta-Poisson model for single-cell RNA-seq data analyses\n",
      "Shape complementarity and hydrogen bond preferences in protein-protein interfaces: implications for antibody modeling and protein-protein docking\n",
      "Muxstep: an open-source C ++ multiplex HMM library for making inferences on multiple data types\n",
      "MINTbase: a framework for the interactive exploration of mitochondrial and nuclear tRNA fragments\n",
      "Efficient identification and analysis of chemical exchange in biomolecules by R1ρ relaxation dispersion with Amaterasu\n",
      "methyLiftover: cross-platform DNA methylation data integration\n",
      "CHiCP: a web-based tool for the integrative and interactive visualization of promoter capture Hi-C datasets\n",
      "Drug-induced adverse events prediction with the LINCS L1000 data\n",
      "Deep models for brain EM image segmentation: novel insights and improved performance\n",
      "OxyBS: estimation of 5-methylcytosine and 5-hydroxymethylcytosine from tandem-treated oxidative bisulfite and bisulfite DNA\n",
      "MGEScan: a Galaxy-based system for identifying retrotransposons in genomes\n",
      "Characterizing gene sets using discriminative random walks with restart on heterogeneous biological networks\n",
      "Ferret: a user-friendly Java tool to extract data from the 1000 Genomes Project\n",
      "jSplice: a high-performance method for accurate prediction of alternative splicing events and its application to large-scale renal cancer transcriptome data\n",
      "MeCorS: Metagenome-enabled error correction of single cell sequencing reads\n",
      "On genomic repeats and reproducibility\n",
      "mDCC_tools: characterizing multi-modal atomic motions in molecular dynamics trajectories\n",
      "ProtAnnot: an App for Integrated Genome Browser to display how alternative splicing and transcription affect proteins\n",
      "Measuring the spatial correlations of protein binding sites\n",
      "A zero-inflated Poisson model for insertion tolerance analysis of genes based on Tn-seq data\n",
      "Integrative analysis for identifying joint modular patterns of gene-expression and drug-response data\n",
      "Library of binding protein scaffolds (LibBP): a computational platform for selection of binding protein scaffolds\n",
      "FastLSU: a more practical approach for the Benjamini-Hochberg FDR controlling procedure for huge-scale testing problems\n",
      "Unipept web services for metaproteomics analysis\n",
      "Computational approaches to define a human milk metaglycome\n",
      "DesignSignatures: a tool for designing primers that yields amplicons with distinct signatures\n",
      "PAA: an R/bioconductor package for biomarker discovery with protein microarrays\n",
      "Quantitative trait association study for mean telomere length in the South Asian genomes\n",
      "Structured sparse canonical correlation analysis for brain imaging genetics: an improved GraphNet method\n",
      "Inclusion of dyad-repeat pattern improves topology prediction of transmembrane β-barrel proteins\n",
      "Orthogonal matrix factorization enables integrative analysis of multiple RNA binding proteins\n",
      "MultiGeMS: detection of SNVs from multiple samples using model selection on high-throughput sequencing data\n",
      "Homology-driven assembly of NOn-redundant protEin sequence sets (NOmESS) for mass spectrometry\n",
      "MAESTROweb: a web server for structure-based protein stability prediction\n",
      "UCSC Data Integrator and Variant Annotation Integrator\n",
      "Modelling-based experiment retrieval: a case study with gene expression clustering\n",
      "Characterizing rate limiting steps in transcription from RNA production times in live cells\n",
      "Utilizing the Jaccard index to reveal population stratification in sequencing data: a simulation study and an application to the 1000 Genomes Project\n",
      "HitWalker2: visual analytics for precision medicine and beyond\n",
      "simDEF: definition-based semantic similarity measure of gene ontology terms for functional similarity analysis of genes\n",
      "MSAcquisitionSimulator: data-dependent acquisition simulator for LC-MS shotgun proteomics\n",
      "CytoGEDEVO-global alignment of biological networks with Cytoscape\n",
      "qsubsec: a lightweight template system for defining sun grid engine workflows\n",
      "An automated workflow for parallel processing of large multiview SPIM recordings\n",
      "Reliable ABC model choice via random forests\n",
      "Optimal design of gene knockout experiments for gene regulatory network inference\n",
      "Uncovering multiloci-ordering by algebraic property of Laplacian matrix and its Fiedler vector\n",
      "Optimal seed solver: optimizing seed selection in read mapping\n",
      "MIEC-SVM: automated pipeline for protein peptide/ligand interaction prediction\n",
      "BRAKER1: Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS\n",
      "WormExp: a web-based application for a Caenorhabditis elegans-specific gene expression enrichment analysis\n",
      "SpectralTDF: transition densities of diffusion processes with time-varying selection parameters, mutation rates and effective population sizes\n",
      "CopyNumber450kCancer: baseline correction for accurate copy number calling from the 450k methylation array\n",
      "Automatic generation of bioinformatics tools for predicting protein-ligand binding sites\n",
      "Evaluation of hierarchical models for integrative genomic analyses\n",
      "Comparison of genetic variants in matched samples using thesaurus annotation\n",
      "Probabilistic models of genetic variation in structured populations applied to global human studies\n",
      "Shape component analysis: structure-preserving dimension reduction on biological shape spaces\n",
      "Riboswitch Scanner: an efficient pHMM-based web-server to detect riboswitches in genomic sequences\n",
      "MMR: a tool for read multi-mapper resolution\n",
      "tcsBU: a tool to extend TCS network layout and visualization\n",
      "MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets\n",
      "EMDomics: a robust and powerful method for the identification of genes differentially expressed between heterogeneous classes\n",
      "LOLA: enrichment analysis for genomic region sets and regulatory elements in R and Bioconductor\n",
      "IsoMIF Finder: online detection of binding site molecular interaction field similarities\n",
      "BGT: efficient and flexible genotype query across many samples\n",
      "funtooNorm: an R package for normalization of DNA methylation data when there are multiple cell or tissue types\n",
      "An algorithm to detect and communicate the differences in computational models describing biological systems\n",
      "RNA structure framework: automated transcriptome-wide reconstruction of RNA secondary structures from high-throughput structure probing data\n",
      "BioSynther: a customized biosynthetic potential explorer\n",
      "A novel bi-level meta-analysis approach: applied to biological pathway analysis\n",
      "Automatic semantic classification of scientific literature according to the hallmarks of cancer\n",
      "FALCON@home: a high-throughput protein structure prediction server based on remote homologue recognition\n",
      "HEALER: homomorphic computation of ExAct Logistic rEgRession for secure rare disease variants analysis in GWAS\n",
      "ALP & FALP: C++ libraries for pairwise local alignment E-values\n",
      "cl-dash: rapid configuration and deployment of Hadoop clusters for bioinformatics research in the cloud\n",
      "Generalized empirical Bayesian methods for discovery of differential data in high-throughput biology\n",
      "Warpgroup: increased precision of metabolomic data processing by consensus integration bound analysis\n",
      "smallWig: parallel compression of RNA-seq WIG files\n",
      "Drug-set enrichment analysis: a novel tool to investigate drug mode of action\n",
      "JBASE: Joint Bayesian Analysis of Subphenotypes and Epistasis\n",
      "PathwaysWeb: a gene pathways API with directional interactions, expanded gene ontology, and versioning\n",
      "ICT: isotope correction toolbox\n",
      "DIVAS: a centralized genetic variant repository representing 150,000 individuals from multiple disease cohorts\n",
      "An algorithm for automated layout of process description maps drawn in SBGN\n",
      "BamHash: a checksum program for verifying the integrity of sequence data\n",
      "A robust AMMI model for the analysis of genotype-by-environment data\n",
      "Hammock: a hidden Markov model-based peptide clustering algorithm to identify protein-interaction consensus motifs in large datasets\n",
      "Computational modeling of development by epithelia, mesenchyme and their interactions: a unified model\n",
      "A hybrid bayesian approach for genome-wide association studies on related individuals\n",
      "BS-SNPer: SNP calling in bisulfite-seq data\n",
      "CoD: inferring immune-cell quantities related to disease states\n",
      "Hierarchical boosting: a machine-learning framework to detect and classify hard selective sweeps in human populations\n",
      "msa: an R package for multiple sequence alignment\n",
      "Identification of hierarchical chromatin domains\n",
      "EPGA2: memory-efficient de novo assembler\n",
      "Bayexer: an accurate and fast Bayesian demultiplexer for Illumina sequences\n",
      "svviz: a read viewer for validating structural variants\n",
      "An integrated structure- and system-based framework to identify new targets of metabolites and known drugs\n",
      "aRrayLasso: a network-based approach to microarray interconversion\n",
      "Protein contact prediction by integrating joint evolutionary coupling analysis and supervised learning\n",
      "motifbreakR: an R/Bioconductor package for predicting variant effects at transcription factor binding sites\n",
      "OVA: integrating molecular and physical phenotype data from multiple biomedical domain ontologies with variant filtering for enhanced variant prioritization\n",
      "BLSSpeller: exhaustive comparative discovery of conserved cis-regulatory elements\n",
      "Accurate disulfide-bonding network predictions improve ab initio structure prediction of cysteine-rich proteins\n",
      "LoopIng: a template-based tool for predicting the structure of protein loops\n",
      "A statistical approach to virtual cellular experiments: improved causal discovery using accumulation IDA (aIDA)\n",
      "SurvCurv database and online survival analysis platform update\n",
      "ASSIsT: an automatic SNP scoring tool for in- and outbreeding species\n",
      "HHalign-Kbest: exploring sub-optimal alignments for remote homology comparative modeling\n",
      "FermiKit: assembly-based variant calling for Illumina resequencing data\n",
      "The MI bundle: enabling network and structural biology in genome visualization tools\n",
      "Alternative preprocessing of RNA-Sequencing data in The Cancer Genome Atlas leads to improved analysis results\n",
      "High-order neural networks and kernel methods for peptide-MHC binding prediction\n",
      "TEtranscripts: a package for including transposable elements in differential expression analysis of RNA-seq datasets\n",
      "GeneCOST: a novel scoring-based prioritization framework for identifying disease causing genes\n",
      "CEMAsuite: open source degenerate PCR primer design\n",
      "Application of learning to rank to protein remote homology detection\n",
      "JASSA: a comprehensive tool for prediction of SUMOylation sites and SIMs\n",
      "Nsite, NsiteH and NsiteM computer tools for studying transcription regulatory elements\n",
      "LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants\n",
      "Estimating beta diversity for under-sampled communities using the variably weighted Odum dissimilarity index and OTUshuff\n",
      "GDFuzz3D: a method for protein 3D structure reconstruction from contact maps, based on a non-Euclidean distance function\n",
      "ProFET: Feature engineering captures high-level protein functions\n",
      "Predicting tumor purity from methylation microarray data\n",
      "Real-time multi-view deconvolution\n",
      "Bandage: interactive visualization of de novo genome assemblies\n",
      "GenomeD3Plot: a library for rich, interactive visualizations of genomic data in web applications\n",
      "LFQC: a lossless compression algorithm for FASTQ files\n",
      "An efficient Bayesian inference framework for coalescent-based nonparametric phylodynamics\n",
      "IBiSS, a versatile and interactive tool for integrated sequence and 3D structure analysis of large macromolecular complexes\n",
      "atSNP: transcription factor binding affinity testing for regulatory SNP detection\n",
      "libRoadRunner: a high performance SBML simulation and analysis library\n",
      "TENET: topological feature-based target characterization in signalling networks\n",
      "PPUS: a web server to predict PUS-specific pseudouridine sites\n",
      "Misassembly detection using paired-end sequence reads and optical mapping data\n",
      "Robust reconstruction of gene expression profiles from reporter gene data using linear inversion\n",
      "Protein (multi-)location prediction: utilizing interdependencies via a generative model\n",
      "Reconstructing 16S rRNA genes in metagenomic data\n",
      "Phenome-driven disease genetics prediction toward drug discovery\n",
      "Integrating different data types by regularized unsupervised multiple kernel learning with application to cancer subtype discovery\n",
      "Inferring models of multiscale copy number evolution for single-tumor phylogenetics\n",
      "Gene network inference by fusing data from diverse distributions\n",
      "A hierarchical Bayesian model for flexible module discovery in three-way time-series data\n",
      "Deconvolving molecular signatures of interactions between microbial colonies\n",
      "MSProGene: integrative proteogenomics beyond six-frames and single nucleotide polymorphisms\n",
      "A parallel and sensitive software tool for methylation analysis on multicore platforms\n",
      "IBS: an illustrator for the presentation and visualization of biological sequences\n",
      "Quantitative frame analysis and the annotation of GC-rich (and other) prokaryotic genomes. An application to Anaeromyxobacter dehalogenans\n",
      "CCLasso: correlation inference for compositional data through Lasso\n",
      "TiQuant: software for tissue analysis, quantification and surface reconstruction\n",
      "Mango: a bias-correcting ChIA-PET analysis pipeline\n",
      "DisVis: quantifying and visualizing accessible interaction space of distance-restrained biomolecular complexes\n",
      "ACE: accurate correction of errors using K-mer tries\n",
      "Message from the ISCB: 2015 ISCB Accomplishment by a Senior Scientist Award: Cyrus Chothia\n",
      "tmle.npvi: targeted, integrative search of associations between DNA copy number and gene expression, accounting for DNA methylation\n",
      "PopGeV: a web-based large-scale population genome browser\n",
      "agplus: a rapid and flexible tool for aggregation plots\n",
      "DSigDB: drug signatures database for gene set analysis\n",
      "Likelihood-based complex trait association testing for arbitrary depth sequencing data\n",
      "GOplot: an R package for visually combining expression data with functional analysis\n",
      "INPS: predicting the impact of non-synonymous variations on protein stability from sequence\n",
      "IonGAP: integrative bacterial genome analysis for Ion Torrent sequence data\n",
      "HTT-DB: horizontally transferred transposable elements database\n",
      "'Flatten plus': a recent implementation in WSxM for biological research\n",
      "kSNP3.0: SNP detection and phylogenetic analysis of genomes without genome alignment or reference genome\n",
      "Data-dependent bucketing improves reference-free compression of sequencing reads\n",
      "Inferring data-specific micro-RNA function through the joint ranking of micro-RNA and pathways from matched micro-RNA and gene expression data\n",
      "Analysis of impedance-based cellular growth assays\n",
      "MultiP-SChlo: multi-label protein subchloroplast localization prediction with Chou's pseudo amino acid composition and a novel multi-label classifier\n",
      "EBSeq-HMM: a Bayesian approach for identifying gene-expression changes in ordered RNA-seq experiments\n",
      "Three minimal sequences found in Ebola virus genomes and absent from human DNA\n",
      "do_x3dna: a tool to analyze structural fluctuations of dsDNA or dsRNA from molecular dynamics simulations\n",
      "ExaML version 3: a tool for phylogenomic analyses on supercomputers\n",
      "MSA-PAD: DNA multiple sequence alignment framework based on PFAM accessed domain information\n",
      "DEOD: uncovering dominant effects of cancer-driver genes based on a partial covariance selection method\n",
      "HAlign: Fast multiple similar DNA/RNA sequence alignment based on the centre star strategy\n",
      "MeDuSa: a multi-draft based scaffolder\n",
      "Integration of somatic mutation, expression and functional data reveals potential driver genes predictive of breast cancer survival\n",
      "Identification of cell types from single-cell transcriptomes using a novel clustering method\n",
      "A robust approach for identifying differentially abundant features in metagenomic samples\n",
      "QSLiMFinder: improved short linear motif prediction using specific query protein data\n",
      "Test set bias affects reproducibility of gene signatures\n",
      "Family genome browser: visualizing genomes with pedigree information\n",
      "4DGenome: a comprehensive database of chromatin interactions\n",
      "SiPAN: simultaneous prediction and alignment of protein-protein interaction networks\n",
      "Structure-PPi: a module for the annotation of cancer-related single-nucleotide variants at protein-protein interfaces\n",
      "BMRF-Net: a software tool for identification of protein interaction subnetworks by a bagging Markov random field-based method\n",
      "EpIC: a rational pipeline for epitope immunogenicity characterization\n",
      "Alternative splicing and co-option of transposable elements: the case of TMPO/LAP2α and ZNF451 in mammals\n",
      "Logical transformation of genome-scale metabolic models for gene level applications and analysis\n",
      "Mining viral proteins for antimicrobial and cell-penetrating drug delivery peptides\n",
      "hiHMM: Bayesian non-parametric joint inference of chromatin state maps\n",
      "Mammalian genome evolution is governed by multiple pacemakers\n",
      "GlycoProfileAssigner: automated structural assignment with error estimation for glycan LC data\n",
      "SimSeq: a nonparametric approach to simulation of RNA-sequence datasets\n",
      "A new method to improve network topological similarity search: applied to fold recognition\n",
      "Automated benchmarking of peptide-MHC class I binding predictions\n",
      "Normalization and noise reduction for single cell RNA-seq experiments\n",
      "rSeqNP: a non-parametric approach for detecting differential expression and splicing from RNA-Seq data\n",
      "EpiToolKit--a web-based workbench for vaccine design\n",
      "Oasis: online analysis of small RNA deep sequencing data\n",
      "CycleFreeFlux: efficient removal of thermodynamically infeasible loops from flux distributions\n",
      "Discriminating precursors of common fragments for large-scale metabolite profiling by triple quadrupole mass spectrometry\n",
      "Characterizing spatial distributions of astrocytes in the mammalian retina\n",
      "Mobility-based prediction of hydration structures of protein surfaces\n",
      "FUN-L: gene prioritization for RNAi screens\n",
      "NxTrim: optimized trimming of Illumina mate pair reads\n",
      "Evolutionary profiles improve protein-protein interaction prediction from sequence\n",
      "Methods for the detection and assembly of novel sequence in high-throughput sequencing data\n",
      "GenoMetric Query Language: a novel approach to large-scale genomic data management\n",
      "Message from the ISCB: ISCB Ebola award for important future research on the computational biology of Ebola virus\n",
      "Transposome: a toolkit for annotation of transposable element families from unassembled sequence reads\n",
      "Where are we at regarding species translation? A review of the sbv IMPROVER challenge\n",
      "Quantitative visualization of alternative exon expression from RNA-seq data\n",
      "ASSIGN: context-specific genomic profiling of multiple heterogeneous biological pathways\n",
      "CypRules: a rule-based P450 inhibition prediction server\n",
      "KMC 2: fast and resource-frugal k-mer counting\n",
      "Topology-function conservation in protein-protein interaction networks\n",
      "More challenges for machine-learning protein interactions\n",
      "CellCODE: a robust latent variable approach to differential expression analysis for heterogeneous cell populations\n",
      "PASPA: a web server for mRNA poly(A) site predictions in plants and algae\n",
      "Using isoelectric point to determine the pH for initial protein crystallization trials\n",
      "Integrating alignment-based and alignment-free sequence similarity measures for biological sequence classification\n",
      "Clonality inference in multiple tumor samples using phylogeny\n",
      "GlycoMine: a machine learning-based approach for predicting N-, C- and O-linked glycosylation in the human proteome\n",
      "rbamtools: an R interface to samtools enabling fast accumulative tabulation of splicing events over multiple RNA-seq samples\n",
      "An automatic tool to analyze and cluster macromolecular conformations based on self-organizing maps\n",
      "Disk-based compression of data from genome sequencing\n",
      "MACE: mutation-oriented profiling of chemical response and gene expression in cancers\n",
      "A Bayesian framework for de novo mutation calling in parents-offspring trios\n",
      "Workflow4Metabolomics: a collaborative research infrastructure for computational metabolomics\n",
      "Epock: rapid analysis of protein pocket dynamics\n",
      "Response to the comment on 'protein-protein binding affinity prediction from amino acid sequence'\n",
      "HYCUD: a computational tool for prediction of effective rotational correlation time in flexible proteins\n",
      "Phy-Mer: a novel alignment-free and reference-independent mitochondrial haplogroup classifier\n",
      "HIPPIE: a high-throughput identification pipeline for promoter interacting enhancer elements\n",
      "Retrieval of Enterobacteriaceae drug targets using singular value decomposition\n",
      "Adaptive settings for the nearest-neighbor particle tracking algorithm\n",
      "An efficient TOF-SIMS image analysis with spatial correlation and alternating non-negativity-constrained least squares\n",
      "GenePainter v. 2.0 resolves the taxonomic distribution of intron positions\n",
      "Computer-assisted curation of a human regulatory core network from the biological literature\n",
      "LigDig: a web server for querying ligand-protein interactions\n",
      "A logical model of HIV-1 interactions with the T-cell activation signalling pathway\n",
      "MetaPSICOV: combining coevolution methods for accurate prediction of contacts and long range hydrogen bonding in proteins\n",
      "MGAS: a powerful tool for multivariate gene-based genome-wide association analysis\n",
      "Mechanotransduction map: simulation model, molecular pathway, gene set\n",
      "QuasR: quantification and annotation of short reads in R\n",
      "Genomation: a toolkit to summarize, annotate and visualize genomic intervals\n",
      "Molecular principles of human virus protein-protein interactions\n",
      "NucleusJ: an ImageJ plugin for quantifying 3D images of interphase nuclei\n",
      "Biological Dynamics Markup Language (BDML): an open format for representing quantitative biological dynamics data\n",
      "SAMNetWeb: identifying condition-specific networks linking signaling and transcription\n",
      "Evolutionary non-linear modelling for selecting vaccines against antigenically variable viruses\n",
      "E-MEM: efficient computation of maximal exact matches for very large genomes\n",
      "SYSBIONS: nested sampling for systems biology\n",
      "mimicMe: a web server for prediction and analysis of host-like proteins in microbial pathogens\n",
      "ViQuaS: an improved reconstruction pipeline for viral quasispecies spectra generated by next-generation sequencing\n",
      "M-path: a compass for navigating potential metabolic pathways\n",
      "UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches\n",
      "SNV-PPILP: refined SNV calling for tumor data using perfect phylogenies and ILP\n",
      "DISOPRED3: precise disordered region predictions with annotated protein-binding activity\n",
      "Optimization method for obtaining nearest-neighbour DNA entropies and enthalpies directly from melting temperatures\n",
      "EVpedia: a community web portal for extracellular vesicles research\n",
      "Comment on 'protein-protein binding affinity prediction from amino acid sequence'\n",
      "Mass spectrometry-based protein identification with accurate statistical significance assignment\n",
      "PALMsiever: a tool to turn raw data into results for single-molecule localization microscopy\n",
      "deML: robust demultiplexing of Illumina sequences using a likelihood-based approach\n",
      "SEABED: Small molEcule activity scanner weB servicE baseD\n",
      "A coalescent-based method for population tree inference with haplotypes\n",
      "Waggawagga: comparative visualization of coiled-coil predictions and detection of stable single α-helices (SAH domains)\n",
      "A model for the expression of gap genes based on the Jeffreys-type equation\n",
      "Context similarity scoring improves protein sequence alignments in the midnight zone\n",
      "LIGSIFT: an open-source tool for ligand structural alignment and virtual screening\n",
      "Statistical significance of variables driving systematic variation in high-dimensional data\n",
      "Seed: a user-friendly tool for exploring and visualizing microbial community data\n",
      "FuncPatch: a web server for the fast Bayesian inference of conserved functional patches in protein 3D structures\n",
      "Genome measures used for quality control are dependent on gene function and ancestry\n",
      "Deviance residuals-based sparse PLS and sparse kernel PLS regression for censored data\n",
      "VCF2Networks: applying genotype networks to single-nucleotide variants data\n",
      "CompMap: a reference-based compression program to speed up read mapping to related reference sequences\n",
      "PFP/ESG: automated protein function prediction servers enhanced with Gene Ontology visualization tool\n",
      "A Python package for parsing, validating, mapping and formatting sequence variants using HGVS nomenclature\n",
      "BioVLAB-MMIA-NGS: microRNA-mRNA integrated analysis using high-throughput sequencing data\n",
      "LongTarget: a tool to predict lncRNA DNA-binding motifs and binding sites via Hoogsteen base-pairing analysis\n",
      "Enlight: web-based integration of GWAS results with biological annotations\n",
      "MTide: an integrated tool for the identification of miRNA-target interaction in plants\n",
      "PHI-DAC: protein homology database through dihedral angle conservation\n",
      "PhaseTank: genome-wide computational identification of phasiRNAs and their regulatory cascades\n",
      "Rcpi: R/Bioconductor package to generate various descriptors of proteins, compounds and their interactions\n",
      "RAMONA: a Web application for gene set analysis on multilevel omics data\n",
      "The Ensembl REST API: Ensembl Data for Any Language\n",
      "Using population data for assessing next-generation sequencing performance\n",
      "JAMM: a peak finder for joint analysis of NGS replicates\n",
      "cddApp: a Cytoscape app for accessing the NCBI conserved domain database\n",
      "Gene regulatory networks by transcription factors and microRNAs in breast cancer\n",
      "ballaxy: web services for structural bioinformatics\n",
      "CCharPPI web server: computational characterization of protein-protein interactions from structure\n",
      "VirVarSeq: a low-frequency virus variant detection pipeline for Illumina sequencing using adaptive base-calling accuracy filtering\n",
      "Individual-level analysis of differential expression of genes and pathways for personalized medicine\n",
      "Inter-species inference of gene set enrichment in lung epithelial cells from proteomic and large transcriptomic datasets\n",
      "Predicting protein phosphorylation from gene expression: top methods from the IMPROVER Species Translation Challenge\n",
      "The Bioinformatics Open Source Conference (BOSC) 2013\n",
      "Inter-species prediction of protein phosphorylation in the sbv IMPROVER species translation challenge\n",
      "A unifying framework for rare variant association testing in family-based designs, including higher criticism approaches, SKATs, and burden tests\n",
      "HiChIP-Peaks: a HiChIP peak calling algorithm\n",
      "Secure multiparty computation for privacy-preserving drug discovery\n",
      "breakpointR: an R/Bioconductor package to localize strand state changes in Strand-seq data\n",
      "DiffNetFDR: differential network analysis with false discovery rate control\n",
      "FastSpar: rapid and scalable correlation estimation for compositional data\n",
      "A network-based zoning for parallel whole-cell simulation\n",
      "LRSSL: predict and interpret drug-disease associations based on data integration using sparse subspace learning\n",
      "Fast and efficient QTL mapper for thousands of molecular phenotypes\n",
      "NBSPred: a support vector machine-based high-throughput pipeline for plant resistance protein NBSLRR prediction\n",
      "Extending gene ontology with gene association networks\n",
      "PBAP: a pipeline for file processing and quality control of pedigree data with dense genetic markers\n",
      "An alternative approach to multiple testing for methylation QTL mapping reduces the proportion of falsely identified CpGs\n",
      "CRAMER: a lightweight, highly customizable web-based genome browser supporting multiple visualization instances\n",
      "Vaxign-ML: supervised machine learning reverse vaccinology model for improved prediction of bacterial protective antigens\n",
      "DL4papers: a deep learning approach for the automatic interpretation of scientific articles\n",
      "PlasGUN: gene prediction in plasmid metagenomic short reads using deep learning\n",
      "PmliPred: a method based on hybrid model and fuzzy decision for plant miRNA-lncRNA interaction prediction\n",
      "netGO: R-Shiny package for network-integrated pathway enrichment analysis\n",
      "Marker controlled superpixel nuclei segmentation and automatic counting on immunohistochemistry staining images\n",
      "ARGminer: a web platform for the crowdsourcing-based curation of antibiotic resistance genes\n",
      "GeneSwitches: ordering gene expression and functional events in single-cell experiments\n",
      "ZIAQ: a quantile regression method for differential expression analysis of single-cell RNA-seq data\n",
      "SurfaceGenie: a web-based application for prioritizing cell-type-specific marker candidates\n",
      "Polar labeling: silver standard algorithm for training disease classifiers\n",
      "A cautionary tale on using imputation methods for inference in matched-pairs design\n",
      "GpABC: a Julia package for approximate Bayesian computation with Gaussian process emulation\n",
      "DEEPrior: a deep learning tool for the prioritization of gene fusions\n",
      "GiniQC: a measure for quantifying noise in single-cell Hi-C data\n",
      "Statistical inference of differential RNA-editing sites from RNA-sequencing data by hierarchical modeling\n",
      "Domain-adversarial multi-task framework for novel therapeutic property prediction of compounds\n",
      "Integrative analysis of cross-modal features for the prognosis prediction of clear cell renal cell carcinoma\n",
      "Morbigenous brain region and gene detection with a genetically evolved random neural network cluster approach in late mild cognitive impairment\n",
      "RxnBLAST: molecular scaffold and reactive chemical environment feature extractor for biochemical reactions\n",
      "Targeted realignment of LC-MS profiles by neighbor-wise compound-specific graphical time warping with misalignment detection\n",
      "3D-Cell-Annotator: an open-source active surface tool for single-cell segmentation in 3D microscopy images\n",
      "BMDx: a graphical Shiny application to perform Benchmark Dose analysis for transcriptomics data\n",
      "The ELIXIR Core Data Resources: fundamental infrastructure for the life sciences\n",
      "AnnoGen: annotating genome-wide pragmatic features\n",
      "MDEHT: a multivariate approach for detecting differential expression of microRNA isoform data in RNA-sequencing studies\n",
      "AlpsNMR: an R package for signal processing of fully untargeted NMR-based metabolomics\n",
      "UDSMProt: universal deep sequence models for protein classification\n",
      "Neural inductive matrix completion with graph convolutional networks for miRNA-disease association prediction\n",
      "FunGeCo: a web-based tool for estimation of functional potential of bacterial genomes and microbiomes using gene context information\n",
      "Benchmarking immunoinformatic tools for the analysis of antibody repertoire sequences\n",
      "PheGWAS: a new dimension to visualize GWAS across multiple phenotypes\n",
      "iMIRAGE: an R package to impute microRNA expression using protein-coding genes\n",
      "Meta-analysis of Caenorhabditis elegans single-cell developmental data reveals multi-frequency oscillation in gene activation\n",
      "Regional imaging genetic enrichment analysis\n",
      "A framework for exhaustive modelling of genetic interaction patterns using Petri nets\n",
      "Platform-integrated mRNA isoform quantification\n",
      "VarGen: an R package for disease-associated variant discovery and annotation\n",
      "pqsfinder web: G-quadruplex prediction using optimized pqsfinder algorithm\n",
      "eCAMI: simultaneous classification and motif identification for enzyme annotation\n",
      "Episo: quantitative estimation of RNA 5-methylcytosine at isoform level by high-throughput sequencing of RNA treated with bisulfite\n",
      "An efficient approach based on multi-sources information to predict circRNA-disease associations using deep convolutional neural network\n",
      "Fusion-Bloom: fusion detection in assembled transcriptomes\n",
      "Incorporating heterogeneous sampling probabilities in continuous phylogeographic inference - Application to H5N1 spread in the Mekong region\n",
      "genBaRcode: a comprehensive R-package for genetic barcode analysis\n",
      "Proactive visual and statistical analysis of genomic data in Epiviz\n",
      "NextPolish: a fast and efficient genome polishing tool for long-read assembly\n",
      "CellBench: R/Bioconductor software for comparing single-cell RNA-seq analysis methods\n",
      "A fast and memory-efficient implementation of the transfer bootstrap\n",
      "Distance measures for tumor evolutionary trees\n",
      "TFmiR2: constructing and analyzing disease-, tissue- and process-specific transcription factor and microRNA co-regulatory networks\n",
      "SCOT: Rethinking the classification of secondary structure elements\n",
      "An integrative approach for fine-mapping chromatin interactions\n",
      "SPDI: data model for variants and applications at NCBI\n",
      "On the inconsistent treatment of gene-protein-reaction rules in context-specific metabolic models\n",
      "Model selection for metabolomics: predicting diagnosis of coronary artery disease using automated machine learning\n",
      "Quantifying functional impact of non-coding variants with multi-task Bayesian neural network\n",
      "Generalized Born radii computation using linear models and neural networks\n",
      "Multi-kernel linear mixed model with adaptive lasso for prediction analysis on high-dimensional multi-omics data\n",
      "Influenza classification from short reads with VAPOR facilitates robust mapping pipelines and zoonotic strain detection for routine surveillance applications\n",
      "GenCLiP 3: mining human genes' functions and regulatory networks from PubMed based on co-occurrences and natural language processing\n",
      "PhyloMagnet: fast and accurate screening of short-read meta-omics data using gene-centric phylogenetics\n",
      "Matched Forest: supervised learning for high-dimensional matched case-control studies\n",
      "Mat_peptide: comprehensive annotation of mature peptides from polyproteins in five virus families\n",
      "Domain-invariant features for mechanism of action prediction in a multi-cell-line drug screen\n",
      "Identification of metabolites from tandem mass spectra with a machine learning approach utilizing structural features\n",
      "DRUDIT: web-based DRUgs DIscovery Tools to design small molecules as modulators of biological targets\n",
      "BCrystal: an interpretable sequence-based protein crystallization predictor\n",
      "MIRUReader: MIRU-VNTR typing directly from long sequencing reads\n",
      "Learning from the ligand: using ligand-based features to improve binding affinity prediction\n",
      "BCSExplorer: a customized biosynthetic chemical space explorer with multifunctional objective function analysis\n",
      "RNAIndel: discovering somatic coding indels from tumor RNA-Seq data\n",
      "Feature selection may improve deep neural networks for the bioinformatics problems\n",
      "ReQTL: identifying correlations between expressed SNVs and gene expression using RNA-sequencing data\n",
      "CircaCompare: a method to estimate and statistically support differences in mesor, amplitude and phase, between circadian rhythms\n",
      "phylogenize: correcting for phylogeny reveals genes associated with microbial distributions\n",
      "A fast and memory efficient MLCS algorithm by character merging for DNA sequences alignment\n",
      "VikNGS: a C++ variant integration kit for next generation sequencing association analysis\n",
      "MetaQUBIC: a computational pipeline for gene-level functional profiling of metagenome and metatranscriptome\n",
      "Zombi: a phylogenetic simulator of trees, genomes and sequences that accounts for dead linages\n",
      "The CroCo cross-link converter: a user-centred tool to convert results from cross-linking mass spectrometry experiments\n",
      "Hexa-Longin domain scaffolds for inter-Rab signalling\n",
      "Metric learning on expression data for gene function prediction\n",
      "Chordomics: a visualization tool for linking function to phylogeny in microbiomes\n",
      "Biogenesis mechanisms of circular RNA can be categorized through feature extraction of a machine learning model\n",
      "Learning a mixture of microbial networks using minorization-maximization\n",
      "ISMB/ECCB 2019 Proceedings\n",
      "Using the structure of genome data in the design of deep neural networks for predicting amyotrophic lateral sclerosis from genotype\n",
      "MCS2: minimal coordinated supports for fast enumeration of minimal cut sets in metabolic networks\n",
      "Inference of clonal selection in cancer populations using single-cell sequencing data\n",
      "scOrange-a tool for hands-on training of concepts from single-cell data analytics\n",
      "Reconstructing signaling pathways using regular language constrained paths\n",
      "Robust network inference using response logic\n",
      "pNovo 3: precise de novo peptide sequencing using a learning-to-rank framework\n",
      "Identifying and ranking potential driver genes of Alzheimer's disease using multiview evidence aggregation\n",
      "Bayesian metabolic flux analysis reveals intracellular flux couplings\n",
      "Rotation equivariant and invariant neural networks for microscopy image analysis\n",
      "Deep learning with multimodal representation for pancancer prognosis prediction\n",
      "A statistical simulator scDesign for rational scRNA-seq experimental design\n",
      "Integrating read-based and population-based phasing for dense and accurate haplotyping of individual genomes\n",
      "Identifying progressive imaging genetic patterns via multi-task sparse canonical correlation analysis: a longitudinal study of the ADNI cohort\n",
      "Fast and accurate correction of optical mapping data via spaced seeds\n",
      "Unification of miRNA and isomiR research: the mirGFF3 format and the mirtop API\n",
      "dbgap2x: an R package to explore and extract data from the database of Genotypes and Phenotypes (dbGaP)\n",
      "A reinforcement-learning-based approach to enhance exhaustive protein loop sampling\n",
      "Non-homogeneous dynamic Bayesian networks with edge-wise sequentially coupled parameters\n",
      "SimkaMin: fast and resource frugal de novo comparative metagenomics\n",
      "DeepCOP: deep learning-based approach to predict gene regulating effects of small molecules\n",
      "EasyModel: user-friendly tool for building and analysis of simple mathematical models in systems biology\n",
      "NExUS: Bayesian simultaneous network estimation across unequal sample sizes\n",
      "A likelihood method for estimating present-day human contamination in ancient male samples using low-depth X-chromosome data\n",
      "Automatic local resolution-based sharpening of cryo-EM maps\n",
      "Improved representation of sequence bloom trees\n",
      "QUBIC2: a novel and robust biclustering algorithm for analyses and interpretation of large-scale RNA-Seq data\n",
      "BioBERT: a pre-trained biomedical language representation model for biomedical text mining\n",
      "EpiSAFARI: sensitive detection of valleys in epigenetic signals for enhancing annotations of functional elements\n",
      "HiLight-PTM: an online application to aid matching peptide pairs with isotopically labelled PTMs\n",
      "LightDock goes information-driven\n",
      "MEPSAnd: minimum energy path surface analysis over n-dimensional surfaces\n",
      "BBKNN: fast batch alignment of single cell transcriptomes\n",
      "pWGBSSimla: a profile-based whole-genome bisulfite sequencing data simulator incorporating methylation QTLs, allele-specific methylations and differentially methylated regions\n",
      "DUETT quantitatively identifies known and novel events in nascent RNA structural dynamics from chemical probing data\n",
      "DNA Rchitect: an R based visualizer for network analysis of chromatin interaction data\n",
      "PIRD: Pan Immune Repertoire Database\n",
      "Predicting the effects of SNPs on transcription factor binding affinity\n",
      "DEEPCON: protein contact prediction using dilated convolutional neural networks with dropout\n",
      "Predicting promoters in phage genomes using PhagePromoter\n",
      "MPRAscore: robust and non-parametric analysis of massively parallel reporter assays\n",
      "Identifying small-effect genetic associations overlooked by the conventional fixed-effect model in a large-scale meta-analysis of coronary artery disease\n",
      "BioNorm: deep learning-based event normalization for the curation of reaction databases\n",
      "Efficient parameterization of large-scale dynamic models based on relative measurements\n",
      "SquiggleKit: a toolkit for manipulating nanopore signal data\n",
      "sAOP: linking chemical stressors to adverse outcomes pathway networks\n",
      "PTM-Logo: a program for generation of sequence logos based on position-specific background amino-acid probabilities\n",
      "Integrating distal and proximal information to predict gene expression via a densely connected convolutional neural network\n",
      "VariantQC: a visual quality control report for variant evaluation\n",
      "QuanTest2: benchmarking multiple sequence alignments using secondary structure prediction\n",
      "MCMCtreeR: functions to prepare MCMCtree analyses and visualize posterior ages on trees\n",
      "Noise-cancelling repeat finder: uncovering tandem repeats in error-prone long-read sequencing data\n",
      "MemBlob database and server for identifying transmembrane regions using cryo-EM maps\n",
      "Look4TRs: a de novo tool for detecting simple tandem repeats using self-supervised hidden Markov models\n",
      "Comparison of single and module-based methods for modeling gene regulatory networks\n",
      "Haplotype assembly of autotetraploid potato using integer linear programing\n",
      "Simultaneous prediction of multiple outcomes using revised stacking algorithms\n",
      "Benchmarking fold detection by DaliLite v.5\n",
      "A helical lock and key model of polyproline II conformation with SH3\n",
      "Sub-dominant principal components inform new vaccine targets for HIV Gag\n",
      "TCR3d: The T cell receptor structural repertoire database\n",
      "Hydra image processor: 5-D GPU image analysis library with MATLAB and python wrappers\n",
      "A fully computational and reasonable representation for karyotypes\n",
      "OncodriveCLUSTL: a sequence-based clustering method to identify cancer drivers\n",
      "Deep representation learning for domain adaptable classification of infrared spectral imaging data\n",
      "Towards reliable named entity recognition in the biomedical domain\n",
      "svtools: population-scale analysis of structural variation\n",
      "A sequential algorithm to detect diffusion switching along intracellular particle trajectories\n",
      "Sequence alignment using machine learning for accurate template-based protein structure prediction\n",
      "Nanopype: a modular and scalable nanopore data processing pipeline\n",
      "A dynamic programing approach to integrate gene expression data and network information for pathway model generation\n",
      "Scaling tree-based automated machine learning to biomedical big data with a feature set selector\n",
      "Gene set enrichment for reproducible science: comparison of CERNO and eight other algorithms\n",
      "CoCo: RNA-seq read assignment correction for nested genes and multimapped reads\n",
      "Smart computational exploration of stochastic gene regulatory network models using human-in-the-loop semi-supervised learning\n",
      "Deep fusion of contextual and object-based representations for delineation of multiple nuclear phenotypes\n",
      "Benchmarking of 4C-seq pipelines based on real and simulated data\n",
      "clustermq enables efficient parallelization of genomic analyses\n",
      "Improving data splitting for classification applications in spectrochemical analyses employing a random-mutation Kennard-Stone algorithm approach\n",
      "deepDR: a network-based deep learning approach to in silico drug repositioning\n",
      "SureTypeSC-a Random Forest and Gaussian mixture predictor of high confidence genotypes in single-cell data\n",
      "ReSimNet: drug response similarity prediction using Siamese neural networks\n",
      "NRStitcher: non-rigid stitching of terapixel-scale volumetric images\n",
      "MetaQUBIC: a computational pipeline for gene-level functional profiling of metagenome and metatranscriptome\n",
      "HyAsP, a greedy tool for plasmids identification\n",
      "CytoPacq: a web-interface for simulating multi-dimensional cell imaging\n",
      "Accurate differential analysis of transcription factor activity from gene expression\n",
      "Multi-SNP mediation intersection-union test\n",
      "MyelinJ: an ImageJ macro for high throughput analysis of myelinating cultures\n",
      "DeepPASTA: deep neural network based polyadenylation site analysis\n",
      "Tibanna: software for scalable execution of portable pipelines on the cloud\n",
      "ProSampler: an ultrafast and accurate motif finder in large ChIP-seq datasets for combinatory motif discovery\n",
      "FP2VEC: a new molecular featurizer for learning molecular properties\n",
      "TractaViewer: a genome-wide tool for preliminary assessment of therapeutic target druggability\n",
      "gpart: human genome partitioning and visualization of high-density SNP data by identifying haplotype blocks\n",
      "Large-scale prediction of binding affinity in protein-small ligand complexes: the PRODIGY-LIG web server\n",
      "Dynamical important residue network (DIRN): network inference via conformational change\n",
      "Develop machine learning-based regression predictive models for engineering protein solubility\n",
      "ClinOmicsTrailbc: a visual analytics tool for breast cancer treatment stratification\n",
      "Overlap graph-based generation of haplotigs for diploids and polyploids\n",
      "Fast detection of maximal exact matches via fixed sampling of query K-mers and Bloom filtering of index K-mers\n",
      "Computational identification of prion-like RNA-binding proteins that form liquid phase-separated condensates\n",
      "Graph-based data integration from bioactive peptide databases of pharmaceutical interest: toward an organized collection enabling visual network analysis\n",
      "A new statistic for efficient detection of repetitive sequences\n",
      "IsoformSwitchAnalyzeR: analysis of changes in genome-wide patterns of alternative splicing and its functional consequences\n",
      "Lipid Mini-On: mining and ontology tool for enrichment analysis of lipidomic data\n",
      "Unsupervised discovery of phenotype-specific multi-omics networks\n",
      "Protein multiple alignments: sequence-based versus structure-based programs\n",
      "Assessing reproducibility of matrix factorization methods in independent transcriptomes\n",
      "Developing structural profile matrices for protein secondary structure and solvent accessibility prediction\n",
      "Kinpute: using identity by descent to improve genotype imputation\n",
      "Simulation-assisted machine learning\n",
      "ASTRAL-MP: scaling ASTRAL to very large datasets using randomization and parallelization\n",
      "IsoCor: isotope correction for high-resolution MS labeling experiments\n",
      "maTE: discovering expressed interactions between microRNAs and their targets\n",
      "Compressed filesystem for managing large genome collections\n",
      "Differential network enrichment analysis reveals novel lipid pathways in chronic kidney disease\n",
      "CyTOFmerge: integrating mass cytometry data across multiple panels\n",
      "mirtronDB: a mirtron knowledge base\n",
      "Protein model quality assessment using 3D oriented convolutional neural networks\n",
      "onlineFDR: an R package to control the false discovery rate for growing data repositories\n",
      "GToTree: a user-friendly workflow for phylogenomics\n",
      "Holistic optimization of an RNA-seq workflow for multi-threaded environments\n",
      "Bit-parallel sequence-to-graph alignment\n",
      "MeSHProbeNet: a self-attentive probe net for MeSH indexing\n",
      "Automated exploration of gene ontology term and pathway networks with ClueGO-REST\n",
      "Bacterial Feature Finder (BaFF)-a system for extracting features overrepresented in sets of prokaryotic organisms\n",
      "Aggrescan3D standalone package for structure-based prediction of protein aggregation properties\n",
      "VERA: agent-based modeling transmission of antibiotic resistance between human pathogens and gut microbiota\n",
      "RECAP reveals the true statistical significance of ChIP-seq peak calls\n",
      "Functional geometry of protein interactomes\n",
      "SinNLRR: a robust subspace clustering method for cell type detection by non-negative and low-rank representation\n",
      "Batch effects correction for microbiome data with Dirichlet-multinomial regression\n",
      "SPIM workflow manager for HPC\n",
      "Missing value estimation methods for DNA methylation data\n",
      "Parameter balancing: consistent parameter sets for kinetic metabolic models\n",
      "GlycanAnalyzer: software for automated interpretation of N-glycan profiles after exoglycosidase digestions\n",
      "Explore, edit and leverage genomic annotations using Python GTF toolkit\n",
      "Driver network as a biomarker: systematic integration and network modeling of multi-omics data to derive driver signaling pathways for drug combination prediction\n",
      "Biomolecular Reaction and Interaction Dynamics Global Environment (BRIDGE)\n",
      "Gsmodutils: a python based framework for test-driven genome scale metabolic model development\n",
      "refineD: improved protein structure refinement using machine learning based restrained relaxation\n",
      "Modeling cell proliferation in human acute myeloid leukemia xenografts\n",
      "Variational infinite heterogeneous mixture model for semi-supervised clustering of heart enhancers\n",
      "Structured crowdsourcing enables convolutional segmentation of histology images\n",
      "G4Hunter web application: a web server for G-quadruplex prediction\n",
      "SaGePhy: an improved phylogenetic simulation framework for gene and subgene evolution\n",
      "GLAD: GLycan Array Dashboard, a visual analytics tool for glycan microarrays\n",
      "Rapid alignment-free phylogenetic identification of metagenomic sequences\n",
      "Predicting runtimes of bioinformatics tools based on historical data: five years of Galaxy usage\n",
      "Reviewer-coerced citation: case report, update on journal policy and suggestions for future prevention\n",
      "NEMO: cancer subtyping by integration of partial multi-omic data\n",
      "Comorbidity4j: a tool for interactive analysis of disease comorbidities over large patient datasets\n",
      "Comparison of statistical algorithms for daily syndromic surveillance aberration detection\n",
      "ensembldb: an R package to create and use Ensembl-based annotation resources\n",
      "The interplay between microRNA and alternative splicing of linear and circular RNAs in eleven plant species\n",
      "Inferring clonal heterogeneity in cancer using SNP arrays and whole genome sequencing\n",
      "Best-first search guided multistage mass spectrometry-based glycan identification\n",
      "SVIM: structural variant identification using mapped long reads\n",
      "Functional data analysis for computational biology\n",
      "RIscoper: a tool for RNA-RNA interaction extraction from the literature\n",
      "OLGA: fast computation of generation probabilities of B- and T-cell receptor amino acid sequences and motifs\n",
      "queryMed: Semantic Web functions for linking pharmacological and medical knowledge to data\n",
      "DIABLO: an integrative approach for identifying key molecular drivers from multi-omics assays\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Xolik: finding cross-linked peptides with maximum paired scores in linear time\n",
      "An accurate and powerful method for copy number variation detection\n",
      "A web application and service for imputing and visualizing missense variant effect maps\n",
      "MSC: a metagenomic sequence classification algorithm\n",
      "ClusterMap: compare multiple single cell RNA-Seq datasets across different experimental conditions\n",
      "Large-scale analysis of human gene expression variability associates highly variable drug targets with lower drug effectiveness and safety\n",
      "AQUAPONY: visualization and interpretation of phylogeographic information on phylogenetic trees\n",
      "CluMSID: an R package for similarity-based clustering of tandem mass spectra to aid feature annotation in metabolomics\n",
      "PyLink: a PyMOL plugin to identify links\n",
      "polymapR-linkage analysis and genetic map construction from F1 populations of outcrossing polyploids\n",
      "PopCluster: an algorithm to identify genetic variants with ethnicity-dependent effects\n",
      "Transcriptogramer: an R/Bioconductor package for transcriptional analysis based on protein-protein interaction\n",
      "Breaking the paradigm: Dr Insight empowers signature-free, enhanced drug repurposing\n",
      "MMseqs2 desktop and local web server app for fast, interactive sequence searches\n",
      "Promoter analysis and prediction in the human genome using sequence-based deep learning models\n",
      "UbiHub: a data hub for the explorers of ubiquitination pathways\n",
      "MoMo: discovery of statistically significant post-translational modification motifs\n",
      "Single-cell RNA-seq interpretations using evolutionary multiobjective ensemble pruning\n",
      "GfaViz: flexible and interactive visualization of GFA sequence graphs\n",
      "Integration of accessibility data from structure probing into RNA-RNA interaction prediction\n",
      "Systematic selection of chemical fingerprint features improves the Gibbs energy prediction of biochemical reactions\n",
      "PredGly: predicting lysine glycation sites for Homo sapiens based on XGboost feature optimization\n",
      "Systematic characterization and prediction of post-translational modification cross-talk between proteins\n",
      "Curse: building expression atlases and co-expression networks from public RNA-Seq data\n",
      "Ancestral sequence reconstruction: accounting for structural information by averaging over replacement matrices\n",
      "DNABarcodeCompatibility: an R-package for optimizing DNA-barcode combinations in multiplex sequencing experiments\n",
      "Inter-residue interactions in alpha-helical transmembrane proteins\n",
      "Comment on: 'Empirical comparison of web-based antimicrobial peptide prediction tools'\n",
      "DensityPath: an algorithm to visualize and reconstruct cell state-transition path on density landscape for single-cell RNA sequencing data\n",
      "Insights on protein thermal stability: a graph representation of molecular interactions\n",
      "WIlsON: Web-based Interactive Omics VisualizatioN\n",
      "Optimal Gene Filtering for Single-Cell data (OGFSC)-a gene filtering algorithm for single-cell RNA-seq data\n",
      "Accurate and efficient estimation of small P-values with the cross-entropy method: applications in genomic data analysis\n",
      "GEMPROT: visualization of the impact on the protein of the genetic variants found on each haplotype\n",
      "Detection of rare disease variants in extended pedigrees using RVS\n",
      "DeeReCT-PolyA: a robust and generic deep learning method for PAS identification\n",
      "TADOSS: computational estimation of tandem domain swap stability\n",
      "PPaxe: easy extraction of protein occurrence and interactions from the scientific literature\n",
      "On the biases in predictions of protein stability changes upon variations: the INPS test case\n",
      "A structural homology approach for computational protein design with flexible backbone\n",
      "EpistasisRank and EpistasisKatz: interaction network centrality methods that integrate prior knowledge networks\n",
      "GenCoF: a graphical user interface to rapidly remove human genome contaminants from metagenomic datasets\n",
      "circtools-a one-stop software solution for circular RNA research\n",
      "CAARS: comparative assembly and annotation of RNA-Seq data\n",
      "Clustering of samples with a tree-shaped dependence structure, with an application to microscopic time lapse imaging\n",
      "A Drug-Side Effect Context-Sensitive Network approach for drug target prediction\n",
      "MDPbiome: microbiome engineering through prescriptive perturbations\n",
      "Single cell network analysis with a mixture of Nested Effects Models\n",
      "piMGM: incorporating multi-source priors in mixed graphical models for learning disease networks\n",
      "A deep neural network approach for learning intrinsic protein-RNA binding preferences\n",
      "A Boolean network inference from time-series gene expression data using a genetic algorithm\n",
      "Towards an accurate and efficient heuristic for species/gene tree co-estimation\n",
      "Higher-order molecular organization as a source of biological function\n",
      "S-Cluster++: a fast program for solving the cluster containment problem for phylogenetic networks\n",
      "OPA2Vec: combining formal and informal content of biomedical ontologies to improve similarity-based prediction\n",
      "Measurement error and variant-calling in deep Illumina sequencing of HIV\n",
      "wgd-simple command line tools for the analysis of ancient whole-genome duplications\n",
      "Improved inference of intermolecular contacts through protein-protein interaction prediction using coevolutionary analysis\n",
      "High-complexity regions in mammalian genomes are enriched for developmental genes\n",
      "Antibody interface prediction with 3D Zernike descriptors and SVM\n",
      "Supervised non-negative matrix factorization methods for MALDI imaging applications\n",
      "gwasurvivr: an R package for genome-wide survival analysis\n",
      "ukbREST: efficient and streamlined data access for reproducible research in large biobanks\n",
      "Partially non-homogeneous dynamic Bayesian networks based on Bayesian regression models with partitioned design matrices\n",
      "Statistical inference of the rate of RNA polymerase II elongation by total RNA sequencing\n",
      "BatchI: Batch effect Identification in high-throughput screening data using a dynamic programming algorithm\n",
      "Simple tricks of convolutional neural network architectures improve DNA-protein binding prediction\n",
      "VCPA: genomic variant calling pipeline and data management tool for Alzheimer's Disease Sequencing Project\n",
      "Alignment-free clustering of UMI tagged DNA molecules\n",
      "SENSE: Siamese neural network for sequence embedding and alignment-free comparison\n",
      "SeeVis-3D space-time cube rendering for visualization of microfluidics image data\n",
      "methylGSA: a Bioconductor package and Shiny app for DNA methylation data length bias adjustment in gene set testing\n",
      "Long-term context-dependent genetic adaptation of the viral genetic cloud\n",
      "BMDExpress 2: enhanced transcriptomic dose-response analysis workflow\n",
      "Integrative cancer patient stratification via subspace merging\n",
      "Quality assessment for the putative intrinsic disorder in proteins\n",
      "CSBFinder: discovery of colinear syntenic blocks across thousands of prokaryotic genomes\n",
      "Joint analysis of individual-level and summary-level GWAS data by leveraging pleiotropy\n",
      "Cross-type biomedical named entity recognition with deep multi-task learning\n",
      "PlateDesigner: a web-based application for the design of microplate experiments\n",
      "A novel approach for drug response prediction in cancer cell lines via network representation learning\n",
      "TEPIC 2-an extended framework for transcription factor binding prediction and integrative epigenomic analysis\n",
      "MetaOmics: analysis pipeline and browser-based software suite for transcriptomic meta-analysis\n",
      "QuagmiR: a cloud-based application for isomiR big data analytics\n",
      "CoMM: a collaborative mixed model to dissecting genetic contributions to complex traits by leveraging regulatory information\n",
      "DIGREM: an integrated web-based platform for detecting effective multi-drug combinations\n",
      "DeepHINT: understanding HIV-1 integration via deep learning with attention\n",
      "De novo pattern discovery enables robust assessment of functional consequences of non-coding variants\n",
      "Squiggle: a user-friendly two-dimensional DNA sequence visualization tool\n",
      "On the viability of unsupervised T-cell receptor sequence clustering for epitope preference\n",
      "Misunderstood parameter of NCBI BLAST impacts the correctness of bioinformatics workflows\n",
      "Exploring sequence-based features for the improved prediction of DNA N4-methylcytosine sites in multiple species\n",
      "A clustering linear combination approach to jointly analyze multiple phenotypes for GWAS\n",
      "The Encyclopedia of Proteome Dynamics: the KinoViewer\n",
      "TOBMI: trans-omics block missing data imputation using a k-nearest neighbor weighted approach\n",
      "Phylolink: phylogenetically-based profiling, visualisations and metrics for biodiversity\n",
      "smCounter2: an accurate low-frequency variant caller for targeted sequencing data with unique molecular identifiers\n",
      "De novo construction of polyploid linkage maps using discrete graphical models\n",
      "Bayesian integrative model for multi-omics data with missingness\n",
      "SCOP: a novel scaffolding algorithm based on contig classification and optimization\n",
      "Deconvoluting essential gene signatures for cancer growth from genomic expression in compound-treated cells\n",
      "A generic deep convolutional neural network framework for prediction of receptor-ligand interactions-NetPhosPan: application to kinase phosphorylation prediction\n",
      "HLA-IMPUTER: an easy to use web application for HLA imputation and association analysis using population-specific reference panels\n",
      "cpPredictor: a web server for template-based prediction of RNA secondary structure\n",
      "A generalized framework for controlling FDR in gene regulatory network inference\n",
      "Traitpedia: a collaborative effort to gather species traits\n",
      "NCycDB: a curated integrative database for fast and accurate metagenomic profiling of nitrogen cycling genes\n",
      "ImaGEO: integrative gene expression meta-analysis from GEO database\n",
      "MCO: towards an ontology and unified vocabulary for a framework-based annotation of microbial growth conditions\n",
      "StructureProfiler: an all-in-one tool for 3D protein structure profiling\n",
      "A quantitative map of protein sequence space for the cis-defensin superfamily\n",
      "Linkage disequilibrium maps to guide contig ordering for genome assembly\n",
      "Global importance of RNA secondary structures in protein-coding sequences\n",
      "TIMING 2.0: high-throughput single-cell profiling of dynamic cell-cell interactions by time-lapse imaging microscopy in nanowell grids\n",
      "Evaluation of cross-platform and interlaboratory concordance via consensus modelling of genomic measurements\n",
      "TAPAS: tool for alternative polyadenylation site analysis\n",
      "The Terabase Search Engine: a large-scale relational database of short-read sequences\n",
      "WAVES: a web application for versatile enhanced bioinformatic services\n",
      "MoDentify: phenotype-driven module identification in metabolomics networks at different resolutions\n",
      "CIRCOAST: a statistical hypothesis test for cellular colocalization with network structures\n",
      "StackDPPred: a stacking based prediction of DNA-binding protein from sequence\n",
      "TreeGrafter: phylogenetic tree-based annotation of proteins with Gene Ontology terms and other annotations\n",
      "MAVIS: merging, annotation, validation, and illustration of structural variants\n",
      "ProteomeVis: a web app for exploration of protein properties from structure to sequence evolution across organisms' proteomes\n",
      "SMSSVD: SubMatrix Selection Singular Value Decomposition\n",
      "Determination of sets of covariating gene expression using graph analysis on pairwise expression ratios\n",
      "Matt: Unix tools for alternative splicing analysis\n",
      "RNAvista: a webserver to assess RNA secondary structures with non-canonical base pairs\n",
      "Noise peak filtering in multi-dimensional NMR spectra using convolutional neural networks\n",
      "Statistical tests for detecting variance effects in quantitative trait studies\n",
      "iS-CellR: a user-friendly tool for analyzing and visualizing single-cell RNA sequencing data\n",
      "MetaboRank: network-based recommendation system to interpret and enrich metabolomics results\n",
      "MrBait: universal identification and design of targeted-enrichment capture probes\n",
      "MetaboAnalystR: an R package for flexible and reproducible analysis of metabolomics data\n",
      "LncmiRSRN: identification and analysis of long non-coding RNA related miRNA sponge regulatory network in human cancer\n",
      "A gene-phenotype relationship extraction pipeline from the biomedical literature using a representation learning approach\n",
      "ISMB 2018 PROCEEDINGS PAPERS COMMITTEE\n",
      "HFSP: high speed homology-driven function annotation of proteins\n",
      "DLBI: deep learning guided Bayesian inference for structure reconstruction of super-resolution fluorescence microscopy\n",
      "Training for translation between disciplines: a philosophy for life and data sciences curricula\n",
      "GSEA-InContext: identifying novel and common patterns in expression experiments\n",
      "SigMat: a classification scheme for gene signature matching\n",
      "Deconvolution and phylogeny inference of structural variations in tumor genomic samples\n",
      "Discriminating early- and late-stage cancers using multiple kernel learning on gene sets\n",
      "An integration of fast alignment and maximum-likelihood methods for electron subtomogram averaging and classification\n",
      "Strand-seq enables reliable separation of long reads by chromosome via expectation maximization\n",
      "Novo&Stitch: accurate reconciliation of genome assemblies via optical maps\n",
      "PrimAlign: PageRank-inspired Markovian alignment for large biological networks\n",
      "aliFreeFold: an alignment-free approach to predict secondary structure from homologous RNA sequences\n",
      "nVenn: generalized, quasi-proportional Venn and Euler diagrams\n",
      "The choice of sequence homologs included in multiple sequence alignments has a dramatic impact on evolutionary conservation analysis\n",
      "appreci8: a pipeline for precise variant calling integrating 8 tools\n",
      "ncdDetect2: improved models of the site-specific mutation rate in cancer and driver detection with robust significance evaluation\n",
      "Prediction of human-Bacillus anthracis protein-protein interactions using multi-layer neural network\n",
      "Predicting miRNA-disease association based on inductive matrix completion\n",
      "runibic: a Bioconductor package for parallel row-based biclustering of gene expression data\n",
      "Learned protein embeddings for machine learning\n",
      "BioStructMap: a Python tool for integration of protein structure and sequence-based features\n",
      "ICBdocker: a Docker image for proteome annotation and visualization\n",
      "GIFT: Guided and Interpretable Factorization for Tensors with an application to large-scale multi-platform cancer analysis\n",
      "Estimating pseudocounts and fold changes for digital expression measurements\n",
      "Deep convolutional networks for quality assessment of protein folds\n",
      "mol2sphere: spherical decomposition of multi-domain molecules for visualization and coarse grained spatial modeling\n",
      "Optimality and identification of dynamic models in systems biology: an inverse optimal control framework\n",
      "DDR: efficient computational method to predict drug-target interactions using graph mining and machine learning approaches\n",
      "GeneSpy, a user-friendly and flexible genomic context visualizer\n",
      "Chemical shift-based identification of monosaccharide spin-systems with NMR spectroscopy to complement untargeted glycomics\n",
      "ClusterScan: simple and generalistic identification of genomic clusters\n",
      "IDP-denovo: de novo transcriptome assembly and isoform annotation by hybrid sequencing\n",
      "SciApps: a cloud-based platform for reproducible bioinformatics workflows\n",
      "miRNACancerMAP: an integrative web server inferring miRNA regulation network for cancer\n",
      "Detection of multi-dimensional co-exclusion patterns in microbial communities\n",
      "Graph-guided multi-task sparse learning model: a method for identifying antigenic variants of influenza A(H3N2) virus\n",
      "ACPred-FL: a sequence-based predictor using effective feature representation to improve the prediction of anti-cancer peptides\n",
      "ProAcePred: prokaryote lysine acetylation sites prediction based on elastic net feature optimization\n",
      "TriPoly: haplotype estimation for polyploids using sequencing data of related individuals\n",
      "LiveKraken--real-time metagenomic classification of illumina data\n",
      "An in silico proteomics screen to predict and prioritize protein-protein interactions dependent on post-translationally modified motifs\n",
      "An automated method for detecting alternatively spliced protein domains\n",
      "deepNF: deep network fusion for protein function prediction\n",
      "PEA: an integrated R toolkit for plant epitranscriptome analysis\n",
      "RIFRAF: a frame-resolving consensus algorithm\n",
      "LncRNAnet: long non-coding RNA identification using deep learning\n",
      "geck: trio-based comparative benchmarking of variant calls\n",
      "PyCoTools: a Python toolbox for COPASI\n",
      "MASCOT: parameter and state inference under the marginal structured coalescent approximation\n",
      "PANDA-view: an easy-to-use tool for statistical analysis and visualization of quantitative proteomics data\n",
      "SoS Notebook: an interactive multi-language data analysis environment\n",
      "pyABC: distributed, likelihood-free inference\n",
      "Transform-MinER: transforming molecules in enzyme reactions\n",
      "Lineage-associated underrepresented permutations (LAUPs) of mammalian genomic sequences based on a Jellyfish-based LAUPs analysis application (JBLA)\n",
      "Base-pair resolution detection of transcription factor binding site by deep deconvolutional network\n",
      "PROBE: analysis and visualization of protein block-level evolution\n",
      "REGGAE: a novel approach for the identification of key transcriptional regulators\n",
      "Recombinational DSBs-intersected genes converge on specific disease- and adaptability-related pathways\n",
      "canvasDesigner: a versatile interactive high-resolution scientific multi-panel visualization toolkit\n",
      "Generic accelerated sequence alignment in SeqAn using vectorization and multi-threading\n",
      "Predicting RNA-protein binding sites and motifs through combining local and global deep convolutional neural networks\n",
      "The UEA sRNA Workbench (version 4.4): a comprehensive suite of tools for analyzing miRNAs and sRNAs\n",
      "polymapR-linkage analysis and genetic map construction from F1 populations of outcrossing polyploids\n",
      "BlockFeST: Bayesian calculation of region-specific FST to detect local adaptation\n",
      "Prediction of lncRNA-disease associations based on inductive matrix completion\n",
      "High precision in protein contact prediction using fully convolutional neural networks and minimal sequence features\n",
      "Exact Bayesian inference for phylogenetic birth-death models\n",
      "TMCrys: predict propensity of success for transmembrane protein crystallization\n",
      "RBind: computational network method to predict RNA binding sites\n",
      "Cluster Locator, online analysis and visualization of gene clustering\n",
      "Two-phase differential expression analysis for single cell RNA-seq\n",
      "Bayesian negative binomial regression for differential expression with confounding factors\n",
      "Comparative analysis of tissue reconstruction algorithms for 3D histology\n",
      "annoPeak: a web application to annotate and visualize peaks from ChIP-seq/ChIP-exo-seq\n",
      "TRUmiCount: correctly counting absolute numbers of molecules using unique molecular identifiers\n",
      "GARFIELD-NGS: Genomic vARiants FIltering by dEep Learning moDels in NGS\n",
      "Using standard microbiome reference groups to simplify beta-diversity analyses and facilitate independent validation\n",
      "Ribopeaks: a web tool for bacterial classification through m/z data from ribosomal proteins\n",
      "The latent geometry of the human protein interaction network\n",
      "Detecting hidden batch factors through data-adaptive adjustment for biological effects\n",
      "PhenotypeSimulator: A comprehensive framework for simulating multi-trait, multi-locus genotype to phenotype relationships\n",
      "De novo haplotype reconstruction in viral quasispecies using paired-end read guided path finding\n",
      "Predicting serious rare adverse reactions of novel chemicals\n",
      "pyNBS: a Python implementation for network-based stratification of tumor mutations\n",
      "Understanding sequencing data as compositions: an outlook and review\n",
      "MetaboCraft: building a Minecraft plugin for metabolomics\n",
      "A Bayesian framework for multiple trait colocalization from summary association statistics\n",
      "iterClust: a statistical framework for iterative clustering analysis\n",
      "ADPredict: ADP-ribosylation site prediction based on physicochemical and structural descriptors\n",
      "Arioc: GPU-accelerated alignment of short bisulfite-treated reads\n",
      "RapidRMSD: rapid determination of RMSDs corresponding to motions of flexible molecules\n",
      "Data mining of effect sizes from PubMed abstracts: a cross-study conceptual replication\n",
      "CoABind: a novel algorithm for Coenzyme A (CoA)- and CoA derivatives-binding residues prediction\n",
      "NeSSie: a tool for the identification of approximate DNA sequence symmetries\n",
      "GAIT: Gene expression Analysis for Interval Time\n",
      "MatrixEpistasis: ultrafast, exhaustive epistasis scan for quantitative traits with covariate adjustment\n",
      "Extrema-weighted feature extraction for functional data\n",
      "MIRA: an R package for DNA methylation-based inference of regulatory activity\n",
      "Parallelization of MAFFT for large-scale multiple sequence alignments\n",
      "ShinyKGode: an interactive application for ODE parameter inference using gradient matching\n",
      "A global network of biomedical relationships derived from text\n",
      "PennDiff: detecting differential alternative splicing and transcription by RNA sequencing\n",
      "Progressive approach for SNP calling and haplotype assembly using single molecular sequencing data\n",
      "flowLearn: fast and precise identification and quality checking of cell populations in flow cytometry\n",
      "BetaSerpentine: a bioinformatics tool for reconstruction of amyloid structures\n",
      "HiCapTools: a software suite for probe design and proximity detection for targeted chromosome conformation capture applications\n",
      "HIITE: HIV-1 incidence and infection time estimator\n",
      "KMgene: a unified R package for gene-based association analysis for complex traits\n",
      "Improving SNP prioritization and pleiotropic architecture estimation by incorporating prior knowledge using graph-GPA\n",
      "L1000FWD: fireworks visualization of drug-induced transcriptomic signatures\n",
      "omicsPrint: detection of data linkage errors in multiple omics studies\n",
      "AbDesigner3D: a structure-guided tool for peptide-based antibody production\n",
      "MutHTP: mutations in human transmembrane proteins\n",
      "Evolutionary relationship between the cysteine and histidine rich domains (CHORDs) and Btk-type zinc fingers\n",
      "TOXsIgN: a cross-species repository for toxicogenomic signatures\n",
      "G2S: a web-service for annotating genomic variants on 3D protein structures\n",
      "DincRNA: a comprehensive web-based bioinformatics toolkit for exploring disease associations and ncRNA function\n",
      "Haystack: systematic analysis of the variation of epigenetic states and cell-type specific regulatory elements\n",
      "DExSI: a new tool for the rapid quantitation of 13C-labelled metabolites detected by GC-MS\n",
      "BAUM: improving genome assembly by adaptive unique mapping and local overlap-layout-consensus approach\n",
      "MAGNAMWAR: an R package for genome-wide association studies of bacterial orthologs\n",
      "Enrichment analysis with EpiAnnotator\n",
      "Algorithmic identification of discrepancies between published ratios and their reported confidence intervals and P-values\n",
      "ChronQC: a quality control monitoring system for clinical next generation sequencing\n",
      "PGA: post-GWAS analysis for disease gene identification\n",
      "Ontological function annotation of long non-coding RNAs through hierarchical multi-label classification\n",
      "ProbAnnoWeb and ProbAnnoPy: probabilistic annotation and gap-filling of metabolic reconstructions\n",
      "A novel approach based on KATZ measure to predict associations of human microbiota with non-infectious diseases\n",
      "K2 and K2*: efficient alignment-free sequence similarity measurement based on Kendall statistics\n",
      "gene2drug: a computational tool for pathway-based rational drug repositioning\n",
      "IndeCut evaluates performance of network motif discovery algorithms\n",
      "New algorithms to represent complex pseudoknotted RNA structures in dot-bracket notation\n",
      "DeepSF: deep convolutional neural network for mapping protein sequences to folds\n",
      "FROGS: Find, Rapidly, OTUs with Galaxy Solution\n",
      "PEITH(Θ): perfecting experiments with information theory in Python with GPU support\n",
      "SANA NetGO: a combinatorial approach to using Gene Ontology (GO) terms to score network alignments\n",
      "SPRING: a kinetic interface for visualizing high dimensional single-cell expression data\n",
      "Integrating 5hmC and gene expression data to infer regulatory mechanisms\n",
      "Model-based design of bistable cell factories for metabolic engineering\n",
      "iDINGO-integrative differential network analysis in genomics with Shiny application\n",
      "bioBakery: a meta'omic analysis environment\n",
      "Genome Context Viewer: visual exploration of multiple annotated genomes using microsynteny\n",
      "SPECTRE: a suite of phylogenetic tools for reticulate evolution\n",
      "Global proteomics profiling improves drug sensitivity prediction: results from a multi-omics, pan-cancer modeling approach\n",
      "Semi-supervised network inference using simulated gene expression dynamics\n",
      "3dRPC: a web server for 3D RNA-protein structure prediction\n",
      "An attention-based BiLSTM-CRF approach to document-level chemical named entity recognition\n",
      "Mapping-free variant calling using haplotype reconstruction from k-mer frequencies\n",
      "scHiCNorm: a software package to eliminate systematic biases in single-cell Hi-C data\n",
      "Extreme learning machines for reverse engineering of gene regulatory networks from expression time series\n",
      "Identification of topological features in renal tumor microenvironment associated with patient survival\n",
      "microRPM: a microRNA prediction model based only on plant small RNA sequencing data\n",
      "Predicting loop conformational ensembles\n",
      "Bringing numerous methods for expression and promoter analysis to a public cloud computing service\n",
      "PREFMD: a web server for protein structure refinement via molecular dynamics simulations\n",
      "TabPath: interactive tables for metabolic pathway analysis\n",
      "FunImageJ: a Lisp framework for scientific image processing\n",
      "Mosdepth: quick coverage calculation for genomes and exomes\n",
      "Simulating the dynamics of targeted capture sequencing with CapSim\n",
      "Machine learning annotation of human branchpoints\n",
      "Efficient comparative phylogenetics on large trees\n",
      "DIEGO: detection of differential alternative splicing using Aitchison's geometry\n",
      "ViewBS: a powerful toolkit for visualization of high-throughput bisulfite sequencing data\n",
      "ChIPWig: a random access-enabling lossless and lossy compression method for ChIP-seq data\n",
      "Reactome enhanced pathway visualization\n",
      "ISEScan: automated identification of insertion sequence elements in prokaryotic genomes\n",
      "Linking metabolic network features to phenotypes using sparse group lasso\n",
      "Quark enables semi-reference-based compression of RNA-seq data\n",
      "netReg: network-regularized linear models for biological association studies\n",
      "DEEPre: sequence-based enzyme EC number prediction by deep learning\n",
      "DISC: DISulfide linkage Characterization from tandem mass spectra\n",
      "Bartender: a fast and accurate clustering algorithm to count barcode reads\n",
      "PESTO: Parameter EStimation TOolbox\n",
      "mORCA: sailing bioinformatics world with mobile devices\n",
      "heatmaply: an R package for creating interactive cluster heatmaps for online publishing\n",
      "PaRSnIP: sequence-based protein solubility prediction using gradient boosting machine\n",
      "Chromatin accessibility prediction via a hybrid deep convolutional neural network\n",
      "iSmaRT: a toolkit for a comprehensive analysis of small RNA-Seq data\n",
      "UC2 search: using unique connectivity of uncharged compounds for metabolite annotation by database searching in mass spectrometry-based metabolomics\n",
      "Robust and sparse correlation matrix estimation for the analysis of high-dimensional genomics data\n",
      "AEGS: identifying aberrantly expressed gene sets for differential variability analysis\n",
      "BiobankUniverse: automatic matchmaking between datasets for biobank data discovery and integration\n",
      "Standardizing biomass reactions and ensuring complete mass balance in genome-scale metabolic models\n",
      "ggseqlogo: a versatile R package for drawing sequence logos\n",
      "VCF-Explorer: filtering and analysing whole genome VCF files\n",
      "A nonparametric significance test for sampled networks\n",
      "LRCstats, a tool for evaluating long reads correction methods\n",
      "myVCF: a desktop application for high-throughput mutations data management\n",
      "powsimR: power analysis for bulk and single cell RNA-seq experiments\n",
      "Lep-MAP3: robust linkage mapping even for low-coverage whole genome sequencing data\n",
      "DiversePathsJ: diverse shortest paths for bioimage analysis\n",
      "diffuStats: an R package to compute diffusion-based scores on biological networks\n",
      "TROVE: a user-friendly tool for visualizing and analyzing cancer hallmarks in signaling networks\n",
      "Sipros Ensemble improves database searching and filtering for complex metaproteomics\n",
      "AQUa: an adaptive framework for compression of sequencing quality scores with random access functionality\n",
      "Nephele: a cloud platform for simplified, standardized and reproducible microbiome data analysis\n",
      "MALDIrppa: quality control and robust analysis for mass spectrometry data\n",
      "RRDB: a comprehensive and non-redundant benchmark for RNA-RNA docking and scoring\n",
      "HoTResDB: host transcriptional response database for viral hemorrhagic fevers\n",
      "Protein-protein interaction specificity is captured by contact preferences and interface composition\n",
      "VariantTools: an extensible framework for developing and testing variant callers\n",
      "pulseR: Versatile computational analysis of RNA turnover from metabolic labeling experiments\n",
      "Gearing up to handle the mosaic nature of life in the quest for orthologs\n",
      "DART: a fast and accurate RNA-seq mapper with a partitioning strategy\n",
      "A novel SCCA approach via truncated ℓ1-norm and truncated group lasso for brain imaging genetics\n",
      "Ms2lda.org: web-based topic modelling for substructure discovery in mass spectrometry\n",
      "Enhanced guide-RNA design and targeting analysis for precise CRISPR genome editing of single and consortia of industrially relevant and non-model organisms\n",
      "Mobi 2.0: an improved method to define intrinsic disorder, mobility and linear binding regions in protein structures\n",
      "LDassoc: an online tool for interactively exploring genome-wide association study results and prioritizing variants for functional investigation\n",
      "Kmer-SSR: a fast and exhaustive SSR search algorithm\n",
      "NetProphet 2.0: mapping transcription factor networks by exploiting scalable data resources\n",
      "Meta-server for automatic analysis, scoring and ranking of docking models\n",
      "LinkageMapView-rendering high-resolution linkage and QTL maps\n",
      "Identification and visualization of differential isoform expression in RNA-seq time series\n",
      "Three-dimensional cardiovascular imaging-genetics: a mass univariate framework\n",
      "NGS-pipe: a flexible, easily extendable and highly configurable framework for NGS analysis\n",
      "tmVar 2.0: integrating genomic variant information from literature with dbSNP and ClinVar for precision medicine\n",
      "Dugong: a Docker image, based on Ubuntu Linux, focused on reproducibility and replicability for bioinformatics analyses\n",
      "MAJIQ-SPEL: web-tool to interrogate classical and complex splicing variations from RNA-Seq data\n",
      "SparseIso: a novel Bayesian approach to identify alternatively spliced isoforms from RNA-seq data\n",
      "PhyloGeoTool: interactively exploring large phylogenies in an epidemiological context\n",
      "Towards clinically more relevant dissection of patient heterogeneity via survival-based Bayesian clustering\n",
      "Enabling reproducible real-time quantitative PCR research: the RDML package\n",
      "CRISPR-DAV: CRISPR NGS data analysis and visualization pipeline\n",
      "Efficient inference for sparse latent variable models of transcriptional regulation\n",
      "ReMILO: reference assisted misassembly detection algorithm using short and long reads\n",
      "FAF-Drugs4: free ADME-tox filtering computations for chemical biology and early stages drug discovery\n",
      "MetaCache: context-aware classification of metagenomic reads using minhashing\n",
      "BSviewer: a genotype-preserving, nucleotide-level visualizer for bisulfite sequencing data\n",
      "EMHP: an accurate automated hole masking algorithm for single-particle cryo-EM image processing\n",
      "Predicting novel metabolic pathways through subgraph mining\n",
      "MWASTools: an R/bioconductor package for metabolome-wide association studies\n",
      "An introduction to deep learning on biological sequence data: examples and solutions\n",
      "Sequence2Vec: a novel embedding approach for modeling transcription factor binding affinity landscape\n",
      "ClusterSignificance: a bioconductor package facilitating statistical analysis of class cluster separations in dimensionality reduced data\n",
      "GraphSpace: stimulating interdisciplinary collaborations in network biology\n",
      "Molecular signatures that can be transferred across different omics platforms\n",
      "CATS (Coordinates of Atoms by Taylor Series): protein design with backbone flexibility in all locally feasible directions\n",
      "miniMDS: 3D structural inference from high-resolution Hi-C data\n",
      "Systematic identification of feature combinations for predicting drug response with Bayesian multi-view multi-task linear regression\n",
      "A new method to study the change of miRNA-mRNA interactions due to environmental exposures\n",
      "Efficient simulation of intrinsic, extrinsic and external noise in biochemical systems\n",
      "TITER: predicting translation initiation sites by deep learning\n",
      "Identification of associations between genotypes and longitudinal phenotypes via temporally-constrained group sparse canonical correlation analysis\n",
      "Increasing the power of meta-analysis of genome-wide association studies to detect heterogeneous effects\n",
      "Modelling haplotypes with respect to reference cohort variation graphs\n",
      "Rectified factor networks for biclustering of omics data\n",
      "Message from the ISCB: 2017 ISCB Innovator Award Given to Aviv Regev\n",
      "MaBoSS 2.0: an environment for stochastic Boolean modeling\n",
      "Oncopression: gene expression compendium for cancer with matched normal tissues\n",
      "ISMB/ECCB 2017 PROCEEDINGS PAPERS COMMITTEE\n",
      "CausalR: extracting mechanistic sense from genome scale data\n",
      "modPhEA: model organism Phenotype Enrichment Analysis of eukaryotic gene sets\n",
      "Identification of transcript regulatory patterns in cell differentiation\n",
      "Multimodal mechanistic signatures for neurodegenerative diseases (NeuroMMSig): a web server for mechanism enrichment\n",
      "UpSetR: an R package for the visualization of intersecting sets and their properties\n",
      "EDEN: evolutionary dynamics within environments\n",
      "Detecting and removing multiplicative spatial bias in high-throughput screening technologies\n",
      "kmerPyramid: an interactive visualization tool for nucleobase and k-mer frequencies\n",
      "TIminer: NGS data mining pipeline for cancer immunology and immunotherapy\n",
      "MotifHyades: expectation maximization for de novo DNA motif pair discovery on paired sequences\n",
      "CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation and visualization\n",
      "Message from the ISCB: 2017 Outstanding Contributions to ISCB Award Given to Fran Lewitter\n",
      "BreakPoint Surveyor: a pipeline for structural variant visualization\n",
      "HUGIn: Hi-C Unifying Genomic Interrogator\n",
      "GLANET: genomic loci annotation and enrichment tool\n",
      "CRISPRcloud: a secure cloud-based pipeline for CRISPR pooled screen deconvolution\n",
      "M2Align: parallel multiple sequence alignment with a multi-objective metaheuristic\n",
      "Predicting accurate contacts in thousands of Pfam domain families using PconsC3\n",
      "Accelerating high-dimensional clustering with lossless data reduction\n",
      "Pattern fusion analysis by adaptive alignment of multiple heterogeneous omics data\n",
      "DIRECTION: a machine learning framework for predicting and characterizing DNA methylation and hydroxymethylation in mammalian genomes\n",
      "False discovery rate control incorporating phylogenetic tree increases detection power in microbiome-wide multiple testing\n",
      "E2FM: an encrypted and compressed full-text index for collections of genomic sequences\n",
      "RankProd 2.0: a refactored bioconductor package for detecting differentially expressed features in molecular profiling datasets\n",
      "MarDRe: efficient MapReduce-based removal of duplicate DNA reads in the cloud\n",
      "PopFly: the Drosophila population genomics browser\n",
      "A deep learning framework for improving long-range residue-residue contact prediction using a hierarchical strategy\n",
      "Differential privacy-based evaporative cooling feature selection and classification with relief-F and random forests\n",
      "BioPAXViz: a cytoscape application for the visual exploration of metabolic pathway evolution\n",
      "A mass graph-based approach for the identification of modified proteoforms using top-down tandem mass spectra\n",
      "Deep mining heterogeneous networks of biomedical linked data to predict novel drug-target associations\n",
      "ClusPro PeptiDock: efficient global docking of peptide recognition motifs using FFT\n",
      "StrandScript: evaluation of Illumina genotyping array design and strand correction\n",
      "TIN-X: target importance and novelty explorer\n",
      "MISA-web: a web server for microsatellite prediction\n",
      "GARN2: coarse-grained prediction of 3D structure of large RNA molecules by regret minimization\n",
      "ploidyNGS: visually exploring ploidy with Next Generation Sequencing data\n",
      "PreMosa: extracting 2D surfaces from 3D microscopy mosaics\n",
      "Mackinac: a bridge between ModelSEED and COBRApy to generate and analyze genome-scale metabolic models\n",
      "CMDR based differential evolution identifies the epistatic interaction in genome-wide association studies\n",
      "The Tomato Expression Atlas\n",
      "cnAnalysis450k: an R package for comparative analysis of 450k/EPIC Illumina methylation array derived copy number data\n",
      "NeBcon: protein contact map prediction using neural network training coupled with naïve Bayes classifiers\n",
      "GenoGAM: genome-wide generalized additive models for ChIP-Seq analysis\n",
      "Entropy-based consensus clustering for patient stratification\n",
      "Aozan: an automated post-sequencing data-processing pipeline\n",
      "EthSEQ: ethnicity annotation from whole exome sequencing data\n",
      "Mining heterogeneous causal effects for personalized cancer treatment\n",
      "A transition-based joint model for disease named entity recognition and normalization\n",
      "ConKit: a python interface to contact predictions\n",
      "biMM: efficient estimation of genetic variances and covariances for cohorts with high-dimensional phenotype measurements\n",
      "poRe GUIs for parallel and real-time processing of MinION sequence data\n",
      "SeqArray-a storage-efficient high-performance data format for WGS variant calls\n",
      "SynMap2 and SynMap3D: web-based whole-genome synteny browsers\n",
      "MetCCS predictor: a web server for predicting collision cross-section values of metabolites in ion mobility-mass spectrometry based metabolomics\n",
      "Fast, clash-free RNA conformational morphing using molecular junctions\n",
      "ProtVista: visualization of protein sequence annotations\n",
      "Computational modeling of in vivo and in vitro protein-DNA interactions by multiple instance learning\n",
      "A network model predicts the intensity of residue-protein thermal coupling\n",
      "Morphologically constrained spectral unmixing by dictionary learning for multiplex fluorescence microscopy\n",
      "Stepwise inference of likely dynamic flux distributions from metabolic time series data\n",
      "SVPV: a structural variant prediction viewer for paired-end sequencing datasets\n",
      "fastER: a user-friendly tool for ultrafast and robust cell segmentation in large-scale microscopy\n",
      "A multi-scale convolutional neural network for phenotyping high-content cellular images\n",
      "Empirical comparison of web-based antimicrobial peptide prediction tools\n",
      "Pln24NT: a web resource for plant 24-nt siRNA producing loci\n",
      "Threshold-seq: a tool for determining the threshold in short RNA-seq datasets\n",
      "nala: text mining natural language mutation mentions\n",
      "Adaptive MCMC in Bayesian phylogenetics: an application to analyzing partitioned data in BEAST\n",
      "Genome-wide genetic heterogeneity discovery with categorical covariates\n",
      "sfinx: an R package for the elimination of false positives from affinity purification-mass spectrometry datasets\n",
      "FELLS: fast estimator of latent local structure\n",
      "Simulome: a genome sequence and variant simulator\n",
      "DiagnoProt: a tool for discovery of new molecules by mass spectrometry\n",
      "OncoSimulR: genetic simulation with arbitrary epistasis and mutator genes in asexual populations\n",
      "GLINT: a user-friendly toolset for the analysis of high-throughput DNA-methylation array data\n",
      "cooccurNet: an R package for co-occurrence network construction and analysis\n",
      "LDAP: a web server for lncRNA-disease association prediction\n",
      "QUBIC: a bioconductor package for qualitative biclustering analysis of gene co-expression data\n",
      "Powerful differential expression analysis incorporating network topology for next-generation sequencing data\n",
      "BASIC: BCR assembly from single cells\n",
      "Image-based surrogate biomarkers for molecular subtypes of colorectal cancer\n",
      "CHRR: coordinate hit-and-run with rounding for uniform sampling of constraint-based models\n",
      "admixturegraph: an R package for admixture graph manipulation and fitting\n",
      "LOCUST: a custom sequence locus typer for classifying microbial isolates\n",
      "The JWS online simulation database\n",
      "Identification of protein complexes by integrating multiple alignment of protein interaction networks\n",
      "H(O)TA: estimation of DNA methylation and hydroxylation levels and efficiencies from time course data\n",
      "ISPRED4: interaction sites PREDiction in protein structures with a refining grammar model\n",
      "Evolutionary design of multiple genes encoding the same protein\n",
      "MetaShot: an accurate workflow for taxon classification of host-associated microbiome from shotgun metagenomic data\n",
      "Unrealistic phylogenetic trees may improve phylogenetic footprinting\n",
      "QCEWAS: automated quality control of results of epigenome-wide association studies\n",
      "ASCIIGenome: a command line genome browser for console terminals\n",
      "GWAR: robust analysis and meta-analysis of genome-wide association studies\n",
      "chainCleaner improves genome alignment specificity and sensitivity\n",
      "Improved orthology inference with Hieranoid 2\n",
      "DeuteRater: a tool for quantifying peptide isotope precision and kinetic proteomics\n",
      "Protein multiple sequence alignment benchmarking through secondary structure prediction\n",
      "Development and application of an algorithm to compute weighted multiple glycan alignments\n",
      "Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R\n",
      "H-BLAST: a fast protein sequence alignment toolkit on heterogeneous computers with GPUs\n",
      "Seeing the trees through the forest: sequence-based homo- and heteromeric protein-protein interaction sites prediction using random forest\n",
      "Advanced Boolean modeling of biological networks applied to systems pharmacology\n",
      "Fast and accurate phylogeny reconstruction using filtered spaced-word matches\n",
      "caspo: a toolbox for automated reasoning on the response of logical signaling networks families\n",
      "PRINCESS: Privacy-protecting Rare disease International Network Collaboration via Encryption through Software guard extensionS\n",
      "Statistical inference of protein structural alignments using information and compression\n",
      "MetaSpark: a spark-based distributed processing tool to recruit metagenomic reads to reference genomes\n",
      "Automated neuron tracing using probability hypothesis density filtering\n",
      "ontologyX: a suite of R packages for working with ontological data\n",
      "pHMM-tree: phylogeny of profile hidden Markov models\n",
      "SINE_scan: an efficient tool to discover short interspersed nuclear elements (SINEs) in large-scale genomic datasets\n",
      "iSmaRT: a toolkit for a comprehensive analysis of small RNA-Seq data\n",
      "ORCAN-a web-based meta-server for real-time detection and functional annotation of orthologs\n",
      "Planning bioinformatics workflows using an expert system\n",
      "ProQ3D: improved model quality assessments using deep learning\n",
      "LncRNA/DNA binding analysis reveals losses and gains and lineage specificity of genomic imprinting in mammals\n",
      "SED-ML web tools: generate, modify and export standard-compliant simulation studies\n",
      "Virtual exploration of early stage atherosclerosis\n",
      "AltORFev facilitates the prediction of alternative open reading frames in eukaryotic mRNAs\n",
      "CAME: identification of chromatin accessibility from nucleosome occupancy and methylome sequencing\n",
      "InMoDe: tools for learning and visualizing intra-motif dependencies of DNA binding sites\n",
      "Statistical significance based on length and position of the local score in a model of i.i.d. sequences\n",
      "Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi\n",
      "SVScore: an impact prediction tool for structural variation\n",
      "Novel probabilistic models of spatial genetic ancestry with applications to stratification correction in genome-wide association studies\n",
      "tranSMART-XNAT Connector tranSMART-XNAT connector-image selection based on clinical phenotypes and genetic profiles\n",
      "DACE: a scalable DP-means algorithm for clustering extremely large sequence data\n",
      "A novel approach based on KATZ measure to predict associations of human microbiota with non-infectious diseases\n",
      "High-throughput interpretation of gene structure changes in human and nonhuman resequencing data, using ACE\n",
      "switchde: inference of switch-like differential expression along single-cell trajectories\n",
      "VarMatch: robust matching of small variant datasets using flexible scoring schemes\n",
      "FUEL-mLoc: feature-unified prediction and explanation of multi-localization of cellular proteins in multiple organisms\n",
      "Fast motif matching revisited: high-order PWMs, SNPs and indels\n",
      "ALTRE: workflow for defining ALTered Regulatory Elements using chromatin accessibility data\n",
      "Controlling the joint local false discovery rate is more powerful than meta-analysis methods in joint analysis of summary statistics from multiple genome-wide association studies\n",
      "Simulation-based estimation of branching models for LTR retrotransposons\n",
      "Structural insight to mutation effects uncover a common allosteric site in class C GPCRs\n",
      "GWAlpha: genome-wide estimation of additive effects (alpha) based on trait quantile distribution from pool-sequencing experiments\n",
      "MEGA-V: detection of variant gene sets in patient cohorts\n",
      "Recycler: an algorithm for detecting plasmids from de novo assembly graphs\n",
      "Agent-based re-engineering of ErbB signaling: a modeling pipeline for integrative systems biology\n",
      "An integer programming framework for inferring disease complexes from network data\n",
      "MRUniNovo: an efficient tool for de novo peptide sequencing utilizing the hadoop distributed computing framework\n",
      "Improved VCF normalization for accurate VCF comparison\n",
      "Message from the ISCB: The 5th ISCB Wikipedia competition: coming to a classroom near you?\n",
      "RBPPred: predicting RNA-binding proteins from sequence using SVM\n",
      "LEAP: constructing gene co-expression networks for single-cell RNA-sequencing data using pseudotime ordering\n",
      "fCCAC: functional canonical correlation analysis to evaluate covariance between nucleic acid sequencing datasets\n",
      "KODAMA: an R package for knowledge discovery and data mining\n",
      "EGAD: ultra-fast functional analysis of gene networks\n",
      "Recognizing metal and acid radical ion-binding sites by integrating ab initio modeling with template-based transferals\n",
      "ImageJ-MATLAB: a bidirectional framework for scientific image analysis interoperability\n",
      "Network-based analysis of omics data: the LEAN method\n",
      "Web-igloo: a web based platform for multivariate data visualization\n",
      "KAT: a K-mer analysis toolkit to quality control NGS datasets and genome assemblies\n",
      "miRNAmeConverter: an R/bioconductor package for translating mature miRNA names to different miRBase versions\n",
      "Rabifier2: an improved bioinformatic classifier of Rab GTPases\n",
      "CLIP Tool Kit (CTK): a flexible and robust pipeline to analyze CLIP sequencing data\n",
      "stAIcalc: tRNA adaptation index calculator based on species-specific weights\n",
      "MIToS.jl: mutual information tools for protein sequence analysis in the Julia language\n",
      "gargammel: a sequence simulator for ancient DNA\n",
      "LAMSA: fast split read alignment with long approximate matches\n",
      "Imbalanced multi-label learning for identifying antimicrobial peptides and their functional types\n",
      "ReadXplorer 2-detailed read mapping analysis and visualization from one single source\n",
      "Towards the knowledge-based design of universal influenza epitope ensemble vaccines\n",
      "Recognizing metal and acid radical ion-binding sites by integrating ab initio modeling with template-based transferals\n",
      "An empirical Bayes method for genotyping and SNP detection using multi-sample next-generation sequencing data\n",
      "LongISLND: in silico sequencing of lengthy and noisy datatypes\n",
      "Robust classification of single-cell transcriptome data by nonnegative matrix factorization\n",
      "PDB2CD: a web-based application for the generation of circular dichroism spectra from protein atomic coordinates\n",
      "BOSS: a novel scaffolding algorithm based on an optimized scaffold graph\n",
      "A probabilistic model for detecting rigid domains in protein structures\n",
      "PeakXus: comprehensive transcription factor binding site discovery from ChIP-Nexus and ChIP-Exo experiments\n",
      "A unified model based multifactor dimensionality reduction framework for detecting gene-gene interactions\n",
      "Assemble CRISPRs from metagenomic sequencing data\n",
      "Information-optimal genome assembly via sparse read-overlap graphs\n",
      "Drug response prediction by inferring pathway-response associations with kernelized Bayesian matrix factorization\n",
      "Complementary feature selection from alternative splicing events and gene expression for phenotype prediction\n",
      "Higher order methylation features for clustering and prediction in epigenomic studies\n",
      "Estimating real cell size distribution from cross-section microscopy imaging\n",
      "LoFtool: a gene intolerance score based on loss-of-function variants in 60 706 individuals\n",
      "Assembly-based inference of B-cell receptor repertoires from short read RNA sequencing data with V'DJer\n",
      "AlmostSignificant: simplifying quality control of high-throughput sequencing data\n",
      "KScons: a Bayesian approach for protein residue contact prediction using the knob-socket model of protein tertiary structure\n",
      "Fast-SNP: a fast matrix pre-processing algorithm for efficient loopless flux optimization of metabolic models\n",
      "Predicting the errors of predicted local backbone angles and non-local solvent- accessibilities of proteins by deep neural networks\n",
      "What time is it? Deep learning approaches for circadian rhythms\n",
      "MSIdV: a versatile tool to visualize biological indices from mass spectrometry imaging data\n",
      "A new correlation clustering method for cancer mutation analysis\n",
      "BatchQC: interactive software for evaluating sample and batch effects in genomic data\n",
      "CSAM: Compressed SAM format\n",
      "ImmQuant: a user-friendly tool for inferring immune cell-type composition from gene-expression data\n",
      "DOMINO: development of informative molecular markers for phylogenetic and genome-wide population genetic studies in non-model organisms\n",
      "iRSpot-EL: identify recombination spots with an ensemble learning approach\n",
      "EnhancerAtlas: a resource for enhancer annotation and analysis in 105 human cell/tissue types\n",
      "PRODIGY: a web server for predicting the binding affinity of protein-protein complexes\n",
      "Rule-based modeling with Virtual Cell\n",
      "Metrics for rapid quality control in RNA structure probing experiments\n",
      "NRGC: a novel referential genome compression algorithm\n",
      "MetaCoMET: a web platform for discovery and visualization of the core microbiome\n",
      "Motif comparison based on similarity of binding affinity profiles\n",
      "plasmidSPAdes: assembling plasmids from whole genome sequencing data\n",
      "MorphoLibJ: integrated library and plugins for mathematical morphology with ImageJ\n",
      "IQRray, a new method for Affymetrix microarray quality control, and the homologous organ conservation score, a new benchmark method for quality control metrics\n",
      "Network-based pathway enrichment analysis with incomplete network information\n",
      "AutoSite: an automated approach for pseudo-ligands prediction-from ligand-binding sites identification to predicting key ligand atoms\n",
      "Structural distinctions of fast and slow bacterial luciferases revealed by phylogenetic analysis\n",
      "pSumo-CD: predicting sumoylation sites in proteins with covariance discriminant algorithm by incorporating sequence-coupled effects into general PseAAC\n",
      "ABAEnrichment: an R package to test for gene set expression enrichment in the adult and developing human brain\n",
      "Vecuum: identification and filtration of false somatic variants caused by recombinant vector contamination\n",
      "compendiumdb: an R package for retrieval and storage of functional genomics data\n",
      "Statistical modeling of isoform splicing dynamics from RNA-seq time series data\n",
      "PhenoScanner: a database of human genotype-phenotype associations\n",
      "ARGON: fast, whole-genome simulation of the discrete time Wright-fisher process\n",
      "Jumping across biomedical contexts using compressive data fusion\n",
      "A novel algorithm for calling mRNA m6A peaks by modeling biological variances in MeRIP-seq data\n",
      "ISMB 2016 Proceedings Papers Committee\n",
      "Linear effects models of signaling pathways from combinatorial perturbation data\n",
      "Data-driven mechanistic analysis method to reveal dynamically evolving regulatory networks\n",
      "Fast metabolite identification with Input Output Kernel Regression\n",
      "Simultaneous prediction of enzyme orthologs from chemical transformation patterns for de novo metabolic pathway reconstruction\n",
      "An integer programming framework for inferring disease complexes from network data\n",
      "Compacting de Bruijn graphs from sequencing data quickly and in low memory\n",
      "A novel method for discovering local spatial clusters of genomic regions with functional relationships from DNA contact maps\n",
      "Comparative analyses of population-scale phenomic data in electronic medical records reveal race-specific disease networks\n",
      "Unbiased probabilistic taxonomic classification for DNA barcoding\n",
      "Web-based network analysis and visualization using CellMaps\n",
      "RTFBSDB: an integrated framework for transcription factor binding site analysis\n",
      "Meshable: searching PubMed abstracts by utilizing MeSH and MeSH-derived topical terms\n",
      "UniCon3D: de novo protein structure prediction using united-residue conformational search via stepwise, probabilistic sampling\n",
      "COCACOLA: binning metagenomic contigs using sequence COmposition, read CoverAge, CO-alignment and paired-end read LinkAge\n",
      "Message from the ISCB: 2016 ISCB Accomplishment by a Senior Scientist Award Given to Søren Brunak\n",
      "Weighted mutual information analysis substantially improves domain-based functional network models\n",
      "TreesimJ: a flexible, forward time population genetic simulator\n",
      "PANTHER-PSEP: predicting disease-causing genetic variants using position-specific evolutionary preservation\n",
      "Computational discovery and in vivo validation of hnf4 as a regulatory gene in planarian regeneration\n",
      "A computational model to predict the immune system activation by citrus-derived vaccine adjuvants\n",
      "EuGene: maximizing synthetic gene design for heterologous expression\n",
      "RVTESTS: an efficient and comprehensive tool for rare variant association analysis using sequence data\n",
      "Efficient privacy-preserving string search and an application in genomics\n",
      "UniProt-DAAC: domain architecture alignment and classification, a new method for automatic functional annotation in UniProtKB\n",
      "IsomiR Bank: a research resource for tracking IsomiRs\n",
      "VariantBam: filtering and profiling of next-generational sequencing data using region-specific rules\n",
      "A mutation profile for top-k patient search exploiting Gene-Ontology and orthogonal non-negative matrix factorization\n",
      "Multiple structure single parameter: analysis of a single protein nano environment descriptor characterizing a shared loci on structurally aligned proteins\n",
      "Comment on 'Discovering hospital admission patterns using models learnt from electronic hospital records'. The importance of using the right codes\n",
      "KiT: a MATLAB package for kinetochore tracking\n",
      "Phasing for medical sequencing using rare variants and large haplotype reference panels\n",
      "SweepFinder2: increased sensitivity, robustness and flexibility\n",
      "dbDSM: a manually curated database for deleterious synonymous mutations\n",
      "ISCB's initial reaction to New England Journal of Medicine editorial on data sharing\n",
      "ll-ACHRB: a scalable algorithm for sampling the feasible solution space of metabolic networks\n",
      "SharpViSu: integrated analysis and segmentation of super-resolution microscopy data\n",
      "A simple method to control over-alignment in the MAFFT multiple sequence alignment program\n",
      "BALL-SNPgp-from genetic variants toward computational diagnostics\n",
      "solarius: an R interface to SOLAR for variance component analysis in pedigrees\n",
      "Isoform-level ribosome occupancy estimation guided by transcript abundance with Ribomap\n",
      "TreeDom: a graphical web tool for analysing domain architecture evolution\n",
      "DOGMA: domain-based transcriptome and proteome quality assessment\n",
      "Informed kmer selection for de novo transcriptome assembly\n",
      "ARACNe-AP: gene network reverse engineering through adaptive partitioning inference of mutual information\n",
      "Partitioning and correlating subgroup characteristics from Aligned Pattern Clusters\n",
      "TnT: a set of libraries for visualizing trees and track-based annotations for the web\n",
      "e23D: database and visualization of A-to-I RNA editing sites mapped to 3D protein structures\n",
      "SCell: integrated analysis of single-cell RNA-seq data\n",
      "flowAI: automatic and interactive anomaly discerning tools for flow cytometry data\n",
      "DyNet: visualization and analysis of dynamic molecular interaction networks\n",
      "iDHS-EL: identifying DNase I hypersensitive sites by fusing three different modes of pseudo nucleotide composition into an ensemble learning framework\n",
      "PREDAC-H3: a user-friendly platform for antigenic surveillance of human influenza a(H3N2) virus based on hemagglutinin sequences\n",
      "ProbFold: a probabilistic method for integration of probing data in RNA secondary structure prediction\n",
      "pileup.js: a JavaScript library for interactive and in-browser visualization of genomic data\n",
      "Canvas: versatile and scalable detection of copy number variants\n",
      "DUDes: a top-down taxonomic profiler for metagenomics\n",
      "chopBAI: BAM index reduction solves I/O bottlenecks in the joint analysis of large sequencing cohorts\n",
      "LS-GKM: a new gkm-SVM for large-scale datasets\n",
      "chromPlot: visualization of genomic data in chromosomal context\n",
      "ARGs-OAP: online analysis pipeline for antibiotic resistance genes detection from metagenomic data using an integrated structured ARG-database\n",
      "Alpha-CENTAURI: assessing novel centromeric repeat sequence variation with long read sequencing\n",
      "Beyond accuracy: creating interoperable and scalable text-mining web services\n",
      "Trinculo: Bayesian and frequentist multinomial logistic regression for genome-wide association studies of multi-category phenotypes\n",
      "TRONCO: an R package for the inference of cancer progression models from heterogeneous genomic data\n",
      "Characterization of clinical signs in the human interactome\n",
      "BCFtools/RoH: a hidden Markov model approach for detecting autozygosity from next-generation sequencing data\n",
      "On the discovery of hospital admission patterns-a clarification\n",
      "Differential methylation analysis for BS-seq data under general experimental design\n",
      "BISQUE: locus- and variant-specific conversion of genomic, transcriptomic and proteomic database identifiers\n",
      "MuCor: mutation aggregation and correlation\n",
      "ELASPIC web-server: proteome-wide structure-based prediction of mutation effects on protein stability and binding affinity\n",
      "ConceptMetab: exploring relationships among metabolite sets to identify links among biomedical concepts\n",
      "Vasohibins: new transglutaminase-like cysteine proteases possessing a non-canonical Cys-His-Ser catalytic triad\n",
      "TFBSTools: an R/bioconductor package for transcription factor binding site analysis\n",
      "EPI-peptide designer: a tool for designing peptide ligand libraries based on epitope-paratope interactions\n",
      "Collaborative analysis of multi-gigapixel imaging data using Cytomine\n",
      "Estimating the number and assignment of clock models in analyses of multigene datasets\n",
      "MMseqs software suite for fast and deep clustering and searching of large protein sequence sets\n",
      "OEFinder: a user interface to identify and visualize ordering effects in single-cell RNA-seq data\n",
      "SV-Bay: structural variant detection in cancer genomes using a Bayesian approach with correction for GC-content and read mappability\n",
      "ProQ2: estimation of model accuracy implemented in Rosetta\n",
      "ChromothripsisDB: a curated database of chromothripsis\n",
      "rCGH: a comprehensive array-based genomic profile platform for precision medicine\n",
      "Fast integration-based prediction bands for ordinary differential equation models\n",
      "traseR: an R package for performing trait-associated SNP enrichment analysis in genomic intervals\n",
      "OMPPM: online multiple palindrome pattern matching\n",
      "Integrating, summarizing and visualizing GWAS-hits and human diversity with DANCE (Disease-ANCEstry networks)\n",
      "DNAshapeR: an R/Bioconductor package for DNA shape prediction and feature encoding\n",
      "INSECT 2.0: a web-server for genome-wide cis-regulatory modules prediction\n",
      "Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications\n",
      "Improved topology prediction using the terminal hydrophobic helices rule\n",
      "IKAP: A heuristic framework for inference of kinase activities from Phosphoproteomics data\n",
      "LedPred: an R/bioconductor package to predict regulatory sequences using support vector machines\n",
      "Accurate continuous geographic assignment from low- to high-density SNP data\n",
      "MetaQUAST: evaluation of metagenome assemblies\n",
      "Identification of drug-target interaction from interactome network with 'guilt-by-association' principle and topology features\n",
      "CDSfold: an algorithm for designing a protein-coding sequence with the most stable secondary structure\n",
      "SomVarIUS: somatic variant identification from unpaired tissue samples\n",
      "CellTracker (not only) for dummies\n",
      "iPQF: a new peptide-to-protein summarization method using peptide spectra characteristics to improve protein quantification\n",
      "KMAD: knowledge-based multiple sequence alignment for intrinsically disordered proteins\n",
      "CHRONOS: a time-varying method for microRNA-mediated subpathway enrichment analysis\n",
      "PAPA: a flexible tool for identifying pleiotropic pathways using genome-wide association study summaries\n",
      "Accurate estimation of isoelectric point of protein and peptide based on amino acid sequences\n",
      "Metabolic network-guided binning of metagenomic sequence fragments\n",
      "Highly accurate sequence-based prediction of half-sphere exposures of amino acid residues in proteins\n",
      "Annotation of rule-based models with formal semantics to enable creation, analysis, reuse and visualization\n",
      "BioNetFit: a fitting tool compatible with BioNetGen, NFsim and distributed computing environments\n",
      "Specific identification and quantification of circular RNAs from sequencing data\n",
      "Flexible data integration and curation using a graph-based approach\n",
      "meRanTK: methylated RNA analysis ToolKit\n",
      "Fast Optimized Cluster Algorithm for Localizations (FOCAL): a spatial cluster analysis for super-resolved microscopy\n",
      "PHYLUCE is a software package for the analysis of conserved genomic loci\n",
      "Two-pass alignment improves novel splice junction quantification\n",
      "Bayesian inference with historical data-based informative priors improves detection of differentially expressed genes\n",
      "SeqSIMLA2_exact: simulate multiple disease sites in large pedigrees with given disease status for diseases with low prevalence\n",
      "Gapped sequence alignment using artificial neural networks: application to the MHC class I system\n",
      "openBIS ELN-LIMS: an open-source database for academic laboratories\n",
      "RVFam: an R package for rare variant association analysis with family data\n",
      "Understanding the fabric of protein crystals: computational classification of biological interfaces and crystal contacts\n",
      "PPIXpress: construction of condition-specific protein interaction networks based on transcript expression\n",
      "Graphical pan-genome analysis with compressed suffix trees and the Burrows-Wheeler transform\n",
      "MT-HESS: an efficient Bayesian approach for simultaneous association detection in OMICS datasets, with application to eQTL mapping in multiple tissues\n",
      "The minimum evolution problem is hard: a link between tree inference and graph clustering problems\n",
      "Scalable clustering algorithms for continuous environmental flow cytometry\n",
      "A multi-objective optimization approach accurately resolves protein domain architectures\n",
      "CSSSCL: a python package that uses combined sequence similarity scores for accurate taxonomic classification of long and short sequence reads\n",
      "I-PV: a CIRCOS module for interactive protein sequence visualization\n",
      "Rapid and enhanced remote homology detection by cascading hidden Markov model searches in sequence space\n",
      "CpGFilter: model-based CpG probe filtering with replicates for epigenome-wide association studies\n",
      "SIMToolbox: a MATLAB toolbox for structured illumination fluorescence microscopy\n",
      "Cell line name recognition in support of the identification of synthetic lethality in cancer from text\n",
      "Pseudoknots in RNA folding landscapes\n",
      "missMethyl: an R package for analyzing data from Illumina's HumanMethylation450 platform\n",
      "Cytoscape.js: a graph theory library for visualisation and analysis\n",
      "Identification of coupling DNA motif pairs on long-range chromatin interactions in human K562 cells\n",
      "Xtalk: a path-based approach for identifying crosstalk between signaling pathways\n",
      "Positive-unlabeled ensemble learning for kinase substrate prediction from dynamic phosphoproteomics data\n",
      "Exploring peptide/MHC detachment processes using hierarchical natural move Monte Carlo\n",
      "ScaffoldScaffolder: solving contig orientation via bidirected to directed graph reduction\n",
      "Component-wise gradient boosting and false discovery control in survival analysis with high-dimensional covariates\n",
      "CD30 cell graphs of Hodgkin lymphoma are not scale-free--an image analysis approach\n",
      "Inferred miRNA activity identifies miRNA-mediated regulatory networks underlying multiple cancers\n",
      "RNF: a general framework to evaluate NGS read mappers\n",
      "SHAPE directed RNA folding\n",
      "Improved large-scale prediction of growth inhibition patterns using the NCI60 cancer cell line panel\n",
      "metaModules identifies key functional subnetworks in microbiome-related disease\n",
      "Deciphering the associations between gene expression and copy number alteration using a sparse double Laplacian shrinkage approach\n",
      "Discovering hospital admission patterns using models learnt from electronic hospital records\n",
      "DIGGIT: a Bioconductor package to infer genetic variants driving cellular phenotypes\n",
      "Learning directed acyclic graphical structures with genetical genomics data\n",
      "Multi-netclust: an efficient tool for finding connected clusters in multi-parametric networks\n",
      "BigBWA: approaching the Burrows-Wheeler aligner to Big Data technologies\n",
      "RASER: reads aligner for SNPs and editing sites of RNA\n",
      "Utilizing de Bruijn graph of metagenome assembly for metatranscriptome analysis\n",
      "An interactive genome browser of association results from the UK10K cohorts project\n",
      "LIBRA: LIgand Binding site Recognition Application\n",
      "database.bio: a web application for interpreting human variations\n",
      "LncRNA-ID: Long non-coding RNA IDentification using balanced random forests\n",
      "SuccFind: a novel succinylation sites online prediction tool via enhanced characteristic strategy\n",
      "Vizardous: interactive analysis of microbial populations with single cell resolution\n",
      "cgmisc: enhanced genome-wide association analyses and visualization\n",
      "NRGsuite: a PyMOL plugin to perform docking simulations in real time using FlexAID\n",
      "Correcting systematic bias and instrument measurement drift with mzRefinery\n",
      "AlignBucket: a tool to speed up 'all-against-all' protein sequence alignments optimizing length constraints\n",
      "MEPSA: minimum energy pathway analysis for energy landscapes\n",
      "ClicO FS: an interactive web-based service of Circos\n",
      "Canonical, stable, general mapping using context schemes\n",
      "Statistically identifying tumor suppressors and oncogenes from pan-cancer genome-sequencing data\n",
      "Mapping networks of light-dark transition in LOV photoreceptors\n",
      "Spaced seeds improve k-mer-based metagenomic classification\n",
      "CRISPR-ERA: a comprehensive design tool for CRISPR-mediated gene editing, repression and activation\n",
      "Identifying kinase dependency in cancer cells by integrating high-throughput drug screening and kinase inhibition data\n",
      "The application of a novel 'rising activity, multi-level mixed effects, indicator emphasis' (RAMMIE) method for syndromic surveillance in England\n",
      "Identifying novel associations between small molecules and miRNAs based on integrated molecular networks\n",
      "WATCLUST: a tool for improving the design of drugs based on protein-water interactions\n",
      "Roary: rapid large-scale prokaryote pan genome analysis\n",
      "Karect: accurate correction of substitution, insertion and deletion errors for next-generation sequencing data\n",
      "Generation of sequence-based data for pedigree-segregating Mendelian or Complex traits\n",
      "SomaticSignatures: inferring mutational signatures from single-nucleotide variants\n",
      "Functional classification of CATH superfamilies: a domain-based approach for protein function annotation\n",
      "A DNA shape-based regulatory score improves position-weight matrix-based recognition of transcription factor binding sites\n",
      "Inference of Markovian properties of molecular sequences from NGS data and applications to comparative genomics\n",
      "ISQuest: finding insertion sequences in prokaryotic sequence fragment data\n",
      "Proteny: discovering and visualizing statistically significant syntenic clusters at the proteome level\n",
      "nextflu: real-time tracking of seasonal influenza virus evolution in humans\n",
      "Data Safe Havens in health research and healthcare\n",
      "MetaQuery: a web server for rapid annotation and quantitative analysis of specific genes in the human gut microbiome\n",
      "Chimira: analysis of small RNA sequencing data and microRNA modifications\n",
      "Fast-SL: an efficient algorithm to identify synthetic lethal sets in metabolic networks\n",
      "TPpred3 detects and discriminates mitochondrial and chloroplastic targeting peptides in eukaryotic proteins\n",
      "REDEMPTION: reduced dimension ensemble modeling and parameter estimation\n",
      "Inferring orthologous gene regulatory networks using interspecies data fusion\n",
      "Reconstructing gene regulatory dynamics from high-dimensional single-cell snapshot data\n",
      "ASTRAL-II: coalescent-based species tree estimation with many hundreds of taxa and thousands of genes\n",
      "Exploiting ontology graph for predicting sparsely annotated gene function\n",
      "In silico phenotyping via co-training for improved phenotype prediction from genotype\n",
      "Dynamic changes of RNA-sequencing expression for precision medicine: N-of-1-pathways Mahalanobis distance within pathways of single subjects predicts breast cancer survival\n",
      "Refined elasticity sampling for Monte Carlo-based identification of stabilizing network patterns\n",
      "Inferring parental genomic ancestries using pooled semi-Markov processes\n",
      "cNMA: a framework of encounter complex-based normal mode analysis to model conformational changes in protein interactions\n",
      "SoloDel: a probabilistic model for detecting low-frequent somatic deletions from unmatched sequencing data\n",
      "oposSOM: R-package for high-dimensional portraying of genome-wide expression landscapes on bioconductor\n",
      "UniAlign: protein structure alignment meets evolution\n",
      "FourCSeq: analysis of 4C sequencing data\n",
      "Differential protein expression and peak selection in mass spectrometry data by binary discriminant analysis\n",
      "antaRNA: ant colony-based RNA sequence design\n",
      "Epigenomic k-mer dictionaries: shedding light on how sequence composition influences in vivo nucleosome positioning\n",
      "IgSimulator: a versatile immunosequencing simulator\n",
      "Trans-species learning of cellular signaling systems with bimodal deep belief networks\n",
      "Repeat- and error-aware comparison of deletions\n",
      "CAPRI: efficient inference of cancer progression models from cross-sectional data\n",
      "ADME SARfari: comparative genomics of drug metabolizing systems\n",
      "PsyGeNET: a knowledge platform on psychiatric disorders and their genes\n",
      "MemGen: a general web server for the setup of lipid membrane simulation systems\n",
      "BinDNase: a discriminatory approach for transcription factor binding prediction using DNase I hypersensitivity data\n",
      "INSPEcT: a computational tool to infer mRNA synthesis, processing and degradation dynamics from RNA- and 4sU-seq time course experiments\n",
      "BFC: correcting Illumina sequencing errors\n",
      "NAIL, a software toolset for inferring, analyzing and visualizing regulatory networks\n",
      "Phylesystem: a git-based data store for community-curated phylogenetic estimates\n",
      "PopIns: population-scale detection of novel sequence insertions\n",
      "Genome-scale strain designs based on regulatory minimal cut sets\n",
      "CiVi: circular genome visualization with unique features to analyze sequence elements\n",
      "Addressing false discoveries in network inference\n",
      "ms-data-core-api: an open-source, metadata-oriented library for computational proteomics\n",
      "edgeRun: an R package for sensitive, functionally relevant differential expression discovery using an unconditional exact test\n",
      "PDIviz: analysis and visualization of protein-DNA binding interfaces\n",
      "StructureFold: genome-wide RNA secondary structure mapping and reconstruction in vivo\n",
      "FastMotif: spectral sequence motif discovery\n",
      "Accurate prediction of RNA nucleotide interactions with backbone k-tree model\n",
      "MetaSV: an accurate and integrative structural-variant caller for next generation sequencing\n",
      "GeneTIER: prioritization of candidate disease genes using tissue-specific gene expression profiles\n",
      "Stratifying tumour subtypes based on copy number alteration profiles using next-generation sequence data\n",
      "MetaMapR: pathway independent metabolomic network analysis incorporating unknowns\n",
      "DREAM: a webserver for the identification of editing sites in mature miRNAs using deep sequencing data\n",
      "SPARSE: quadratic time simultaneous alignment and folding of RNAs without sequence-based heuristics\n",
      "Using neighborhood cohesiveness to infer interactions between protein domains\n",
      "Power and sample-size estimation for microbiome studies using pairwise distances and PERMANOVA\n",
      "bio-samtools 2: a package for analysis and visualization of sequence and alignment data with SAMtools in Ruby\n",
      "KeBABS: an R package for kernel-based analysis of biological sequences\n",
      "RiboTools: a Galaxy toolbox for qualitative ribosome profiling analysis\n",
      "Proper evaluation of alignment-free network comparison methods\n",
      "DISSCO: direct imputation of summary statistics allowing covariates\n",
      "Automatic determination of NET (neutrophil extracellular traps) coverage in fluorescent microscopy images\n",
      "MAGNA++: Maximizing Accuracy in Global Network Alignment via both node and edge conservation\n",
      "iTagPlot: an accurate computation and interactive drawing tool for tag density plot\n",
      "SOMP: web server for in silico prediction of sites of metabolism for drug-like compounds\n",
      "Cardinal: an R package for statistical analysis of mass spectrometry-based imaging experiments\n",
      "Computational approaches towards understanding human long non-coding RNA biology\n",
      "LBIBCell: a cell-based simulation environment for morphogenetic problems\n",
      "ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization\n",
      "MUGBAS: a species free gene-based programme suite for post-GWAS analysis\n",
      "MIRA: mutual information-based reporter algorithm for metabolic networks\n",
      "IVA: accurate de novo assembly of RNA virus genomes\n",
      "An event-driven approach for studying gene block evolution in bacteria\n",
      "Guidance for RNA-seq co-expression network construction and analysis: safety in numbers\n",
      "GlycanAnalysis Plug-in: a database search tool for N-glycan structures using mass spectrometry\n",
      "tEFMA: computing thermodynamically feasible elementary flux modes in metabolic networks\n",
      "Population-scale three-dimensional reconstruction and quantitative profiling of microglia arbors\n",
      "AIDA: ab initio domain assembly for automated multi-domain protein structure prediction and domain-domain interaction prediction\n",
      "TarPred: a web application for predicting therapeutic and side effect targets of chemical compounds\n",
      "A cautionary note on using binary calls for analysis of DNA methylation\n",
      "A novel statistical method for quantitative comparison of multiple ChIP-seq datasets\n",
      "A trimming-and-retrieving alignment scheme for reduced representation bisulfite sequencing\n",
      "MOST: a software environment for constraint-based metabolic modeling and strain design\n",
      "BioTextQuest+: a knowledge integration platform for literature mining and concept discovery\n",
      "Application of clinical text data for phenome-wide association studies (PheWASs)\n",
      "mFASD: a structure-based algorithm for discriminating different types of metal-binding sites\n",
      "Expitope: a web server for epitope expression\n",
      "CNOGpro: detection and quantification of CNVs in prokaryotic whole-genome sequencing data\n",
      "Starcode: sequence clustering based on all-pairs search\n",
      "Factor graph analysis of live cell-imaging data reveals mechanisms of cell fate decisions\n",
      "KAPPA, a simple algorithm for discovery and clustering of proteins defined by a key amino acid pattern: a case study of the cysteine-rich proteins\n",
      "GenoExp: a web tool for predicting gene expression levels from single nucleotide polymorphisms\n",
      "RAMPART: a workflow management system for de novo genome assembly\n",
      "MetAmp: combining amplicon data from multiple markers for OTU analysis\n",
      "Bayesian feature selection for high-dimensional linear regression via the Ising approximation with applications to genomics\n",
      "ENVIRONMENTS and EOL: identification of Environment Ontology terms in text and the annotation of the Encyclopedia of Life\n",
      "Prediction of potential disease-associated microRNAs based on random walk\n",
      "bbcontacts: prediction of β-strand pairing from direct coupling patterns\n",
      "Plant photosynthesis phenomics data quality control\n",
      "dslice: an R package for nonparametric testing of associations with application in QTL and gene set analysis\n",
      "Ultrafast SNP analysis using the Burrows-Wheeler transform of short-read data\n",
      "WALTZ-DB: a benchmark database of amyloidogenic hexapeptides\n",
      "Functional Gene Networks: R/Bioc package to generate and analyse gene networks derived from functional enrichment and clustering\n",
      "scrm: efficiently simulating long sequences using the approximated coalescent with recombination\n",
      "Repulsive parallel MCMC algorithm for discovering diverse motifs from large sequence sets\n",
      "An integrative approach to predicting the functional effects of non-coding and coding sequence variation\n",
      "Shifted Hamming distance: a fast and accurate SIMD-friendly filter to accelerate alignment verification in read mapping\n",
      "MADGiC: a model-based approach for identifying driver genes in cancer\n",
      "RNA-Rocket: an RNA-Seq analysis resource for infectious disease research\n",
      "LINKPHASE3: an improved pedigree-based phasing algorithm robust to genotyping and map errors\n",
      "A haplotype-based framework for group-wise transmission/disequilibrium tests for rare variant association analysis\n",
      "The chemical component dictionary: complete descriptions of constituent molecules in experimentally determined 3D macromolecules in the Protein Data Bank\n",
      "CONSRANK: a server for the analysis, comparison and ranking of docking models based on inter-residue contacts\n",
      "Population-based structural variation discovery with Hydra-Multi\n",
      "MicroRNA modules prefer to bind weak and unconventional target sites\n",
      "Empowering biologists with multi-omics data: colorectal cancer as a paradigm\n",
      "VarSim: a high-fidelity simulation and validation framework for high-throughput genome sequencing with cancer applications\n",
      "diXa: a data infrastructure for chemical safety assessment\n",
      "NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy\n",
      "Identifying cancer-related microRNAs based on gene expression data\n",
      "GSDS 2.0: an upgraded gene feature visualization server\n",
      "repDNA: a Python package to generate various modes of feature vectors for DNA sequences by incorporating user-defined physicochemical properties and sequence-order effects\n",
      "Clustering-based model of cysteine co-evolution improves disulfide bond connectivity prediction and reduces homologous sequence requirements\n",
      "Achievements and challenges in structural bioinformatics and computational biophysics\n",
      "flowCL: ontology-based cell population labelling in flow cytometry\n",
      "Improved gene tree error correction in the presence of horizontal gene transfer\n",
      "PEASE: predicting B-cell epitopes utilizing antibody sequence\n",
      "Tissue-aware data integration approach for the inference of pathway interactions in metazoan organisms\n",
      "Promiscuous mRNA splicing under the control of AIRE in medullary thymic epithelial cells\n",
      "The Victor C++ library for protein representation and advanced manipulation\n",
      "Bioimaging-based detection of mislocalized proteins in human cancers by semi-supervised learning\n",
      "CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites\n",
      "HyDRA: gene prioritization via hybrid distance-score rank aggregation\n",
      "jNMFMA: a joint non-negative matrix factorization meta-analysis of transcriptomics data\n",
      "XenoSite server: a web-available site of metabolism prediction tool\n",
      "EPGA: de novo assembly using the distributions of reads and insert size\n",
      "M3D: a kernel-based test for spatially correlated changes in methylation profiles\n",
      "Rclick: a web server for comparison of RNA 3D structures\n",
      "MUSCLE: automated multi-objective evolutionary optimization of targeted LC-MS/MS analysis\n",
      "CytoCom: a Cytoscape app to visualize, query and analyse disease comorbidity networks\n",
      "IntSide: a web server for the chemical and biological examination of drug side effects\n",
      "TreeEFM: calculating elementary flux modes using linear optimization in a tree-based algorithm\n",
      "flowDensity: reproducing manual gating of flow cytometry data by automated density-based cell population identification\n",
      "Measuring the wisdom of the crowds in network-based gene function inference\n",
      "Automated structural classification of lipids by machine learning\n",
      "Automatic prediction of polysaccharide utilization loci in Bacteroidetes species\n",
      "WebPSN: a web server for high-throughput investigation of structural communication in biomacromolecules\n",
      "MetMSLine: an automated and fully integrated pipeline for rapid processing of high-resolution LC-MS metabolomic datasets\n",
      "DIANA--algorithmic improvements for analysis of data-independent acquisition MS data\n",
      "MP-GeneticSynth: inferring biological network regulations from time series\n",
      "minimac2: faster genotype imputation\n",
      "MACOED: a multi-objective ant colony optimization algorithm for SNP epistasis detection in genome-wide association studies\n",
      "GeneNet Toolbox for MATLAB: a flexible platform for the analysis of gene connectivity in biological networks\n",
      "Novel function discovery with GeneMANIA: a new integrated resource for gene function prediction in Escherichia coli\n",
      "Log-odds sequence logos\n",
      "A crowd-sourcing approach for the construction of species-specific cell signaling networks\n",
      "Computing autocatalytic sets to unravel inconsistencies in metabolic network reconstructions\n",
      "SIST: stress-induced structural transitions in superhelical DNA\n",
      "MulRF: a software package for phylogenetic analysis using multi-copy gene trees\n",
      "PROPER: comprehensive power evaluation for differential expression using RNA-seq\n",
      "The RNA shapes studio\n",
      "Sigma: strain-level inference of genomes from metagenomic analysis for biosurveillance\n",
      "METAINTER: meta-analysis of multiple regression models in genome-wide association studies\n",
      "PIP-DB: the Protein Isoelectric Point database\n",
      "Translating bioinformatics in oncology: guilt-by-profiling analysis and identification of KIF18B and CDCA3 as novel driver genes in carcinogenesis\n",
      "Improved rat genome gene prediction by integration of ESTs with RNA-Seq information\n",
      "Snowball: resampling combined with distance-based regression to discover transcriptional consequences of a driver mutation\n",
      "SignalSpider: probabilistic pattern discovery on multiple normalized ChIP-Seq signal profiles\n",
      "The GOBLET training portal: a global repository of bioinformatics training materials, courses and trainers\n",
      "Over-representation of correlation analysis (ORCA): a method for identifying associations between variable sets\n",
      "Prediction of Indel flanking regions in protein sequences using a variable-order Markov model\n",
      "GenPlay Multi-Genome, a tool to compare and analyze multiple human genomes in a graphical interface\n",
      "poRe: an R package for the visualization and analysis of nanopore sequencing data\n",
      "Inter-species pathway perturbation prediction via data-driven detection of functional homology\n",
      "RETRACTED: Analysis and prediction of β-turn types using multinomial logistic regression and artificial neural network\n",
      "Virxicon: A Lexicon Of Viral Sequences\n",
      "Prediction Of Histone Post-Translational Modifications Using Deep Learning\n",
      "The ortholog conjecture revisited: the value of orthologs and paralogs in function prediction\n",
      "Testing hypotheses about the microbiome using the linear decomposition model (LDM)\n",
      "Block HSIC Lasso: model-free biomarker detection for ultra-high dimensional data\n",
      "CROSSMAPPER: estimating cross-mapping rates and optimizing experimental design in multi-species sequencing studies\n",
      "RTNduals: an R/Bioconductor package for analysis of co-regulation and inference of dual regulons\n",
      "RERconverge: an R package for associating evolutionary rates with convergent traits\n",
      "DeepSymmetry: using 3D convolutional networks for identification of tandem repeats and internal symmetries in protein structures\n",
      "Genome-wide identification of the essential protein-coding genes and long non-coding RNAs for human pan-cancer\n",
      "MetWork: a web server for natural products anticipation\n",
      "Integrating hypertension phenotype and genotype with hybrid non-negative matrix factorization\n",
      "MR4Cancer: a web server prioritizing master regulators for cancer\n",
      "On the power of epigenome-wide association studies using a disease-discordant twin design\n",
      "TomoEED: fast edge-enhancing denoising of tomographic volumes\n",
      "Phantom: investigating heterogeneous gene sets in time-course data\n",
      "cBinderDB: a covalent binding agent database\n",
      "OfftargetFinder: a web tool for species-specific RNAi design\n",
      "FluxModeCalculator: an efficient tool for large-scale flux mode computation\n",
      "SpotCaliper: fast wavelet-based spot detection with accurate size estimation\n",
      "MichelaNglo: sculpting protein views on web pages without coding\n",
      "scBatch: batch-effect correction of RNA-seq data through sample distance matrix adjustment\n",
      "ColocML: machine learning quantifies co-localization between mass spectrometry images\n",
      "QMEANDisCo-distance constraints applied on model quality estimation\n",
      "TeSS: a platform for discovering life-science training opportunities\n",
      "DeepKinZero: zero-shot learning for predicting kinase-phosphosite associations involving understudied kinases\n",
      "Estimating the power of sequence covariation for detecting conserved RNA structure\n",
      "Census of halide-binding sites in protein structures\n",
      "FTIP: an accurate and efficient method for global protein surface comparison\n",
      "A Bayesian approach to accurate and robust signature detection on LINCS L1000 data\n",
      "MODER2: first-order Markov modeling and discovery of monomeric and dimeric binding motifs\n",
      "G4Killer web application: a tool to design G-quadruplex mutations\n",
      "UMI-VarCal: a new UMI-based variant caller that efficiently improves low-frequency variant detection in paired-end sequencing NGS libraries\n",
      "On the bias of H-scores for comparing biclusters, and how to correct it\n",
      "Network-based prediction of drug-target interactions using an arbitrary-order proximity embedded deep forest\n",
      "Joint reconstruction of multiple gene networks by simultaneously capturing inter-tumor and intra-tumor heterogeneity\n",
      "sismonr: simulation of in silico multi-omic networks with adjustable ploidy and post-transcriptional regulation in R\n",
      "Inference of gene regulatory networks based on nonlinear ordinary differential equations\n",
      "BAlaS: fast, interactive and accessible computational alanine-scanning using BudeAlaScan\n",
      "Adaptive boosting-based computational model for predicting potential miRNA-disease associations\n",
      "CROP: correlation-based reduction of feature multiplicities in untargeted metabolomic data\n",
      "FilTar: using RNA-Seq data to improve microRNA target prediction accuracy in animals\n",
      "RiboFlow, RiboR and RiboPy: an ecosystem for analyzing ribosome profiling data at read length resolution\n",
      "The open targets post-GWAS analysis pipeline\n",
      "ABEMUS: platform-specific and data-informed detection of somatic SNVs in cfDNA\n",
      "InterPep2: global peptide-protein docking using interaction surface templates\n",
      "Gapsplit: efficient random sampling for non-convex constraint-based models\n",
      "ChemBioServer 2.0: an advanced web server for filtering, clustering and networking of chemical compounds facilitating both drug discovery and repurposing\n",
      "BiORSEO: a bi-objective method to predict RNA secondary structures with pseudoknots using RNA 3D modules\n",
      "BEM: Mining Coregulation Patterns in Transcriptomics via Boolean Matrix Factorization\n",
      "A U-statistics for integrative analysis of multilayer omics data\n",
      "A novel method SEProm for prokaryotic promoter prediction based on DNA structure and energetics\n",
      "BCdatabaser: on-the-fly reference database creation for (meta-)barcoding\n",
      "LiPLike: towards gene regulatory network predictions of high certainty\n",
      "SMART: SuperMaximal approximate repeats tool\n",
      "Multi-omics reveals functional genomic and metabolic mechanisms of milk production and quality in dairy cows\n",
      "A haplotype-aware de novo assembly of related individuals using pedigree sequence graph\n",
      "LOBSTER: an environment to design bioimage analysis workflows for large and complex fluorescence microscopy data\n",
      "LPM: a latent probit model to characterize the relationship among complex traits using summary statistics from multiple GWASs and functional annotations\n",
      "SAMMI: a semi-automated tool for the visualization of metabolic networks\n",
      "scTIM: seeking cell-type-indicative marker from single cell RNA-seq data by consensus optimization\n",
      "GlyMDB: Glycan Microarray Database and analysis toolset\n",
      "ngsReports: a Bioconductor package for managing FastQC reports and other NGS related log files\n",
      "ClinVAP: a reporting strategy from variants to therapeutic options\n",
      "SSIPe: accurately estimating protein-protein binding affinity change upon mutations using evolutionary profiles in combination with an optimized physical energy function\n",
      "GABAC: an arithmetic coding solution for genomic data\n",
      "Logomaker: beautiful sequence logos in Python\n",
      "Formal axioms in biomedical ontologies improve analysis and interpretation of associated data\n",
      "SpaCell: integrating tissue morphology and spatial gene expression to predict disease cells\n",
      "pydca v1.0: a comprehensive software for direct coupling analysis of RNA and protein sequences\n",
      "Antibody complementarity determining region design using high-capacity machine learning\n",
      "RNASeq_similarity_matrix: visually identify sample mix-ups in RNASeq data using a 'genomic' sequence similarity matrix\n",
      "Detection of differentially methylated CpG sites between tumor samples with uneven tumor purities\n",
      "ADDO: a comprehensive toolkit to detect, classify and visualize additive and non-additive quantitative trait loci\n",
      "Cerebro: interactive visualization of scRNA-seq data\n",
      "CRISPRitz: rapid, high-throughput and variant-aware in silico off-target site identification for CRISPR genome editing\n",
      "Accurate detection of short and long active ORFs using Ribo-seq data\n",
      "Prediction of mutation effects using a deep temporal convolutional network\n",
      "Discovery of disease- and drug-specific pathways through community structures of a literature network\n",
      "Learning complex subcellular distribution patterns of proteins via analysis of immunohistochemistry images\n",
      "The Cytoscape BioGateway App: explorative network building from the BioGateway triple store\n",
      "scGEAToolbox: a Matlab toolbox for single-cell RNA sequencing data analysis\n",
      "MDiNE: a model to estimate differential co-occurrence networks in microbiome studies\n",
      "QMEANDisCo-distance constraints applied on model quality estimation\n",
      "Improved dropClust R package with integrative analysis support for scRNA-seq data\n",
      "Network-based multi-task learning models for biomarker selection and cancer outcome prediction\n",
      "YY1 is a cis-regulator in the organoid models of high mammographic density\n",
      "CASPR, an analysis pipeline for single and paired guide RNA CRISPR screens, reveals optimal target selection for long non-coding RNAs\n",
      "mCSM-AB2: guiding rational antibody design using graph-based signatures\n",
      "Set cover-based methods for motif selection\n",
      "bWGR: Bayesian Whole-Genome Regression\n",
      "Deep annotation of untargeted LC-MS metabolomics data with Binner\n",
      "Topological data analysis quantifies biological nano-structure from single molecule localization microscopy\n",
      "CSMD: a computational subtraction-based microbiome discovery pipeline for species-level characterization of clinical metagenomic samples\n",
      "Patch-DCA: improved protein interface prediction by utilizing structural information and clustering DCA scores\n",
      "Dynalogo: an interactive sequence logo with dynamic thresholding of matched quantitative proteomic data\n",
      "BioSwitch: a tool for the detection of bistability and multi-steady state behaviour in signalling and gene regulatory networks\n",
      "GPseudoClust: deconvolution of shared pseudo-profiles at single-cell resolution\n",
      "Studying 3D genome evolution using genomic sequence\n",
      "AggreRATE-Pred: a mathematical model for the prediction of change in aggregation rate upon point mutation\n",
      "BiomeNet: a database for construction and analysis of functional interaction networks for any species with a sequenced genome\n",
      "SPARSim single cell: a count data simulator for scRNA-seq data\n",
      "SynQuant: an automatic tool to quantify synapses from microscopy images\n",
      "NITUMID: Nonnegative matrix factorization-based Immune-TUmor MIcroenvironment Deconvolution\n",
      "Novel algorithms for efficient subsequence searching and mapping in nanopore raw signals towards targeted sequencing\n",
      "iATC-NRAKEL: an efficient multi-label classifier for recognizing anatomical therapeutic chemical classes of drugs\n",
      "Bayesian weighted Mendelian randomization for causal inference based on summary statistics\n",
      "Soft windowing application to improve analysis of high-throughput phenotyping data\n",
      "VISOR: a versatile haplotype-aware structural variant simulator for short- and long-read sequencing\n",
      "γ-TRIS: a graph-algorithm for comprehensive identification of vector genomic insertion sites\n",
      "Topology-based classification of tetrads and quadruplex structures\n",
      "L1EM: a tool for accurate locus specific LINE-1 RNA quantification\n",
      "Thermodynamically consistent estimation of Gibbs free energy from data: data reconciliation approach\n",
      "Towards next-generation diagnostics for tuberculosis: identification of novel molecular targets by large-scale comparative genomics\n",
      "T-lex3: an accurate tool to genotype and estimate population frequencies of transposable elements using the latest short-read whole genome sequencing data\n",
      "MitoTrack, a user-friendly semi-automatic software for lineage tracking in living embryos\n",
      "scDAPA: detection and visualization of dynamic alternative polyadenylation from single cell RNA-seq data\n",
      "MUFold-SSW: a new web server for predicting protein secondary structures, torsion angles and turns\n",
      "AssessORF: combining evolutionary conservation and proteomics to assess prokaryotic gene predictions\n",
      "PPNID: a reference database and molecular identification pipeline for plant-parasitic nematodes\n",
      "Weighted elastic net for unsupervised domain adaptation with application to age prediction from DNA methylation data\n",
      "TADA: phylogenetic augmentation of microbiome samples enhances phenotype classification\n",
      "DIFFUSE: predicting isoform functions from sequences and expression profiles via deep learning\n",
      "Prediction of mRNA subcellular localization using deep recurrent neural networks\n",
      "Precise modelling and interpretation of bioactivities of ligands targeting G protein-coupled receptors\n",
      "Enhancing the drug discovery process: Bayesian inference for the analysis and comparison of dose-response experiments\n",
      "Efficient merging of genome profile alignments\n",
      "Large scale microbiome profiling in the cloud\n",
      "SCRIBER: accurate and partner type-specific prediction of protein-binding residues from proteins sequences\n",
      "Learning signaling networks from combinatorial perturbations by exploiting siRNA off-target effects\n",
      "Unsupervised segmentation of mass spectrometric ion images characterizes morphology of tissues\n",
      "LinearFold: linear-time approximate RNA folding by 5'-to-3' dynamic programming and beam search\n",
      "Large-scale mammalian genome rearrangements coincide with chromatin interactions\n",
      "GkmExplain: fast and accurate interpretation of nonlinear gapped k-mer SVMs\n",
      "PRECISE: a domain adaptation approach to transfer predictors of drug response from pre-clinical models to tumors\n",
      "Identifying molecular recognition features in intrinsically disordered regions of proteins by transfer learning\n",
      "DDAP: docking domain affinity and biosynthetic pathway prediction tool for type I polyketide synthases\n",
      "MeLAD: an integrated resource for metalloenzyme-ligand associations\n",
      "Phylogenetic tree-based microbiome association test\n",
      "ChaperISM: improved chaperone binding prediction using position-independent scoring matrices\n",
      "A powerful and flexible weighted distance-based method incorporating interactions between DNA methylation and environmental factors on health outcomes\n",
      "pCRM1exportome: database of predicted CRM1-dependent Nuclear Export Signal (NES) motifs in cancer-related genes\n",
      "Regulatory annotation of genomic intervals based on tissue-specific expression QTLs\n",
      "MOOMIN - Mathematical explOration of 'Omics data on a MetabolIc Network\n",
      "hypeR: an R package for geneset enrichment workflows\n",
      "neoepiscope improves neoepitope prediction with multivariant phasing\n",
      "StrainHub: a phylogenetic tool to construct pathogen transmission networks\n",
      "MultiGuideScan: a multi-processing tool for designing CRISPR guide RNA libraries\n",
      "Topology-independent and global protein structure alignment through an FFT-based algorithm\n",
      "RAG-Web: RNA structure prediction/design using RNA-As-Graphs\n",
      "Sequence tube maps: making graph genomes intuitive to commuters\n",
      "ATEN: And/Or tree ensemble for inferring accurate Boolean network topology and dynamics\n",
      "DeepGOPlus: improved protein function prediction from sequence\n",
      "THETA: a new genotypic approach for predicting HIV-1 CRF02-AG coreceptor usage\n",
      "ISoLDE: a data-driven statistical method for the inference of allelic imbalance in datasets with reciprocal crosses\n",
      "TFEA.ChIP: a tool kit for transcription factor binding site enrichment analysis capitalizing on ChIP-seq datasets\n",
      "PHANOTATE: a novel approach to gene identification in phage genomes\n",
      "Estimating and testing the microbial causal mediation effect with high-dimensional and compositional microbiome data\n",
      "diSTruct v1.0: generating biomolecular structures from distance constraints\n",
      "The influence of heterogeneous codon frequencies along sequences on the estimation of molecular adaptation\n",
      "DDT - Drug Discovery Tool: a fast and intuitive graphics user interface for docking and molecular dynamics analysis\n",
      "Identification of expression signatures for non-small-cell lung carcinoma subtype classification\n",
      "MEpurity: estimating tumor purity using DNA methylation data\n",
      "circDeep: deep learning approach for circular RNA classification from other long non-coding RNA\n",
      "ValTrendsDB: bringing Protein Data Bank validation information closer to the user\n",
      "Pygenprop: a Python library for programmatic exploration and comparison of organism genome properties\n",
      "Increasing the efficiency and accuracy of the ABACUS protein sequence design method\n",
      "Interactive exploration of heterogeneous biological networks with Biomine Explorer\n",
      "Genetic cooperativity in multi-layer networks implicates cell survival and senescence in the striatum of Huntington's disease mice synchronous to symptoms\n",
      "GTShark: genotype compression in large projects\n",
      "sefOri: selecting the best-engineered sequence features to predict DNA replication origins\n",
      "iCn3D, a web-based 3D viewer for sharing 1D/2D/3D representations of biomolecular structures\n",
      "MRLR: unraveling high-resolution meiotic recombination by linked reads\n",
      "Positive multistate protein design\n",
      "CoCoScore: context-aware co-occurrence scoring for text mining applications using distant supervision\n",
      "GeneNoteBook, a collaborative notebook for comparative genomics\n",
      "MetOSite: an integrated resource for the study of methionine residues sulfoxidation\n",
      "A scalable method for parameter identification in kinetic models of metabolism using steady-state data\n",
      "Chicdiff: a computational pipeline for detecting differential chromosomal interactions in Capture Hi-C data\n",
      "SCANVIS: a tool for SCoring, ANnotating and VISualizing splice junctions\n",
      "DNN-Dom: predicting protein domain boundary from sequence alone by deep neural network\n",
      "vulcanSpot: a tool to prioritize therapeutic vulnerabilities in cancer\n",
      "RAbHIT: R Antibody Haplotype Inference Tool\n",
      "RBPSponge: genome-wide identification of lncRNAs that sponge RBPs\n",
      "deconvSeq: deconvolution of cell mixture distribution in sequencing data\n",
      "Assessing concordance among human, in silico predictions and functional assays on genetic variant classification\n",
      "PSI-Sigma: a comprehensive splicing-detection method for short-read and long-read RNA-seq analysis\n",
      "EROS-DOCK: protein-protein docking using exhaustive branch-and-bound rotational search\n",
      "EnImpute: imputing dropout events in single-cell RNA-sequencing data via ensemble learning\n",
      "AlphaFold at CASP13\n",
      "Estimating the strength of expression conservation from high throughput RNA-seq data\n",
      "epiTAD: a web application for visualizing chromosome conformation capture data in the context of genetic epidemiology\n",
      "NGSEP3: accurate variant calling across species and sequencing protocols\n",
      "Incorporating spatial-anatomical similarity into the VGWAS framework for AD biomarker detection\n",
      "A coordinate descent approach for sparse Bayesian learning in high dimensional QTL mapping and genome-wide association studies\n",
      "Another look at matrix correlations\n",
      "Accurate peptide fragmentation predictions allow data driven approaches to replace and improve upon proteomics search engine scoring functions\n",
      "A Bayesian two-way latent structure model for genomic data integration reveals few pan-genomic cluster subtypes in a breast cancer cohort\n",
      "iRNAD: a computational tool for identifying D modification sites in RNA sequence\n",
      "Exact hypothesis testing for shrinkage-based Gaussian graphical models\n",
      "The barcode, UMI, set format and BUStools\n",
      "G-OnRamp: a Galaxy-based platform for collaborative annotation of eukaryotic genomes\n",
      "Identification of disease-associated loci using machine learning for genotype and network data integration\n",
      "HiCNN: a very deep convolutional neural network to better enhance the resolution of Hi-C data\n",
      "High precision protein functional site detection using 3D convolutional neural networks\n",
      "kmcEx: memory-frugal and retrieval-efficient encoding of counted k-mers\n",
      "Adaptive boosting-based computational model for predicting potential miRNA-disease associations\n",
      "Cell-level somatic mutation detection from single-cell RNA sequencing\n",
      "The EntOptLayout Cytoscape plug-in for the efficient visualization of major protein complexes in protein-protein interaction and signalling networks\n",
      "FuncTree2: an interactive radial tree for functional hierarchies and omics data visualization\n",
      "ORCA: a comprehensive bioinformatics container environment for education and research\n",
      "A framework for space-efficient variable-order Markov models\n",
      "SurVIndel: improving CNV calling from high-throughput sequencing data through statistical testing\n",
      "Modelling G×E with historical weather information improves genomic prediction in new environments\n",
      "A learning-based framework for miRNA-disease association identification using neural networks\n",
      "AbPredict 2: a server for accurate and unstrained structure prediction of antibody variable domains\n",
      "Nano-GLADIATOR: real-time detection of copy number alterations from nanopore sequencing data\n",
      "HLA*LA-HLA typing from linearly projected graph alignments\n",
      "Open Agile text mining for bioinformatics: the PubAnnotation ecosystem\n",
      "Discovery of tandem and interspersed segmental duplications using high-throughput sequencing\n",
      "Benefits of dimension reduction in penalized regression methods for high-dimensional grouped data: a case study in low sample size\n",
      "RTNsurvival: an R/Bioconductor package for regulatory network survival analysis\n",
      "Pinetree: a step-wise gene expression simulator with codon-specific translation rates\n",
      "Determining parameters for non-linear models of multi-loop free energy change\n",
      "Shouji: a fast and efficient pre-alignment filter for sequence alignment\n",
      "CliqueMS: a computational tool for annotating in-source metabolite ions from LC-MS untargeted metabolomics data based on a coelution similarity network\n",
      "PedigreeNet: a web-based pedigree viewer for biological databases\n",
      "Differential proteostatic regulation of insoluble and abundant proteins\n",
      "CytoBackBone: an algorithm for merging of phenotypic information from different cytometric profiles\n",
      "SArKS: de novo discovery of gene expression regulatory motif sites and domains by suffix array kernel smoothing\n",
      "ERVcaller: identifying polymorphic endogenous retrovirus and other transposable element insertions using whole-genome sequencing data\n",
      "GladiaTOX: GLobal Assessment of Dose-IndicAtor in TOXicology\n",
      "Lemon: a framework for rapidly mining structural information from the Protein Data Bank\n",
      "CABS-dock standalone: a toolbox for flexible protein-peptide docking\n",
      "Stable H3K4me3 is associated with transcription initiation during early embryo development\n",
      "Gene-based association tests using GWAS summary statistics\n",
      "GREP: genome for REPositioning drugs\n",
      "Choice of species affects phylogenetic stability of deep nodes: an empirical example in Terrabacteria\n",
      "VIMCO: variational inference for multiple correlated outcomes in genome-wide association studies\n",
      "Dr.VAE: improving drug response prediction via modeling of drug perturbation effects\n",
      "PyFeat: a Python-based effective feature generation tool for DNA, RNA and protein sequences\n",
      "G2P: a Genome-Wide-Association-Study simulation tool for genotype simulation, phenotype simulation and power evaluation\n",
      "metagenomeFeatures: an R package for working with 16S rRNA reference databases and marker-gene survey feature data\n",
      "Benchmarking optimization methods for parameter estimation in large kinetic models\n",
      "C3D: a tool to predict 3D genomic interactions between cis-regulatory elements\n",
      "Nucleotide Archival Format (NAF) enables efficient lossless reference-free compression of DNA sequences\n",
      "Easybiotics: a GUI for 3D physical modelling of multi-species bacterial populations\n",
      "The TMCrys server for supporting crystallization of transmembrane proteins\n",
      "Inter-Modular Linkers play a crucial role in governing the biosynthesis of non-ribosomal peptides\n",
      "A framework and an algorithm to detect low-abundance DNA by a handy sequencer and a palm-sized computer\n",
      "BioKEEN: a library for learning and evaluating biological knowledge graph embeddings\n",
      "DeepAffinity: interpretable deep learning of compound-protein affinity through unified recurrent and convolutional neural networks\n",
      "Identification of caveolin-1 domain signatures via machine learning and graphlet analysis of single-molecule super-resolution data\n",
      "Gag-protease coevolution shapes the outcome of lopinavir-inclusive treatment regimens in chronically infected HIV-1 subtype C patients\n",
      "RiboD: a comprehensive database for prokaryotic riboswitches\n",
      "IGLOSS: iterative gapless local similarity search\n",
      "ChimeraUGEM: unsupervised gene expression modeling in any given organism\n",
      "Assembly Graph Browser: interactive visualization of assembly graphs\n",
      "DeepAMR for predicting co-occurrent resistance of Mycobacterium tuberculosis\n",
      "Haplotype assembly of autotetraploid potato using integer linear programing\n",
      "Efficient implementation of convolutional neural networks in the data processing of two-photon in vivo imaging\n",
      "Protein fold recognition based on multi-view modeling\n",
      "The ReproGenomics Viewer: a multi-omics and cross-species resource compatible with single-cell studies for the reproductive science community\n",
      "admixr-R package for reproducible analyses using ADMIXTOOLS\n",
      "Benchmarking time-series data discretization on inference methods\n",
      "PSiTE: a Phylogeny guided Simulator for Tumor Evolution\n",
      "A network of networks approach for modeling interconnected brain tissue-specific networks\n",
      "EBIC: an open source software for high-dimensional and big data analyses\n",
      "TopoLink: evaluation of structural models using chemical crosslinking distance constraints\n",
      "Streamlined analysis of LINCS L1000 data with the slinky package for R\n",
      "TOPAS: network-based structural alignment of RNA sequences\n",
      "KASPspoon: an in vitro and in silico PCR analysis tool for high-throughput SNP genotyping\n",
      "Benchmark problems for dynamic modeling of intracellular processes\n",
      "Boost-HiC: computational enhancement of long-range contacts in chromosomal contact maps\n",
      "MetFlow: an interactive and integrated workflow for metabolomics data cleaning and differential metabolite discovery\n",
      "SIAN: software for structural identifiability analysis of ODE models\n",
      "DeepPhos: prediction of protein phosphorylation sites with deep learning\n",
      "Linking sequence patterns and functionality of alpha-helical antimicrobial peptides\n",
      "M3Drop: dropout-based feature selection for scRNASeq\n",
      "Fully unsupervised symmetry-based mitosis detection in time-lapse cell microscopy\n",
      "KinVis: a visualization tool to detect cryptic relatedness in genetic datasets\n",
      "Reply to the paper: Misunderstood parameters of NCBI BLAST impacts the correctness of bioinformatics workflows\n",
      "mAHTPred: a sequence-based meta-predictor for improving the prediction of anti-hypertensive peptides using effective feature representation\n",
      "Computing irreversible minimal cut sets in genome-scale metabolic networks via flux cone projection\n",
      "Seave: a comprehensive web platform for storing and interrogating human genomic variation\n",
      "Response to comments on 'Empirical comparison of web-based antimicrobial peptide prediction tools'\n",
      "On what to permute in test-based approaches for variable importance measures in Random Forests\n",
      "CNValidator: validating somatic copy-number inference\n",
      "Implementation of a Stirling number estimator enables direct calculation of population genetics tests for large sequence datasets\n",
      "CASMAP: detection of statistically significant combinations of SNPs in association mapping\n",
      "Commonly misunderstood parameters of NCBI BLAST and important considerations for users\n",
      "Gene expression models based on transcription factor binding events confer insight into functional cis-regulatory variants\n",
      "Increasing the accuracy of protein loop structure prediction with evolutionary constraints\n",
      "PopViz: a webserver for visualizing minor allele frequencies and damage prediction scores of human genetic variations\n",
      "GlycanFormatConverter: a conversion tool for translating the complexities of glycans\n",
      "AStrap: identification of alternative splicing from transcript sequences without a reference genome\n",
      "Improving prediction of protein secondary structure, backbone angles, solvent accessibility and contact numbers by using predicted contact maps and an ensemble of recurrent and residual convolutional neural networks\n",
      "atSNP Search: a web resource for statistically evaluating influence of human genetic variation on transcription factor binding\n",
      "BIOLITMAP: a web-based geolocated, temporal and thematic visualization of the evolution of bioinformatics publications\n",
      "miRDeep-P2: accurate and fast analysis of the microRNA transcriptome in plants\n",
      "Capsule network for protein post-translational modification site prediction\n",
      "Alfred: interactive multi-sample BAM alignment statistics, feature counting and feature annotation for long- and short-read sequencing\n",
      "GWASpro: a high-performance genome-wide association analysis server\n",
      "Drug Targetor: a web interface to investigate the human druggome for over 500 phenotypes\n",
      "Validation of electron microscopy initial models via small angle X-ray scattering curves\n",
      "CRISPR-Local: a local single-guide RNA (sgRNA) design tool for non-reference plant genomes\n",
      "DiTaxa: nucleotide-pair encoding of 16S rRNA for host phenotype and biomarker detection\n",
      "Computational modeling reveals MAP3K8 as mediator of resistance to vemurafenib in thyroid cancer stem cells\n",
      "An equivariant Bayesian convolutional network predicts recombination hotspots and accurately resolves binding motifs\n",
      "SkeletalVis: an exploration and meta-analysis data portal of cross-species skeletal transcriptomics data\n",
      "Integrate multiple traits to detect novel trait-gene association using GWAS summary data with an adaptive test approach\n",
      "SGTK: a toolkit for visualization and assessment of scaffold graphs\n",
      "Bias in the intervention in prediction measure in random forests: illustrations and recommendations\n",
      "DeepCrystal: a deep learning framework for sequence-based protein crystallization prediction\n",
      "TAIJI: approaching experimental replicates-level accuracy for drug synergy prediction\n",
      "BGSA: a bit-parallel global sequence alignment toolkit for multi-core and many-core architectures\n",
      "G4HunterApps\n",
      "Semi-supervised learning of Hidden Markov Models for biological sequence analysis\n",
      "FactorialHMM: fast and exact inference in factorial hidden Markov models\n",
      "Identify origin of replication in Saccharomyces cerevisiae using two-step feature selection technique\n",
      "SeQuiLa: an elastic, fast and scalable SQL-oriented solution for processing and querying genomic intervals\n",
      "Branch-recombinant Gaussian processes for analysis of perturbations in biological time series\n",
      "Topology independent structural matching discovers novel templates for protein interfaces\n",
      "Approximate, simultaneous comparison of microbial genome architectures via syntenic anchoring of quiver representations\n",
      "Fast characterization of segmental duplications in genome assemblies\n",
      "Predicting protein-protein interactions through sequence-based deep learning\n",
      "fastp: an ultra-fast all-in-one FASTQ preprocessor\n",
      "Accurate and adaptive imputation of summary statistics in mixed-ethnicity cohorts\n",
      "DREAM-Yara: an exact read mapper for very large databases with short update time\n",
      "iCFN: an efficient exact algorithm for multistate protein design\n",
      "Off-target predictions in CRISPR-Cas9 gene editing using deep learning\n",
      "SPhyR: tumor phylogeny estimation from single-cell sequencing data under loss and error\n",
      "Scalable and exhaustive screening of metabolic functions carried out by microbial consortia\n",
      "An ontology-based method for assessing batch effect adjustment approaches in heterogeneous datasets\n",
      "BayMAP: a Bayesian hierarchical model for the analysis of PAR-CLIP data\n",
      "DeepT3: deep convolutional neural networks accurately identify Gram-negative bacterial type III secreted effectors using the N-terminal sequence\n",
      "dtangle: accurate and robust cell type deconvolution\n",
      "Computational modeling of cellular structures using conditional deep generative networks\n",
      "Discovering network phenotype between genetic risk factors and disease status via diagnosis-aligned multi-modality regression method in Alzheimer's disease\n",
      "BiXGBoost: a scalable, flexible boosting-based method for reconstructing gene regulatory networks\n",
      "Heavy-tailed prior distributions for sequence count data: removing the noise and preserving large differences\n",
      "DrugThatGene: integrative analysis to streamline the identification of druggable genes, pathways and protein complexes from CRISPR screens\n",
      "Set of approaches based on 3D structure and position specific-scoring matrix for predicting DNA-binding proteins\n",
      "Batch effects correction for microbiome data with Dirichlet-multinomial regression\n",
      "VarSome: the human genomic variant search engine\n",
      "simGWAS: a fast method for simulation of large scale case-control GWAS summary statistics\n",
      "ELMER v.2: an R/Bioconductor package to reconstruct gene regulatory networks from DNA methylation and transcriptome profiles\n",
      "Measuring phylogenetic signal between categorical traits and phylogenies\n",
      "pJRES Binning Algorithm (JBA): a new method to facilitate the recovery of metabolic information from pJRES 1H NMR spectra\n",
      "GeneLab: Omics database for spaceflight experiments\n",
      "Landscape of multi-tissue global gene expression reveals the regulatory signatures of feed efficiency in beef cattle\n",
      "Enhanced prediction of RNA solvent accessibility with long short-term memory neural networks and improved sequence profiles\n",
      "SCALOP: sequence-based antibody canonical loop structure annotation\n",
      "MetaSMC: a coalescent-based shotgun sequence simulator for evolving microbial populations\n",
      "CANDI: an R package and Shiny app for annotating radiographs and evaluating computer-aided diagnosis\n",
      "Pisces: an accurate and versatile variant caller for somatic and germline next-generation sequencing data\n",
      "RiboProP: a probabilistic ribosome positioning algorithm for ribosome profiling\n",
      "Accurate detection of chimeric contigs via Bionano optical maps\n",
      "Bayesian negative binomial regression for differential expression with confounding factors\n",
      "sRNAPrimerDB: comprehensive primer design and search web service for small non-coding RNAs\n",
      "Inferring disease-associated long non-coding RNAs using genome-wide tissue expression profiles\n",
      "Strain-GeMS: optimized subspecies identification from microbiome data based on accurate variant modeling\n",
      "SanXoT: a modular and versatile package for the quantitative analysis of high-throughput proteomics experiments\n",
      "MSPminer: abundance-based reconstitution of microbial pan-genomes from shotgun metagenomic data\n",
      "PySFD: comprehensive molecular insights from significant feature differences detected among many simulated ensembles\n",
      "iTerm-PseKNC: a sequence-based tool for predicting bacterial transcriptional terminators\n",
      "CINNA: an R/CRAN package to decipher Central Informative Nodes in Network Analysis\n",
      "STatistical Inference Relief (STIR) feature selection\n",
      "Combining count- and length-based z-scores leads to improved predictions in non-invasive prenatal testing\n",
      "A Bayesian framework for high-throughput T cell receptor pairing\n",
      "The GCTx format and cmap{Py, R, M, J} packages: resources for optimized storage and integrated traversal of annotated dense matrices\n",
      "CATANA: a tool for generating comprehensive annotations of alternative transcript events\n",
      "SAFE-clustering: Single-cell Aggregated (from Ensemble) clustering for single-cell RNA-seq data\n",
      "Prediction of protein group function by iterative classification on functional relevance network\n",
      "novoCaller: a Bayesian network approach for de novo variant calling from pedigree and population sequence data\n",
      "NanoShaper-VMD interface: computing and visualizing surfaces, pockets and channels in molecular systems\n",
      "Comparative analysis of methods for evaluation of protein models against native structures\n",
      "HitPickV2: a web server to predict targets of chemical compounds\n",
      "Synergizing CRISPR/Cas9 off-target predictions for ensemble insights and practical applications\n",
      "Resolution-exchanged structural modeling and simulations jointly unravel that subunit rolling underlies the mechanism of programmed ribosomal frameshifting\n",
      "Efficient computation of the kinship coefficients\n",
      "SKmDB: an integrated database of next generation sequencing information in skeletal muscle\n",
      "Variable selection and validation in multivariate modelling\n",
      "Inferring gene expression networks with hubs using a degree weighted Lasso approach\n",
      "Penalized co-inertia analysis with applications to -omics data\n",
      "gemBS: high throughput processing for DNA methylation data from bisulfite sequencing\n",
      "DCARS: differential correlation across ranked samples\n",
      "isiKnock: in silico knockouts in signaling pathways\n",
      "Clustering of temporal gene expression data with mixtures of mixed effects models with a penalized likelihood\n",
      "ASGART: fast and parallel genome scale segmental duplications mapping\n",
      "Detection of de novo copy number deletions from targeted sequencing of trios\n",
      "JustOrthologs: a fast, accurate and user-friendly ortholog identification algorithm\n",
      "DiscoverSL: an R package for multi-omic data driven prediction of synthetic lethality in cancers\n",
      "The epiGenomic Efficient Correlator (epiGeEC) tool allows fast comparison of user datasets with thousands of public epigenomic datasets\n",
      "LipidIMMS Analyzer: integrating multi-dimensional information to support lipid identification in ion mobility-mass spectrometry based lipidomics\n",
      "GPseudoRank: a permutation sampler for single cell orderings\n",
      "iSwathX: an interactive web-based application for extension of DIA peptide reference libraries\n",
      "MendelProb: probability and sample size calculations for Mendelian studies of exome and whole genome sequence data\n",
      "Cryfa: a secure encryption tool for genomic data\n",
      "SKEMPI 2.0: an updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation\n",
      "Scaling read aligners to hundreds of threads on general-purpose processors\n",
      "BIPSPI: a method for the prediction of partner-specific protein-protein interfaces\n",
      "Reproducible and replicable comparisons using SummarizedBenchmark\n",
      "ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R\n",
      "GIGSEA: genotype imputed gene set enrichment analysis using GWAS summary level data\n",
      "Broom: application for non-redundant storage of high throughput sequencing data\n",
      "A parallel computational framework for ultra-large-scale sequence clustering analysis\n",
      "Hierarchical optimization for the efficient parametrization of ODE models\n",
      "Uncertainty quantification, propagation and characterization by Bayesian analysis combined with global sensitivity analysis applied to dynamical intracellular pathway models\n",
      "Crumble: reference free lossy compression of sequence quality values\n",
      "LigVoxel: inpainting binding pockets using 3D-convolutional neural networks\n",
      "Compound-protein interaction prediction with end-to-end learning of neural networks for graphs and sequences\n",
      "GATK PathSeq: a customizable computational tool for the discovery and identification of microbial sequences in libraries from eukaryotic hosts\n",
      "PepBDB: a comprehensive structural database of biological peptide-protein interactions\n",
      "SLIDE - a web-based tool for interactive visualization of large-scale - omics data\n",
      "Hybrid correction of highly noisy long reads using a variable-order de Bruijn graph\n",
      "ISMB 2018 proceedings\n",
      "Deep neural networks and distant supervision for geographic location mention extraction\n",
      "Random forest based similarity learning for single cell RNA sequencing data\n",
      "Personalized regression enables sample-specific pan-cancer analysis\n",
      "A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity\n",
      "Finding associated variants in genome-wide association studies on multiple traits\n",
      "A space and time-efficient index for the compacted colored de Bruijn graph\n",
      "Covariate-dependent negative binomial factor analysis of RNA sequencing data\n",
      "Protein threading using residue co-variation and deep learning\n",
      "Learning with multiple pairwise kernels for drug bioactivity prediction\n",
      "An optimization framework for network annotation\n",
      "Quantifying the similarity of topological domains across normal and cancer human cell types\n",
      "DisruPPI: structure-based computational redesign algorithm for protein binding disruption\n",
      "COSSMO: predicting competitive alternative splice site selection using deep learning\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BEL2ABM: agent-based simulation of static models in Biological Expression Language\n",
      "PartsGenie: an integrated tool for optimizing and sharing synthetic biology parts\n",
      "BMC3C: binning metagenomic contigs using codon usage, sequence composition and read coverage\n",
      "GeneCT: a generalizable cancerous status and tissue origin classifier for pan-cancer biopsies\n",
      "Practical dynamic de Bruijn graphs\n",
      "SOPanG: online text searching over a pan-genome\n",
      "A descriptive marker gene approach to single-cell pseudotime inference\n",
      "MetaPOAP: presence or absence of metabolic pathways in metagenome-assembled genomes\n",
      "SCIP: a single-cell image processor toolbox\n",
      "snpAD: an ancient DNA genotype caller\n",
      "Meffil: efficient normalization and analysis of very large DNA methylation datasets\n",
      "MolArt: a molecular structure annotation and visualization tool\n",
      "iLoc-lncRNA: predict the subcellular location of lncRNAs by incorporating octamer composition into general PseKNC\n",
      "Prediction of kinetics of protein folding with non-redundant contact information\n",
      "panISa: ab initio detection of insertion sequences in bacterial genomes from short read sequence data\n",
      "GWASinlps: non-local prior based iterative SNP selection tool for genome-wide association studies\n",
      "iMetaLab 1.0: a web platform for metaproteomics data analysis\n",
      "RiboSketch: versatile visualization of multi-stranded RNA and DNA secondary structure\n",
      "TranscriptClean: variant-aware correction of indels, mismatches and splice junctions in long-read transcripts\n",
      "ePCR: an R-package for survival and time-to-event prediction in advanced prostate cancer, applied to real-world patient cohorts\n",
      "gmxapi: a high-level interface for advanced control and extension of molecular dynamics simulations\n",
      "Evaluating statistical approaches to leverage large clinical datasets for uncovering therapeutic and adverse medication effects\n",
      "comoRbidity: an R package for the systematic analysis of disease comorbidities\n",
      "SSMART: sequence-structure motif identification for RNA-binding proteins\n",
      "iEnhancer-EL: identifying enhancers and their strength with ensemble learning approach\n",
      "MetQy-an R package to query metabolic functions of genes and genomes\n",
      "MIDORI server: a webserver for taxonomic assignment of unknown metazoan mitochondrial-encoded sequences using a curated database\n",
      "Axe: rapid, competitive sequence read demultiplexing using a trie\n",
      "ggCyto: next generation open-source visualization software for cytometry\n",
      "Improved design and analysis of CRISPR knockout screens\n",
      "DeepMirTar: a deep-learning approach for predicting human miRNA targets\n",
      "MODE-TASK: large-scale protein motion tools\n",
      "Enhanced functionalities for annotating and indexing clinical text with the NCBO Annotator\n",
      "SimExTargId: a comprehensive package for real-time LC-MS data acquisition and analysis\n",
      "Using meshes for MeSH term enrichment and semantic analyses\n",
      "Application of network smoothing to glycan LC-MS profiling\n",
      "Combining probabilistic alignments with read pair information improves accuracy of split-alignments\n",
      "Single cell clustering based on cell-pair differentiability correlation and variance analysis\n",
      "AC-DIAMOND v1: accelerating large-scale DNA-protein alignment\n",
      "Identification and characterization of moonlighting long non-coding RNAs based on RNA and protein interactome\n",
      "ConSpeciFix: classifying prokaryotic species based on gene flow\n",
      "RecPhyloXML: a format for reconciled gene trees\n",
      "Correcting mistakes in predicting distributions\n",
      "MRMAssayDB: an integrated resource for validated targeted proteomics assays\n",
      "The revival of the Gini importance?\n",
      "Grouper: graph-based clustering and annotation for improved de novo transcriptome analysis\n",
      "LZW-Kernel: fast kernel utilizing variable length code blocks from LZW compressors for protein sequence classification\n",
      "Jwalk and MNXL web server: model validation using restraints from crosslinking mass spectrometry\n",
      "Modeling one thousand intron length distributions with fitild\n",
      "JRmGRN: joint reconstruction of multiple gene regulatory networks with common hub genes using data from multiple tissues or conditions\n",
      "Quantification of biases in predictions of protein stability changes upon mutations\n",
      "Deep reinforcement learning of cell movement in the early stage of C.elegans embryogenesis\n",
      "Automatic selection of verification tools for efficient analysis of biochemical models\n",
      "RWEN: response-weighted elastic net for prediction of chemosensitivity of cancer cell lines\n",
      "EMUDRA: Ensemble of Multiple Drug Repositioning Approaches to improve prediction accuracy\n",
      "DEsingle for detecting three types of differential expression in single-cell RNA-seq data\n",
      "Ultra-fast global homology detection with Discrete Cosine Transform and Dynamic Time Warping\n",
      "Parapred: antibody paratope prediction using convolutional and recurrent neural networks\n",
      "DeepEfflux: a 2D convolutional neural network model for identifying families of efflux proteins in transporters\n",
      "CaSiAn: a Calcium Signaling Analyzer tool\n",
      "JAMI: fast computation of conditional mutual information for ceRNA network analysis\n",
      "A sequence family database built on ECOD structural domains\n",
      "SnapperDB: a database solution for routine sequencing analysis of bacterial isolates\n",
      "IRscope: an online program to visualize the junction sites of chloroplast genomes\n",
      "B-cell receptor reconstruction from single-cell RNA-seq with VDJPuzzle\n",
      "Inference of the human polyadenylation code\n",
      "AGORA: organellar genome annotation from the amino acid and nucleotide references\n",
      "Genome-by-genome approach for fast bacterial genealogical relationship evaluation\n",
      "scanPAV: a pipeline for extracting presence-absence variations in genome pairs\n",
      "BART: a transcription factor prediction tool with query gene sets or epigenomic profiles\n",
      "SimPhospho: a software tool enabling confident phosphosite assignment\n",
      "SPV: a JavaScript Signaling Pathway Visualizer\n",
      "Deep learning improves antimicrobial peptide recognition\n",
      "ALPHA: a toolkit for Automated Local PHylogenomic Analyses\n",
      "GenomeDISCO: a concordance score for chromosome conformation capture experiments using random walks on contact map graphs\n",
      "CMV: visualization for RNA and protein family models and their comparisons\n",
      "CGManalyzer: an R package for analyzing continuous glucose monitoring studies\n",
      "Integrating long-range connectivity information into de Bruijn graphs\n",
      "Using combined evidence from replicates to evaluate ChIP-seq peaks\n",
      "A novel computational approach for drug repurposing using systems biology\n",
      "METHCOMP: a special purpose compression platform for DNA methylation data\n",
      "Optimality and identification of dynamic models in systems biology: an inverse optimal control framework\n",
      "GOLabeler: improving sequence-based large-scale protein function prediction by learning to rank\n",
      "AnaCoDa: analyzing codon data with Bayesian mixture models\n",
      "eMolTox: prediction of molecular toxicity with confidence\n",
      "BPRMeth: a flexible Bioconductor package for modelling methylation profiles\n",
      "FasParser2: a graphical platform for batch manipulation of tremendous amount of sequence data\n",
      "Mitigating the adverse impact of batch effects in sample pattern detection\n",
      "SubRecon: ancestral reconstruction of amino acid substitutions along a branch in a phylogeny\n",
      "PhyloMAd: efficient assessment of phylogenomic model adequacy\n",
      "Bacmeta: simulator for genomic evolution in bacterial metapopulations\n",
      "STAR Chimeric Post for rapid detection of circular RNA and fusion transcripts\n",
      "IWTomics: testing high-resolution sequence-based 'Omics' data at multiple locations and scales\n",
      "ViCTree: an automated framework for taxonomic classification from protein sequences\n",
      "Cost function network-based design of protein-protein interactions: predicting changes in binding affinity\n",
      "Multiobjective multifactor dimensionality reduction to detect SNP-SNP interactions\n",
      "The lncLocator: a subcellular localization predictor for long non-coding RNAs based on a stacked ensemble classifier\n",
      "A new approach for interpreting Random Forest models and its application to the biology of ageing\n",
      "ThreaDNA: predicting DNA mechanics' contribution to sequence selectivity of proteins along whole genomes\n",
      "ssbio: a Python framework for structural systems biology\n",
      "chromswitch: a flexible method to detect chromatin state switches\n",
      "WDL-RF: predicting bioactivities of ligand molecules acting with G protein-coupled receptors by combining weighted deep learning and random forest\n",
      "INfORM: Inference of NetwOrk Response Modules\n",
      "scEpath: energy landscape-based inference of transition probabilities and cellular trajectories from single-cell transcriptomic data\n",
      "Novel overlapping subgraph clustering for the detection of antigen epitopes\n",
      "Aligning dynamic networks with DynaWAVE\n",
      "Co-complex protein membership evaluation using Maximum Entropy on GO ontology and InterPro annotation\n",
      "pepKalc: scalable and comprehensive calculation of electrostatic interactions in random coil polypeptides\n",
      "An easy and efficient approach for testing identifiability\n",
      "Sequanix: a dynamic graphical interface for Snakemake workflows\n",
      "Primer3_masker: integrating masking of template sequence with primer design software\n",
      "A distance-based approach for testing the mediation effect of the human microbiome\n",
      "PEATH: single-individual haplotyping by a probabilistic evolutionary algorithm with toggling\n",
      "iPat: intelligent prediction and association tool for genomic research\n",
      "Discovering personalized driver mutation profiles of single samples in cancer by network control strategy\n",
      "Sensitive and specific post-call filtering of genetic variants in xenograft and primary tumors\n",
      "DiffGraph: an R package for identifying gene network rewiring using differential graphical models\n",
      "CHROMIXS: automatic and interactive analysis of chromatography-coupled small-angle X-ray scattering data\n",
      "polyPK: an R package for pharmacokinetic analysis of multi-component drugs using a metabolomics approach\n",
      "An automated benchmarking platform for MHC class II binding prediction methods\n",
      "A high-resolution map of the human small non-coding transcriptome\n",
      "Multiple hot-deck imputation for network inference from RNA sequencing data\n",
      "DeepSig: deep learning improves signal peptide detection in proteins\n",
      "DPMIND: degradome-based plant miRNA-target interaction and network database\n",
      "Missing value imputation for LC-MS metabolomics data by incorporating metabolic network and adduct ion relations\n",
      "Personal Cancer Genome Reporter: variant interpretation report for precision oncology\n",
      "GRAM-CNN: a deep learning approach with local context for named entity recognition in biomedical text\n",
      "Computational identification of micro-structural variations and their proteogenomic consequences in cancer\n",
      "GIGI-Quick: a fast approach to impute missing genotypes in genome-wide association family data\n",
      "Rcupcake: an R package for querying and analyzing biomedical data through the BD2K PIC-SURE RESTful API\n",
      "JASPAR RESTful API: accessing JASPAR data from any programming language\n",
      "Curatr: a web application for creating, curating and sharing a mass spectral library\n",
      "GeoBoost: accelerating research involving the geospatial metadata of virus GenBank records\n",
      "Machine learning for classifying tuberculosis drug-resistance from DNA sequencing data\n",
      "NGLview-interactive molecular graphics for Jupyter notebooks\n",
      "TarHunter, a tool for predicting conserved microRNA targets and target mimics in plants\n",
      "Matrix factorization-based data fusion for the prediction of lncRNA-disease associations\n",
      "LitPathExplorer: a confidence-based visual text analytics tool for exploring literature-enriched pathway models\n",
      "Aether: leveraging linear programming for optimal cloud computing in genomics\n",
      "DNCON2: improved protein contact prediction using two-level deep convolutional neural networks\n",
      "Bicycle: a bioinformatics pipeline to analyze bisulfite sequencing data\n",
      "RecoverY: k-mer-based read classification for Y-chromosome-specific sequencing and assembly\n",
      "ThetaMater: Bayesian estimation of population size parameter θ from genomic data\n",
      "shinyCircos: an R/Shiny application for interactive creation of Circos plot\n",
      "FUn: a framework for interactive visualizations of large, high-dimensional datasets on the web\n",
      "The systematic analysis of ultraconserved genomic regions in the budding yeast\n",
      "DDR: efficient computational method to predict drug-target interactions using graph mining and machine learning approaches\n",
      "SimulaTE: simulating complex landscapes of transposable elements of populations\n",
      "Myriads: P-value-based multiple testing correction\n",
      "Sequential search leads to faster, more efficient fragment-based de novo protein structure prediction\n",
      "LIBRA-WA: a web application for ligand binding site detection and protein function recognition\n",
      "Complex analyses of inverted repeats in mitochondrial genomes revealed their importance and variability\n",
      "RepLong: de novo repeat identification using long read sequencing data\n",
      "Cost-effective and accurate method of measuring fetal fraction using SNP imputation\n",
      "pLoc-mHum: predict subcellular localization of multi-location human proteins via general PseAAC to winnow out the crucial GO information\n",
      "MetaCherchant: analyzing genomic context of antibiotic resistance genes in gut microbiota\n",
      "pStab: prediction of stable mutants, unfolding curves, stability maps and protein electrostatic frustration\n",
      "An exact test for comparing a fixed quantitative property between gene sets\n",
      "TMEM132: an ancient architecture of cohesin and immunoglobulin domains define a new family of neural adhesion molecules\n",
      "Drug-drug interaction extraction via hierarchical RNNs on sequence and shortest dependency paths\n",
      "Unsupervised multiple kernel learning for heterogeneous data integration\n",
      "ArachnoServer 3.0: an online resource for automated discovery, analysis and annotation of spider toxins\n",
      "SeqBox: RNAseq/ChIPseq reproducible analysis on a consumer game computer\n",
      "xMWAS: a data-driven integration and differential network analysis tool\n",
      "PyBEL: a computational framework for Biological Expression Language\n",
      "SliceMap: an algorithm for automated brain region annotation\n",
      "A probabilistic pathway score (PROPS) for classification with applications to inflammatory bowel disease\n",
      "SAGE2: parallel human genome assembly\n",
      "Detection of sputum by interpreting the time-frequency distribution of respiratory sound signal using image processing techniques\n",
      "Machine learning accelerates MD-based binding pose prediction between ligands and proteins\n",
      "Compound image segmentation of published biomedical figures\n",
      "SELANSI: a toolbox for simulation of stochastic gene regulatory networks\n",
      "Generalized correlation measure using count statistics for gene expression data with ordered samples\n",
      "Using uncertainty to link and rank evidence from biomedical literature for model curation\n",
      "DeepLoc: prediction of protein subcellular localization using deep learning\n",
      "Faucet: streaming de novo assembly graph construction\n",
      "DelPhiForce web server: electrostatic forces and energy calculations and visualization\n",
      "cgHeliParm: analysis of dsDNA helical parameters for coarse-grained MARTINI molecular dynamics simulations\n",
      "wft4galaxy: a workflow testing tool for galaxy\n",
      "DynaPho: a web platform for inferring the dynamics of time-series phosphoproteomics\n",
      "ScanGEO: parallel mining of high-throughput gene expression data\n",
      "FOLD: a method to optimize power in meta-analysis of genetic association studies with overlapping subjects\n",
      "Phylotyper: in silico predictor of gene subtypes\n",
      "PySeqLab: an open source Python package for sequence labeling and segmentation\n",
      "A robust DF-REML framework for variance components estimation in genetic studies\n",
      "On expert curation and scalability: UniProtKB/Swiss-Prot as a case study\n",
      "In silico structural modeling of multiple epigenetic marks on DNA\n",
      "Towards unified quality verification of synthetic count data with countsimQC\n",
      "DeepGO: predicting protein functions from sequence and interactions using a deep ontology-aware classifier\n",
      "The systematic analysis of ultraconserved genomic regions in the budding yeast\n",
      "OncomiR: an online resource for exploring pan-cancer microRNA dysregulation\n",
      "Quality of biological images, reconstructed using localization microscopy data\n",
      "A collaborative filtering-based approach to biomedical knowledge discovery\n",
      "On the mechanisms of protein interactions: predicting their affinity from unbound tertiary structures\n",
      "Segway 2.0: Gaussian mixture models and minibatch training\n",
      "Identification of cancer driver genes in focal genomic aberrations from whole-exome sequencing data\n",
      "CircPro: an integrated tool for the identification of circRNAs with protein-coding potential\n",
      "Panoptes: web-based exploration of large scale genome variation data\n",
      "High dimensional classification with combined adaptive sparse PLS and logistic regression\n",
      "iPromoter-2L: a two-layer predictor for identifying promoters and their types by multi-window-based PseKNC\n",
      "Integrating splice-isoform expression into genome-scale models characterizes breast cancer metabolism\n",
      "Novel features and enhancements in BioBin, a tool for the biologically inspired binning and association analysis of rare variants\n",
      "A non-negative matrix factorization based method for predicting disease-associated miRNAs in miRNA-disease bilayer network\n",
      "MetExploreViz: web component for interactive metabolic network visualization\n",
      "Interactive network visualization in Jupyter notebooks: visJS2jupyter\n",
      "IntPred: a structure-based predictor of protein-protein interaction sites\n",
      "CGmapTools improves the precision of heterozygous SNV calls and supports allele-specific methylation detection and visualization in bisulfite-sequencing data\n",
      "MemBrain-contact 2.0: a new two-stage machine learning model for the prediction enhancement of transmembrane protein residue contacts in the full chain\n",
      "PiMP my metabolome: an integrated, web-based tool for LC-MS metabolomics data\n",
      "ALIGNSEC: viewing protein secondary structure predictions within large multiple sequence alignments\n",
      "HiDi: an efficient reverse engineering schema for large-scale dynamic regulatory network reconstruction using adaptive differentiation\n",
      "Conditional asymptotic inference for the kernel association test\n",
      "In silico identification of rescue sites by double force scanning\n",
      "pgRNAFinder: a web-based tool to design distance independent paired-gRNA\n",
      "Reference genome assessment from a population scale perspective: an accurate profile of variability and noise\n",
      "psygenet2r: a R/Bioconductor package for the analysis of psychiatric disease genes\n",
      "clustComp, a bioconductor package for the comparison of clustering results\n",
      "An improved compound Poisson model for the number of motif hits in DNA sequences\n",
      "Genome-scale regression analysis reveals a linear relationship for promoters and enhancers after combinatorial drug treatment\n",
      "ncDR: a comprehensive resource of non-coding RNAs involved in drug resistance\n",
      "iDNA4mC: identifying DNA N4-methylcytosine sites based on nucleotide chemical properties\n",
      "Efficient approximations of RNA kinetics landscape using non-redundant sampling\n",
      "Predicting phenotypes from microarrays using amplified, initially marginal, eigenvector regression\n",
      "deBGR: an efficient and near-exact representation of the weighted de Bruijn graph\n",
      "Multiple network-constrained regressions expand insights into influenza vaccination responses\n",
      "Image-based spatiotemporal causality inference for protein signaling networks\n",
      "popFBA: tackling intratumour heterogeneity with Flux Balance Analysis\n",
      "A scalable moment-closure approximation for large-scale biochemical reaction networks\n",
      "Incorporating interaction networks into the determination of functionally related hit genes in genomic experiments with Markov random fields\n",
      "Improving the performance of minimizers and winnowing schemes\n",
      "HopLand: single-cell pseudotime recovery using continuous Hopfield network-based modeling of Waddington's epigenetic landscape\n",
      "DextMP: deep dive into text for predicting moonlighting proteins\n",
      "Orthologous Matrix (OMA) algorithm 2.0: more robust to asymmetric evolutionary rates and more scalable hierarchical orthologous group inference\n",
      "Deep learning with word embeddings improves biomedical named entity recognition\n",
      "SVM-dependent pairwise HMM: an application to protein pairwise alignments\n",
      "Inferring transcriptional logic from multiple dynamic experiments\n",
      "TSGSIS: a high-dimensional grouped variable selection approach for detection of whole-genome SNP-SNP interactions\n",
      "High-speed and high-ratio referential genome compression\n",
      "NucliTrack: an integrated nuclei tracking application\n",
      "BPP: a sequence-based algorithm for branch point prediction\n",
      "BioQueue: a novel pipeline framework to accelerate bioinformatics analysis\n",
      "The International Society for Computational Biology and WikiProject Computational Biology: celebrating 10 years of collaboration towards open access\n",
      "The interfacial character of antibody paratopes: analysis of antibody-antigen structures\n",
      "matter: an R package for rapid prototyping with larger-than-memory datasets on disk\n",
      "Pheno4J: a gene to phenotype graph database\n",
      "EpiCompare: an online tool to define and explore genomic regions with tissue or cell type-specific epigenomic features\n",
      "MoDMaps3D: an interactive webtool for the quantification and 3D visualization of interrelationships in a dataset of DNA sequences\n",
      "A spectral algorithm for fast de novo layout of uncorrected long nanopore reads\n",
      "FQC Dashboard: integrates FastQC results into a web-based, interactive, and extensible FASTQ quality control tool\n",
      "COMPASS: the COMPletely Arbitrary Sequence Simulator\n",
      "Model-based branching point detection in single-cell data by K-branches clustering\n",
      "DNA sequence+shape kernel enables alignment-free modeling of transcription factor binding\n",
      "MetaSRA: normalized human sample-specific metadata for the Sequence Read Archive\n",
      "PIXiE: an algorithm for automated ion mobility arrival time extraction and collision cross section calculation using global data association\n",
      "IGESS: a statistical approach to integrating individual-level genotype data and summary statistics in genome-wide association studies\n",
      "Application of the cghRA framework to the genomic characterization of Diffuse Large B-Cell Lymphoma\n",
      "Foldit Standalone: a video game-derived protein structure manipulation interface using Rosetta\n",
      "Assembling draft genomes using contiBAIT\n",
      "twoddpcr: an R/Bioconductor package and Shiny app for Droplet Digital PCR analysis\n",
      "SinCHet: a MATLAB toolbox for single cell heterogeneity analysis in cancer\n",
      "STOPGAP: a database for systematic target opportunity assessment by genetic association predictions\n",
      "Annotating function to differentially expressed LincRNAs in myelodysplastic syndrome using a network-based method\n",
      "OMSim: a simulator for optical map data\n",
      "MobiDB-lite: fast and highly specific consensus prediction of intrinsic disorder in proteins\n",
      "DistributedFBA.jl: high-level, high-performance flux balance analysis in Julia\n",
      "The genetic map comparator: a user-friendly application to display and compare genetic maps\n",
      "Chainy: an universal tool for standardized relative quantification in real-time PCR\n",
      "AFS: identification and quantification of species composition by metagenomic sequencing\n",
      "fastMitoCalc: an ultra-fast program to estimate mitochondrial DNA copy number from whole-genome sequences\n",
      "CircosVCF: circos visualization of whole-genome sequence variations stored in VCF files\n",
      "Primerize-2D: automated primer design for RNA multidimensional chemical mapping\n",
      "Wright-Fisher exact solver (WFES): scalable analysis of population genetic models without simulation or diffusion theory\n",
      "GppFst: genomic posterior predictive simulations of FST and dXY for identifying outlier loci from population genomic data\n",
      "Image-based correction of continuous and discontinuous non-planar axial distortion in serial section microscopy\n",
      "Linearity of network proximity measures: implications for set-based queries and significance testing\n",
      "Multi-label classifier based on histogram of gradients for predicting the anatomical therapeutic chemical class/classes of a given compound\n",
      "Capturing non-local interactions by long short-term memory bidirectional recurrent neural networks for improving prediction of protein secondary structure, backbone angles, contact numbers and solvent accessibility\n",
      "MOST-visualization: software for producing automated textbook-style maps of genome-scale metabolic networks\n",
      "Removal of batch effects using distribution-matching residual networks\n",
      "DMINDA 2.0: integrated and systematic views of regulatory DNA motif identification and analyses\n",
      "Chainy: an universal tool for standardized relative quantification in real-time PCR\n",
      "Green listed-a CRISPR screen tool\n",
      "SubCons: a new ensemble method for improved human subcellular localization predictions\n",
      "IcyTree: rapid browser-based visualization for phylogenetic trees and networks\n",
      "BioCIDER: a Contextualisation InDEx for biological Resources discovery\n",
      "MetCirc: navigating mass spectral similarity in high-resolution MS/MS metabolomics data\n",
      "RTK: efficient rarefaction analysis of large datasets\n",
      "EzMap: a simple pipeline for reproducible analysis of the human virome\n",
      "TASIC: determining branching models from time series single cell data\n",
      "SCODE: an efficient regulatory network inference algorithm from single-cell RNA-Seq during differentiation\n",
      "A penalized regression approach to haplotype reconstruction of viral populations arising in early HIV/SIV infection\n",
      "Fuento: functional enrichment for bioinformatics\n",
      "Gracob: a novel graph-based constant-column biclustering method for mining growth phenotype data\n",
      "HiC-spector: a matrix library for spectral and reproducibility analysis of Hi-C contact maps\n",
      "annotatr: genomic regions in context\n",
      "Sequential computation of elementary modes and minimal cut sets in genome-scale metabolic networks using alternate integer linear programming\n",
      "Trainable Weka Segmentation: a machine learning tool for microscopy pixel classification\n",
      "Smoldyn: particle-based simulation with rule-based modeling, improved molecular interaction and a library interface\n",
      "SIApopr: a computational method to simulate evolutionary branching trees for analysis of tumor clonal evolution\n",
      "Quantifying circular RNA expression from RNA-seq data using model-based framework\n",
      "VCNet: vector-based gene co-expression network construction and its application to RNA-seq data\n",
      "Tagging SNP-set selection with maximum information based on linkage disequilibrium structure in genome-wide association studies\n",
      "PeptideMapper: efficient and versatile amino acid sequence and tag mapping\n",
      "Dynamix: dynamic visualization by automatic selection of informative tracks from hundreds of genomic datasets\n",
      "SmartR: an open-source platform for interactive visual analytics for translational research data\n",
      "IntegratedMRF: random forest-based framework for integrating prediction from different data types\n",
      "ResistoMap-online visualization of human gut microbiota antibiotic resistome\n",
      "Phenopolis: an open platform for harmonization and analysis of genetic and phenotypic data\n",
      "A generalized association test based on U statistics\n",
      "Fast admixture analysis and population tree estimation for SNP and NGS data\n",
      "FUNNEL-GSEA: FUNctioNal ELastic-net regression in time-course gene set enrichment analysis\n",
      "GARLIC: Genomic Autozygosity Regions Likelihood-based Inference and Classification\n",
      "MapReduce for accurate error correction of next-generation sequencing data\n",
      "Correlated mutations select misfolded from properly folded proteins\n",
      "Changes associated with Ebola virus adaptation to novel species\n",
      "Abstracting the dynamics of biological pathways using information theory: a case study of apoptosis pathway\n",
      "Binning_refiner: improving genome bins through the combination of different binning programs\n",
      "VALORATE: fast and accurate log-rank test in balanced and unbalanced comparisons of survival curves and cancer genomics\n",
      "PatternMarkers & GWCoGAPS for novel data-driven biomarkers via whole transcriptome NMF\n",
      "iATC-mISF: a multi-label classifier for predicting the classes of anatomical therapeutic chemicals\n",
      "Assocplots: a Python package for static and interactive visualization of multiple-group GWAS results\n",
      "P3BSseq: parallel processing pipeline software for automatic analysis of bisulfite sequencing data\n",
      "CIIPro: a new read-across portal to fill data gaps using public large-scale chemical and biological data\n",
      "The START App: a web-based RNAseq analysis and visualization resource\n",
      "Granular clustering of de novo protein models\n",
      "Genotyping inversions and tandem duplications\n",
      "Bayesian network-response regression\n",
      "cyvcf2: fast, flexible variant analysis with Python\n",
      "Pathway Inspector: a pathway based web application for RNAseq analysis of model and non-model organisms\n",
      "A Cytoscape app for motif enumeration with ISMAGS\n",
      "Automated cell type discovery and classification through knowledge transfer\n",
      "NSSRF: global network similarity search with subgraph signatures and its applications\n",
      "FIT: statistical modeling tool for transcriptome dynamics under fluctuating field conditions\n",
      "Sparse network modeling and metscape-based visualization methods for the analysis of large-scale metabolomics data\n",
      "'COV'COP' allows to detect CNVs responsible for inherited diseases among amplicons sequencing data\n",
      "Dscam1 web server: online prediction of Dscam1 self- and hetero-affinity\n",
      "Classification of RNA structure change by 'gazing' at experimental data\n",
      "A comprehensive quality control workflow for paired tumor-normal NGS experiments\n",
      "NetLand: quantitative modeling and visualization of Waddington's epigenetic landscape using probabilistic potential\n",
      "Quantification of fibrous spatial point patterns from single-molecule localization microscopy (SMLM) data\n",
      "aRNApipe: a balanced, efficient and distributed pipeline for processing RNA-seq data in high-performance computing environments\n",
      "FASTdoop: a versatile and efficient library for the input of FASTA and FASTQ files for MapReduce Hadoop bioinformatics applications\n",
      "The super-n-motifs model: a novel alignment-free approach for representing and comparing RNA secondary structures\n",
      "Modeling gene-wise dependencies improves the identification of drug response biomarkers in cancer studies\n",
      "A DNA intercalation methodology for an efficient prediction of ligand binding pose and energetics\n",
      "Rebuttal to the Letter to the Editor in response to the paper: proper evaluation of alignment-free network comparison methods\n",
      "SigMod: an exact and efficient method to identify a strongly interconnected disease-associated module in a gene network\n",
      "RENT+: an improved method for inferring local genealogical trees from haplotypes with recombination\n",
      "Exploring the predictive power of polygenic scores derived from genome-wide association studies: a study of 10 complex traits\n",
      "GE-mini: a mobile APP for large-scale gene expression visualization\n",
      "Cross-React: a new structural bioinformatics method for predicting allergen cross-reactivity\n",
      "Robust parameter estimation for dynamical systems from outlier-corrupted data\n",
      "AFRESh: an adaptive framework for compression of reads and assembled sequences with random access functionality\n",
      "RImmPort: an R/Bioconductor package that enables ready-for-analysis immunology research data\n",
      "phyloSkeleton: taxon selection, data retrieval and marker identification for phylogenomics\n",
      "PanViz: interactive visualization of the structure of functionally annotated pangenomes\n",
      "An ensemble approach to protein fold classification by integration of template-based assignment and support vector machine classifier\n",
      "PBIT: Pipeline Builder for Identification of drug Targets for infectious diseases\n",
      "Training alignment parameters for arbitrary sequencers with LAST-TRAIN\n",
      "geneAttribution: trait agnostic identification of candidate genes associated with noncoding variation\n",
      "seXY: a tool for sex inference from genotype arrays\n",
      "PhosD: inferring kinase-substrate interactions based on protein domains\n",
      "MetaDCN: meta-analysis framework for differential co-expression network detection with an application in breast cancer\n",
      "Accelerating Bayesian inference for evolutionary biology models\n",
      "Calypso: a user-friendly web-server for mining and visualizing microbiome-environment interactions\n",
      "myGenomeBrowser: building and sharing your own genome browser\n",
      "A new method for decontamination of de novo transcriptomes using a hierarchical clustering algorithm\n",
      "cerebroViz: an R package for anatomical visualization of spatiotemporal brain data\n",
      "Expanding the toolkit for membrane protein modeling in Rosetta\n",
      "SarConfoCal: simultaneous sarcomere length and cytoplasmic calcium measurements for laser scanning confocal microscopy images\n",
      "MetaboSignal: a network-based approach for topological analysis of metabotype regulation via metabolic and signaling pathways\n",
      "A general framework for association analysis of microbial communities on a taxonomic tree\n",
      "Defining the clonality of peripheral T cell lymphomas using RNA-seq\n",
      "Innovations and challenges in detecting long read overlaps: an evaluation of the state-of-the-art\n",
      "The Ark: a customizable web-based data management tool for health and medical research\n",
      "HYSYS: have you swapped your samples?\n",
      "A pipeline for local assembly of minisatellite alleles from single-molecule sequencing data\n",
      "CellSort: a support vector machine tool for optimizing fluorescence-activated cell sorting and reducing experimental effort\n",
      "DeFCoM: analysis and modeling of transcription factor binding sites using a motif-centric genomic footprinter\n",
      "ReconMap: an interactive visualization of human metabolism\n",
      "Uncertainty estimation of predictions of peptides' chromatographic retention times in shotgun proteomics\n",
      "LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis\n",
      "PyBoolNet: a python package for the generation, analysis and visualization of boolean networks\n",
      "tcpl: the ToxCast pipeline for high-throughput screening data\n",
      "INTEGRATE-neo: a pipeline for personalized gene fusion neoantigen discovery\n",
      "Optimizing ChIP-seq peak detectors using visual labels and supervised machine learning\n",
      "Imputing gene expression to maximize platform compatibility\n",
      "Isomorphic semantic mapping of variant call format (VCF2RDF)\n",
      "Improving cross-study prediction through addon batch effect adjustment or addon normalization\n",
      "Molecular mechanisms underlying COPD-muscle dysfunction unveiled through a systems medicine approach\n",
      "Detecting subnetwork-level dynamic correlations\n",
      "FastRFS: fast and accurate Robinson-Foulds Supertrees using constrained exact optimization\n",
      "A generative model for the behavior of RNA polymerase\n",
      "CGDM: collaborative genomic data model for molecular profiling data using NoSQL\n",
      "LightAssembler: fast and memory-efficient assembly algorithm for high-throughput sequencing reads\n",
      "BioNetGen 2.2: advances in rule-based modeling\n",
      "PyPanda: a Python package for gene regulatory network reconstruction\n",
      "SIFORM: shared informative factor models for integration of multi-platform bioinformatic data\n",
      "MetaCycle: an integrated R package to evaluate periodicity in large scale data\n",
      "deBGA: read alignment with de Bruijn graph-based seed and extension\n",
      "Application of the MAFFT sequence alignment program to large data-reexamination of the usefulness of chained guide trees\n",
      "SwiSpot: modeling riboswitches by spotting out switching sequences\n",
      "TwoPhaseInd: an R package for estimating gene-treatment interactions and discovering predictive markers in randomized clinical trials\n",
      "Accounting for pairwise distance restraints in FFT-based protein-protein docking\n",
      "Visualizing the geography of genetic variants\n",
      "bTSSfinder: a novel tool for the prediction of promoters in cyanobacteria and Escherichia coli\n",
      "Combining multiple tools outperforms individual methods in gene set enrichment analyses\n",
      "FoldAtlas: a repository for genome-wide RNA structure probing data\n",
      "AcCNET (Accessory Genome Constellation Network): comparative genomics software for accessory genome analysis using bipartite networks\n",
      "FBB: a fast Bayesian-bound tool to calibrate RNA-seq aligners\n",
      "Evolutionary conservation of Ebola virus proteins predicts important functions at residue level\n",
      "GeneEvolve: a fast and memory efficient forward-time simulator of realistic whole-genome sequence and SNP data\n",
      "MSAProbs-MPI: parallel multiple sequence aligner for distributed-memory systems\n",
      "Seeksv: an accurate tool for somatic structural variation and virus integration detection\n",
      "IsotopicLabelling: an R package for the analysis of MS isotopic patterns of labelled analytes\n",
      "D-VASim: an interactive virtual laboratory environment for the simulation and analysis of genetic circuits\n",
      "The OGCleaner: filtering false-positive homology clusters\n",
      "A novel method for predicting activity of cis-regulatory modules, based on a diverse training set\n",
      "tHapMix: simulating tumour samples through haplotype mixtures\n",
      "stringMLST: a fast k-mer based tool for multilocus sequence typing\n",
      "MPRAnator: a web-based tool for the design of massively parallel reporter assay experiments\n",
      "Rail-RNA: scalable analysis of RNA-seq splicing and coverage\n",
      "On cross-conditional and fluctuation correlations in competitive RNA networks\n",
      "Edge-based sensitivity analysis of signaling networks by using Boolean dynamics\n",
      "Genome wide predictions of miRNA regulation by transcription factors\n",
      "Simulated linear test applied to quantitative proteomics\n",
      "AUCpreD: proteome-level protein disorder prediction by AUC-maximized deep convolutional neural fields\n",
      "Snowball: strain aware gene assembly of metagenomes\n",
      "DeepChrome: deep-learning for predicting gene expression from histone modifications\n",
      "MetaProb: accurate metagenomic reads binning based on probabilistic sequence signatures\n",
      "Extending partial haplotypes to full genome haplotypes using chromosome conformation capture data\n",
      "Fast genotyping of known SNPs through approximate k-mer matching\n",
      "A program for verification of phylogenetic network models\n",
      "PanTools: representation, storage and exploration of pan-genomic data\n",
      "GTRAC: fast retrieval from compressed collections of genomic variants\n",
      "Causality modeling for directed disease network\n",
      "Combining dependent P-values with an empirical adaptation of Brown's method\n",
      "TANDEM: a two-stage approach to maximize interpretability of drug response models based on multiple molecular data types\n",
      "A statistical method for the detection of variants from next-generation resequencing of DNA pools\n",
      "PseKRAAC: a flexible web server for generating pseudo K-tuple reduced amino acids composition\n",
      "PhenomeScape: a cytoscape app to identify differentially regulated sub-networks using known disease associations\n",
      "Cas-analyzer: an online tool for assessing genome editing results using NGS data\n",
      "PPI4DOCK: large scale assessment of the use of homology models in free docking over more than 1000 realistic targets\n",
      "DEsubs: an R package for flexible identification of differentially expressed subpathways using RNA-seq experiments\n",
      "A computational strategy to adjust for copy number in tumor Hi-C data\n",
      "Corpus domain effects on distributional semantic modeling of medical terms\n",
      "SiNVICT: ultra-sensitive detection of single nucleotide variants and indels in circulating tumour DNA\n",
      "Extensive complementarity between gene function prediction methods\n",
      "oxBS-MLE: an efficient method to estimate 5-methylcytosine and 5-hydroxymethylcytosine in paired bisulfite and oxidative bisulfite treated DNA\n",
      "Gene Slider: sequence logo interactive data-visualization for education and research\n",
      "DTMiner: identification of potential disease targets through biomedical literature mining\n",
      "RNAcommender: genome-wide recommendation of RNA-protein interactions\n",
      "BiDiFuse: a FIJI plugin for fusing bi-directionally recorded microscopic image volumes\n",
      "scphaser: haplotype inference using single-cell RNA-seq data\n",
      "Detection of differentially methylated regions in whole genome bisulfite sequencing data using local Getis-Ord statistics\n",
      "ReactPRED: a tool to predict and analyze biochemical reactions\n",
      "NET-GE: a web-server for NETwork-based human gene enrichment\n",
      "Functional classification of CATH superfamilies: a domain-based approach for protein function annotation\n",
      "Joint sparse canonical correlation analysis for detecting differential imaging genetics modules\n",
      "WALT: fast and accurate read mapping for bisulfite sequencing\n",
      "Simultaneous gene finding in multiple genomes\n",
      "Variance adaptive shrinkage (vash): flexible empirical Bayes estimation of variances\n",
      "PReFerSim: fast simulation of demography and selection under the Poisson Random Field model\n",
      "LEVER: software tools for segmentation, tracking and lineaging of proliferating cells\n",
      "ntHash: recursive nucleotide hashing\n",
      "Estimating evolutionary rates using time-structured data: a general comparison of phylogenetic methods\n",
      "LAMPLINK: detection of statistically significant SNP combinations from GWAS data\n",
      "BioPartsDB: a synthetic biology workflow web-application for education and research\n",
      "Local versus global biological network alignment\n",
      "GeneCodeq: quality score compression and improved genotyping using a Bayesian framework\n",
      "Conpair: concordance and contamination estimator for matched tumor-normal pairs\n",
      "A probabilistic approach for collective similarity-based drug-drug interaction prediction\n",
      "Accurate in silico prediction of species-specific methylation sites based on information gain feature optimization\n",
      "Inheritance-mode specific pathogenicity prioritization (ISPP) for human protein coding genes\n",
      "Revealing aperiodic aspects of solenoid proteins from sequence information\n",
      "CAGEd-oPOSSUM: motif enrichment analysis from CAGE-derived TSSs\n",
      "A simple model predicts UGT-mediated metabolism\n",
      "New KEGG pathway-based interpretable features for classifying ageing-related mouse proteins\n",
      "SELAM: simulation of epistasis and local adaptation during admixture with mate choice\n",
      "The structural effects of mutations can aid in differential phenotype prediction of beta-myosin heavy chain (Myosin-7) missense variants\n",
      "PHOCOS: inferring multi-feature phenotypic crosstalk networks\n",
      "Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction\n",
      "RNAiFold2T: Constraint Programming design of thermo-IRES switches\n",
      "DFLpred: High-throughput prediction of disordered flexible linker regions in protein sequences\n",
      "Faster and more accurate graphical model identification of tandem mass spectra using trellises\n",
      "Read-based phasing of related individuals\n",
      "An algorithm for computing the gene tree probability under the multispecies coalescent and its application in the inference of population tree\n",
      "Genome assembly from synthetic long read clouds\n",
      "phRAIDER: Pattern-Hunter based Rapid Ab Initio Detection of Elementary Repeats\n",
      "Convolutional neural network architectures for predicting DNA-protein binding\n",
      "Exploring the structure and function of temporal networks with dynamic graphlets\n",
      "CONDOP: an R package for CONdition-Dependent Operon Predictions\n",
      "GenVisR: Genomic Visualizations in R\n",
      "Prediction of conserved long-range RNA-RNA interactions in full viral genomes\n",
      "Inferring the perturbation time from biological time course data\n",
      "Accurate self-correction of errors in long reads using de Bruijn graphs\n",
      "MetalPredator: a web server to predict iron-sulfur cluster binding proteomes\n",
      "DBSI server: DNA binding site identifier\n",
      "methylFlow: cell-specific methylation pattern reconstruction from high-throughput bisulfite-converted DNA sequencing\n",
      "Complex heatmaps reveal patterns and correlations in multidimensional genomic data\n",
      "PERMANOVA-S: association test for microbial community composition that accommodates confounders and multiple distances\n",
      "PRESS: PRotEin S-Sulfenylation server\n",
      "A simple yet accurate correction for winner's curse can predict signals discovered in much larger genome scans\n",
      "Top-down analysis of protein samples by de novo sequencing techniques\n",
      "SA-SSR: a suffix array-based algorithm for exhaustive and efficient SSR discovery in large genetic sequences\n",
      "Evaluation of hybrid and non-hybrid methods for de novo assembly of nanopore reads\n",
      "Rapid genotype refinement for whole-genome sequencing data using multi-variate normal distributions\n",
      "NMRPro: an integrated web component for interactive processing and visualization of NMR spectra\n",
      "cit: hypothesis testing software for mediation analysis in genomic applications\n",
      "NSimScan: DNA comparison tool with increased speed, sensitivity and accuracy\n",
      "Tally: a scoring tool for boundary determination between repetitive and non-repetitive protein sequences\n",
      "fqtools: an efficient software suite for modern FASTQ file manipulation\n",
      "FRAGSION: ultra-fast protein fragment library generation by IOHMM sampling\n",
      "SASpy: a PyMOL plugin for manipulation and refinement of hybrid models against small angle X-ray scattering data\n",
      "XIBD: software for inferring pairwise identity by descent on the X chromosome\n",
      "metaCCA: summary statistics-based multivariate meta-analysis of genome-wide association studies using canonical correlation analysis\n",
      "Transcript structure and domain display: a customizable transcript visualization tool\n",
      "PhamDB: a web-based application for building Phamerator databases\n",
      "WeSME: uncovering mutual exclusivity of cancer drivers and beyond\n",
      "OrfM: a fast open reading frame predictor for metagenomic data\n",
      "FastHiC: a fast and accurate algorithm to detect long-range chromosomal interactions from Hi-C data\n",
      "BAM-matcher: a tool for rapid NGS sample matching\n",
      "XLinkDB 2.0: integrated, large-scale structural analysis of protein crosslinking data\n",
      "BRAT-nova: fast and accurate mapping of bisulfite-treated reads\n",
      "Confidence assignment for mass spectrometry based peptide identifications via the extreme value distribution\n",
      "APPAGATO: an APproximate PArallel and stochastic GrAph querying TOol for biological networks\n",
      "Incorporating organelle correlations into semi-supervised learning for protein subcellular localization prediction\n",
      "Estimating IBD tracts from low coverage NGS data\n",
      "Romulus: robust multi-state identification of transcription factor binding sites from DNase-seq data\n",
      "SAMFIRE: multi-locus variant calling for time-resolved sequence data\n",
      "Mollack: a web server for the automated creation of conformational ensembles for intrinsically disordered proteins\n",
      "hotspot: software to support sperm-typing for investigating recombination hotspots\n",
      "TOPDOM: database of conservatively located domains and motifs in proteins\n",
      "INPS-MD: a web server to predict stability of protein variants from sequence and structure\n",
      "MEMO: multi-experiment mixture model analysis of censored data\n",
      "Memdock: an α-helical membrane protein docking algorithm\n",
      "R2C: improving ab initio residue contact map prediction using dynamic fusion strategy and Gaussian noise filter\n",
      "SBtab: a flexible table format for data exchange in systems biology\n",
      "Cell-free DNA fragment-size distribution analysis for non-invasive prenatal CNV prediction\n",
      "Rail-dbGaP: analyzing dbGaP-protected data in the cloud with Amazon Elastic MapReduce\n",
      "Quality control of single-cell RNA-seq by SinQC\n",
      "Identifying cell-specific microRNA transcriptional start sites\n",
      "ProbOnto: ontology and knowledge base of probability distributions\n",
      "Genome-scale prediction of moonlighting proteins using diverse protein association information\n",
      "HilbertCurve: an R/Bioconductor package for high-resolution visualization of genomic data\n",
      "FREGAT: an R package for region-based association analysis\n",
      "gEVAL - a web-based browser for evaluating genome assemblies\n",
      "HLaffy: estimating peptide affinities for Class-1 HLA molecules by learning position-specific pair potentials\n",
      "Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences\n",
      "Inferring gene targets of drugs and chemical compounds from gene expression profiles\n",
      "D3M: detection of differential distributions of methylation levels\n",
      "PinaColada: peptide-inhibitor ant colony ad-hoc design algorithm\n",
      "Analysis of CFSE time-series data using division-, age- and label-structured population models\n",
      "Pathway analysis by randomization incorporating structure-PARIS: an update\n",
      "The spatiotemporal order of signaling events unveils the logic of development signaling\n",
      "Visual Omics Explorer (VOE): a cross-platform portal for interactive data visualization\n",
      "A quantitative model for oxygen uptake and release in a family of hemeproteins\n",
      "Integrated genome browser: visual analytics platform for genomics\n",
      "Computationally expanding infinium HumanMethylation450 BeadChip array data to reveal distinct DNA methylation patterns of rheumatoid arthritis\n",
      "An empirical Bayes change-point model for identifying 3' and 5' alternative splicing by next-generation RNA sequencing\n",
      "Privacy-preserving microbiome analysis using secure computation\n",
      "Gene expression inference with deep learning\n",
      "MOCCASIN: converting MATLAB ODE models to SBML\n",
      "Optimization of alignment-based methods for taxonomic binning of metagenomics reads\n",
      "GiANT: gene set uncertainty in enrichment analysis\n",
      "The geospatial data quality REST API for primary biodiversity data\n",
      "Alloscore: a method for predicting allosteric ligand-protein interactions\n",
      "ParDRe: faster parallel duplicated reads removal tool for sequencing studies\n",
      "lodGWAS: a software package for genome-wide association analysis of biomarkers with a limit of detection\n",
      "Multi-task consensus clustering of genome-wide transcriptomes from related biological conditions\n",
      "Protein backbone ensemble generation explores the local structural space of unseen natural homologs\n",
      "GeneValidator: identify problems with protein-coding gene predictions\n",
      "FINEMAP: efficient variable selection using summary data from genome-wide association studies\n",
      "Realizing privacy preserving genome-wide association studies\n",
      "Computational discovery of pathway-level genetic vulnerabilities in non-small-cell lung cancer\n",
      "VIRALpro: a tool to identify viral capsid and tail sequences\n",
      "SPMM: estimating infection duration of multivariant HIV-1 infections\n",
      "MOGEN: a tool for reconstructing 3D models of genomes from chromosomal conformation capturing data\n",
      "Hierarchical block matrices as efficient representations of chromosome topologies and their application for 3C data integration\n",
      "PaxtoolsR: pathway analysis in R using Pathway Commons\n",
      "Positive and negative forms of replicability in gene network analysis\n",
      "Fuse: multiple network alignment via data fusion\n",
      "destiny: diffusion maps for large-scale single-cell data in R\n",
      "BioFVM: an efficient, parallelized diffusive transport solver for 3-D biological simulations\n",
      "Sequence database versioning for command line and Galaxy bioinformatics servers\n",
      "PharmacoGx: an R package for analysis of large pharmacogenomic datasets\n",
      "Sampling ARG of multiple populations under complex configurations of subdivision and admixture\n",
      "rcellminer: exploring molecular profiles and drug response of the NCI-60 cell lines in R\n",
      "An integrative somatic mutation analysis to identify pathways linked with survival outcomes across 19 cancer types\n",
      "Large-scale machine learning for metagenomics sequence classification\n",
      "hybridSPAdes: an algorithm for hybrid assembly of short and long reads\n",
      "FoCuS-point: software for STED fluorescence correlation and time-gated single photon counting\n",
      "Analyzing synergistic and non-synergistic interactions in signalling pathways using Boolean Nested Effect Models\n",
      "The intrinsic combinatorial organization and information theoretic content of a sequence are correlated to the DNA encoded nucleosome organization of eukaryotic genomes\n",
      "rHAT: fast alignment of noisy long reads with regional hashing\n",
      "Identification of differentially methylated loci using wavelet-based functional mixed models\n",
      "Realtime analysis and visualization of MinION sequencing data with npReader\n",
      "Exact quantification of cellular robustness in genome-scale metabolic networks\n",
      "No Promoter Left Behind (NPLB): learn de novo promoter architectures from genome-wide transcription start sites\n",
      "Specific small-RNA signatures in the amygdala at premotor and motor stages of Parkinson's disease revealed by deep sequencing analysis\n",
      "The discordant method: a novel approach for differential correlation\n",
      "Enumeration and extension of non-equivalent deterministic update schedules in Boolean networks\n",
      "catRAPID signature: identification of ribonucleoproteins and RNA-binding regions\n",
      "Protael: protein data visualization library for the web\n",
      "DelPhiPKa web server: predicting pKa of proteins, RNAs and DNAs\n",
      "SylvX: a viewer for phylogenetic tree reconciliations\n",
      "PRIMUS: improving pedigree reconstruction using mitochondrial and Y haplotypes\n",
      "Detecting dispersed duplications in high-throughput sequencing data using a database-free approach\n",
      "NMRe: a web server for NMR protein structure refinement with high-quality structure validation scores\n",
      "PDID: database of molecular-level putative protein-drug interactions in the structural human proteome\n",
      "Web-based Gene Pathogenicity Analysis (WGPA): a web platform to interpret gene pathogenicity from personal genome data\n",
      "A-DaGO-Fun: an adaptable Gene Ontology semantic similarity-based functional analysis tool\n",
      "GERV: a statistical method for generative evaluation of regulatory variants for transcription factor binding\n",
      "3DRobot: automated generation of diverse and well-packed protein structure decoys\n",
      "Unified tests for fine-scale mapping and identifying sparse high-dimensional sequence associations\n",
      "SUPER-FOCUS: a tool for agile functional analysis of shotgun metagenomic data\n",
      "Fast and accurate non-sequential protein structure alignment using a new asymmetric linear sum assignment heuristic\n",
      "miTRATA: a web-based tool for microRNA Truncation and Tailing Analysis\n",
      "Jflow: a workflow management system for web applications\n",
      "HapFlow: visualizing haplotypes in sequencing data\n",
      "Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data\n",
      "Computational probing protein-protein interactions targeting small molecules\n",
      "ScreenBEAM: a novel meta-analysis algorithm for functional genomics screens via Bayesian hierarchical modeling\n",
      "XLmap: an R package to visualize and score protein structure models based on sites of protein cross-linking\n",
      "P-SAMS: a web site for plant artificial microRNA and synthetic trans-acting small interfering RNA design\n",
      "A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data\n",
      "markophylo: Markov chain analysis on phylogenetic trees\n",
      "Human cell structure-driven model construction for predicting protein subcellular location from biological images\n",
      "Reveel: large-scale population genotyping using low-coverage sequencing data\n",
      "Inter-functional analysis of high-throughput phenotype data by non-parametric clustering and its application to photosynthesis\n",
      "Large-scale extraction of gene interactions from full-text literature using DeepDive\n",
      "NgsRelate: a software tool for estimating pairwise relatedness from next-generation sequencing data\n",
      "BiopLib and BiopTools--a C programming library and toolset for manipulating protein structure\n",
      "RNASeqMetaDB: a database and web server for navigating metadata of publicly available mouse RNA-Seq datasets\n",
      "Integrated structure- and ligand-based in silico approach to predict inhibition of cytochrome P450 2D6\n",
      "glmgraph: an R package for variable selection and predictive modeling of structured genomic data\n",
      "HiXCorr: a portable high-speed XCorr engine for high-resolution tandem mass spectrometry\n",
      "Fast and accurate approximate inference of transcript expression from RNA-seq data\n",
      "InteractiveROSETTA: a graphical user interface for the PyRosetta protein modeling suite\n",
      "Pairwise RNA secondary structure alignment with conserved stem pattern\n",
      "Determining conserved metabolic biomarkers from a million database queries\n",
      "Mutadelic: mutation analysis using description logic inferencing capabilities\n",
      "cyNeo4j: connecting Neo4j and Cytoscape\n",
      "TIPR: transcription initiation pattern recognition on a genome scale\n",
      "Impact of normalization methods on high-throughput screening data with high hit rates and drug testing with dose-response data\n",
      "Strategies to improve the performance of rare variant association studies by optimizing the selection of controls\n",
      "ARResT/AssignSubsets: a novel application for robust subclassification of chronic lymphocytic leukemia based on B cell receptor IG stereotypy\n",
      "An efficient algorithm for the extraction of HGVS variant descriptions from sequences\n",
      "SynLinker: an integrated system for designing linkers and synthetic fusion proteins\n",
      "MetTailor: dynamic block summary and intensity normalization for robust analysis of mass spectrometry data in metabolomics\n",
      "Designing alternative splicing RNA-seq studies. Beyond generic guidelines\n",
      "GeneVetter: a web tool for quantitative monogenic assessment of rare diseases\n",
      "dendextend: an R package for visualizing, adjusting and comparing trees of hierarchical clustering\n",
      "C-It-Loci: a knowledge database for tissue-enriched loci\n",
      "LayerCake: a tool for the visual comparison of viral deep sequencing data\n",
      "Data2Dynamics: a modeling environment tailored to parameter estimation in dynamical systems\n",
      "Error filtering, pair assembly and error correction for next-generation sequencing reads\n",
      "ERGC: an efficient referential genome compression algorithm\n",
      "damidseq_pipeline: an automated pipeline for processing DamID sequencing datasets\n",
      "FoldNucleus: web server for the prediction of RNA and protein folding nuclei from their 3D structures\n",
      "Sincell: an R/Bioconductor package for statistical assessment of cell-state hierarchies from single-cell RNA-seq\n",
      "Forna (force-directed RNA): Simple and effective online RNA secondary structure diagrams\n",
      "Investigating microbial co-occurrence patterns based on metagenomic compositional data\n",
      "JSBML 1.0: providing a smorgasbord of options to encode systems biology models\n",
      "MetaPathways v2.5: quantitative functional, taxonomic and usability improvements\n",
      "Reconstruction of clonal trees and tumor composition from multi-sample sequencing data\n",
      "Knowledge-driven geospatial location resolution for phylogeographic models of virus migration\n",
      "A generic methodological framework for studying single cell motility in high-throughput time-lapse data\n",
      "FERAL: network-based classifier with application to breast cancer outcome prediction\n",
      "Improving compound-protein interaction prediction by building up highly credible negative samples\n",
      "Identification of causal genes for complex traits\n",
      "Adapt-Mix: learning local genetic correlation structure improves summary statistics-based analyses\n",
      "Exploring the structure and function of temporal networks with dynamic graphlets\n",
      "Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data\n",
      "DISTMIX: direct imputation of summary statistics for unmeasured SNPs from mixed ethnicity cohorts\n",
      "Sparse multi-view matrix factorization: a multivariate approach to multiple tissue comparisons\n",
      "A web application for the unspecific detection of differentially expressed DNA regions in strand-specific expression data\n",
      "FinisherSC: a repeat-aware tool for upgrading de novo assembly using long reads\n",
      "lpNet: a linear programming approach to reconstruct signal transduction networks\n",
      "xHeinz: an algorithm for mining cross-species network modules under a flexible conservation model\n",
      "GC3-biased gene domains in mammalian genomes\n",
      "A novel essential domain perspective for exploring gene essentiality\n",
      "Diffusion maps for high-dimensional single-cell analysis of differentiation data\n",
      "Reverse engineering of logic-based differential equation models using a mixed-integer dynamic optimization approach\n",
      "Computer vision-based automated peak picking applied to protein NMR spectra\n",
      "Message from ISCB: Outstanding contributions to ISCB award\n",
      "CoRegNet: reconstruction and integrated analysis of co-regulatory networks\n",
      "WGE: a CRISPR database for genome engineering\n",
      "Seq2pathway: an R/Bioconductor package for pathway analysis of next-generation sequencing data\n",
      "GLASS: a comprehensive database for experimentally validated GPCR-ligand associations\n",
      "Gener: a minimal programming module for chemical controllers based on DNA strand displacement\n",
      "Automated band annotation for RNA structure probing experiments with numerous capillary electrophoresis profiles\n",
      "Polyester: simulating RNA-seq datasets with differential transcript expression\n",
      "phylogeo: an R package for geographic analysis and visualization of microbiome data\n",
      "DockStar: a novel ILP-based integrative method for structural modeling of multimolecular protein complexes\n",
      "chipPCR: an R package to pre-process raw data of amplification curves\n",
      "iFoldRNA v2: folding RNA with constraints\n",
      "PDBest: a user-friendly platform for manipulating and enhancing protein structures\n",
      "Protein homology reveals new targets for bioactive small molecules\n",
      "ScaffMatch: scaffolding algorithm based on maximum weight matching\n",
      "GREGOR: evaluating global enrichment of trait-associated variants in epigenomic features using a systematic, data-driven approach\n",
      "Applying stability selection to consistently estimate sparse principal components in high-dimensional molecular data\n",
      "AVIA v2.0: annotation, visualization and impact analysis of genomic variants and genes\n",
      "GS-align for glycan structure alignment and similarity measurement\n",
      "PROVEAN web server: a tool to predict the functional effect of amino acid substitutions and indels\n",
      "Gene selection for the reconstruction of stem cell differentiation trees: a linear programming approach\n",
      "Regulatory network inferred using expression data of small sample size: application and validation in erythroid system\n",
      "Global optimization-based inference of chemogenomic features from drug-target interactions\n",
      "Halvade: scalable sequence analysis with MapReduce\n",
      "A statistical physics perspective on alignment-independent protein sequence comparison\n",
      "Computational modeling of the expansion of human cord blood CD133+ hematopoietic stem/progenitor cells with different cytokine combinations\n",
      "Learning chromatin states with factorized information criteria\n",
      "ASBench: benchmarking sets for allosteric discovery\n",
      "Solubis: optimize your protein\n",
      "Efficient visualization of high-throughput targeted proteomics experiments: TAPIR\n",
      "A novel motif-discovery algorithm to identify co-regulatory motifs in large transcription factor and microRNA co-regulatory networks in human\n",
      "Probing the binding affinity of amyloids to reduce toxicity of oligomers in diabetes\n",
      "VisualCNA: a GUI for interactive constraint network analysis and protein engineering for improving thermostability\n",
      "miR2GO: comparative functional analysis for microRNAs\n",
      "Reducing the search space for causal genetic variants with VASP\n",
      "14-3-3-Pred: improved methods to predict 14-3-3-binding phosphopeptides\n",
      "GeLL: a generalized likelihood library for phylogenetic models\n",
      "RepExplore: addressing technical replicate variance in proteomics and metabolomics data analysis\n",
      "Fast, accurate, and reliable molecular docking with QuickVina 2\n",
      "GeNOSA: inferring and experimentally supporting quantitative gene regulatory networks in prokaryotes\n",
      "specL--an R/Bioconductor package to prepare peptide spectrum matches for use in targeted proteomics\n",
      "Comparative study of the effectiveness and limitations of current methods for detecting sequence coevolution\n",
      "The GPMDB REST interface\n",
      "Folding RaCe: a robust method for predicting changes in protein folding rates upon point mutations\n",
      "A multiobjective memetic algorithm for PPI network alignment\n",
      "Incorporating peak grouping information for alignment of multiple liquid chromatography-mass spectrometry datasets\n",
      "PolyMarker: A fast polyploid primer design pipeline\n",
      "ClassifyR: an R package for performance assessment of classification with applications to transcriptomics\n",
      "Learning HMMs for nucleotide sequences from amino acid alignments\n",
      "SPARQL-enabled identifier conversion with Identifiers.org\n",
      "TIMMA-R: an R package for predicting synergistic multi-targeted drug combinations in cancer cell lines or patient-derived samples\n",
      "Computational identification of MoRFs in protein sequences\n",
      "Omics Pipe: a community-based framework for reproducible multi-omics data analysis\n",
      "3USS: a web server for detecting alternative 3'UTRs from RNA-seq experiments\n",
      "Gaussian process test for high-throughput sequencing time series: application to experimental evolution\n",
      "Large-scale extraction of brain connectivity from the neuroscientific literature\n",
      "MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph\n",
      "GrammR: graphical representation and modeling of count data with application in metagenomics\n",
      "Bias in microRNA functional enrichment analysis\n",
      "Deep profiling of multitube flow cytometry data\n",
      "ENViz: a Cytoscape App for integrated statistical analysis and visualization of sample-matched data with multiple data types\n",
      "Acquire: an open-source comprehensive cancer biobanking system\n",
      "PVAAS: identify variants associated with aberrant splicing from RNA-seq\n",
      "Inferring single-cell gene expression mechanisms using stochastic simulation\n",
      "Selection of models for the analysis of risk-factor trees: leveraging biological knowledge to mine large sets of risk factors with application to microbiome data\n",
      "Modified screening and ranking algorithm for copy number variation detection\n",
      "Knowledge-based modeling of peptides at protein interfaces: PiPreD\n",
      "J-Circos: an interactive Circos plotter\n",
      "The QDREC web server: determining dose-response characteristics of complex macroparasites in phenotypic drug screens\n",
      "PBOOST: a GPU-based tool for parallel permutation tests in genome-wide association studies\n",
      "mirPub: a database for searching microRNA publications\n",
      "CDvist: a webserver for identification and visualization of conserved domains in protein sequences\n",
      "The Cyni framework for network inference in Cytoscape\n",
      "LocNES: a computational tool for locating classical NESs in CRM1 cargo proteins\n",
      "Detection of circular permutations within protein structures using CE-CP\n",
      "JEPEG: a summary statistics based tool for gene-level joint testing of functional variants\n",
      "3Dmol.js: molecular visualization with WebGL\n",
      "Sputnik: ad hoc distributed computation\n",
      "State of the art prediction of HIV-1 protease cleavage sites\n",
      "Identification of deleterious synonymous variants in human genomes\n",
      "SNPlice: variants that modulate Intron retention from RNA-sequencing data\n",
      "ICMA: an integrated cardiac modeling and analysis platform\n",
      "Kablammo: an interactive, web-based BLAST results visualizer\n",
      "GenomeCons: a web server for manipulating multiple genome sequence alignments and their consensus sequences\n",
      "QuASAR: quantitative allele-specific analysis of reads\n",
      "Graphical algorithm for integration of genetic and biological data: proof of principle using psoriasis as a model\n",
      "Semiconductor sequencing: how many flows do you need?\n",
      "Exploiting hidden information interleaved in the redundancy of the genetic code without prior knowledge\n",
      "Faster sequence homology searches by clustering subsequences\n",
      "SNiPA: an interactive, genetic variant-centered annotation browser\n",
      "LuciPHOr2: site localization of generic post-translational modifications from tandem mass spectrometry data\n",
      "Ustiloxins, fungal cyclic peptides, are ribosomally synthesized in Ustilaginoidea virens\n",
      "Unsupervised discovery of information structure in biomedical documents\n",
      "Graphical model for joint segmentation and tracking of multiple dividing cells\n",
      "WemIQ: an accurate and robust isoform quantification method for RNA-seq data\n",
      "Local statistics allow quantification of cell-to-cell variability from high-throughput microscope images\n",
      "GASS: identifying enzyme active sites with genetic algorithms\n",
      "Taxator-tk: precise taxonomic assignment of metagenomes by fast approximation of evolutionary neighborhoods\n",
      "Distinct profiling of antimicrobial peptide families\n",
      "GlycoRDF: an ontology to standardize glycomics data in RDF\n",
      "Ulysses: accurate detection of low-frequency structural variations in large insert-size sequencing libraries\n",
      "Joint amalgamation of most parsimonious reconciled gene trees\n",
      "JAMSS: proteomics mass spectrometry simulation in Java\n",
      "Fast and accurate site frequency spectrum estimation from low coverage sequence data\n",
      "eQTL epistasis: detecting epistatic effects and inferring hierarchical relationships of genes in biological pathways\n",
      "Large-scale binding ligand prediction by improved patch-based method Patch-Surfer2.0\n",
      "PPDMs-a resource for mapping small molecule bioactivities from ChEMBL to Pfam-A protein domains\n",
      "Navigating protected genomics data with UCSC Genome Browser in a Box\n",
      "iDoComp: a compression scheme for assembled genomes\n",
      "Using 2k + 2 bubble searches to find single nucleotide polymorphisms in k-mer graphs\n",
      "A novel representation of genomic sequences for taxonomic clustering and visualization by means of self-organizing maps\n",
      "piPipes: a set of pipelines for piRNA and transposon analysis via small RNA-seq, RNA-seq, degradome- and CAGE-seq, ChIP-seq and genomic DNA sequencing\n",
      "A5-miseq: an updated pipeline to assemble microbial genomes from Illumina MiSeq data\n",
      "DANN: a deep learning approach for annotating the pathogenicity of genetic variants\n",
      "Simple, rapid and accurate genotyping-by-sequencing from aligned whole genomes with ArrayMaker\n",
      "An efficient algorithm for the blocked pattern matching problem\n",
      "Rcount: simple and flexible RNA-Seq read counting\n",
      "SNPsnap: a Web-based tool for identification and annotation of matched SNPs\n",
      "Pooled assembly of marine metagenomic datasets: enriching annotation through chimerism\n",
      "PhosphoPICK: modelling cellular context to map kinase-substrate phosphorylation events\n",
      "Integrative data analysis indicates an intrinsic disordered domain character of Argonaute-binding motifs\n",
      "Tabhu: tools for antibody humanization\n",
      "PDB-wide collection of binding data: current status of the PDBbind database\n",
      "NOVA: a software to analyze complexome profiling data\n",
      "Using the plurality of codon positions to identify deleterious variants in human exomes\n",
      "LocalAli: an evolutionary-based local alignment approach to identify functionally conserved modules in multiple networks\n",
      "Inference of protein kinetics by stochastic modeling and simulation of fluorescence recovery after photobleaching experiments\n",
      "ExportAid: database of RNA elements regulating nuclear RNA export in mammals\n",
      "OrthoInspector 2.0: Software and database updates\n",
      "WhopGenome: high-speed access to whole-genome variation and sequence data in R\n",
      "Consensus Genotyper for Exome Sequencing (CGES): improving the quality of exome variant genotypes\n",
      "Hybrid Bayesian-rank integration approach improves the predictive power of genomic dataset aggregation\n",
      "GLAD: a mixed-membership model for heterogeneous tumor subtype classification\n",
      "Shiny-phyloseq: Web application for interactive microbiome analysis with provenance tracking\n",
      "switchBox: an R package for k-Top Scoring Pairs classifier development\n",
      "A two-stage statistical procedure for feature selection and comparison in functional analysis of metagenomes\n",
      "Understanding the limits of animal models as predictors of human biology: lessons learned from the sbv IMPROVER Species Translation Challenge\n",
      "UNDO: a Bioconductor R package for unsupervised deconvolution of mixed gene expressions in tumor samples\n",
      "BalestraWeb: efficient online evaluation of drug-target interactions\n",
      "Genomon ITDetector: a tool for somatic internal tandem duplication detection from cancer genome sequencing data\n",
      "Insights into structural variations and genome rearrangements in prokaryotic genomes\n",
      "Summary of the BioLINK SIG 2013 meeting at ISMB/ECCB 2013\n",
      "Conditional out-of-distribution generation for unpaired data using transfer VAE\n",
      "SVIM-asm: Structural variant detection from haploid and diploid genome assemblies\n",
      "MHCAttnNet: predicting MHC-peptide bindings for MHC alleles classes I and II using an attention-based deep neural model\n",
      "Audit logs to enforce document integrity in Skyline and Panorama\n",
      "MUM&Co: accurate detection of all SV types through whole-genome alignment\n",
      "popSTR2 enables clinical and population-scale genotyping of microsatellites\n",
      "Graph embedding on biomedical networks: methods, applications and evaluations\n",
      "A latent unknown clustering integrating multi-omics data (LUCID) with phenotypic traits\n",
      "Haplotype-aware graph indexes\n",
      "snpEnrichR: analyzing co-localization of SNPs and their proxies in genomic regions\n",
      "PriLive: privacy-preserving real-time filtering for next-generation sequencing\n",
      "chimeraviz: a tool for visualizing chimeric RNA\n",
      "SNPhood: investigate, quantify and visualise the epigenomic neighbourhood of SNPs using NGS data\n",
      "MCAST: scanning for cis-regulatory motif clusters\n",
      "Foldalign 2.5: multithreaded implementation for pairwise structural RNA alignment\n",
      "Prediction of the permeability of neutral drugs inferred from their solvation properties\n"
     ]
    }
   ],
   "source": [
    "for i in titles:\n",
    "    print(i)\n",
    "    print"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "62bef738",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
